<!DOCTYPE HTML>
<html lang="pt">
<head>
<!-- Generated by javadoc (17) on Sun May 15 18:36:12 WEST 2022 -->
<title>C-Index</title>
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta name="dc.created" content="2022-05-15">
<meta name="description" content="index: C">
<meta name="generator" content="javadoc/IndexWriter">
<link rel="stylesheet" type="text/css" href="../stylesheet.css" title="Style">
<link rel="stylesheet" type="text/css" href="../script-dir/jquery-ui.min.css" title="Style">
<link rel="stylesheet" type="text/css" href="../jquery-ui.overrides.css" title="Style">
<script type="text/javascript" src="../script.js"></script>
<script type="text/javascript" src="../script-dir/jquery-3.5.1.min.js"></script>
<script type="text/javascript" src="../script-dir/jquery-ui.min.js"></script>
</head>
<body class="index-page">
<script type="text/javascript">var pathtoroot = "../";
loadScripts(document, 'script');</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<div class="flex-box">
<header role="banner" class="flex-header">
<nav role="navigation">
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="top-nav" id="navbar-top">
<div class="skip-nav"><a href="#skip-navbar-top" title="Skip navigation links">Skip navigation links</a></div>
<ul id="navbar-top-firstrow" class="nav-list" title="Navigation">
<li><a href="../index.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li>Use</li>
<li><a href="../overview-tree.html">Tree</a></li>
<li><a href="../deprecated-list.html">Deprecated</a></li>
<li class="nav-bar-cell1-rev">Index</li>
<li><a href="../help-doc.html#index">Help</a></li>
</ul>
</div>
<div class="sub-nav">
<div class="nav-list-search"><label for="search-input">SEARCH:</label>
<input type="text" id="search-input" value="search" disabled="disabled">
<input type="reset" id="reset-button" value="reset" disabled="disabled">
</div>
</div>
<!-- ========= END OF TOP NAVBAR ========= -->
<span class="skip-nav" id="skip-navbar-top"></span></nav>
</header>
<div class="flex-content">
<main role="main">
<div class="header">
<h1>Index</h1>
</div>
<a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Z</a>&nbsp;<br><a href="../allclasses-index.html">All&nbsp;Classes&nbsp;and&nbsp;Interfaces</a><span class="vertical-separator">|</span><a href="../allpackages-index.html">All&nbsp;Packages</a><span class="vertical-separator">|</span><a href="../constant-values.html">Constant&nbsp;Field&nbsp;Values</a><span class="vertical-separator">|</span><a href="../serialized-form.html">Serialized&nbsp;Form</a>
<h2 class="title" id="I:C">C</h2>
<dl class="index">
<dt><a href="../org/biojava/nbio/core/util/InputStreamProvider.html#CACHE_PROPERTY" class="member-name-link">CACHE_PROPERTY</a> - Static variable in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/InputStreamProvider.html" title="class in org.biojava.nbio.core.util">InputStreamProvider</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/JoiningSequenceReaderTest.html#canScan()" class="member-name-link">canScan()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/JoiningSequenceReaderTest.html" title="class in org.biojava.nbio.core.sequence">JoiningSequenceReaderTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a> - Class in <a href="../org/biojava/nbio/core/sequence/transcription/package-summary.html">org.biojava.nbio.core.sequence.transcription</a></dt>
<dd>
<div class="block">Attempts to wrap compounds so it is possible to view them
 in a case insensitive manner</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#%3Cinit%3E(org.biojava.nbio.core.sequence.compound.NucleotideCompound)" class="member-name-link">CaseInsensitiveCompound(NucleotideCompound)</a> - Constructor for class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html#%3Cinit%3E(org.biojava.nbio.core.sequence.compound.NucleotideCompound,org.biojava.nbio.core.sequence.compound.NucleotideCompound,org.biojava.nbio.core.sequence.compound.NucleotideCompound)" class="member-name-link">CaseInsensitiveTriplet(NucleotideCompound, NucleotideCompound, NucleotideCompound)</a> - Constructor for class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.CaseInsensitiveTriplet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">CasePreservingProteinSequenceCreator</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">A sequence creator which preserves the case of its input string in
 the user collection of the returned ProteinSequence.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#%3Cinit%3E(org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">CasePreservingProteinSequenceCreator(CompoundSet&lt;AminoAcidCompound&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">CasePreservingProteinSequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/CDSComparator.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence">CDSComparator</a> - Class in <a href="../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/CDSComparator.html#%3Cinit%3E()" class="member-name-link">CDSComparator()</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/CDSComparator.html" title="class in org.biojava.nbio.core.sequence">CDSComparator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/CDSSequence.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence">CDSSequence</a> - Class in <a href="../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">Represents a exon or coding sequence in a gene.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/CDSSequence.html#%3Cinit%3E(org.biojava.nbio.core.sequence.TranscriptSequence,int,int,int)" class="member-name-link">CDSSequence(TranscriptSequence, int, int, int)</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/CDSSequence.html" title="class in org.biojava.nbio.core.sequence">CDSSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankReaderTest.html#CDStest()" class="member-name-link">CDStest()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankReaderTest.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReaderTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html#checksum(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">checksum(Sequence&lt;C&gt;)</a> - Static method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Performs a simple CRC64 checksum on any given sequence.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNASequence.DNAType.html#CHROMOSOME" class="member-name-link">CHROMOSOME</a> - Enum constant in enum org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DNASequence.DNAType.html" title="enum in org.biojava.nbio.core.sequence">DNASequence.DNAType</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/ChromosomeSequence.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence">ChromosomeSequence</a> - Class in <a href="../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">A ChromosomeSequence is a DNASequence but keeps track of geneSequences</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/ChromosomeSequence.html#%3Cinit%3E()" class="member-name-link">ChromosomeSequence()</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/ChromosomeSequence.html" title="class in org.biojava.nbio.core.sequence">ChromosomeSequence</a></dt>
<dd>
<div class="block">Empty constructor used by tools that need a proper Bean that allows the actual
 sequence data to be set after construction.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/ChromosomeSequence.html#%3Cinit%3E(java.lang.String)" class="member-name-link">ChromosomeSequence(String)</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/ChromosomeSequence.html" title="class in org.biojava.nbio.core.sequence">ChromosomeSequence</a></dt>
<dd>
<div class="block">String is king and assume DNA</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/ChromosomeSequence.html#%3Cinit%3E(java.lang.String,org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">ChromosomeSequence(String, CompoundSet&lt;NucleotideCompound&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/ChromosomeSequence.html" title="class in org.biojava.nbio.core.sequence">ChromosomeSequence</a></dt>
<dd>
<div class="block">Allows the creation of a ChromosomeSequence using String for the sequence with a custom CompoundSet</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/ChromosomeSequence.html#%3Cinit%3E(org.biojava.nbio.core.sequence.template.SequenceReader)" class="member-name-link">ChromosomeSequence(SequenceReader&lt;NucleotideCompound&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/ChromosomeSequence.html" title="class in org.biojava.nbio.core.sequence">ChromosomeSequence</a></dt>
<dd>
<div class="block">Fairly important constructor given the size of a ChromsomeSequence where the
 ProxySequenceReader could load from disk via RandomAccessFile so that the sequence
 doesn't need to be kept in memory.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/ChromosomeSequence.html#%3Cinit%3E(org.biojava.nbio.core.sequence.template.SequenceReader,org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">ChromosomeSequence(SequenceReader&lt;NucleotideCompound&gt;, CompoundSet&lt;NucleotideCompound&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/ChromosomeSequence.html" title="class in org.biojava.nbio.core.sequence">ChromosomeSequence</a></dt>
<dd>
<div class="block">Allows the creation of a ChromosomeSequence using a ProxyResequenceReader for the sequence with a custom CompoundSet</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#CIRCULAR_DNA" class="member-name-link">CIRCULAR_DNA</a> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html" title="class in org.biojava.nbio.core.sequence.io">GenbankWriterHelper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/LocationHelper.html#circularLocation(int,int,org.biojava.nbio.core.sequence.Strand,int)" class="member-name-link">circularLocation(int, int, Strand, int)</a> - Static method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/LocationHelper.html" title="class in org.biojava.nbio.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Converts a location which defines the outer bounds of a circular
 location and splits it into the required portions.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/Location.Tools.html#circularLocation(int,int,org.biojava.nbio.core.sequence.Strand,int)" class="member-name-link">circularLocation(int, int, Strand, int)</a> - Static method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/Location.Tools.html" title="class in org.biojava.nbio.core.sequence.location.template">Location.Tools</a></dt>
<dd>
<div class="block">Converts a location which defines the outer bounds of a circular
 location and splits it into the required portions.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/Equals.html#classEqual(java.lang.Object,java.lang.Object)" class="member-name-link">classEqual(Object, Object)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/Equals.html" title="class in org.biojava.nbio.core.util">Equals</a></dt>
<dd>
<div class="block">This method should be called before beginning any equals methods.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/util/ClasspathResource.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io.util">ClasspathResource</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/util/package-summary.html">org.biojava.nbio.core.sequence.io.util</a></dt>
<dd>
<div class="block">This object represents a classpath resource on the local system.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#%3Cinit%3E(java.lang.String)" class="member-name-link">ClasspathResource(String)</a> - Constructor for class org.biojava.nbio.core.sequence.io.util.<a href="../org/biojava/nbio/core/sequence/io/util/ClasspathResource.html" title="class in org.biojava.nbio.core.sequence.io.util">ClasspathResource</a></dt>
<dd>
<div class="block">Basic constructor only allowing you to specify where to find the file.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#%3Cinit%3E(java.lang.String,boolean)" class="member-name-link">ClasspathResource(String, boolean)</a> - Constructor for class org.biojava.nbio.core.sequence.io.util.<a href="../org/biojava/nbio/core/sequence/io/util/ClasspathResource.html" title="class in org.biojava.nbio.core.sequence.io.util">ClasspathResource</a></dt>
<dd>
<div class="block">Advanced constructor which allows you to optionally pre-cache the
 data</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#%3Cinit%3E(java.lang.String,boolean,boolean)" class="member-name-link">ClasspathResource(String, boolean, boolean)</a> - Constructor for class org.biojava.nbio.core.sequence.io.util.<a href="../org/biojava/nbio/core/sequence/io/util/ClasspathResource.html" title="class in org.biojava.nbio.core.sequence.io.util">ClasspathResource</a></dt>
<dd>
<div class="block">Advanced constructor which lets you set the preCache variable and to
 force the type of file we are decompressing.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/FlatFileCache.html#clear()" class="member-name-link">clear()</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/FlatFileCache.html" title="class in org.biojava.nbio.core.util">FlatFileCache</a></dt>
<dd>
<div class="block">Removes all elements from the cache</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/SoftHashMap.html#clear()" class="member-name-link">clear()</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SoftHashMap.html" title="class in org.biojava.nbio.core.util">SoftHashMap</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#clearCache()" class="member-name-link">clearCache()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html#clearCache()" class="member-name-link">clearCache()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Nullifies cached arrays/objects.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SimplePoint.html#clonePoint()" class="member-name-link">clonePoint()</a> - Method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/Point.html#clonePoint()" class="member-name-link">clonePoint()</a> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/Point.html" title="interface in org.biojava.nbio.core.sequence.location.template">Point</a></dt>
<dd>
<div class="block">Returns a copy of this point</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#close()" class="member-name-link">close()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava.nbio.core.sequence.io">BufferedReaderBytesRead</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaReader.html#close()" class="member-name-link">close()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaReader.html" title="class in org.biojava.nbio.core.sequence.io">FastaReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankReader.html#close()" class="member-name-link">close()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankReader.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/PrettyXMLWriter.html#close()" class="member-name-link">close()</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/PrettyXMLWriter.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/XMLWriter.html#close()" class="member-name-link">close()</a> - Method in interface org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/XMLWriter.html" title="interface in org.biojava.nbio.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Close this XMLWriter, and its underlying stream.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html#close(java.io.Closeable)" class="member-name-link">close(Closeable)</a> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Closes any Object which implements the interface <a href="https://docs.oracle.com/javase/8/docs/api/java/io/Closeable.html" title="class or interface in java.io" class="external-link"><code>Closeable</code></a> and
 sending any error to the logger but not forcing any explicit catching of
 stream errors.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/PrettyXMLWriter.html#closeTag(java.lang.String)" class="member-name-link">closeTag(String)</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/PrettyXMLWriter.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/XMLWriter.html#closeTag(java.lang.String)" class="member-name-link">closeTag(String)</a> - Method in interface org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/XMLWriter.html" title="interface in org.biojava.nbio.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Closes an un-qualified element.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/PrettyXMLWriter.html#closeTag(java.lang.String,java.lang.String)" class="member-name-link">closeTag(String, String)</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/PrettyXMLWriter.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/XMLWriter.html#closeTag(java.lang.String,java.lang.String)" class="member-name-link">closeTag(String, String)</a> - Method in interface org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/XMLWriter.html" title="interface in org.biojava.nbio.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Closes an element</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.StringFormat.html#CLUSTALW" class="member-name-link">CLUSTALW</a> - Enum constant in enum org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.StringFormat.html" title="enum in org.biojava.nbio.core.alignment.template">Profile.StringFormat</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/LightweightProfile.StringFormat.html#CLUSTALW" class="member-name-link">CLUSTALW</a> - Enum constant in enum org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/LightweightProfile.StringFormat.html" title="enum in org.biojava.nbio.core.sequence.template">LightweightProfile.StringFormat</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/SingleLinkageClustererProduct.html#clusterIt(org.biojava.nbio.core.util.SingleLinkageClusterer)" class="member-name-link">clusterIt(SingleLinkageClusterer)</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SingleLinkageClustererProduct.html" title="class in org.biojava.nbio.core.util">SingleLinkageClustererProduct</a></dt>
<dd>
<div class="block">Calculate the hierarchical clustering and store it in dendrogram array This is the naive implementation (o(n3)) of single linkage clustering as outlined in wikipedia: http://en.wikipedia.org/wiki/Single-linkage_clustering</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html#%3Cinit%3E(org.biojava.nbio.core.sequence.transcription.Table.CaseInsensitiveTriplet)" class="member-name-link">Codon(Table.CaseInsensitiveTriplet)</a> - Constructor for class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html#%3Cinit%3E(org.biojava.nbio.core.sequence.transcription.Table.CaseInsensitiveTriplet,org.biojava.nbio.core.sequence.compound.AminoAcidCompound,boolean,boolean)" class="member-name-link">Codon(Table.CaseInsensitiveTriplet, AminoAcidCompound, boolean, boolean)</a> - Constructor for class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a> - Class in <a href="../org/biojava/nbio/core/sequence/compound/package-summary.html">org.biojava.nbio.core.sequence.compound</a></dt>
<dd>
<div class="block">Define a codon</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html#%3Cinit%3E(org.biojava.nbio.core.sequence.compound.NucleotideCompound,org.biojava.nbio.core.sequence.compound.NucleotideCompound,org.biojava.nbio.core.sequence.compound.NucleotideCompound,boolean)" class="member-name-link">CodonCompound(NucleotideCompound, NucleotideCompound, NucleotideCompound, boolean)</a> - Constructor for class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACTableProduct.html#codonStrings()" class="member-name-link">codonStrings()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/IUPACTableProduct.html" title="class in org.biojava.nbio.core.sequence.io">IUPACTableProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/CDSComparator.html#compare(org.biojava.nbio.core.sequence.CDSSequence,org.biojava.nbio.core.sequence.CDSSequence)" class="member-name-link">compare(CDSSequence, CDSSequence)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/CDSComparator.html" title="class in org.biojava.nbio.core.sequence">CDSComparator</a></dt>
<dd>
<div class="block">Used to sort two CDSSequences where Negative Strand makes it tough</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/ExonComparator.html#compare(org.biojava.nbio.core.sequence.ExonSequence,org.biojava.nbio.core.sequence.ExonSequence)" class="member-name-link">compare(ExonSequence, ExonSequence)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/ExonComparator.html" title="class in org.biojava.nbio.core.sequence">ExonComparator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/SequenceComparator.html#compare(org.biojava.nbio.core.sequence.template.AbstractSequence,org.biojava.nbio.core.sequence.template.AbstractSequence)" class="member-name-link">compare(AbstractSequence&lt;?&gt;, AbstractSequence&lt;?&gt;)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/SequenceComparator.html" title="class in org.biojava.nbio.core.sequence">SequenceComparator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/FuzzyPoint.html#compareTo(org.biojava.nbio.core.sequence.location.template.Point)" class="member-name-link">compareTo(Point)</a> - Method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/FuzzyPoint.html" title="class in org.biojava.nbio.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SimplePoint.html#compareTo(org.biojava.nbio.core.sequence.location.template.Point)" class="member-name-link">compareTo(Point)</a> - Method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNATest.html#complement()" class="member-name-link">complement()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DNATest.html" title="class in org.biojava.nbio.core.sequence">DNATest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/ComplementCompound.html" class="type-name-link" title="interface in org.biojava.nbio.core.sequence.template">ComplementCompound</a> - Interface in <a href="../org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/views/ComplementSequenceView.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.views">ComplementSequenceView</a>&lt;<a href="../org/biojava/nbio/core/sequence/views/ComplementSequenceView.html" title="type parameter in ComplementSequenceView">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/ComplementCompound.html" title="interface in org.biojava.nbio.core.sequence.template">ComplementCompound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/views/package-summary.html">org.biojava.nbio.core.sequence.views</a></dt>
<dd>
<div class="block">For a given sequence this class will create a view over the top of it
 and for every request the code will return the complement of the underlying
 base e.g. base A will become base T</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/views/ComplementSequenceView.html#%3Cinit%3E(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">ComplementSequenceView(Sequence&lt;C&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.views.<a href="../org/biojava/nbio/core/sequence/views/ComplementSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">ComplementSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/LocationHelper.html#completeCircularPasses(int,int)" class="member-name-link">completeCircularPasses(int, int)</a> - Static method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/LocationHelper.html" title="class in org.biojava.nbio.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Works in a similar way to modulateCircularLocation but returns
 the number of complete passes over a Sequence length a circular
 location makes i.e. if we have a sequence of length 10 and the
 location 3..52 we make 4 complete passes through the genome to
 go from position 3 to position 52.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/Location.Tools.html#completeCircularPasses(int,int)" class="member-name-link">completeCircularPasses(int, int)</a> - Static method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/Location.Tools.html" title="class in org.biojava.nbio.core.sequence.location.template">Location.Tools</a></dt>
<dd>
<div class="block">Works in a similar way to modulateCircularLocation but returns
 the number of complete passes over a Sequence length a circular
 location makes i.e. if we have a sequence of length 10 and the
 location 3..52 we make 4 complete passes through the genome to
 go from position 3 to position 52.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/LocationTest.html#completePasses()" class="member-name-link">completePasses()</a> - Method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/LocationTest.html" title="class in org.biojava.nbio.core.sequence.location">LocationTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNATest.html#composition()" class="member-name-link">composition()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DNATest.html" title="class in org.biojava.nbio.core.sequence">DNATest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/Compound.html" class="type-name-link" title="interface in org.biojava.nbio.core.sequence.template">Compound</a> - Interface in <a href="../org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.Step.html#COMPOUND" class="member-name-link">COMPOUND</a> - Enum constant in enum org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.Step.html" title="enum in org.biojava.nbio.core.alignment.template">AlignedSequence.Step</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/exceptions/CompoundNotFoundException.html" class="type-name-link" title="class in org.biojava.nbio.core.exceptions">CompoundNotFoundException</a> - Exception in <a href="../org/biojava/nbio/core/exceptions/package-summary.html">org.biojava.nbio.core.exceptions</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/exceptions/CompoundNotFoundException.html#%3Cinit%3E(java.lang.String)" class="member-name-link">CompoundNotFoundException(String)</a> - Constructor for exception org.biojava.nbio.core.exceptions.<a href="../org/biojava/nbio/core/exceptions/CompoundNotFoundException.html" title="class in org.biojava.nbio.core.exceptions">CompoundNotFoundException</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#compoundsEqual(C,C)" class="member-name-link">compoundsEqual(C, C)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#compoundsEquivalent(C,C)" class="member-name-link">compoundsEquivalent(C, C)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/CompoundSet.html#compoundsEquivalent(C,C)" class="member-name-link">compoundsEquivalent(C, C)</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#compoundsEquivalent(org.biojava.nbio.core.sequence.compound.AminoAcidCompound,org.biojava.nbio.core.sequence.compound.AminoAcidCompound)" class="member-name-link">compoundsEquivalent(AminoAcidCompound, AminoAcidCompound)</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#compoundSet" class="member-name-link">compoundSet</a> - Variable in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/CompoundSet.html" class="type-name-link" title="interface in org.biojava.nbio.core.sequence.template">CompoundSet</a>&lt;<a href="../org/biojava/nbio/core/sequence/template/CompoundSet.html" title="type parameter in CompoundSet">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="../org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html#compoundToInt(org.biojava.nbio.core.sequence.compound.NucleotideCompound)" class="member-name-link">compoundToInt(NucleotideCompound)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.CaseInsensitiveTriplet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/CompoundTranslator.html" class="type-name-link" title="interface in org.biojava.nbio.core.sequence.template">CompoundTranslator</a>&lt;<a href="../org/biojava/nbio/core/sequence/template/CompoundTranslator.html" title="type parameter in CompoundTranslator">F</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>,<wbr><a href="../org/biojava/nbio/core/sequence/template/CompoundTranslator.html" title="type parameter in CompoundTranslator">T</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="../org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/ConcurrencyTools.html" class="type-name-link" title="class in org.biojava.nbio.core.util">ConcurrencyTools</a> - Class in <a href="../org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>
<div class="block">Static utility to easily share a thread pool for concurrent/parallel/lazy execution.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/LocationHelper.html#consistentAccessions(java.util.List)" class="member-name-link">consistentAccessions(List&lt;Location&gt;)</a> - Static method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/LocationHelper.html" title="class in org.biojava.nbio.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Scans a list of locations and returns true if all the given locations
 are linked to the same sequence.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/StringManipulationHelper.html#convertStreamToString(java.io.InputStream)" class="member-name-link">convertStreamToString(InputStream)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/StringManipulationHelper.html" title="class in org.biojava.nbio.core.util">StringManipulationHelper</a></dt>
<dd>
<div class="block">Converts an InputStream of text to a String, closing the stream
 before returning.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/FileDownloadUtils.html#copy(java.io.File,java.io.File)" class="member-name-link">copy(File, File)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/FileDownloadUtils.html" title="class in org.biojava.nbio.core.util">FileDownloadUtils</a></dt>
<dd>
<div class="block">Copy the content of file src to dst TODO since java 1.7 this is provided
 in java.nio.file.Files</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html#copy(java.io.InputStream,java.io.OutputStream)" class="member-name-link">copy(InputStream, OutputStream)</a> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Moves the bytes from input to output using a 4KB byte array.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html#copyEmblReference(org.biojava.nbio.core.sequence.io.embl.EmblReference)" class="member-name-link">copyEmblReference(EmblReference)</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReference</a></dt>
<dd>
<div class="block">return copy of EmblReference</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReaderProduct.html#copyInputStreamToFile(java.io.InputStream,java.io.File)" class="member-name-link">copyInputStreamToFile(InputStream, File)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReaderProduct.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReaderProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReaderTest.html#copyTestFiles()" class="member-name-link">copyTestFiles()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReaderTest.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReaderTest</a></dt>
<dd>
<div class="block">In <a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader"><code>GenbankProxySequenceReader</code></a> there is a check to see if the requested files are already in the temp
 directory before attempting to retrieve them from the remote server. so simply copying the test files to the temp
 directory avoids calling out to the server and hitting a 429 status code from the server which fails the build.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html#countAT(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">countAT(Sequence&lt;NucleotideCompound&gt;)</a> - Static method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Returns the count of AT in the given sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#countCompounds(C...)" class="member-name-link">countCompounds(C...)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#countCompounds(C...)" class="member-name-link">countCompounds(C...)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#countCompounds(C...)" class="member-name-link">countCompounds(C...)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#countCompounds(C...)" class="member-name-link">countCompounds(C...)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#countCompounds(C...)" class="member-name-link">countCompounds(C...)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#countCompounds(C...)" class="member-name-link">countCompounds(C...)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Counts the number of times a compound appears in this sequence store</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#countCompounds(C...)" class="member-name-link">countCompounds(C...)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#countCompounds(C...)" class="member-name-link">countCompounds(C...)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Delegates to <code>SequenceMixin#countCompounds(org.biojava.nbio.core.sequence.template.Sequence, C[])</code></div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#countCompounds(C...)" class="member-name-link">countCompounds(C...)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/Sequence.html#countCompounds(C...)" class="member-name-link">countCompounds(C...)</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Returns the number of times we found a compound in the Sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html#countCompounds(C...)" class="member-name-link">countCompounds(C...)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html#countCompounds(org.biojava.nbio.core.sequence.template.Sequence,C...)" class="member-name-link">countCompounds(Sequence&lt;C&gt;, C...)</a> - Static method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">For the given vargs of compounds this method counts the number of
 times those compounds appear in the given sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html#countGC(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">countGC(Sequence&lt;NucleotideCompound&gt;)</a> - Static method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Returns the count of GC in the given sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/CRC64Checksum.html" class="type-name-link" title="class in org.biojava.nbio.core.util">CRC64Checksum</a> - Class in <a href="../org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>
<div class="block">Utility class that calculates a CRC64 checksum on a stream of bytes.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/CRC64Checksum.html#%3Cinit%3E()" class="member-name-link">CRC64Checksum()</a> - Constructor for class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/CRC64Checksum.html" title="class in org.biojava.nbio.core.util">CRC64Checksum</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastHitBuilder.html#createBlastHit()" class="member-name-link">createBlastHit()</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastHitBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHitBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html#createBlastHsp()" class="member-name-link">createBlastHsp()</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHspBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastResultBuilder.html#createBlastResult()" class="member-name-link">createBlastResult()</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastResultBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastResultBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html#createIterator(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">createIterator(Sequence&lt;C&gt;)</a> - Static method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Creates a simple sequence iterator which moves through a sequence going
 from 1 to the length of the Sequence.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastTabularParser.html#createObjects(double)" class="member-name-link">createObjects(double)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastTabularParser.html" title="class in org.biojava.nbio.core.search.io.blast">BlastTabularParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParser.html#createObjects(double)" class="member-name-link">createObjects(double)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParser.html" title="class in org.biojava.nbio.core.search.io.blast">BlastXMLParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/ResultFactory.html#createObjects(double)" class="member-name-link">createObjects(double)</a> - Method in interface org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/ResultFactory.html" title="interface in org.biojava.nbio.core.search.io">ResultFactory</a></dt>
<dd>
<div class="block">Launch the parsing and get back a list of Result objects representing the
 search result in the specified file.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#createSequence(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">createSequence(Sequence&lt;F&gt;)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/CompoundTranslator.html#createSequence(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">createSequence(Sequence&lt;F&gt;)</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/CompoundTranslator.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#createSequence(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">createSequence(Sequence&lt;NucleotideCompound&gt;)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html" title="class in org.biojava.nbio.core.sequence.transcription">DNAToRNATranslator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#createSequence(org.biojava.nbio.core.sequence.template.Sequence,org.biojava.nbio.core.sequence.transcription.Frame)" class="member-name-link">createSequence(Sequence&lt;NucleotideCompound&gt;, Frame)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html" title="class in org.biojava.nbio.core.sequence.transcription">DNAToRNATranslator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#createSequences(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">createSequences(Sequence&lt;F&gt;)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/CompoundTranslator.html#createSequences(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">createSequences(Sequence&lt;F&gt;)</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/CompoundTranslator.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#createSequences(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">createSequences(Sequence&lt;NucleotideCompound&gt;)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html" title="class in org.biojava.nbio.core.sequence.transcription">DNAToRNATranslator</a></dt>
<dd>
<div class="block">Overloaded local version which delegates to an optional translator
 when told to (specified during construction).</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#createSequences(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">createSequences(Sequence&lt;NucleotideCompound&gt;)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html" title="class in org.biojava.nbio.core.sequence.transcription">RNAToAminoAcidTranslator</a></dt>
<dd>
<div class="block">Performs the core conversion of RNA to Peptide.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html#createSubSequence(org.biojava.nbio.core.sequence.template.Sequence,int,int)" class="member-name-link">createSubSequence(Sequence&lt;C&gt;, int, int)</a> - Static method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Creates a simple sub sequence view delimited by the given start and end.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.AnnotationType.html#CURATED" class="member-name-link">CURATED</a> - Enum constant in enum org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.AnnotationType.html" title="enum in org.biojava.nbio.core.sequence.template">AbstractSequence.AnnotationType</a></dt>
<dd>&nbsp;</dd>
</dl>
<a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Z</a>&nbsp;<br><a href="../allclasses-index.html">All&nbsp;Classes&nbsp;and&nbsp;Interfaces</a><span class="vertical-separator">|</span><a href="../allpackages-index.html">All&nbsp;Packages</a><span class="vertical-separator">|</span><a href="../constant-values.html">Constant&nbsp;Field&nbsp;Values</a><span class="vertical-separator">|</span><a href="../serialized-form.html">Serialized&nbsp;Form</a></main>
</div>
</div>
</body>
</html>
