<!DOCTYPE HTML>
<html lang="pt">
<head>
<!-- Generated by javadoc (17) on Sun May 15 18:36:12 WEST 2022 -->
<title>H-Index</title>
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta name="dc.created" content="2022-05-15">
<meta name="description" content="index: H">
<meta name="generator" content="javadoc/IndexWriter">
<link rel="stylesheet" type="text/css" href="../stylesheet.css" title="Style">
<link rel="stylesheet" type="text/css" href="../script-dir/jquery-ui.min.css" title="Style">
<link rel="stylesheet" type="text/css" href="../jquery-ui.overrides.css" title="Style">
<script type="text/javascript" src="../script.js"></script>
<script type="text/javascript" src="../script-dir/jquery-3.5.1.min.js"></script>
<script type="text/javascript" src="../script-dir/jquery-ui.min.js"></script>
</head>
<body class="index-page">
<script type="text/javascript">var pathtoroot = "../";
loadScripts(document, 'script');</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<div class="flex-box">
<header role="banner" class="flex-header">
<nav role="navigation">
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="top-nav" id="navbar-top">
<div class="skip-nav"><a href="#skip-navbar-top" title="Skip navigation links">Skip navigation links</a></div>
<ul id="navbar-top-firstrow" class="nav-list" title="Navigation">
<li><a href="../index.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li>Use</li>
<li><a href="../overview-tree.html">Tree</a></li>
<li><a href="../deprecated-list.html">Deprecated</a></li>
<li class="nav-bar-cell1-rev">Index</li>
<li><a href="../help-doc.html#index">Help</a></li>
</ul>
</div>
<div class="sub-nav">
<div class="nav-list-search"><label for="search-input">SEARCH:</label>
<input type="text" id="search-input" value="search" disabled="disabled">
<input type="reset" id="reset-button" value="reset" disabled="disabled">
</div>
</div>
<!-- ========= END OF TOP NAVBAR ========= -->
<span class="skip-nav" id="skip-navbar-top"></span></nav>
</header>
<div class="flex-content">
<main role="main">
<div class="header">
<h1>Index</h1>
</div>
<a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Z</a>&nbsp;<br><a href="../allclasses-index.html">All&nbsp;Classes&nbsp;and&nbsp;Interfaces</a><span class="vertical-separator">|</span><a href="../allpackages-index.html">All&nbsp;Packages</a><span class="vertical-separator">|</span><a href="../constant-values.html">Constant&nbsp;Field&nbsp;Values</a><span class="vertical-separator">|</span><a href="../serialized-form.html">Serialized&nbsp;Form</a>
<h2 class="title" id="I:H">H</h2>
<dl class="index">
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#hasCompound(C)" class="member-name-link">hasCompound(C)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/CompoundSet.html#hasCompound(C)" class="member-name-link">hasCompound(C)</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#hasCompound(org.biojava.nbio.core.sequence.compound.AminoAcidCompound)" class="member-name-link">hasCompound(AminoAcidCompound)</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html#hasGap(int)" class="member-name-link">hasGap(int)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.html#hasGap(int)" class="member-name-link">hasGap(int)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns true if any <a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template"><code>AlignedSequence</code></a> has a gap at the given index.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/Hashcoder.html#hash(int,boolean)" class="member-name-link">hash(int, boolean)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/Hashcoder.html" title="class in org.biojava.nbio.core.util">Hashcoder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/Hashcoder.html#hash(int,char)" class="member-name-link">hash(int, char)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/Hashcoder.html" title="class in org.biojava.nbio.core.util">Hashcoder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/Hashcoder.html#hash(int,double)" class="member-name-link">hash(int, double)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/Hashcoder.html" title="class in org.biojava.nbio.core.util">Hashcoder</a></dt>
<dd>
<div class="block">double support which is done using the same techinque as float hashing
 except we convert to a long not to an int.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/Hashcoder.html#hash(int,float)" class="member-name-link">hash(int, float)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/Hashcoder.html" title="class in org.biojava.nbio.core.util">Hashcoder</a></dt>
<dd>
<div class="block">float support done via <a href="https://docs.oracle.com/javase/8/docs/api/java/lang/Float.html#floatToIntBits(float)" title="class or interface in java.lang" class="external-link"><code>Float.floatToIntBits(float)</code></a> which allows
 the encoding of a float as an int.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/Hashcoder.html#hash(int,int)" class="member-name-link">hash(int, int)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/Hashcoder.html" title="class in org.biojava.nbio.core.util">Hashcoder</a></dt>
<dd>
<div class="block">Used for ints, bytes and shorts</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/Hashcoder.html#hash(int,long)" class="member-name-link">hash(int, long)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/Hashcoder.html" title="class in org.biojava.nbio.core.util">Hashcoder</a></dt>
<dd>
<div class="block">long support done by shifting by 32 (using unsigned shift)</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/Hashcoder.html#hash(int,java.lang.Object)" class="member-name-link">hash(int, Object)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/Hashcoder.html" title="class in org.biojava.nbio.core.util">Hashcoder</a></dt>
<dd>
<div class="block"><code>o</code> is a possibly-null object field, and possibly an
 array.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#hashCode()" class="member-name-link">hashCode()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hit.html#hashCode()" class="member-name-link">hashCode()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hit.html" title="class in org.biojava.nbio.core.search.io">Hit</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hsp.html#hashCode()" class="member-name-link">hashCode()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hsp.html" title="class in org.biojava.nbio.core.search.io">Hsp</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Result.html#hashCode()" class="member-name-link">hashCode()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Result.html" title="class in org.biojava.nbio.core.search.io">Result</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/AccessionID.html#hashCode()" class="member-name-link">hashCode()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/AccessionID.html" title="class in org.biojava.nbio.core.sequence">AccessionID</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html#hashCode()" class="member-name-link">hashCode()</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html#hashCode()" class="member-name-link">hashCode()</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#hashCode()" class="member-name-link">hashCode()</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound">NucleotideCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#hashCode()" class="member-name-link">hashCode()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#hashCode()" class="member-name-link">hashCode()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#hashCode()" class="member-name-link">hashCode()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/FuzzyPoint.html#hashCode()" class="member-name-link">hashCode()</a> - Method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/FuzzyPoint.html" title="class in org.biojava.nbio.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SimplePoint.html#hashCode()" class="member-name-link">hashCode()</a> - Method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#hashCode()" class="member-name-link">hashCode()</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#hashCode()" class="member-name-link">hashCode()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#hashCode()" class="member-name-link">hashCode()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#hashCode()" class="member-name-link">hashCode()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#hashCode()" class="member-name-link">hashCode()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#hashCode()" class="member-name-link">hashCode()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html#hashCode()" class="member-name-link">hashCode()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#hashCode()" class="member-name-link">hashCode()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#hashCode()" class="member-name-link">hashCode()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html#hashCode()" class="member-name-link">hashCode()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#hashCode()" class="member-name-link">hashCode()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html#hashCode()" class="member-name-link">hashCode()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.CaseInsensitiveTriplet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html#hashCode()" class="member-name-link">hashCode()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/Hashcoder.html" class="type-name-link" title="class in org.biojava.nbio.core.util">Hashcoder</a> - Class in <a href="../org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>
<div class="block">Contains helper methods for generating a HashCode without having to resort to
 the commons lang hashcode builders.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/Hashcoder.html#%3Cinit%3E()" class="member-name-link">Hashcoder()</a> - Constructor for class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/Hashcoder.html" title="class in org.biojava.nbio.core.util">Hashcoder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.SequenceIterator.html#hasNext()" class="member-name-link">hasNext()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.SequenceIterator.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin.SequenceIterator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#header" class="member-name-link">header</a> - Variable in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#headerParser" class="member-name-link">headerParser</a> - Variable in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hit.html" class="type-name-link" title="class in org.biojava.nbio.core.search.io">Hit</a> - Class in <a href="../org/biojava/nbio/core/search/io/package-summary.html">org.biojava.nbio.core.search.io</a></dt>
<dd>
<div class="block">This class models a search Hit.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hit.html#%3Cinit%3E(int,java.lang.String,java.lang.String,java.lang.String,int,java.util.List,org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">Hit(int, String, String, String, int, List&lt;Hsp&gt;, Sequence)</a> - Constructor for class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hit.html" title="class in org.biojava.nbio.core.search.io">Hit</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hsp.html" class="type-name-link" title="class in org.biojava.nbio.core.search.io">Hsp</a>&lt;<a href="../org/biojava/nbio/core/search/io/Hsp.html" title="type parameter in Hsp">S</a> extends <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="../org/biojava/nbio/core/search/io/Hsp.html" title="type parameter in Hsp">C</a>&gt;,<wbr><a href="../org/biojava/nbio/core/search/io/Hsp.html" title="type parameter in Hsp">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/search/io/package-summary.html">org.biojava.nbio.core.search.io</a></dt>
<dd>
<div class="block">This class models a search Hsp.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hsp.html#%3Cinit%3E(int,double,int,double,int,int,int,int,int,int,int,int,int,int,java.lang.String,java.lang.String,java.lang.String,java.lang.Double,java.lang.Integer)" class="member-name-link">Hsp(int, double, int, double, int, int, int, int, int, int, int, int, int, int, String, String, String, Double, Integer)</a> - Constructor for class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hsp.html" title="class in org.biojava.nbio.core.search.io">Hsp</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/HspTest.html" class="type-name-link" title="class in org.biojava.nbio.core.search.io">HspTest</a> - Class in <a href="../org/biojava/nbio/core/search/io/package-summary.html">org.biojava.nbio.core.search.io</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/HspTest.html#%3Cinit%3E()" class="member-name-link">HspTest()</a> - Constructor for class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/HspTest.html" title="class in org.biojava.nbio.core.search.io">HspTest</a></dt>
<dd>&nbsp;</dd>
</dl>
<a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Z</a>&nbsp;<br><a href="../allclasses-index.html">All&nbsp;Classes&nbsp;and&nbsp;Interfaces</a><span class="vertical-separator">|</span><a href="../allpackages-index.html">All&nbsp;Packages</a><span class="vertical-separator">|</span><a href="../constant-values.html">Constant&nbsp;Field&nbsp;Values</a><span class="vertical-separator">|</span><a href="../serialized-form.html">Serialized&nbsp;Form</a></main>
</div>
</div>
</body>
</html>
