<!DOCTYPE HTML>
<html lang="pt">
<head>
<!-- Generated by javadoc (17) on Sun May 15 18:36:12 WEST 2022 -->
<title>F-Index</title>
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta name="dc.created" content="2022-05-15">
<meta name="description" content="index: F">
<meta name="generator" content="javadoc/IndexWriter">
<link rel="stylesheet" type="text/css" href="../stylesheet.css" title="Style">
<link rel="stylesheet" type="text/css" href="../script-dir/jquery-ui.min.css" title="Style">
<link rel="stylesheet" type="text/css" href="../jquery-ui.overrides.css" title="Style">
<script type="text/javascript" src="../script.js"></script>
<script type="text/javascript" src="../script-dir/jquery-3.5.1.min.js"></script>
<script type="text/javascript" src="../script-dir/jquery-ui.min.js"></script>
</head>
<body class="index-page">
<script type="text/javascript">var pathtoroot = "../";
loadScripts(document, 'script');</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<div class="flex-box">
<header role="banner" class="flex-header">
<nav role="navigation">
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="top-nav" id="navbar-top">
<div class="skip-nav"><a href="#skip-navbar-top" title="Skip navigation links">Skip navigation links</a></div>
<ul id="navbar-top-firstrow" class="nav-list" title="Navigation">
<li><a href="../index.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li>Use</li>
<li><a href="../overview-tree.html">Tree</a></li>
<li><a href="../deprecated-list.html">Deprecated</a></li>
<li class="nav-bar-cell1-rev">Index</li>
<li><a href="../help-doc.html#index">Help</a></li>
</ul>
</div>
<div class="sub-nav">
<div class="nav-list-search"><label for="search-input">SEARCH:</label>
<input type="text" id="search-input" value="search" disabled="disabled">
<input type="reset" id="reset-button" value="reset" disabled="disabled">
</div>
</div>
<!-- ========= END OF TOP NAVBAR ========= -->
<span class="skip-nav" id="skip-navbar-top"></span></nav>
</header>
<div class="flex-content">
<main role="main">
<div class="header">
<h1>Index</h1>
</div>
<a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Z</a>&nbsp;<br><a href="../allclasses-index.html">All&nbsp;Classes&nbsp;and&nbsp;Interfaces</a><span class="vertical-separator">|</span><a href="../allpackages-index.html">All&nbsp;Packages</a><span class="vertical-separator">|</span><a href="../constant-values.html">Constant&nbsp;Field&nbsp;Values</a><span class="vertical-separator">|</span><a href="../serialized-form.html">Serialized&nbsp;Form</a>
<h2 class="title" id="I:F">F</h2>
<dl class="index">
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.StringFormat.html#FASTA" class="member-name-link">FASTA</a> - Enum constant in enum org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.StringFormat.html" title="enum in org.biojava.nbio.core.alignment.template">Profile.StringFormat</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/LightweightProfile.StringFormat.html#FASTA" class="member-name-link">FASTA</a> - Enum constant in enum org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/LightweightProfile.StringFormat.html" title="enum in org.biojava.nbio.core.sequence.template">LightweightProfile.StringFormat</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaGeneWriter.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">FastaGeneWriter</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">A Gene sequence has a Positive or Negative Strand where we want to write out to a stream the 5 to 3 prime version.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#%3Cinit%3E(java.io.OutputStream,java.util.Collection,org.biojava.nbio.core.sequence.io.template.FastaHeaderFormatInterface,boolean)" class="member-name-link">FastaGeneWriter(OutputStream, Collection&lt;GeneSequence&gt;, FastaHeaderFormatInterface&lt;GeneSequence, NucleotideCompound&gt;, boolean)</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaGeneWriter.html" title="class in org.biojava.nbio.core.sequence.io">FastaGeneWriter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#%3Cinit%3E(java.io.OutputStream,java.util.Collection,org.biojava.nbio.core.sequence.io.template.FastaHeaderFormatInterface,boolean,int)" class="member-name-link">FastaGeneWriter(OutputStream, Collection&lt;GeneSequence&gt;, FastaHeaderFormatInterface&lt;GeneSequence, NucleotideCompound&gt;, boolean, int)</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaGeneWriter.html" title="class in org.biojava.nbio.core.sequence.io">FastaGeneWriter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/template/FastaHeaderFormatInterface.html" class="type-name-link" title="interface in org.biojava.nbio.core.sequence.io.template">FastaHeaderFormatInterface</a>&lt;<a href="../org/biojava/nbio/core/sequence/io/template/FastaHeaderFormatInterface.html" title="type parameter in FastaHeaderFormatInterface">S</a> extends <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;?&gt;,<wbr><a href="../org/biojava/nbio/core/sequence/io/template/FastaHeaderFormatInterface.html" title="type parameter in FastaHeaderFormatInterface">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="../org/biojava/nbio/core/sequence/io/template/package-summary.html">org.biojava.nbio.core.sequence.io.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaReader.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">FastaReader</a>&lt;<a href="../org/biojava/nbio/core/sequence/io/FastaReader.html" title="type parameter in FastaReader">S</a> extends <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;?&gt;,<wbr><a href="../org/biojava/nbio/core/sequence/io/FastaReader.html" title="type parameter in FastaReader">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">Use FastaReaderHelper as an example of how to use this class where FastaReaderHelper should be the
 primary class used to read Fasta files</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaReader.html#%3Cinit%3E(java.io.File,org.biojava.nbio.core.sequence.io.template.SequenceHeaderParserInterface,org.biojava.nbio.core.sequence.io.template.SequenceCreatorInterface)" class="member-name-link">FastaReader(File, SequenceHeaderParserInterface&lt;S, C&gt;, SequenceCreatorInterface&lt;C&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaReader.html" title="class in org.biojava.nbio.core.sequence.io">FastaReader</a></dt>
<dd>
<div class="block">If you are going to use the FileProxyProteinSequenceCreator then you
 need to use this constructor because we need details about
 the location of the file.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaReader.html#%3Cinit%3E(java.io.InputStream,org.biojava.nbio.core.sequence.io.template.SequenceHeaderParserInterface,org.biojava.nbio.core.sequence.io.template.SequenceCreatorInterface)" class="member-name-link">FastaReader(InputStream, SequenceHeaderParserInterface&lt;S, C&gt;, SequenceCreatorInterface&lt;C&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaReader.html" title="class in org.biojava.nbio.core.sequence.io">FastaReader</a></dt>
<dd>
<div class="block">If you are going to use FileProxyProteinSequenceCreator then do not use this constructor because we need details about
 local file offsets for quick reads.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaReaderHelper.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">FastaReaderHelper</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaReaderHelper.html#%3Cinit%3E()" class="member-name-link">FastaReaderHelper()</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaReaderHelper.html" title="class in org.biojava.nbio.core.sequence.io">FastaReaderHelper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaReaderTest.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">FastaReaderTest</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaReaderTest.html#%3Cinit%3E()" class="member-name-link">FastaReaderTest()</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaReaderTest.html" title="class in org.biojava.nbio.core.sequence.io">FastaReaderTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaSequenceParser.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">FastaSequenceParser</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">Used to parse a stream of a fasta file to get the sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaSequenceParser.html#%3Cinit%3E()" class="member-name-link">FastaSequenceParser()</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">FastaSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaWriter.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">FastaWriter</a>&lt;<a href="../org/biojava/nbio/core/sequence/io/FastaWriter.html" title="type parameter in FastaWriter">S</a> extends <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;?&gt;,<wbr><a href="../org/biojava/nbio/core/sequence/io/FastaWriter.html" title="type parameter in FastaWriter">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">The FastaWriter writes a collection of sequences to an outputStream.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaWriter.html#%3Cinit%3E(java.io.OutputStream,java.util.Collection,org.biojava.nbio.core.sequence.io.template.FastaHeaderFormatInterface)" class="member-name-link">FastaWriter(OutputStream, Collection&lt;S&gt;, FastaHeaderFormatInterface&lt;S, C&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaWriter.html" title="class in org.biojava.nbio.core.sequence.io">FastaWriter</a></dt>
<dd>
<div class="block">Use default line length of 60</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaWriter.html#%3Cinit%3E(java.io.OutputStream,java.util.Collection,org.biojava.nbio.core.sequence.io.template.FastaHeaderFormatInterface,int)" class="member-name-link">FastaWriter(OutputStream, Collection&lt;S&gt;, FastaHeaderFormatInterface&lt;S, C&gt;, int)</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaWriter.html" title="class in org.biojava.nbio.core.sequence.io">FastaWriter</a></dt>
<dd>
<div class="block">Set custom lineLength</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaWriterHelper.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">FastaWriterHelper</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">The class that should be used to write out fasta file of a sequence collection</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaWriterHelper.html#%3Cinit%3E()" class="member-name-link">FastaWriterHelper()</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaWriterHelper.html" title="class in org.biojava.nbio.core.sequence.io">FastaWriterHelper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaWriterTest.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">FastaWriterTest</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaWriterTest.html#%3Cinit%3E()" class="member-name-link">FastaWriterTest()</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaWriterTest.html" title="class in org.biojava.nbio.core.sequence.io">FastaWriterTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a>&lt;<a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="type parameter in FeatureDbReferenceInfo">S</a> extends <a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="type parameter in FeatureDbReferenceInfo">C</a>&gt;,<wbr><a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="type parameter in FeatureDbReferenceInfo">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/features/package-summary.html">org.biojava.nbio.core.sequence.features</a></dt>
<dd>
<div class="block">It is <a href="../org/biojava/nbio/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features"><code>DBReferenceInfo</code></a> which implements <a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features"><code>FeatureInterface</code></a>.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#%3Cinit%3E(java.lang.String,java.lang.String)" class="member-name-link">FeatureDbReferenceInfo(String, String)</a> - Constructor for class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html" class="type-name-link" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a>&lt;<a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="type parameter in FeatureInterface">S</a> extends <a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="type parameter in FeatureInterface">C</a>&gt;,<wbr><a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="type parameter in FeatureInterface">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="../org/biojava/nbio/core/sequence/features/package-summary.html">org.biojava.nbio.core.sequence.features</a></dt>
<dd>
<div class="block">Interface class to handle describing arbitrary features.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureRetriever.html" class="type-name-link" title="interface in org.biojava.nbio.core.sequence.features">FeatureRetriever</a>&lt;<a href="../org/biojava/nbio/core/sequence/features/FeatureRetriever.html" title="type parameter in FeatureRetriever">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="../org/biojava/nbio/core/sequence/features/package-summary.html">org.biojava.nbio.core.sequence.features</a></dt>
<dd>
<div class="block">If a SequenceProxyReader implements this interface then that external source
 has a list features</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#features" class="member-name-link">features</a> - Variable in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeaturesKeyWordInterface.html" class="type-name-link" title="interface in org.biojava.nbio.core.sequence.features">FeaturesKeyWordInterface</a> - Interface in <a href="../org/biojava/nbio/core/sequence/features/package-summary.html">org.biojava.nbio.core.sequence.features</a></dt>
<dd>
<div class="block">Models the keywords that are annotated for a protein sequence at Uniprot.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReaderProduct.html#fetchFromCache(java.lang.String)" class="member-name-link">fetchFromCache(String)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReaderProduct.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReaderProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReaderProduct.html#fetchUniprotXML(java.lang.String)" class="member-name-link">fetchUniprotXML(String)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReaderProduct.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReaderProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/FileDownloadUtils.html" class="type-name-link" title="class in org.biojava.nbio.core.util">FileDownloadUtils</a> - Class in <a href="../org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/FileDownloadUtils.html#%3Cinit%3E()" class="member-name-link">FileDownloadUtils()</a> - Constructor for class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/FileDownloadUtils.html" title="class in org.biojava.nbio.core.util">FileDownloadUtils</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">FileProxyDNASequenceCreator</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">This class is a good example of using the SequenceCreatorInterface where during parsing of the stream
 the sequence and the offset index are passed to create a Protein sequence that will be loaded in lazily.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#%3Cinit%3E(java.io.File,org.biojava.nbio.core.sequence.template.CompoundSet,org.biojava.nbio.core.sequence.io.template.SequenceParserInterface)" class="member-name-link">FileProxyDNASequenceCreator(File, CompoundSet&lt;NucleotideCompound&gt;, SequenceParserInterface)</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">FileProxyDNASequenceCreator</a></dt>
<dd>
<div class="block">Need File so that we can store full path name in SequenceFileProxyLoader for Random File access as a quick read</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">FileProxyProteinSequenceCreator</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">This class is a good example of using the SequenceCreatorInterface where during parsing of the stream
 the sequence and the offset index are passed to create a Protein sequence that will be loaded in lazily.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#%3Cinit%3E(java.io.File,org.biojava.nbio.core.sequence.template.CompoundSet,org.biojava.nbio.core.sequence.io.template.SequenceParserInterface)" class="member-name-link">FileProxyProteinSequenceCreator(File, CompoundSet&lt;AminoAcidCompound&gt;, SequenceParserInterface)</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">FileProxyProteinSequenceCreator</a></dt>
<dd>
<div class="block">Need File so that we can store full path name in SequenceFileProxyLoader for Random File access as a quick read</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FileProxyRNASequenceCreator.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">FileProxyRNASequenceCreator</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">This class is a good example of using the SequenceCreatorInterface where during parsing of the stream
 the sequence and the offset index are passed to create a Protein sequence that will be loaded in lazily.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FileProxyRNASequenceCreator.html#%3Cinit%3E(java.io.File,org.biojava.nbio.core.sequence.template.CompoundSet,org.biojava.nbio.core.sequence.io.template.SequenceParserInterface)" class="member-name-link">FileProxyRNASequenceCreator(File, CompoundSet&lt;NucleotideCompound&gt;, SequenceParserInterface)</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FileProxyRNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">FileProxyRNASequenceCreator</a></dt>
<dd>
<div class="block">Need File so that we can store full path name in SequenceFileProxyLoader for Random File access as a quick read</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/UncompressInputStreamData.html#finchar" class="member-name-link">finchar</a> - Variable in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/UncompressInputStreamData.html" title="class in org.biojava.nbio.core.util">UncompressInputStreamData</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/SingleLinkageClustererProduct.html#firstClusterId(java.util.Map,int,org.biojava.nbio.core.util.SingleLinkageClusterer)" class="member-name-link">firstClusterId(Map&lt;Integer, Set&lt;Integer&gt;&gt;, int, SingleLinkageClusterer)</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SingleLinkageClustererProduct.html" title="class in org.biojava.nbio.core.util">SingleLinkageClustererProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/FlatFileCache.html" class="type-name-link" title="class in org.biojava.nbio.core.util">FlatFileCache</a> - Class in <a href="../org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>
<div class="block">Provides a cache for storing multiple small files in memory.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNATest.html#fourBit()" class="member-name-link">fourBit()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DNATest.html" title="class in org.biojava.nbio.core.sequence">DNATest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html#%3Cinit%3E(java.lang.String,org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">FourBitArrayWorker(String, CompoundSet&lt;C&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">FourBitSequenceReader.FourBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html#%3Cinit%3E(org.biojava.nbio.core.sequence.template.CompoundSet,int)" class="member-name-link">FourBitArrayWorker(CompoundSet&lt;C&gt;, int)</a> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">FourBitSequenceReader.FourBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html#%3Cinit%3E(org.biojava.nbio.core.sequence.template.CompoundSet,int%5B%5D)" class="member-name-link">FourBitArrayWorker(CompoundSet&lt;C&gt;, int[])</a> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">FourBitSequenceReader.FourBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html#%3Cinit%3E(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">FourBitArrayWorker(Sequence&lt;C&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">FourBitSequenceReader.FourBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.storage">FourBitSequenceReader</a>&lt;<a href="../org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html" title="type parameter in FourBitSequenceReader">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/storage/package-summary.html">org.biojava.nbio.core.sequence.storage</a></dt>
<dd>
<div class="block">Four bit encoding of the bit formats.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#%3Cinit%3E(java.lang.String,org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">FourBitSequenceReader(String, CompoundSet&lt;C&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">FourBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#%3Cinit%3E(java.lang.String,org.biojava.nbio.core.sequence.template.CompoundSet,org.biojava.nbio.core.sequence.AccessionID)" class="member-name-link">FourBitSequenceReader(String, CompoundSet&lt;C&gt;, AccessionID)</a> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">FourBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#%3Cinit%3E(org.biojava.nbio.core.sequence.storage.FourBitSequenceReader.FourBitArrayWorker)" class="member-name-link">FourBitSequenceReader(FourBitSequenceReader.FourBitArrayWorker&lt;C&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">FourBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#%3Cinit%3E(org.biojava.nbio.core.sequence.storage.FourBitSequenceReader.FourBitArrayWorker,org.biojava.nbio.core.sequence.AccessionID)" class="member-name-link">FourBitSequenceReader(FourBitSequenceReader.FourBitArrayWorker&lt;C&gt;, AccessionID)</a> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">FourBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html#%3Cinit%3E(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">FourBitSequenceReader(Sequence&lt;C&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">FourBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.storage">FourBitSequenceReader.FourBitArrayWorker</a>&lt;<a href="../org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="type parameter in FourBitSequenceReader.FourBitArrayWorker">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/storage/package-summary.html">org.biojava.nbio.core.sequence.storage</a></dt>
<dd>
<div class="block">A four bit per compound implementation of the bit array worker code.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Frame.html" class="type-name-link" title="enum in org.biojava.nbio.core.sequence.transcription">Frame</a> - Enum in <a href="../org/biojava/nbio/core/sequence/transcription/package-summary.html">org.biojava.nbio.core.sequence.transcription</a></dt>
<dd>
<div class="block">Indicates a way of translating a sequence.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/UncompressInputStreamData.html#free_ent" class="member-name-link">free_ent</a> - Variable in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/UncompressInputStreamData.html" title="class in org.biojava.nbio.core.util">UncompressInputStreamData</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/SequenceOptimizationHints.SequenceUsage.html#FULL_SEQUENCE_DATA" class="member-name-link">FULL_SEQUENCE_DATA</a> - Enum constant in enum org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/SequenceOptimizationHints.SequenceUsage.html" title="enum in org.biojava.nbio.core.sequence">SequenceOptimizationHints.SequenceUsage</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/FuzzyPoint.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.location">FuzzyPoint</a> - Class in <a href="../org/biojava/nbio/core/sequence/location/package-summary.html">org.biojava.nbio.core.sequence.location</a></dt>
<dd>
<div class="block">Implementation for resolving fuzzy locations.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/FuzzyPoint.html#%3Cinit%3E(int,int)" class="member-name-link">FuzzyPoint(int, int)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/FuzzyPoint.html" title="class in org.biojava.nbio.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/FuzzyPoint.html#%3Cinit%3E(int,int,org.biojava.nbio.core.sequence.location.template.Point.Resolver)" class="member-name-link">FuzzyPoint(int, int, Point.Resolver&lt;FuzzyPoint&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/FuzzyPoint.html" title="class in org.biojava.nbio.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/FuzzyPoint.html#%3Cinit%3E(int,int,org.biojava.nbio.core.sequence.location.template.Point.Resolver,boolean,boolean)" class="member-name-link">FuzzyPoint(int, int, Point.Resolver&lt;FuzzyPoint&gt;, boolean, boolean)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/FuzzyPoint.html" title="class in org.biojava.nbio.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
</dl>
<a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Z</a>&nbsp;<br><a href="../allclasses-index.html">All&nbsp;Classes&nbsp;and&nbsp;Interfaces</a><span class="vertical-separator">|</span><a href="../allpackages-index.html">All&nbsp;Packages</a><span class="vertical-separator">|</span><a href="../constant-values.html">Constant&nbsp;Field&nbsp;Values</a><span class="vertical-separator">|</span><a href="../serialized-form.html">Serialized&nbsp;Form</a></main>
</div>
</div>
</body>
</html>
