<!DOCTYPE HTML>
<html lang="pt">
<head>
<!-- Generated by javadoc (17) on Sun May 15 18:36:12 WEST 2022 -->
<title>S-Index</title>
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta name="dc.created" content="2022-05-15">
<meta name="description" content="index: S">
<meta name="generator" content="javadoc/IndexWriter">
<link rel="stylesheet" type="text/css" href="../stylesheet.css" title="Style">
<link rel="stylesheet" type="text/css" href="../script-dir/jquery-ui.min.css" title="Style">
<link rel="stylesheet" type="text/css" href="../jquery-ui.overrides.css" title="Style">
<script type="text/javascript" src="../script.js"></script>
<script type="text/javascript" src="../script-dir/jquery-3.5.1.min.js"></script>
<script type="text/javascript" src="../script-dir/jquery-ui.min.js"></script>
</head>
<body class="index-page">
<script type="text/javascript">var pathtoroot = "../";
loadScripts(document, 'script');</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<div class="flex-box">
<header role="banner" class="flex-header">
<nav role="navigation">
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="top-nav" id="navbar-top">
<div class="skip-nav"><a href="#skip-navbar-top" title="Skip navigation links">Skip navigation links</a></div>
<ul id="navbar-top-firstrow" class="nav-list" title="Navigation">
<li><a href="../index.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li>Use</li>
<li><a href="../overview-tree.html">Tree</a></li>
<li><a href="../deprecated-list.html">Deprecated</a></li>
<li class="nav-bar-cell1-rev">Index</li>
<li><a href="../help-doc.html#index">Help</a></li>
</ul>
</div>
<div class="sub-nav">
<div class="nav-list-search"><label for="search-input">SEARCH:</label>
<input type="text" id="search-input" value="search" disabled="disabled">
<input type="reset" id="reset-button" value="reset" disabled="disabled">
</div>
</div>
<!-- ========= END OF TOP NAVBAR ========= -->
<span class="skip-nav" id="skip-navbar-top"></span></nav>
</header>
<div class="flex-content">
<main role="main">
<div class="header">
<h1>Index</h1>
</div>
<a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Z</a>&nbsp;<br><a href="../allclasses-index.html">All&nbsp;Classes&nbsp;and&nbsp;Interfaces</a><span class="vertical-separator">|</span><a href="../allpackages-index.html">All&nbsp;Packages</a><span class="vertical-separator">|</span><a href="../constant-values.html">Constant&nbsp;Field&nbsp;Values</a><span class="vertical-separator">|</span><a href="../serialized-form.html">Serialized&nbsp;Form</a>
<h2 class="title" id="I:S">S</h2>
<dl class="index">
<dt><a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html" class="type-name-link" title="class in org.biojava.nbio.core.alignment.matrices">ScaledSubstitutionMatrix</a> - Class in <a href="../org/biojava/nbio/core/alignment/matrices/package-summary.html">org.biojava.nbio.core.alignment.matrices</a></dt>
<dd>
<div class="block">The biojava-alignment module represents substitution matrices with short
 values.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html#%3Cinit%3E()" class="member-name-link">ScaledSubstitutionMatrix()</a> - Constructor for class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/SearchIO.html" class="type-name-link" title="class in org.biojava.nbio.core.search.io">SearchIO</a> - Class in <a href="../org/biojava/nbio/core/search/io/package-summary.html">org.biojava.nbio.core.search.io</a></dt>
<dd>
<div class="block">Designed by Paolo Pavan.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/SearchIO.html#%3Cinit%3E(java.io.File)" class="member-name-link">SearchIO(File)</a> - Constructor for class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/SearchIO.html" title="class in org.biojava.nbio.core.search.io">SearchIO</a></dt>
<dd>
<div class="block">Build a SearchIO reader and tries to select the appropriate parser inspecting
 file extension.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/SearchIO.html#%3Cinit%3E(java.io.File,org.biojava.nbio.core.search.io.ResultFactory)" class="member-name-link">SearchIO(File, ResultFactory)</a> - Constructor for class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/SearchIO.html" title="class in org.biojava.nbio.core.search.io">SearchIO</a></dt>
<dd>
<div class="block">Build a SearchIO reader and specify a ResultFactory object to be used
 for parsing</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/SearchIO.html#%3Cinit%3E(java.io.File,org.biojava.nbio.core.search.io.ResultFactory,double)" class="member-name-link">SearchIO(File, ResultFactory, double)</a> - Constructor for class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/SearchIO.html" title="class in org.biojava.nbio.core.search.io">SearchIO</a></dt>
<dd>
<div class="block">Build a SearchIO reader, specify a ResultFactory object to be used for parsing
 and filter hsp retrieved by a e-value threshold.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/SearchIOTest.html" class="type-name-link" title="class in org.biojava.nbio.core.search.io">SearchIOTest</a> - Class in <a href="../org/biojava/nbio/core/search/io/package-summary.html">org.biojava.nbio.core.search.io</a></dt>
<dd>
<div class="block">Designed by Paolo Pavan.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/SearchIOTest.html#%3Cinit%3E()" class="member-name-link">SearchIOTest()</a> - Constructor for class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/SearchIOTest.html" title="class in org.biojava.nbio.core.search.io">SearchIOTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/exceptions/Messages.html#SECTIONKEYNULL" class="member-name-link">SECTIONKEYNULL</a> - Static variable in class org.biojava.nbio.core.exceptions.<a href="../org/biojava/nbio/core/exceptions/Messages.html" title="class in org.biojava.nbio.core.exceptions">Messages</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/Hashcoder.html#SEED" class="member-name-link">SEED</a> - Static variable in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/Hashcoder.html" title="class in org.biojava.nbio.core.util">Hashcoder</a></dt>
<dd>
<div class="block">An initial value for a <code>hashCode</code>, to which we add
 contributions from fields.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/XMLHelper.html#selectElements(org.w3c.dom.Element,java.lang.String)" class="member-name-link">selectElements(Element, String)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/XMLHelper.html" title="class in org.biojava.nbio.core.util">XMLHelper</a></dt>
<dd>
<div class="block">Gets a list of elements matching xpathExpression.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/XMLHelper.html#selectParentElement(org.w3c.dom.Element,java.lang.String)" class="member-name-link">selectParentElement(Element, String)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/XMLHelper.html" title="class in org.biojava.nbio.core.util">XMLHelper</a></dt>
<dd>
<div class="block">Given an element, searches upwards through ancestor Elements till the first Element
 matching the requests parentName is found.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/XMLHelper.html#selectSingleElement(org.w3c.dom.Element,java.lang.String)" class="member-name-link">selectSingleElement(Element, String)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/XMLHelper.html" title="class in org.biojava.nbio.core.util">XMLHelper</a></dt>
<dd>
<div class="block">If xpathExpression is a plain string with no '/' characterr, this is 
 interpreted as a child element name to search for.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#seqArraySize(int)" class="member-name-link">seqArraySize(int)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/Sequence.html" class="type-name-link" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="type parameter in Sequence">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="../org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>
<div class="block">Main interface for defining a collection of Compounds and accessing them
 using biological indexes</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.storage">SequenceAsStringHelper</a>&lt;<a href="../org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html" title="type parameter in SequenceAsStringHelper">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/storage/package-summary.html">org.biojava.nbio.core.sequence.storage</a></dt>
<dd>
<div class="block">This is a common method that can be used across multiple storage/proxy implementations to
 handle Negative strand and other interesting elements of sequence data.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#%3Cinit%3E()" class="member-name-link">SequenceAsStringHelper()</a> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html" title="class in org.biojava.nbio.core.sequence.storage">SequenceAsStringHelper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/SequenceComparator.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence">SequenceComparator</a> - Class in <a href="../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">Used to sort sequences in ascending order of  bioBegin property.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/SequenceComparator.html#%3Cinit%3E()" class="member-name-link">SequenceComparator()</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/SequenceComparator.html" title="class in org.biojava.nbio.core.sequence">SequenceComparator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html" class="type-name-link" title="interface in org.biojava.nbio.core.sequence.io.template">SequenceCreatorInterface</a>&lt;<a href="../org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html" title="type parameter in SequenceCreatorInterface">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="../org/biojava/nbio/core/sequence/io/template/package-summary.html">org.biojava.nbio.core.sequence.io.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNATest.html#sequenceEquality()" class="member-name-link">sequenceEquality()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DNATest.html" title="class in org.biojava.nbio.core.sequence">DNATest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html#sequenceEquality(org.biojava.nbio.core.sequence.template.Sequence,org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">sequenceEquality(Sequence&lt;C&gt;, Sequence&lt;C&gt;)</a> - Static method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">A case-sensitive manner of comparing two sequence objects together.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html#sequenceEqualityIgnoreCase(org.biojava.nbio.core.sequence.template.Sequence,org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">sequenceEqualityIgnoreCase(Sequence&lt;C&gt;, Sequence&lt;C&gt;)</a> - Static method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">A case-insensitive manner of comparing two sequence objects together.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a>&lt;<a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="type parameter in SequenceFileProxyLoader">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/loader/package-summary.html">org.biojava.nbio.core.sequence.loader</a></dt>
<dd>
<div class="block">This class represents the storage container of a sequence stored in a fasta file where
 the initial parsing of the file we store the offset and length of the sequence.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#%3Cinit%3E(java.io.File,org.biojava.nbio.core.sequence.io.template.SequenceParserInterface,long,int,org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">SequenceFileProxyLoader(File, SequenceParserInterface, long, int, CompoundSet&lt;C&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/template/SequenceHeaderParserInterface.html" class="type-name-link" title="interface in org.biojava.nbio.core.sequence.io.template">SequenceHeaderParserInterface</a>&lt;<a href="../org/biojava/nbio/core/sequence/io/template/SequenceHeaderParserInterface.html" title="type parameter in SequenceHeaderParserInterface">S</a> extends <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;?&gt;,<wbr><a href="../org/biojava/nbio/core/sequence/io/template/SequenceHeaderParserInterface.html" title="type parameter in SequenceHeaderParserInterface">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="../org/biojava/nbio/core/sequence/io/template/package-summary.html">org.biojava.nbio.core.sequence.io.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.SequenceIterator.html#%3Cinit%3E(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">SequenceIterator(Sequence&lt;C&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.SequenceIterator.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin.SequenceIterator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SequenceLocation.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.location">SequenceLocation</a>&lt;<a href="../org/biojava/nbio/core/sequence/location/SequenceLocation.html" title="type parameter in SequenceLocation">S</a> extends <a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="../org/biojava/nbio/core/sequence/location/SequenceLocation.html" title="type parameter in SequenceLocation">C</a>&gt;,<wbr><a href="../org/biojava/nbio/core/sequence/location/SequenceLocation.html" title="type parameter in SequenceLocation">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/location/package-summary.html">org.biojava.nbio.core.sequence.location</a></dt>
<dd>
<div class="block">A location in a sequence that keeps a reference to its parent sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SequenceLocation.html#%3Cinit%3E(int,int,S)" class="member-name-link">SequenceLocation(int, int, S)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SequenceLocation.html" title="class in org.biojava.nbio.core.sequence.location">SequenceLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SequenceLocation.html#%3Cinit%3E(int,int,S,org.biojava.nbio.core.sequence.Strand)" class="member-name-link">SequenceLocation(int, int, S, Strand)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SequenceLocation.html" title="class in org.biojava.nbio.core.sequence.location">SequenceLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SequenceLocation.html#%3Cinit%3E(int,int,S,org.biojava.nbio.core.sequence.Strand,boolean,java.util.List)" class="member-name-link">SequenceLocation(int, int, S, Strand, boolean, List&lt;Location&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SequenceLocation.html" title="class in org.biojava.nbio.core.sequence.location">SequenceLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SequenceLocation.html#%3Cinit%3E(org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.location.template.Point,S,org.biojava.nbio.core.sequence.Strand)" class="member-name-link">SequenceLocation(Point, Point, S, Strand)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SequenceLocation.html" title="class in org.biojava.nbio.core.sequence.location">SequenceLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a> - Class in <a href="../org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>
<div class="block">Provides a set of static methods to be used as static imports when needed
 across multiple Sequence implementations but inheritance gets in the way.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html#%3Cinit%3E()" class="member-name-link">SequenceMixin()</a> - Constructor for class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.SequenceIterator.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin.SequenceIterator</a>&lt;<a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.SequenceIterator.html" title="type parameter in SequenceMixin.SequenceIterator">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>
<div class="block">A basic sequence iterator which iterates over the given Sequence by
 biological index.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/SequenceOptimizationHints.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence">SequenceOptimizationHints</a> - Class in <a href="../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">A static class that provides optimization hints for memory or performance handling of sequence data.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#%3Cinit%3E()" class="member-name-link">SequenceOptimizationHints()</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/SequenceOptimizationHints.html" title="class in org.biojava.nbio.core.sequence">SequenceOptimizationHints</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/SequenceOptimizationHints.SequenceCollection.html" class="type-name-link" title="enum in org.biojava.nbio.core.sequence">SequenceOptimizationHints.SequenceCollection</a> - Enum in <a href="../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/SequenceOptimizationHints.SequenceUsage.html" class="type-name-link" title="enum in org.biojava.nbio.core.sequence">SequenceOptimizationHints.SequenceUsage</a> - Enum in <a href="../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/SequencePair.html" class="type-name-link" title="interface in org.biojava.nbio.core.alignment.template">SequencePair</a>&lt;<a href="../org/biojava/nbio/core/alignment/template/SequencePair.html" title="type parameter in SequencePair">S</a> extends <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="../org/biojava/nbio/core/alignment/template/SequencePair.html" title="type parameter in SequencePair">C</a>&gt;,<wbr><a href="../org/biojava/nbio/core/alignment/template/SequencePair.html" title="type parameter in SequencePair">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="../org/biojava/nbio/core/alignment/template/package-summary.html">org.biojava.nbio.core.alignment.template</a></dt>
<dd>
<div class="block">Defines a data structure for the results of pairwise sequence alignment.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/template/SequenceParserInterface.html" class="type-name-link" title="interface in org.biojava.nbio.core.sequence.io.template">SequenceParserInterface</a> - Interface in <a href="../org/biojava/nbio/core/sequence/io/template/package-summary.html">org.biojava.nbio.core.sequence.io.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a>&lt;<a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="type parameter in SequenceProxyView">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html#%3Cinit%3E()" class="member-name-link">SequenceProxyView()</a> - Constructor for class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html#%3Cinit%3E(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">SequenceProxyView(Sequence&lt;C&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html#%3Cinit%3E(org.biojava.nbio.core.sequence.template.Sequence,java.lang.Integer,java.lang.Integer)" class="member-name-link">SequenceProxyView(Sequence&lt;C&gt;, Integer, Integer)</a> - Constructor for class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>
<div class="block">Main constructor for working with SequenceProxyViews</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceReader.html" class="type-name-link" title="interface in org.biojava.nbio.core.sequence.template">SequenceReader</a>&lt;<a href="../org/biojava/nbio/core/sequence/template/SequenceReader.html" title="type parameter in SequenceReader">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="../org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/SequenceTestUtils.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence">SequenceTestUtils</a> - Class in <a href="../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">Test utility methods for classes in this package</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/SequenceTestUtils.html#%3Cinit%3E()" class="member-name-link">SequenceTestUtils()</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/SequenceTestUtils.html" title="class in org.biojava.nbio.core.sequence">SequenceTestUtils</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/SequenceTools.html" class="type-name-link" title="class in org.biojava.nbio.core.util">SequenceTools</a> - Class in <a href="../org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/SequenceTools.html#%3Cinit%3E()" class="member-name-link">SequenceTools()</a> - Constructor for class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SequenceTools.html" title="class in org.biojava.nbio.core.util">SequenceTools</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceView.html" class="type-name-link" title="interface in org.biojava.nbio.core.sequence.template">SequenceView</a>&lt;<a href="../org/biojava/nbio/core/sequence/template/SequenceView.html" title="type parameter in SequenceView">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="../org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/SequenceViewTest.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence">SequenceViewTest</a> - Class in <a href="../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/SequenceViewTest.html#%3Cinit%3E()" class="member-name-link">SequenceViewTest()</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/SequenceViewTest.html" title="class in org.biojava.nbio.core.sequence">SequenceViewTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/AAindexFactory.html#setAAIndexProvider(org.biojava.nbio.core.alignment.matrices.AAIndexProvider)" class="member-name-link">setAAIndexProvider(AAIndexProvider)</a> - Static method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/AAindexFactory.html" title="class in org.biojava.nbio.core.alignment.matrices">AAindexFactory</a></dt>
<dd>
<div class="block">Sets the singleton provider.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#setAccession(java.lang.String)" class="member-name-link">setAccession(String)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">GenericGenbankHeaderParser</a></dt>
<dd>
<div class="block">
 The last accession passed to this routine will always be the one used.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#setAccession(org.biojava.nbio.core.sequence.AccessionID)" class="member-name-link">setAccession(AccessionID)</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#setAccession(org.biojava.nbio.core.sequence.AccessionID)" class="member-name-link">setAccession(AccessionID)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#setAccessionNumber(java.util.List)" class="member-name-link">setAccessionNumber(List&lt;String&gt;)</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#setAnnotationType(org.biojava.nbio.core.sequence.template.AbstractSequence.AnnotationType)" class="member-name-link">setAnnotationType(AbstractSequence.AnnotationType)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#setAssemblyHeader(java.lang.String)" class="member-name-link">setAssemblyHeader(String)</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#setAssemblyInformation(java.lang.String)" class="member-name-link">setAssemblyInformation(String)</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/reference/AbstractReference.html#setAuthors(java.lang.String)" class="member-name-link">setAuthors(String)</a> - Method in class org.biojava.nbio.core.sequence.reference.<a href="../org/biojava/nbio/core/sequence/reference/AbstractReference.html" title="class in org.biojava.nbio.core.sequence.reference">AbstractReference</a></dt>
<dd>
<div class="block">Set The authors are a list of Inventors that retrieved from the Reference section.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/reference/GenbankReference.html#setAuthors(java.lang.String)" class="member-name-link">setAuthors(String)</a> - Method in class org.biojava.nbio.core.sequence.reference.<a href="../org/biojava/nbio/core/sequence/reference/GenbankReference.html" title="class in org.biojava.nbio.core.sequence.reference">GenbankReference</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/reference/ReferenceInterface.html#setAuthors(java.lang.String)" class="member-name-link">setAuthors(String)</a> - Method in interface org.biojava.nbio.core.sequence.reference.<a href="../org/biojava/nbio/core/sequence/reference/ReferenceInterface.html" title="interface in org.biojava.nbio.core.sequence.reference">ReferenceInterface</a></dt>
<dd>
<div class="block">Set the authors that retrieved from Reference section.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#setBetweenCompounds(boolean)" class="member-name-link">setBetweenCompounds(boolean)</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#setBioBegin(java.lang.Integer)" class="member-name-link">setBioBegin(Integer)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#setBioEnd(java.lang.Integer)" class="member-name-link">setBioEnd(Integer)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html#setBioEnd(java.lang.Integer)" class="member-name-link">setBioEnd(Integer)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html#setBioStart(java.lang.Integer)" class="member-name-link">setBioStart(Integer)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html#setChildrenFeatures(java.util.List)" class="member-name-link">setChildrenFeatures(List&lt;FeatureInterface&lt;S, C&gt;&gt;)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">Set the children features</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#setChildrenFeatures(java.util.List)" class="member-name-link">setChildrenFeatures(List&lt;FeatureInterface&lt;S, C&gt;&gt;)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html#setChildrenFeatures(java.util.List)" class="member-name-link">setChildrenFeatures(List&lt;FeatureInterface&lt;S, C&gt;&gt;)</a> - Method in interface org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Set the children features</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/ChromosomeSequence.html#setChromosomeNumber(int)" class="member-name-link">setChromosomeNumber(int)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/ChromosomeSequence.html" title="class in org.biojava.nbio.core.sequence">ChromosomeSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html#setCols(java.util.List)" class="member-name-link">setCols(List&lt;AminoAcidCompound&gt;)</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#setComment(java.lang.String)" class="member-name-link">setComment(String)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">GenericGenbankHeaderParser</a></dt>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#setComments(java.util.List)" class="member-name-link">setComments(List&lt;String&gt;)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Set comments.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/InsdcParser.html#setComplexFeaturesAppendMode(org.biojava.nbio.core.sequence.location.InsdcParser.complexFeaturesAppendEnum)" class="member-name-link">setComplexFeaturesAppendMode(InsdcParser.complexFeaturesAppendEnum)</a> - Method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/InsdcParser.html" title="class in org.biojava.nbio.core.sequence.location">InsdcParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#setCompoundAt(char,int)" class="member-name-link">setCompoundAt(char, int)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Converts from char to Compound and sets it at the given biological index</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#setCompoundAt(C,int)" class="member-name-link">setCompoundAt(C, int)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Sets the compound at the specified biological index</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#setCompoundSet(org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">setCompoundSet(CompoundSet&lt;C&gt;)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#setCompoundSet(org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">setCompoundSet(CompoundSet&lt;C&gt;)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#setCompoundSet(org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">setCompoundSet(CompoundSet&lt;C&gt;)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#setCompoundSet(org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">setCompoundSet(CompoundSet&lt;C&gt;)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#setCompoundSet(org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">setCompoundSet(CompoundSet&lt;C&gt;)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Class is immutable &amp; so this is unsupported</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#setCompoundSet(org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">setCompoundSet(CompoundSet&lt;C&gt;)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#setCompoundSet(org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">setCompoundSet(CompoundSet&lt;C&gt;)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Unsupported</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#setCompoundSet(org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">setCompoundSet(CompoundSet&lt;C&gt;)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceReader.html#setCompoundSet(org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">setCompoundSet(CompoundSet&lt;C&gt;)</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceReader.html" title="interface in org.biojava.nbio.core.sequence.template">SequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/views/RnaSequenceView.html#setCompoundSet(org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">setCompoundSet(CompoundSet&lt;NucleotideCompound&gt;)</a> - Method in class org.biojava.nbio.core.sequence.views.<a href="../org/biojava/nbio/core/sequence/views/RnaSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#setConstructedSequence(java.lang.String)" class="member-name-link">setConstructedSequence(String)</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#setContents(java.lang.String)" class="member-name-link">setContents(String)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#setContents(java.lang.String)" class="member-name-link">setContents(String)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#setContents(java.lang.String)" class="member-name-link">setContents(String)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">Once the sequence is retrieved set the contents and make sure everything this is valid
 Some uniprot records contain white space in the sequence.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#setContents(java.lang.String)" class="member-name-link">setContents(String)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#setContents(java.lang.String)" class="member-name-link">setContents(String)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Class is immutable &amp; so this is unsupported</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#setContents(java.lang.String)" class="member-name-link">setContents(String)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#setContents(java.lang.String)" class="member-name-link">setContents(String)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Unsupported</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceReader.html#setContents(java.lang.String)" class="member-name-link">setContents(String)</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceReader.html" title="interface in org.biojava.nbio.core.sequence.template">SequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/views/RnaSequenceView.html#setContents(java.lang.String)" class="member-name-link">setContents(String)</a> - Method in class org.biojava.nbio.core.sequence.views.<a href="../org/biojava/nbio/core/sequence/views/RnaSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#setContents(java.lang.String,java.util.ArrayList)" class="member-name-link">setContents(String, ArrayList)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#setContents(java.util.List)" class="member-name-link">setContents(List&lt;C&gt;)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#setCreatedDate(java.lang.String)" class="member-name-link">setCreatedDate(String)</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#setDatabase(java.lang.String)" class="member-name-link">setDatabase(String)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">DBReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#setDatabaseCrossReference(java.lang.String)" class="member-name-link">setDatabaseCrossReference(String)</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastTabularParser.html#setDatabaseReferences(java.util.List)" class="member-name-link">setDatabaseReferences(List&lt;Sequence&gt;)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastTabularParser.html" title="class in org.biojava.nbio.core.search.io.blast">BlastTabularParser</a></dt>
<dd>
<div class="block">Intended for use with run module.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParser.html#setDatabaseReferences(java.util.List)" class="member-name-link">setDatabaseReferences(List&lt;Sequence&gt;)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParser.html" title="class in org.biojava.nbio.core.search.io.blast">BlastXMLParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserProduct.html#setDatabaseReferences(java.util.List)" class="member-name-link">setDatabaseReferences(List&lt;Sequence&gt;)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserProduct.html" title="class in org.biojava.nbio.core.search.io.blast">BlastXMLParserProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/ResultFactory.html#setDatabaseReferences(java.util.List)" class="member-name-link">setDatabaseReferences(List&lt;Sequence&gt;)</a> - Method in interface org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/ResultFactory.html" title="interface in org.biojava.nbio.core.search.io">ResultFactory</a></dt>
<dd>
<div class="block">Specify the collection of sequences objects used as database in the Search run.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#setDatabaseReferences(org.biojava.nbio.core.sequence.features.DatabaseReferenceInterface)" class="member-name-link">setDatabaseReferences(DatabaseReferenceInterface)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastResultBuilder.html#setDbFile(java.lang.String)" class="member-name-link">setDbFile(String)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastResultBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastResultBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/SingleLinkageClusterer.html#setDendrogram(org.biojava.nbio.core.util.SingleLinkageClusterer.LinkedPair%5B%5D)" class="member-name-link">setDendrogram(SingleLinkageClusterer.LinkedPair[])</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SingleLinkageClusterer.html" title="class in org.biojava.nbio.core.util">SingleLinkageClusterer</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html#setDescription(java.lang.String)" class="member-name-link">setDescription(String)</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html#setDescription(java.lang.String)" class="member-name-link">setDescription(String)</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/SubstitutionMatrix.html#setDescription(java.lang.String)" class="member-name-link">setDescription(String)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava.nbio.core.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Sets the description of this matrix.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html#setDescription(java.lang.String)" class="member-name-link">setDescription(String)</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html#setDescription(java.lang.String)" class="member-name-link">setDescription(String)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#setDescription(java.lang.String)" class="member-name-link">setDescription(String)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html#setDescription(java.lang.String)" class="member-name-link">setDescription(String)</a> - Method in interface org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Set the description that can be used to describe the feature</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#setDescription(java.lang.String)" class="member-name-link">setDescription(String)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">GenericGenbankHeaderParser</a></dt>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html#setDescription(java.lang.String)" class="member-name-link">setDescription(String)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#setDescription(java.lang.String)" class="member-name-link">setDescription(String)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/Compound.html#setDescription(java.lang.String)" class="member-name-link">setDescription(String)</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#setDescription(java.lang.String)" class="member-name-link">setDescription(String)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html#setDescription(java.lang.String)" class="member-name-link">setDescription(String)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNASequence.html#setDNAType(org.biojava.nbio.core.sequence.DNASequence.DNAType)" class="member-name-link">setDNAType(DNASequence.DNAType)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#setEmblId(org.biojava.nbio.core.sequence.io.embl.EmblId)" class="member-name-link">setEmblId(EmblId)</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#setEmblReference(java.util.List)" class="member-name-link">setEmblReference(List&lt;EmblReference&gt;)</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#setFeatureHeader(java.lang.String)" class="member-name-link">setFeatureHeader(String)</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#setFeatureRetriever(org.biojava.nbio.core.sequence.features.FeatureRetriever)" class="member-name-link">setFeatureRetriever(FeatureRetriever)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#setFeaturesKeyWord(org.biojava.nbio.core.sequence.features.FeaturesKeyWordInterface)" class="member-name-link">setFeaturesKeyWord(FeaturesKeyWordInterface)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#setFeatureTable(java.lang.String)" class="member-name-link">setFeatureTable(String)</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastTabularParser.html#setFile(java.io.File)" class="member-name-link">setFile(File)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastTabularParser.html" title="class in org.biojava.nbio.core.search.io.blast">BlastTabularParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParser.html#setFile(java.io.File)" class="member-name-link">setFile(File)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParser.html" title="class in org.biojava.nbio.core.search.io.blast">BlastXMLParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/ResultFactory.html#setFile(java.io.File)" class="member-name-link">setFile(File)</a> - Method in interface org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/ResultFactory.html" title="interface in org.biojava.nbio.core.search.io">ResultFactory</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#setGenbankDirectoryCache(java.lang.String)" class="member-name-link">setGenbankDirectoryCache(String)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastHitBuilder.html#setHitAccession(java.lang.String)" class="member-name-link">setHitAccession(String)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastHitBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHitBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastHitBuilder.html#setHitDef(java.lang.String)" class="member-name-link">setHitDef(String)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastHitBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHitBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastHitBuilder.html#setHitId(java.lang.String)" class="member-name-link">setHitId(String)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastHitBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHitBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastHitBuilder.html#setHitLen(int)" class="member-name-link">setHitLen(int)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastHitBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHitBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastHitBuilder.html#setHitNum(int)" class="member-name-link">setHitNum(int)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastHitBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHitBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastResultBuilder.html#setHits(java.util.List)" class="member-name-link">setHits(List&lt;Hit&gt;)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastResultBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastResultBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastHitBuilder.html#setHitSequence(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">setHitSequence(Sequence)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastHitBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHitBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html#setHspAlignLen(int)" class="member-name-link">setHspAlignLen(int)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHspBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html#setHspBitScore(double)" class="member-name-link">setHspBitScore(double)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHspBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html#setHspEvalue(double)" class="member-name-link">setHspEvalue(double)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHspBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html#setHspGaps(int)" class="member-name-link">setHspGaps(int)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHspBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html#setHspHitFrame(int)" class="member-name-link">setHspHitFrame(int)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHspBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html#setHspHitFrom(int)" class="member-name-link">setHspHitFrom(int)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHspBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html#setHspHitTo(int)" class="member-name-link">setHspHitTo(int)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHspBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html#setHspHseq(java.lang.String)" class="member-name-link">setHspHseq(String)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHspBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html#setHspIdentity(int)" class="member-name-link">setHspIdentity(int)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHspBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html#setHspIdentityString(java.lang.String)" class="member-name-link">setHspIdentityString(String)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHspBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html#setHspNum(int)" class="member-name-link">setHspNum(int)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHspBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html#setHspPositive(int)" class="member-name-link">setHspPositive(int)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHspBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html#setHspQseq(java.lang.String)" class="member-name-link">setHspQseq(String)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHspBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html#setHspQueryFrame(int)" class="member-name-link">setHspQueryFrame(int)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHspBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html#setHspQueryFrom(int)" class="member-name-link">setHspQueryFrom(int)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHspBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html#setHspQueryTo(int)" class="member-name-link">setHspQueryTo(int)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHspBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastHitBuilder.html#setHsps(java.util.List)" class="member-name-link">setHsps(List&lt;Hsp&gt;)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastHitBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHitBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html#setHspScore(int)" class="member-name-link">setHspScore(int)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHspBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#setId(java.lang.String)" class="member-name-link">setId(String)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">DBReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/AccessionID.html#setIdentifier(java.lang.String)" class="member-name-link">setIdentifier(String)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/AccessionID.html" title="class in org.biojava.nbio.core.sequence">AccessionID</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#setIdentifier(java.lang.String)" class="member-name-link">setIdentifier(String)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">GenericGenbankHeaderParser</a></dt>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACParser.html#setIdLookup(java.util.Map)" class="member-name-link">setIdLookup(Map&lt;Integer, IUPACParser.IUPACTable&gt;)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/IUPACParser.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/SingleLinkageClustererProductProduct.html#setIndicesToCheck(java.util.ArrayList)" class="member-name-link">setIndicesToCheck(ArrayList&lt;Integer&gt;)</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SingleLinkageClustererProductProduct.html" title="class in org.biojava.nbio.core.util">SingleLinkageClustererProductProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACParserProduct.html#setIs(java.io.InputStream)" class="member-name-link">setIs(InputStream)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/IUPACParserProduct.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParserProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/SingleLinkageClustererProduct.html#setIsScoreMatrix(boolean)" class="member-name-link">setIsScoreMatrix(boolean)</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SingleLinkageClustererProduct.html" title="class in org.biojava.nbio.core.util">SingleLinkageClustererProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/SingleLinkageClustererProductProduct.html#setIsScoreMatrix(boolean)" class="member-name-link">setIsScoreMatrix(boolean)</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SingleLinkageClustererProductProduct.html" title="class in org.biojava.nbio.core.util">SingleLinkageClustererProductProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastResultBuilder.html#setIterationNumber(int)" class="member-name-link">setIterationNumber(int)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastResultBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastResultBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/reference/AbstractReference.html#setJournal(java.lang.String)" class="member-name-link">setJournal(String)</a> - Method in class org.biojava.nbio.core.sequence.reference.<a href="../org/biojava/nbio/core/sequence/reference/AbstractReference.html" title="class in org.biojava.nbio.core.sequence.reference">AbstractReference</a></dt>
<dd>
<div class="block">Set The journal usually contains the Publication Number, Publication Date and Assignee</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/reference/GenbankReference.html#setJournal(java.lang.String)" class="member-name-link">setJournal(String)</a> - Method in class org.biojava.nbio.core.sequence.reference.<a href="../org/biojava/nbio/core/sequence/reference/GenbankReference.html" title="class in org.biojava.nbio.core.sequence.reference">GenbankReference</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/reference/ReferenceInterface.html#setJournal(java.lang.String)" class="member-name-link">setJournal(String)</a> - Method in interface org.biojava.nbio.core.sequence.reference.<a href="../org/biojava/nbio/core/sequence/reference/ReferenceInterface.html" title="interface in org.biojava.nbio.core.sequence.reference">ReferenceInterface</a></dt>
<dd>
<div class="block">Set the journal that retrieved from Reference section.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#setKeyword(java.util.List)" class="member-name-link">setKeyword(List&lt;String&gt;)</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#setLastUpdatedDate(java.lang.String)" class="member-name-link">setLastUpdatedDate(String)</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#setLineLength(int)" class="member-name-link">setLineLength(int)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaGeneWriter.html" title="class in org.biojava.nbio.core.sequence.io">FastaGeneWriter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaWriter.html#setLineLength(int)" class="member-name-link">setLineLength(int)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaWriter.html" title="class in org.biojava.nbio.core.sequence.io">FastaWriter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankWriter.html#setLineLength(int)" class="member-name-link">setLineLength(int)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankWriter.html" title="class in org.biojava.nbio.core.sequence.io">GenbankWriter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaWriter.html#setLineSeparator(java.lang.String)" class="member-name-link">setLineSeparator(String)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaWriter.html" title="class in org.biojava.nbio.core.sequence.io">FastaWriter</a></dt>
<dd>
<div class="block">Allow an override of operating system line separator for programs that needs a specific CRLF or CR or LF option</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html#setLocation(org.biojava.nbio.core.sequence.location.template.AbstractLocation)" class="member-name-link">setLocation(AbstractLocation)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">A feature could be a single sequence position like a mutation or a post translational modification of an amino acid.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#setLocation(org.biojava.nbio.core.sequence.location.template.AbstractLocation)" class="member-name-link">setLocation(AbstractLocation)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html#setLocation(org.biojava.nbio.core.sequence.location.template.AbstractLocation)" class="member-name-link">setLocation(AbstractLocation)</a> - Method in interface org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">The new location for this feature.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/MutableAlignedSequence.html#setLocationInAlignment(org.biojava.nbio.core.sequence.location.template.Location)" class="member-name-link">setLocationInAlignment(Location)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/MutableAlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template">MutableAlignedSequence</a></dt>
<dd>
<div class="block">Sets the position of the <a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template"><code>AlignedSequence</code></a> to the given <a href="../org/biojava/nbio/core/sequence/location/template/Location.html" title="interface in org.biojava.nbio.core.sequence.location.template"><code>Location</code></a> (start, gaps, end).</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html#setLongName(java.lang.String)" class="member-name-link">setLongName(String)</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html#setLongName(java.lang.String)" class="member-name-link">setLongName(String)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/Compound.html#setLongName(java.lang.String)" class="member-name-link">setLongName(String)</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#setLongName(java.lang.String)" class="member-name-link">setLongName(String)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html#setLongName(java.lang.String)" class="member-name-link">setLongName(String)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#setLowercaseToNull(org.biojava.nbio.core.sequence.ProteinSequence,java.lang.Object%5B%5D)" class="member-name-link">setLowercaseToNull(ProteinSequence, Object[])</a> - Static method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">CasePreservingProteinSequenceCreator</a></dt>
<dd>
<div class="block">Takes a <a href="../org/biojava/nbio/core/sequence/ProteinSequence.html" title="class in org.biojava.nbio.core.sequence"><code>ProteinSequence</code></a> which was created by a
 <a href="../org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io"><code>CasePreservingProteinSequenceCreator</code></a>.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/SingleLinkageClustererProduct.html#setMatrix(double%5B%5D%5B%5D)" class="member-name-link">setMatrix(double[][])</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SingleLinkageClustererProduct.html" title="class in org.biojava.nbio.core.util">SingleLinkageClustererProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html#setMatrix(short%5B%5D%5B%5D)" class="member-name-link">setMatrix(short[][])</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html#setMax(short)" class="member-name-link">setMax(short)</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html#setMin(short)" class="member-name-link">setMin(short)</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html#setMismatchCount(java.lang.Integer)" class="member-name-link">setMismatchCount(Integer)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHspBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html#setMolecularWeight(java.lang.Float)" class="member-name-link">setMolecularWeight(Float)</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html#setMolecularWeight(java.lang.Float)" class="member-name-link">setMolecularWeight(Float)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/Compound.html#setMolecularWeight(java.lang.Float)" class="member-name-link">setMolecularWeight(Float)</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#setMolecularWeight(java.lang.Float)" class="member-name-link">setMolecularWeight(Float)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html#setMolecularWeight(java.lang.Float)" class="member-name-link">setMolecularWeight(Float)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html#setName(java.lang.String)" class="member-name-link">setName(String)</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html#setName(java.lang.String)" class="member-name-link">setName(String)</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/SubstitutionMatrix.html#setName(java.lang.String)" class="member-name-link">setName(String)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava.nbio.core.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Sets the name (short description) of this matrix.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/Qualifier.html#setName(java.lang.String)" class="member-name-link">setName(String)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/Qualifier.html" title="class in org.biojava.nbio.core.sequence.features">Qualifier</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#setName(java.lang.String)" class="member-name-link">setName(String)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">GenericGenbankHeaderParser</a></dt>
<dt><a href="../org/biojava/nbio/core/sequence/features/Qualifier.html#setNeedsQuotes(boolean)" class="member-name-link">setNeedsQuotes(boolean)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/Qualifier.html" title="class in org.biojava.nbio.core.sequence.features">Qualifier</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#setNotesList(java.util.ArrayList)" class="member-name-link">setNotesList(ArrayList&lt;String&gt;)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/SingleLinkageClustererProduct.html#setNumItems(int)" class="member-name-link">setNumItems(int)</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SingleLinkageClustererProduct.html" title="class in org.biojava.nbio.core.util">SingleLinkageClustererProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#setOrGanelle(java.lang.String)" class="member-name-link">setOrGanelle(String)</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#setOrganismClassification(java.lang.String)" class="member-name-link">setOrganismClassification(String)</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#setOrganismSpecies(java.lang.String)" class="member-name-link">setOrganismSpecies(String)</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#setOriginalHeader(java.lang.String)" class="member-name-link">setOriginalHeader(String)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/MutableSequencePair.html#setPair(org.biojava.nbio.core.alignment.template.AlignedSequence,org.biojava.nbio.core.alignment.template.AlignedSequence)" class="member-name-link">setPair(AlignedSequence&lt;S, C&gt;, AlignedSequence&lt;S, C&gt;)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/MutableSequencePair.html" title="interface in org.biojava.nbio.core.alignment.template">MutableSequencePair</a></dt>
<dd>
<div class="block">Sets both <a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template"><code>AlignedSequence</code></a>s of the pair.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/MutableProfilePair.html#setPair(org.biojava.nbio.core.alignment.template.Profile,org.biojava.nbio.core.alignment.template.Profile)" class="member-name-link">setPair(Profile&lt;S, C&gt;, Profile&lt;S, C&gt;)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/MutableProfilePair.html" title="interface in org.biojava.nbio.core.alignment.template">MutableProfilePair</a></dt>
<dd>
<div class="block">Sets both <a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template"><code>Profile</code></a>s of the pair.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/ProteinSequence.html#setParentDNASequence(org.biojava.nbio.core.sequence.template.AbstractSequence,java.lang.Integer,java.lang.Integer)" class="member-name-link">setParentDNASequence(AbstractSequence&lt;NucleotideCompound&gt;, Integer, Integer)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/ProteinSequence.html" title="class in org.biojava.nbio.core.sequence">ProteinSequence</a></dt>
<dd>
<div class="block">A Protein sequence can be stand alone or loaded from a transcript
 sequence.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html#setParentFeature(org.biojava.nbio.core.sequence.features.FeatureInterface)" class="member-name-link">setParentFeature(FeatureInterface&lt;S, C&gt;)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">A feature can be the child or contained by a parent feature.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#setParentFeature(org.biojava.nbio.core.sequence.features.FeatureInterface)" class="member-name-link">setParentFeature(FeatureInterface&lt;S, C&gt;)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html#setParentFeature(org.biojava.nbio.core.sequence.features.FeatureInterface)" class="member-name-link">setParentFeature(FeatureInterface&lt;S, C&gt;)</a> - Method in interface org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Set the parent feature</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#setParentSequence(org.biojava.nbio.core.sequence.template.AbstractSequence)" class="member-name-link">setParentSequence(AbstractSequence&lt;?&gt;)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastTabularParser.html#setParsingConsistency(org.biojava.nbio.core.search.io.blast.BlastTabularParser.PARSING_CONSISTENCY)" class="member-name-link">setParsingConsistency(BlastTabularParser.PARSING_CONSISTENCY)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastTabularParser.html" title="class in org.biojava.nbio.core.search.io.blast">BlastTabularParser</a></dt>
<dd>
<div class="block">Tries to define a different level of consistency during parsing.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#setPartialOn3prime(boolean)" class="member-name-link">setPartialOn3prime(boolean)</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocationProduct.html#setPartialOn3prime(boolean)" class="member-name-link">setPartialOn3prime(boolean)</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocationProduct.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocationProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#setPartialOn5prime(boolean)" class="member-name-link">setPartialOn5prime(boolean)</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocationProduct.html#setPartialOn5prime(boolean)" class="member-name-link">setPartialOn5prime(boolean)</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocationProduct.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocationProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html#setPercentageIdentity(java.lang.Double)" class="member-name-link">setPercentageIdentity(Double)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHspBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastResultBuilder.html#setProgram(java.lang.String)" class="member-name-link">setProgram(String)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastResultBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastResultBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastResultBuilder.html#setProgramSpecificParameters(java.util.HashMap)" class="member-name-link">setProgramSpecificParameters(HashMap&lt;String, String&gt;)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastResultBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastResultBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#setProjectIdentifier(java.lang.String)" class="member-name-link">setProjectIdentifier(String)</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#setProperties(java.util.LinkedHashMap)" class="member-name-link">setProperties(LinkedHashMap&lt;String, String&gt;)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">DBReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#setProxySequenceReader(org.biojava.nbio.core.sequence.template.SequenceReader)" class="member-name-link">setProxySequenceReader(SequenceReader&lt;C&gt;)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Very important method that allows external mappings of sequence data and features.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html#setQualifiers(java.util.Map)" class="member-name-link">setQualifiers(Map&lt;String, List&lt;Qualifier&gt;&gt;)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#setQualifiers(java.util.Map)" class="member-name-link">setQualifiers(Map&lt;String, List&lt;Qualifier&gt;&gt;)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html#setQualifiers(java.util.Map)" class="member-name-link">setQualifiers(Map&lt;String, List&lt;Qualifier&gt;&gt;)</a> - Method in interface org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Set the qualifiers</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/QualityFeature.html#setQualities(java.util.List)" class="member-name-link">setQualities(List&lt;Number&gt;)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/QualityFeature.html" title="class in org.biojava.nbio.core.sequence.features">QualityFeature</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/QuantityFeature.html#setQuantities(java.util.List)" class="member-name-link">setQuantities(List&lt;Number&gt;)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/QuantityFeature.html" title="class in org.biojava.nbio.core.sequence.features">QuantityFeature</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/MutableSequencePair.html#setQuery(org.biojava.nbio.core.alignment.template.AlignedSequence)" class="member-name-link">setQuery(AlignedSequence&lt;S, C&gt;)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/MutableSequencePair.html" title="interface in org.biojava.nbio.core.alignment.template">MutableSequencePair</a></dt>
<dd>
<div class="block">Sets the first <a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template"><code>AlignedSequence</code></a> of the pair.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/MutableProfilePair.html#setQuery(org.biojava.nbio.core.alignment.template.Profile)" class="member-name-link">setQuery(Profile&lt;S, C&gt;)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/MutableProfilePair.html" title="interface in org.biojava.nbio.core.alignment.template">MutableProfilePair</a></dt>
<dd>
<div class="block">Sets the first <a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template"><code>Profile</code></a> of the pair.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastResultBuilder.html#setQueryDef(java.lang.String)" class="member-name-link">setQueryDef(String)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastResultBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastResultBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastResultBuilder.html#setQueryID(java.lang.String)" class="member-name-link">setQueryID(String)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastResultBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastResultBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastResultBuilder.html#setQueryLength(int)" class="member-name-link">setQueryLength(int)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastResultBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastResultBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastTabularParser.html#setQueryReferences(java.util.List)" class="member-name-link">setQueryReferences(List&lt;Sequence&gt;)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastTabularParser.html" title="class in org.biojava.nbio.core.search.io.blast">BlastTabularParser</a></dt>
<dd>
<div class="block">Intended for use with run module.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParser.html#setQueryReferences(java.util.List)" class="member-name-link">setQueryReferences(List&lt;Sequence&gt;)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParser.html" title="class in org.biojava.nbio.core.search.io.blast">BlastXMLParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserProduct.html#setQueryReferences(java.util.List)" class="member-name-link">setQueryReferences(List&lt;Sequence&gt;)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserProduct.html" title="class in org.biojava.nbio.core.search.io.blast">BlastXMLParserProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/ResultFactory.html#setQueryReferences(java.util.List)" class="member-name-link">setQueryReferences(List&lt;Sequence&gt;)</a> - Method in interface org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/ResultFactory.html" title="interface in org.biojava.nbio.core.search.io">ResultFactory</a></dt>
<dd>
<div class="block">Specify the collection of sequences objects used as queries in the Search run.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastResultBuilder.html#setQuerySequence(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">setQuerySequence(Sequence)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastResultBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastResultBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastResultBuilder.html#setReference(java.lang.String)" class="member-name-link">setReference(String)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastResultBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastResultBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html#setReferenceAuthor(java.lang.String)" class="member-name-link">setReferenceAuthor(String)</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReference</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html#setReferenceComment(java.lang.String)" class="member-name-link">setReferenceComment(String)</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReference</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html#setReferenceCrossReference(java.lang.String)" class="member-name-link">setReferenceCrossReference(String)</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReference</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html#setReferenceGroup(java.lang.String)" class="member-name-link">setReferenceGroup(String)</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReference</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html#setReferenceLocation(java.lang.String)" class="member-name-link">setReferenceLocation(String)</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReference</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html#setReferenceNumber(java.lang.String)" class="member-name-link">setReferenceNumber(String)</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReference</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html#setReferencePosition(java.lang.String)" class="member-name-link">setReferencePosition(String)</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReference</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#setReferences(java.util.List)" class="member-name-link">setReferences(List&lt;AbstractReference&gt;)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Set the list of <a href="../org/biojava/nbio/core/sequence/reference/AbstractReference.html" title="class in org.biojava.nbio.core.sequence.reference"><code>AbstractReference</code></a></div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html#setReferenceTitle(java.lang.String)" class="member-name-link">setReferenceTitle(String)</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReference</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html#setRows(java.util.List)" class="member-name-link">setRows(List&lt;AminoAcidCompound&gt;)</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html#setScale(int)" class="member-name-link">setScale(int)</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#setSequence(java.lang.String)" class="member-name-link">setSequence(String)</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SequenceLocation.html#setSequence(S)" class="member-name-link">setSequence(S)</a> - Method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SequenceLocation.html" title="class in org.biojava.nbio.core.sequence.location">SequenceLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/InsdcParser.html#setSequenceCircular(boolean)" class="member-name-link">setSequenceCircular(boolean)</a> - Method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/InsdcParser.html" title="class in org.biojava.nbio.core.sequence.location">InsdcParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#setSequenceCollection(org.biojava.nbio.core.sequence.SequenceOptimizationHints.SequenceCollection)" class="member-name-link">setSequenceCollection(SequenceOptimizationHints.SequenceCollection)</a> - Static method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/SequenceOptimizationHints.html" title="class in org.biojava.nbio.core.sequence">SequenceOptimizationHints</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#setSequenceDescription(java.lang.String)" class="member-name-link">setSequenceDescription(String)</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#setSequenceHeader(java.lang.String)" class="member-name-link">setSequenceHeader(String)</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/InsdcParser.html#setSequenceLength(long)" class="member-name-link">setSequenceLength(long)</a> - Method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/InsdcParser.html" title="class in org.biojava.nbio.core.sequence.location">InsdcParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/MutableProfile.html#setSequences(java.util.List)" class="member-name-link">setSequences(List&lt;AlignedSequence&lt;S, C&gt;&gt;)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/MutableProfile.html" title="interface in org.biojava.nbio.core.alignment.template">MutableProfile</a></dt>
<dd>
<div class="block">Sets entirely new set of <a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template"><code>AlignedSequence</code></a>s.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#setSequenceScore(java.lang.Double)" class="member-name-link">setSequenceScore(Double)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#setSequenceUsage(org.biojava.nbio.core.sequence.SequenceOptimizationHints.SequenceUsage)" class="member-name-link">setSequenceUsage(SequenceOptimizationHints.SequenceUsage)</a> - Static method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/SequenceOptimizationHints.html" title="class in org.biojava.nbio.core.sequence">SequenceOptimizationHints</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html#setShortDescription(java.lang.String)" class="member-name-link">setShortDescription(String)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#setShortDescription(java.lang.String)" class="member-name-link">setShortDescription(String)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html#setShortDescription(java.lang.String)" class="member-name-link">setShortDescription(String)</a> - Method in interface org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Set the short description that can be used to describe the feature</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html#setShortName(java.lang.String)" class="member-name-link">setShortName(String)</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html#setShortName(java.lang.String)" class="member-name-link">setShortName(String)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/Compound.html#setShortName(java.lang.String)" class="member-name-link">setShortName(String)</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#setShortName(java.lang.String)" class="member-name-link">setShortName(String)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html#setShortName(java.lang.String)" class="member-name-link">setShortName(String)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html#setSource(java.lang.String)" class="member-name-link">setSource(String)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">Set the feature source</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#setSource(java.lang.String)" class="member-name-link">setSource(String)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html#setSource(java.lang.String)" class="member-name-link">setSource(String)</a> - Method in interface org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Change the source of the FeatureInterface.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#setSource(java.lang.String)" class="member-name-link">setSource(String)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Added support for the source of this sequence for GFF3 export</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/GeneSequence.html#setStrand(org.biojava.nbio.core.sequence.Strand)" class="member-name-link">setStrand(Strand)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#setStrand(org.biojava.nbio.core.sequence.Strand)" class="member-name-link">setStrand(Strand)</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#setSubLocations(java.util.List)" class="member-name-link">setSubLocations(List&lt;Location&gt;)</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/MutableSequencePair.html#setTarget(org.biojava.nbio.core.alignment.template.AlignedSequence)" class="member-name-link">setTarget(AlignedSequence&lt;S, C&gt;)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/MutableSequencePair.html" title="interface in org.biojava.nbio.core.alignment.template">MutableSequencePair</a></dt>
<dd>
<div class="block">Sets the second <a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template"><code>AlignedSequence</code></a> of the pair.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/MutableProfilePair.html#setTarget(org.biojava.nbio.core.alignment.template.Profile)" class="member-name-link">setTarget(Profile&lt;S, C&gt;)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/MutableProfilePair.html" title="interface in org.biojava.nbio.core.alignment.template">MutableProfilePair</a></dt>
<dd>
<div class="block">Sets the second <a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template"><code>Profile</code></a> of the pair.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#setTaxonomy(org.biojava.nbio.core.sequence.TaxonomyID)" class="member-name-link">setTaxonomy(TaxonomyID)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/ConcurrencyTools.html#setThreadPool(java.util.concurrent.ThreadPoolExecutor)" class="member-name-link">setThreadPool(ThreadPoolExecutor)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/ConcurrencyTools.html" title="class in org.biojava.nbio.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Sets thread pool to any given <a href="https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ThreadPoolExecutor.html" title="class or interface in java.util.concurrent" class="external-link"><code>ThreadPoolExecutor</code></a> to allow use of an alternative execution style.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/ConcurrencyTools.html#setThreadPoolCPUsAvailable(int)" class="member-name-link">setThreadPoolCPUsAvailable(int)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/ConcurrencyTools.html" title="class in org.biojava.nbio.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Sets thread pool to reserve a given number of processor cores for foreground or other use.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/ConcurrencyTools.html#setThreadPoolCPUsFraction(float)" class="member-name-link">setThreadPoolCPUsFraction(float)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/ConcurrencyTools.html" title="class in org.biojava.nbio.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Sets thread pool to a given fraction of the available processors.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/ConcurrencyTools.html#setThreadPoolDefault()" class="member-name-link">setThreadPoolDefault()</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/ConcurrencyTools.html" title="class in org.biojava.nbio.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Sets thread pool to default of 1 background thread for each processor core.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/ConcurrencyTools.html#setThreadPoolSingle()" class="member-name-link">setThreadPoolSingle()</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/ConcurrencyTools.html" title="class in org.biojava.nbio.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Sets thread pool to a single background thread.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/ConcurrencyTools.html#setThreadPoolSize(int)" class="member-name-link">setThreadPoolSize(int)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/ConcurrencyTools.html" title="class in org.biojava.nbio.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Sets thread pool to given size.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/reference/AbstractReference.html#setTitle(java.lang.String)" class="member-name-link">setTitle(String)</a> - Method in class org.biojava.nbio.core.sequence.reference.<a href="../org/biojava/nbio/core/sequence/reference/AbstractReference.html" title="class in org.biojava.nbio.core.sequence.reference">AbstractReference</a></dt>
<dd>
<div class="block">Set The title that retrieved from the Reference section.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/reference/GenbankReference.html#setTitle(java.lang.String)" class="member-name-link">setTitle(String)</a> - Method in class org.biojava.nbio.core.sequence.reference.<a href="../org/biojava/nbio/core/sequence/reference/GenbankReference.html" title="class in org.biojava.nbio.core.sequence.reference">GenbankReference</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/reference/ReferenceInterface.html#setTitle(java.lang.String)" class="member-name-link">setTitle(String)</a> - Method in interface org.biojava.nbio.core.sequence.reference.<a href="../org/biojava/nbio/core/sequence/reference/ReferenceInterface.html" title="interface in org.biojava.nbio.core.sequence.reference">ReferenceInterface</a></dt>
<dd>
<div class="block">Set the title that retrieved from Reference section.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html#setType(java.lang.String)" class="member-name-link">setType(String)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">Set the feature type</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#setType(java.lang.String)" class="member-name-link">setType(String)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html#setType(java.lang.String)" class="member-name-link">setType(String)</a> - Method in interface org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Change the type of this feature.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#setUniprotbaseURL(java.lang.String)" class="member-name-link">setUniprotbaseURL(String)</a> - Static method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#setUniprotDirectoryCache(java.lang.String)" class="member-name-link">setUniprotDirectoryCache(String)</a> - Static method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#setup()" class="member-name-link">setup()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#setup()" class="member-name-link">setup()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#setup()" class="member-name-link">setup()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastTabularParserTest.html#setUp()" class="member-name-link">setUp()</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastTabularParserTest.html" title="class in org.biojava.nbio.core.search.io.blast">BlastTabularParserTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserTest.html#setUp()" class="member-name-link">setUp()</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserTest.html" title="class in org.biojava.nbio.core.search.io.blast">BlastXMLParserTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/HspTest.html#setUp()" class="member-name-link">setUp()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/HspTest.html" title="class in org.biojava.nbio.core.search.io">HspTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/SearchIOTest.html#setUp()" class="member-name-link">setUp()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/SearchIOTest.html" title="class in org.biojava.nbio.core.search.io">SearchIOTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaReaderTest.html#setUp()" class="member-name-link">setUp()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaReaderTest.html" title="class in org.biojava.nbio.core.sequence.io">FastaReaderTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankCookbookTest.html#setUp()" class="member-name-link">setUp()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankCookbookTest.html" title="class in org.biojava.nbio.core.sequence.io">GenbankCookbookTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastTabularParserTest.html#setUpClass()" class="member-name-link">setUpClass()</a> - Static method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastTabularParserTest.html" title="class in org.biojava.nbio.core.search.io.blast">BlastTabularParserTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserTest.html#setUpClass()" class="member-name-link">setUpClass()</a> - Static method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserTest.html" title="class in org.biojava.nbio.core.search.io.blast">BlastXMLParserTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/HspTest.html#setUpClass()" class="member-name-link">setUpClass()</a> - Static method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/HspTest.html" title="class in org.biojava.nbio.core.search.io">HspTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/SearchIOTest.html#setUpClass()" class="member-name-link">setUpClass()</a> - Static method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/SearchIOTest.html" title="class in org.biojava.nbio.core.search.io">SearchIOTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaReaderTest.html#setUpClass()" class="member-name-link">setUpClass()</a> - Static method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaReaderTest.html" title="class in org.biojava.nbio.core.sequence.io">FastaReaderTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankCookbookTest.html#setUpClass()" class="member-name-link">setUpClass()</a> - Static method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankCookbookTest.html" title="class in org.biojava.nbio.core.sequence.io">GenbankCookbookTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#setUserCollection(java.util.Collection)" class="member-name-link">setUserCollection(Collection&lt;Object&gt;)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html#setUserObject(java.lang.Object)" class="member-name-link">setUserObject(Object)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">Allow the user to associate an object with the feature.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#setUserObject(java.lang.Object)" class="member-name-link">setUserObject(Object)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html#setUserObject(java.lang.Object)" class="member-name-link">setUserObject(Object)</a> - Method in interface org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/Qualifier.html#setValue(java.lang.String)" class="member-name-link">setValue(String)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/Qualifier.html" title="class in org.biojava.nbio.core.sequence.features">Qualifier</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#setVersion(int)" class="member-name-link">setVersion(int)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">GenericGenbankHeaderParser</a></dt>
<dt><a href="../org/biojava/nbio/core/sequence/AccessionID.html#setVersion(java.lang.Integer)" class="member-name-link">setVersion(Integer)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/AccessionID.html" title="class in org.biojava.nbio.core.sequence">AccessionID</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastResultBuilder.html#setVersion(java.lang.String)" class="member-name-link">setVersion(String)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastResultBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastResultBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/MutableAlignedSequence.html#shiftAtAlignmentLocation(org.biojava.nbio.core.sequence.location.template.Location,int)" class="member-name-link">shiftAtAlignmentLocation(Location, int)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/MutableAlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template">MutableAlignedSequence</a></dt>
<dd>
<div class="block">Slides a part of the <a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template"><code>AlignedSequence</code></a>.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/MutableAlignedSequence.html#shiftAtSequenceLocation(org.biojava.nbio.core.sequence.location.template.Location,int)" class="member-name-link">shiftAtSequenceLocation(Location, int)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/MutableAlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template">MutableAlignedSequence</a></dt>
<dd>
<div class="block">Slides a part of the <a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template"><code>AlignedSequence</code></a>.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html#shuffle(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">shuffle(Sequence&lt;C&gt;)</a> - Static method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Implements sequence shuffling by first materializing the given
 <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a> into a <a href="https://docs.oracle.com/javase/8/docs/api/java/util/List.html" title="class or interface in java.util" class="external-link"><code>List</code></a>, applying
 <a href="https://docs.oracle.com/javase/8/docs/api/java/util/Collections.html#shuffle(java.util.List)" title="class or interface in java.util" class="external-link"><code>Collections.shuffle(List)</code></a> and then returning the shuffled
 elements in a new instance of <code>SequenceBackingStore</code> which behaves
 as a <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a>.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/ConcurrencyTools.html#shutdown()" class="member-name-link">shutdown()</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/ConcurrencyTools.html" title="class in org.biojava.nbio.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Disables new tasks from being submitted and closes the thread pool cleanly.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/ConcurrencyTools.html#shutdownAndAwaitTermination()" class="member-name-link">shutdownAndAwaitTermination()</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/ConcurrencyTools.html" title="class in org.biojava.nbio.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Closes the thread pool.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" class="type-name-link" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a>&lt;<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="type parameter in SimpleAlignedSequence">S</a> extends <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="type parameter in SimpleAlignedSequence">C</a>&gt;,<wbr><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="type parameter in SimpleAlignedSequence">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/alignment/package-summary.html">org.biojava.nbio.core.alignment</a></dt>
<dd>
<div class="block">Implements a data structure for a <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a> within an alignment.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#%3Cinit%3E(org.biojava.nbio.core.alignment.template.AlignedSequence,java.util.List)" class="member-name-link">SimpleAlignedSequence(AlignedSequence&lt;S, C&gt;, List&lt;AlignedSequence.Step&gt;)</a> - Constructor for class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>
<div class="block">Creates a new <a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template"><code>AlignedSequence</code></a> for the given <a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template"><code>AlignedSequence</code></a> in a global alignment.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#%3Cinit%3E(org.biojava.nbio.core.alignment.template.AlignedSequence,java.util.List,int,int)" class="member-name-link">SimpleAlignedSequence(AlignedSequence&lt;S, C&gt;, List&lt;AlignedSequence.Step&gt;, int, int)</a> - Constructor for class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>
<div class="block">Creates a new <a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template"><code>AlignedSequence</code></a> for the given <a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template"><code>AlignedSequence</code></a> in a local alignment.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#%3Cinit%3E(S,java.util.List)" class="member-name-link">SimpleAlignedSequence(S, List&lt;AlignedSequence.Step&gt;)</a> - Constructor for class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>
<div class="block">Creates an <a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template"><code>AlignedSequence</code></a> for the given <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a> in a global alignment.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#%3Cinit%3E(S,java.util.List,int,int)" class="member-name-link">SimpleAlignedSequence(S, List&lt;AlignedSequence.Step&gt;, int, int)</a> - Constructor for class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>
<div class="block">Creates an <a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template"><code>AlignedSequence</code></a> for the given <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a> in a local alignment.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" class="type-name-link" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a> - Class in <a href="../org/biojava/nbio/core/alignment/package-summary.html">org.biojava.nbio.core.alignment</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#%3Cinit%3E()" class="member-name-link">SimpleAlignedSequenceTest()</a> - Constructor for class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/SimpleGenbankProxySequenceReaderTest.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.loader">SimpleGenbankProxySequenceReaderTest</a> - Class in <a href="../org/biojava/nbio/core/sequence/loader/package-summary.html">org.biojava.nbio.core.sequence.loader</a></dt>
<dd>
<div class="block">Testing example for issue #834</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/SimpleGenbankProxySequenceReaderTest.html#%3Cinit%3E()" class="member-name-link">SimpleGenbankProxySequenceReaderTest()</a> - Constructor for class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/SimpleGenbankProxySequenceReaderTest.html" title="class in org.biojava.nbio.core.sequence.loader">SimpleGenbankProxySequenceReaderTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SimpleLocation.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a> - Class in <a href="../org/biojava/nbio/core/sequence/location/package-summary.html">org.biojava.nbio.core.sequence.location</a></dt>
<dd>
<div class="block">Very basic implementation of the Location interface which defines a series
 of simple constructors.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SimpleLocation.html#%3Cinit%3E(int,int)" class="member-name-link">SimpleLocation(int, int)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SimpleLocation.html#%3Cinit%3E(int,int,org.biojava.nbio.core.sequence.Strand)" class="member-name-link">SimpleLocation(int, int, Strand)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SimpleLocation.html#%3Cinit%3E(int,int,org.biojava.nbio.core.sequence.Strand,java.util.List)" class="member-name-link">SimpleLocation(int, int, Strand, List&lt;Location&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SimpleLocation.html#%3Cinit%3E(int,int,org.biojava.nbio.core.sequence.Strand,org.biojava.nbio.core.sequence.location.template.Location...)" class="member-name-link">SimpleLocation(int, int, Strand, Location...)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SimpleLocation.html#%3Cinit%3E(org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.location.template.Point)" class="member-name-link">SimpleLocation(Point, Point)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SimpleLocation.html#%3Cinit%3E(org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.Strand)" class="member-name-link">SimpleLocation(Point, Point, Strand)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SimpleLocation.html#%3Cinit%3E(org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.Strand,boolean,boolean)" class="member-name-link">SimpleLocation(Point, Point, Strand, boolean, boolean)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SimpleLocation.html#%3Cinit%3E(org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.Strand,boolean,boolean,java.util.List)" class="member-name-link">SimpleLocation(Point, Point, Strand, boolean, boolean, List&lt;Location&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SimpleLocation.html#%3Cinit%3E(org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.Strand,boolean,java.util.List)" class="member-name-link">SimpleLocation(Point, Point, Strand, boolean, List&lt;Location&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SimpleLocation.html#%3Cinit%3E(org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.Strand,boolean,org.biojava.nbio.core.sequence.AccessionID)" class="member-name-link">SimpleLocation(Point, Point, Strand, boolean, AccessionID)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SimpleLocation.html#%3Cinit%3E(org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.Strand,boolean,org.biojava.nbio.core.sequence.location.template.Location...)" class="member-name-link">SimpleLocation(Point, Point, Strand, boolean, Location...)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SimpleLocation.html#%3Cinit%3E(org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.Strand,java.util.List)" class="member-name-link">SimpleLocation(Point, Point, Strand, List&lt;Location&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SimpleLocation.html#%3Cinit%3E(org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.Strand,org.biojava.nbio.core.sequence.AccessionID)" class="member-name-link">SimpleLocation(Point, Point, Strand, AccessionID)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SimpleLocation.html#%3Cinit%3E(org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.Strand,org.biojava.nbio.core.sequence.location.template.Location...)" class="member-name-link">SimpleLocation(Point, Point, Strand, Location...)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SimplePoint.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a> - Class in <a href="../org/biojava/nbio/core/sequence/location/package-summary.html">org.biojava.nbio.core.sequence.location</a></dt>
<dd>
<div class="block">Basic implementation of the Point interface.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SimplePoint.html#%3Cinit%3E(int)" class="member-name-link">SimplePoint(int)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SimplePoint.html#%3Cinit%3E(int,boolean,boolean)" class="member-name-link">SimplePoint(int, boolean, boolean)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" class="type-name-link" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a>&lt;<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="type parameter in SimpleProfile">S</a> extends <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="type parameter in SimpleProfile">C</a>&gt;,<wbr><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="type parameter in SimpleProfile">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/alignment/package-summary.html">org.biojava.nbio.core.alignment</a></dt>
<dd>
<div class="block">Implements a data structure for the results of sequence alignment.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html#%3Cinit%3E(java.util.Collection)" class="member-name-link">SimpleProfile(Collection&lt;AlignedSequence&lt;S, C&gt;&gt;)</a> - Constructor for class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a></dt>
<dd>
<div class="block">Creates a profile for the already aligned sequences.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html#%3Cinit%3E(S)" class="member-name-link">SimpleProfile(S)</a> - Constructor for class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a></dt>
<dd>
<div class="block">Creates a profile from a single sequence.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfilePair.html" class="type-name-link" title="class in org.biojava.nbio.core.alignment">SimpleProfilePair</a>&lt;<a href="../org/biojava/nbio/core/alignment/SimpleProfilePair.html" title="type parameter in SimpleProfilePair">S</a> extends <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="../org/biojava/nbio/core/alignment/SimpleProfilePair.html" title="type parameter in SimpleProfilePair">C</a>&gt;,<wbr><a href="../org/biojava/nbio/core/alignment/SimpleProfilePair.html" title="type parameter in SimpleProfilePair">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/alignment/package-summary.html">org.biojava.nbio.core.alignment</a></dt>
<dd>
<div class="block">Implements a data structure for the results of the alignment of a pair of <a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template"><code>Profile</code></a>s.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfilePair.html#%3Cinit%3E(org.biojava.nbio.core.alignment.template.Profile,org.biojava.nbio.core.alignment.template.Profile,java.util.List,java.util.List)" class="member-name-link">SimpleProfilePair(Profile&lt;S, C&gt;, Profile&lt;S, C&gt;, List&lt;AlignedSequence.Step&gt;, List&lt;AlignedSequence.Step&gt;)</a> - Constructor for class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfilePair.html" title="class in org.biojava.nbio.core.alignment">SimpleProfilePair</a></dt>
<dd>
<div class="block">Creates a pair profile for the given profiles.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" class="type-name-link" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a> - Class in <a href="../org/biojava/nbio/core/alignment/package-summary.html">org.biojava.nbio.core.alignment</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#%3Cinit%3E()" class="member-name-link">SimpleProfileTest()</a> - Constructor for class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html" class="type-name-link" title="class in org.biojava.nbio.core.alignment">SimpleSequencePair</a>&lt;<a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html" title="type parameter in SimpleSequencePair">S</a> extends <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html" title="type parameter in SimpleSequencePair">C</a>&gt;,<wbr><a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html" title="type parameter in SimpleSequencePair">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/alignment/package-summary.html">org.biojava.nbio.core.alignment</a></dt>
<dd>
<div class="block">Implements a data structure for the results of pairwise sequence alignment.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html#%3Cinit%3E(org.biojava.nbio.core.alignment.template.AlignedSequence,org.biojava.nbio.core.alignment.template.AlignedSequence)" class="member-name-link">SimpleSequencePair(AlignedSequence&lt;S, C&gt;, AlignedSequence&lt;S, C&gt;)</a> - Constructor for class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePair</a></dt>
<dd>
<div class="block">Creates a pair profile for the given already aligned sequences.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html#%3Cinit%3E(S,S,java.util.List,int,int,java.util.List,int,int)" class="member-name-link">SimpleSequencePair(S, S, List&lt;AlignedSequence.Step&gt;, int, int, List&lt;AlignedSequence.Step&gt;, int, int)</a> - Constructor for class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePair</a></dt>
<dd>
<div class="block">Creates a pair profile for the given sequences with a local alignment.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html#%3Cinit%3E(S,S,java.util.List,java.util.List)" class="member-name-link">SimpleSequencePair(S, S, List&lt;AlignedSequence.Step&gt;, List&lt;AlignedSequence.Step&gt;)</a> - Constructor for class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePair</a></dt>
<dd>
<div class="block">Creates a pair profile for the given sequences with a global alignment.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" class="type-name-link" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a> - Class in <a href="../org/biojava/nbio/core/alignment/package-summary.html">org.biojava.nbio.core.alignment</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#%3Cinit%3E()" class="member-name-link">SimpleSequencePairTest()</a> - Constructor for class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html" class="type-name-link" title="class in org.biojava.nbio.core.alignment.matrices">SimpleSubstitutionMatrix</a>&lt;<a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html" title="type parameter in SimpleSubstitutionMatrix">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/alignment/matrices/package-summary.html">org.biojava.nbio.core.alignment.matrices</a></dt>
<dd>
<div class="block">Implements a data structure which holds the score (penalty or bonus) given during alignment for the exchange of one
 <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Compound</code></a> in a sequence for another.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html#%3Cinit%3E(org.biojava.nbio.core.sequence.template.CompoundSet,short,short)" class="member-name-link">SimpleSubstitutionMatrix(CompoundSet&lt;C&gt;, short, short)</a> - Constructor for class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">SimpleSubstitutionMatrix</a></dt>
<dd>
<div class="block">Creates an identity substitution matrix from match and replace values.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html#%3Cinit%3E(org.biojava.nbio.core.sequence.template.CompoundSet,java.io.File)" class="member-name-link">SimpleSubstitutionMatrix(CompoundSet&lt;C&gt;, File)</a> - Constructor for class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">SimpleSubstitutionMatrix</a></dt>
<dd>
<div class="block">Creates a substitution matrix by reading in a file.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html#%3Cinit%3E(org.biojava.nbio.core.sequence.template.CompoundSet,java.io.Reader,java.lang.String)" class="member-name-link">SimpleSubstitutionMatrix(CompoundSet&lt;C&gt;, Reader, String)</a> - Constructor for class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">SimpleSubstitutionMatrix</a></dt>
<dd>
<div class="block">Creates a substitution matrix by parsing some input.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html#%3Cinit%3E(org.biojava.nbio.core.sequence.template.CompoundSet,java.lang.String,java.lang.String)" class="member-name-link">SimpleSubstitutionMatrix(CompoundSet&lt;C&gt;, String, String)</a> - Constructor for class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">SimpleSubstitutionMatrix</a></dt>
<dd>
<div class="block">Creates a substitution matrix by parsing a String.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNATest.html#singleCompoundSequence()" class="member-name-link">singleCompoundSequence()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DNATest.html" title="class in org.biojava.nbio.core.sequence">DNATest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a>&lt;<a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="type parameter in SingleCompoundSequenceReader">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/storage/package-summary.html">org.biojava.nbio.core.sequence.storage</a></dt>
<dd>
<div class="block">An implementation of the SequenceReader interface which for every
 call will return only 1 compound (given to it during construction; a String
 is also valid but will require a CompoundSet).</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#%3Cinit%3E(C,org.biojava.nbio.core.sequence.template.CompoundSet,int)" class="member-name-link">SingleCompoundSequenceReader(C, CompoundSet&lt;C&gt;, int)</a> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Build the object with a compound rather than a String</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#%3Cinit%3E(java.lang.String,org.biojava.nbio.core.sequence.template.CompoundSet,int)" class="member-name-link">SingleCompoundSequenceReader(String, CompoundSet&lt;C&gt;, int)</a> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Public constructor to be used with String based constructor</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/SingleLinkageClusterer.html" class="type-name-link" title="class in org.biojava.nbio.core.util">SingleLinkageClusterer</a> - Class in <a href="../org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>
<div class="block">An implementation of a single linkage clusterer

 See http://en.wikipedia.org/wiki/Single-linkage_clustering</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/SingleLinkageClusterer.html#%3Cinit%3E(double%5B%5D%5B%5D,boolean)" class="member-name-link">SingleLinkageClusterer(double[][], boolean)</a> - Constructor for class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SingleLinkageClusterer.html" title="class in org.biojava.nbio.core.util">SingleLinkageClusterer</a></dt>
<dd>
<div class="block">Constructs a new SingleLinkageClusterer
 Subsequently use <a href="../org/biojava/nbio/core/util/SingleLinkageClusterer.html#getDendrogram()"><code>SingleLinkageClusterer.getDendrogram()</code></a> to get the full tree
 or <a href="../org/biojava/nbio/core/util/SingleLinkageClusterer.html#getClusters(double)"><code>SingleLinkageClusterer.getClusters(double)</code></a> to get the clusters at a certain cutoff in the tree
 Please note that the matrix will be altered during the clustering procedure.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/SingleLinkageClusterer.LinkedPair.html" class="type-name-link" title="class in org.biojava.nbio.core.util">SingleLinkageClusterer.LinkedPair</a> - Class in <a href="../org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/SingleLinkageClustererProduct.html" class="type-name-link" title="class in org.biojava.nbio.core.util">SingleLinkageClustererProduct</a> - Class in <a href="../org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/SingleLinkageClustererProduct.html#%3Cinit%3E()" class="member-name-link">SingleLinkageClustererProduct()</a> - Constructor for class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SingleLinkageClustererProduct.html" title="class in org.biojava.nbio.core.util">SingleLinkageClustererProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/SingleLinkageClustererProductProduct.html" class="type-name-link" title="class in org.biojava.nbio.core.util">SingleLinkageClustererProductProduct</a> - Class in <a href="../org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/SingleLinkageClustererProductProduct.html#%3Cinit%3E()" class="member-name-link">SingleLinkageClustererProductProduct()</a> - Constructor for class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SingleLinkageClustererProductProduct.html" title="class in org.biojava.nbio.core.util">SingleLinkageClustererProductProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/FlatFileCache.html#size()" class="member-name-link">size()</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/FlatFileCache.html" title="class in org.biojava.nbio.core.util">FlatFileCache</a></dt>
<dd>
<div class="block">Returns the number of items in the cache.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/SoftHashMap.html#size()" class="member-name-link">size()</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SoftHashMap.html" title="class in org.biojava.nbio.core.util">SoftHashMap</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#skip(long)" class="member-name-link">skip(long)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava.nbio.core.sequence.io">BufferedReaderBytesRead</a></dt>
<dd>
<div class="block">Skips characters.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/UncompressInputStream.html#skip(long)" class="member-name-link">skip(long)</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/UncompressInputStream.html" title="class in org.biojava.nbio.core.util">UncompressInputStream</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/SoftHashMap.html" class="type-name-link" title="class in org.biojava.nbio.core.util">SoftHashMap</a>&lt;<a href="../org/biojava/nbio/core/util/SoftHashMap.html" title="type parameter in SoftHashMap">K</a>,<wbr><a href="../org/biojava/nbio/core/util/SoftHashMap.html" title="type parameter in SoftHashMap">V</a>&gt; - Class in <a href="../org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>
<div class="block">A in memory cache using soft references.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/SoftHashMap.html#%3Cinit%3E()" class="member-name-link">SoftHashMap()</a> - Constructor for class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SoftHashMap.html" title="class in org.biojava.nbio.core.util">SoftHashMap</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/SoftHashMap.html#%3Cinit%3E(int)" class="member-name-link">SoftHashMap(int)</a> - Constructor for class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SoftHashMap.html" title="class in org.biojava.nbio.core.util">SoftHashMap</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/UncompressInputStreamData.html#stack" class="member-name-link">stack</a> - Variable in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/UncompressInputStreamData.html" title="class in org.biojava.nbio.core.util">UncompressInputStreamData</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/UncompressInputStreamData.html#stackp" class="member-name-link">stackp</a> - Variable in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/UncompressInputStreamData.html" title="class in org.biojava.nbio.core.util">UncompressInputStreamData</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/StartCodonSequence.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence">StartCodonSequence</a> - Class in <a href="../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">Used to map the start codon feature on a gene</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/StartCodonSequence.html#%3Cinit%3E(org.biojava.nbio.core.sequence.TranscriptSequence,int,int)" class="member-name-link">StartCodonSequence(TranscriptSequence, int, int)</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/StartCodonSequence.html" title="class in org.biojava.nbio.core.sequence">StartCodonSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#stopAtStopCodons(boolean)" class="member-name-link">stopAtStopCodons(boolean)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>
<div class="block">If set, then the last codon translated in the resulting peptide
 sequence will be the stop codon</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/StopCodonSequence.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence">StopCodonSequence</a> - Class in <a href="../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">Used to map the stop codon sequence on a gene</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/StopCodonSequence.html#%3Cinit%3E(org.biojava.nbio.core.sequence.TranscriptSequence,int,int)" class="member-name-link">StopCodonSequence(TranscriptSequence, int, int)</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/StopCodonSequence.html" title="class in org.biojava.nbio.core.sequence">StopCodonSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastTabularParser.html#storeObjects(java.util.List)" class="member-name-link">storeObjects(List&lt;Result&gt;)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastTabularParser.html" title="class in org.biojava.nbio.core.search.io.blast">BlastTabularParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParser.html#storeObjects(java.util.List)" class="member-name-link">storeObjects(List&lt;Result&gt;)</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParser.html" title="class in org.biojava.nbio.core.search.io.blast">BlastXMLParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/ResultFactory.html#storeObjects(java.util.List)" class="member-name-link">storeObjects(List&lt;Result&gt;)</a> - Method in interface org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/ResultFactory.html" title="interface in org.biojava.nbio.core.search.io">ResultFactory</a></dt>
<dd>
<div class="block">The factory that implements this method will be able to save the Search results
 to a file in the same format that it is able to read.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/Strand.html" class="type-name-link" title="enum in org.biojava.nbio.core.sequence">Strand</a> - Enum in <a href="../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">Provides a way of representing the strand of a sequence, location
 hit or feature.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/StringManipulationHelper.html" class="type-name-link" title="class in org.biojava.nbio.core.util">StringManipulationHelper</a> - Class in <a href="../org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>
<div class="block">A utility class for common <a href="https://docs.oracle.com/javase/8/docs/api/java/lang/String.html" title="class or interface in java.lang" class="external-link"><code>String</code></a> manipulation tasks.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a>&lt;<a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="type parameter in StringProxySequenceReader">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/loader/package-summary.html">org.biojava.nbio.core.sequence.loader</a></dt>
<dd>
<div class="block">An example of a ProxySequenceReader that is created from a String.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#%3Cinit%3E()" class="member-name-link">StringProxySequenceReader()</a> - Constructor for class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#%3Cinit%3E(java.lang.String,org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">StringProxySequenceReader(String, CompoundSet&lt;C&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/SequenceOptimizationHints.SequenceUsage.html#SUB_SEQUENCE_DATA" class="member-name-link">SUB_SEQUENCE_DATA</a> - Enum constant in enum org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/SequenceOptimizationHints.SequenceUsage.html" title="enum in org.biojava.nbio.core.sequence">SequenceOptimizationHints.SequenceUsage</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/ConcurrencyTools.html#submit(java.util.concurrent.Callable)" class="member-name-link">submit(Callable&lt;T&gt;)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/ConcurrencyTools.html" title="class in org.biojava.nbio.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Queues up a task and adds a default log entry.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/ConcurrencyTools.html#submit(java.util.concurrent.Callable,java.lang.String)" class="member-name-link">submit(Callable&lt;T&gt;, String)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/ConcurrencyTools.html" title="class in org.biojava.nbio.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Queues up a task and adds a log entry.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNATest.html#subSequence()" class="member-name-link">subSequence()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DNATest.html" title="class in org.biojava.nbio.core.sequence">DNATest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/EditSequenceTest.html#substitute()" class="member-name-link">substitute()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/EditSequenceTest.html" title="class in org.biojava.nbio.core.sequence">EditSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/edits/Edit.Substitute.html#%3Cinit%3E(java.lang.String,int)" class="member-name-link">Substitute(String, int)</a> - Constructor for class org.biojava.nbio.core.sequence.edits.<a href="../org/biojava/nbio/core/sequence/edits/Edit.Substitute.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Substitute</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/edits/Edit.Substitute.html#%3Cinit%3E(org.biojava.nbio.core.sequence.template.Sequence,int)" class="member-name-link">Substitute(Sequence&lt;C&gt;, int)</a> - Constructor for class org.biojava.nbio.core.sequence.edits.<a href="../org/biojava/nbio/core/sequence/edits/Edit.Substitute.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Substitute</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/SubstitutionMatrix.html" class="type-name-link" title="interface in org.biojava.nbio.core.alignment.template">SubstitutionMatrix</a>&lt;<a href="../org/biojava/nbio/core/alignment/template/SubstitutionMatrix.html" title="type parameter in SubstitutionMatrix">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="../org/biojava/nbio/core/alignment/template/package-summary.html">org.biojava.nbio.core.alignment.template</a></dt>
<dd>
<div class="block">Defines a data structure which holds the score (penalty or bonus) given during alignment for the exchange of one
 <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Compound</code></a> in a sequence for another.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html" class="type-name-link" title="class in org.biojava.nbio.core.alignment.matrices">SubstitutionMatrixHelper</a> - Class in <a href="../org/biojava/nbio/core/alignment/matrices/package-summary.html">org.biojava.nbio.core.alignment.matrices</a></dt>
<dd>
<div class="block">Static utility to access substitution matrices that come bundled with BioJava.</div>
</dd>
</dl>
<a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Z</a>&nbsp;<br><a href="../allclasses-index.html">All&nbsp;Classes&nbsp;and&nbsp;Interfaces</a><span class="vertical-separator">|</span><a href="../allpackages-index.html">All&nbsp;Packages</a><span class="vertical-separator">|</span><a href="../constant-values.html">Constant&nbsp;Field&nbsp;Values</a><span class="vertical-separator">|</span><a href="../serialized-form.html">Serialized&nbsp;Form</a></main>
</div>
</div>
</body>
</html>
