<!DOCTYPE HTML>
<html lang="pt">
<head>
<!-- Generated by javadoc (17) on Sun May 15 18:36:12 WEST 2022 -->
<title>T-Index</title>
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta name="dc.created" content="2022-05-15">
<meta name="description" content="index: T">
<meta name="generator" content="javadoc/IndexWriter">
<link rel="stylesheet" type="text/css" href="../stylesheet.css" title="Style">
<link rel="stylesheet" type="text/css" href="../script-dir/jquery-ui.min.css" title="Style">
<link rel="stylesheet" type="text/css" href="../jquery-ui.overrides.css" title="Style">
<script type="text/javascript" src="../script.js"></script>
<script type="text/javascript" src="../script-dir/jquery-3.5.1.min.js"></script>
<script type="text/javascript" src="../script-dir/jquery-ui.min.js"></script>
</head>
<body class="index-page">
<script type="text/javascript">var pathtoroot = "../";
loadScripts(document, 'script');</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<div class="flex-box">
<header role="banner" class="flex-header">
<nav role="navigation">
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="top-nav" id="navbar-top">
<div class="skip-nav"><a href="#skip-navbar-top" title="Skip navigation links">Skip navigation links</a></div>
<ul id="navbar-top-firstrow" class="nav-list" title="Navigation">
<li><a href="../index.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li>Use</li>
<li><a href="../overview-tree.html">Tree</a></li>
<li><a href="../deprecated-list.html">Deprecated</a></li>
<li class="nav-bar-cell1-rev">Index</li>
<li><a href="../help-doc.html#index">Help</a></li>
</ul>
</div>
<div class="sub-nav">
<div class="nav-list-search"><label for="search-input">SEARCH:</label>
<input type="text" id="search-input" value="search" disabled="disabled">
<input type="reset" id="reset-button" value="reset" disabled="disabled">
</div>
</div>
<!-- ========= END OF TOP NAVBAR ========= -->
<span class="skip-nav" id="skip-navbar-top"></span></nav>
</header>
<div class="flex-content">
<main role="main">
<div class="header">
<h1>Index</h1>
</div>
<a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Z</a>&nbsp;<br><a href="../allclasses-index.html">All&nbsp;Classes&nbsp;and&nbsp;Interfaces</a><span class="vertical-separator">|</span><a href="../allpackages-index.html">All&nbsp;Packages</a><span class="vertical-separator">|</span><a href="../constant-values.html">Constant&nbsp;Field&nbsp;Values</a><span class="vertical-separator">|</span><a href="../serialized-form.html">Serialized&nbsp;Form</a>
<h2 class="title" id="I:T">T</h2>
<dl class="index">
<dt><a href="../org/biojava/nbio/core/util/UncompressInputStreamData.html#tab_prefix" class="member-name-link">tab_prefix</a> - Variable in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/UncompressInputStreamData.html" title="class in org.biojava.nbio.core.util">UncompressInputStreamData</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/UncompressInputStreamData.html#tab_suffix" class="member-name-link">tab_suffix</a> - Variable in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/UncompressInputStreamData.html" title="class in org.biojava.nbio.core.util">UncompressInputStreamData</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#table(java.lang.Integer)" class="member-name-link">table(Integer)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>
<div class="block">Uses the static instance of <a href="../org/biojava/nbio/core/sequence/io/IUPACParser.html" title="class in org.biojava.nbio.core.sequence.io"><code>IUPACParser</code></a> to find instances of
 <a href="../org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava.nbio.core.sequence.io"><code>IUPACParser.IUPACTable</code></a>s by ID.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#table(java.lang.String)" class="member-name-link">table(String)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>
<div class="block">Uses the static instance of <a href="../org/biojava/nbio/core/sequence/io/IUPACParser.html" title="class in org.biojava.nbio.core.sequence.io"><code>IUPACParser</code></a> to find instances of
 <a href="../org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava.nbio.core.sequence.io"><code>IUPACParser.IUPACTable</code></a>s by its String name</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#table(org.biojava.nbio.core.sequence.transcription.Table)" class="member-name-link">table(Table)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.html" class="type-name-link" title="interface in org.biojava.nbio.core.sequence.transcription">Table</a> - Interface in <a href="../org/biojava/nbio/core/sequence/transcription/package-summary.html">org.biojava.nbio.core.sequence.transcription</a></dt>
<dd>
<div class="block">Provides a way of separating us from the specific <a href="../org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava.nbio.core.sequence.io"><code>IUPACParser.IUPACTable</code></a> even
 though this is the only implementing class for the interface.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.transcription">Table.CaseInsensitiveTriplet</a> - Class in <a href="../org/biojava/nbio/core/sequence/transcription/package-summary.html">org.biojava.nbio.core.sequence.transcription</a></dt>
<dd>
<div class="block">Class used to hold three nucleotides together and allow for equality
 to be assessed in a case insensitive manner.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a> - Class in <a href="../org/biojava/nbio/core/sequence/transcription/package-summary.html">org.biojava.nbio.core.sequence.transcription</a></dt>
<dd>
<div class="block">Instance of a Codon which is 3 <a href="../org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound"><code>NucleotideCompound</code></a>s, its
 corresponding <a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava.nbio.core.sequence.compound"><code>AminoAcidCompound</code></a> and if it is a start or stop codon.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/TargetedLocationParserTest.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.location">TargetedLocationParserTest</a> - Class in <a href="../org/biojava/nbio/core/sequence/location/package-summary.html">org.biojava.nbio.core.sequence.location</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/TargetedLocationParserTest.html#%3Cinit%3E(org.biojava.nbio.core.sequence.location.TargetedLocationParserTest.Data)" class="member-name-link">TargetedLocationParserTest(TargetedLocationParserTest.Data)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/TargetedLocationParserTest.html" title="class in org.biojava.nbio.core.sequence.location">TargetedLocationParserTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/TargetedLocationParserTest.Data.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.location">TargetedLocationParserTest.Data</a> - Class in <a href="../org/biojava/nbio/core/sequence/location/package-summary.html">org.biojava.nbio.core.sequence.location</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TaxonomyID.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence">TaxonomyID</a> - Class in <a href="../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">A sequence can be associated with a species or Taxonomy ID</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TaxonomyID.html#%3Cinit%3E(java.lang.String,org.biojava.nbio.core.sequence.DataSource)" class="member-name-link">TaxonomyID(String, DataSource)</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TaxonomyID.html" title="class in org.biojava.nbio.core.sequence">TaxonomyID</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastTabularParserTest.html#tearDown()" class="member-name-link">tearDown()</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastTabularParserTest.html" title="class in org.biojava.nbio.core.search.io.blast">BlastTabularParserTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserTest.html#tearDown()" class="member-name-link">tearDown()</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserTest.html" title="class in org.biojava.nbio.core.search.io.blast">BlastXMLParserTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/HspTest.html#tearDown()" class="member-name-link">tearDown()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/HspTest.html" title="class in org.biojava.nbio.core.search.io">HspTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/SearchIOTest.html#tearDown()" class="member-name-link">tearDown()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/SearchIOTest.html" title="class in org.biojava.nbio.core.search.io">SearchIOTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaReaderTest.html#tearDown()" class="member-name-link">tearDown()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaReaderTest.html" title="class in org.biojava.nbio.core.sequence.io">FastaReaderTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankCookbookTest.html#tearDown()" class="member-name-link">tearDown()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankCookbookTest.html" title="class in org.biojava.nbio.core.sequence.io">GenbankCookbookTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastTabularParserTest.html#tearDownClass()" class="member-name-link">tearDownClass()</a> - Static method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastTabularParserTest.html" title="class in org.biojava.nbio.core.search.io.blast">BlastTabularParserTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserTest.html#tearDownClass()" class="member-name-link">tearDownClass()</a> - Static method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserTest.html" title="class in org.biojava.nbio.core.search.io.blast">BlastXMLParserTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/HspTest.html#tearDownClass()" class="member-name-link">tearDownClass()</a> - Static method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/HspTest.html" title="class in org.biojava.nbio.core.search.io">HspTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/SearchIOTest.html#tearDownClass()" class="member-name-link">tearDownClass()</a> - Static method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/SearchIOTest.html" title="class in org.biojava.nbio.core.search.io">SearchIOTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaReaderTest.html#tearDownClass()" class="member-name-link">tearDownClass()</a> - Static method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaReaderTest.html" title="class in org.biojava.nbio.core.sequence.io">FastaReaderTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankCookbookTest.html#tearDownClass()" class="member-name-link">tearDownClass()</a> - Static method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankCookbookTest.html" title="class in org.biojava.nbio.core.sequence.io">GenbankCookbookTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AmbiguityDNACompoundTest.html#testAmbiguity()" class="member-name-link">testAmbiguity()</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/AmbiguityDNACompoundTest.html" title="class in org.biojava.nbio.core.sequence.compound">AmbiguityDNACompoundTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/TestAmbiguityCompoundSet.html" class="type-name-link" title="class in org.biojava.nbio.core">TestAmbiguityCompoundSet</a> - Class in <a href="../org/biojava/nbio/core/package-summary.html">org.biojava.nbio.core</a></dt>
<dd>
<div class="block">A Test case for https://github.com/biojava/biojava/issues/344

 Created by andreas on 12/4/15.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/TestAmbiguityCompoundSet.html#%3Cinit%3E()" class="member-name-link">TestAmbiguityCompoundSet()</a> - Constructor for class org.biojava.nbio.core.<a href="../org/biojava/nbio/core/TestAmbiguityCompoundSet.html" title="class in org.biojava.nbio.core">TestAmbiguityCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AmbiguityDNACompoundTest.html#testBasicAmbiguity()" class="member-name-link">testBasicAmbiguity()</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/AmbiguityDNACompoundTest.html" title="class in org.biojava.nbio.core.sequence.compound">AmbiguityDNACompoundTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/LocationTest.html#testBasicCircularLocation()" class="member-name-link">testBasicCircularLocation()</a> - Method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/LocationTest.html" title="class in org.biojava.nbio.core.sequence.location">LocationTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/TestAmbiguityCompoundSet.html#testCompountSet()" class="member-name-link">testCompountSet()</a> - Method in class org.biojava.nbio.core.<a href="../org/biojava/nbio/core/TestAmbiguityCompoundSet.html" title="class in org.biojava.nbio.core">TestAmbiguityCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/SearchIOTest.html#testConstructorWithEvalueHspFilter()" class="member-name-link">testConstructorWithEvalueHspFilter()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/SearchIOTest.html" title="class in org.biojava.nbio.core.search.io">SearchIOTest</a></dt>
<dd>
<div class="block">Constructor test specifying Factory and using a evalue threshold filter</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/SearchIOTest.html#testConstructorWithFactoryGuess()" class="member-name-link">testConstructorWithFactoryGuess()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/SearchIOTest.html" title="class in org.biojava.nbio.core.search.io">SearchIOTest</a></dt>
<dd>
<div class="block">Constructor test with GuessFactory</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/SearchIOTest.html#testConstructorWithoutFactoryGuess()" class="member-name-link">testConstructorWithoutFactoryGuess()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/SearchIOTest.html" title="class in org.biojava.nbio.core.search.io">SearchIOTest</a></dt>
<dd>
<div class="block">Constructor test specifying Factory</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testCountCompounds()" class="member-name-link">testCountCompounds()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastTabularParserTest.html#testCreateObjects()" class="member-name-link">testCreateObjects()</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastTabularParserTest.html" title="class in org.biojava.nbio.core.search.io.blast">BlastTabularParserTest</a></dt>
<dd>
<div class="block">Test of createObjects method, of class BlastTabularParser.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserTest.html#testCreateObjects()" class="member-name-link">testCreateObjects()</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserTest.html" title="class in org.biojava.nbio.core.search.io.blast">BlastXMLParserTest</a></dt>
<dd>
<div class="block">Test of createObjects method, of class BlastXMLParser.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/HspTest.html#testEquals()" class="member-name-link">testEquals()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/HspTest.html" title="class in org.biojava.nbio.core.search.io">HspTest</a></dt>
<dd>
<div class="block">Test of equals method, of class Hsp.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/fasta/TestFASTAReader.html" class="type-name-link" title="class in org.biojava.nbio.core.fasta">TestFASTAReader</a> - Class in <a href="../org/biojava/nbio/core/fasta/package-summary.html">org.biojava.nbio.core.fasta</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/fasta/TestFASTAReader.html#%3Cinit%3E()" class="member-name-link">TestFASTAReader()</a> - Constructor for class org.biojava.nbio.core.fasta.<a href="../org/biojava/nbio/core/fasta/TestFASTAReader.html" title="class in org.biojava.nbio.core.fasta">TestFASTAReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReaderTest.html#testFeatures()" class="member-name-link">testFeatures()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReaderTest.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReaderTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testGetAccession()" class="member-name-link">testGetAccession()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetAlignedSequenceInt()" class="member-name-link">testGetAlignedSequenceInt()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetAlignedSequenceIntOutOfBounds()" class="member-name-link">testGetAlignedSequenceIntOutOfBounds()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetAlignedSequenceIntOutOfBounds2()" class="member-name-link">testGetAlignedSequenceIntOutOfBounds2()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetAlignedSequenceIntOutOfBounds3()" class="member-name-link">testGetAlignedSequenceIntOutOfBounds3()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetAlignedSequenceIntOutOfBounds4()" class="member-name-link">testGetAlignedSequenceIntOutOfBounds4()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetAlignedSequenceIntOutOfBounds5()" class="member-name-link">testGetAlignedSequenceIntOutOfBounds5()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetAlignedSequenceIntOutOfBounds6()" class="member-name-link">testGetAlignedSequenceIntOutOfBounds6()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetAlignedSequences()" class="member-name-link">testGetAlignedSequences()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetAlignedSequenceS()" class="member-name-link">testGetAlignedSequenceS()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetAlignedSequencesIntArray()" class="member-name-link">testGetAlignedSequencesIntArray()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetAlignedSequencesSArray()" class="member-name-link">testGetAlignedSequencesSArray()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/HspTest.html#testGetAlignment()" class="member-name-link">testGetAlignment()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/HspTest.html" title="class in org.biojava.nbio.core.search.io">HspTest</a></dt>
<dd>
<div class="block">Test of getAlignment method, of class Hsp.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testGetAlignmentIndexAt()" class="member-name-link">testGetAlignmentIndexAt()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testGetAlignmentIndexAtOutOfBounds()" class="member-name-link">testGetAlignmentIndexAtOutOfBounds()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testGetAlignmentIndexAtOutOfBounds2()" class="member-name-link">testGetAlignmentIndexAtOutOfBounds2()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testGetAlignmentIndexAtOutOfBounds3()" class="member-name-link">testGetAlignmentIndexAtOutOfBounds3()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testGetAlignmentIndexAtOutOfBounds4()" class="member-name-link">testGetAlignmentIndexAtOutOfBounds4()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testGetAsList()" class="member-name-link">testGetAsList()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testGetCompoundAt()" class="member-name-link">testGetCompoundAt()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/SequenceViewTest.html#testGetCompoundAt()" class="member-name-link">testGetCompoundAt()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/SequenceViewTest.html" title="class in org.biojava.nbio.core.sequence">SequenceViewTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetCompoundAtIntInt()" class="member-name-link">testGetCompoundAtIntInt()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetCompoundAtIntIntOutOfBounds()" class="member-name-link">testGetCompoundAtIntIntOutOfBounds()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetCompoundAtIntIntOutOfBounds2()" class="member-name-link">testGetCompoundAtIntIntOutOfBounds2()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetCompoundAtIntIntOutOfBounds3()" class="member-name-link">testGetCompoundAtIntIntOutOfBounds3()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetCompoundAtIntIntOutOfBounds4()" class="member-name-link">testGetCompoundAtIntIntOutOfBounds4()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetCompoundAtIntIntOutOfBounds5()" class="member-name-link">testGetCompoundAtIntIntOutOfBounds5()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetCompoundAtIntIntOutOfBounds6()" class="member-name-link">testGetCompoundAtIntIntOutOfBounds6()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetCompoundAtIntIntOutOfBounds7()" class="member-name-link">testGetCompoundAtIntIntOutOfBounds7()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetCompoundAtIntIntOutOfBounds8()" class="member-name-link">testGetCompoundAtIntIntOutOfBounds8()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetCompoundAtIntIntOutOfBounds9()" class="member-name-link">testGetCompoundAtIntIntOutOfBounds9()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetCompoundAtSInt()" class="member-name-link">testGetCompoundAtSInt()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetCompoundAtSIntOutOfBounds()" class="member-name-link">testGetCompoundAtSIntOutOfBounds()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetCompoundAtSIntOutOfBounds2()" class="member-name-link">testGetCompoundAtSIntOutOfBounds2()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetCompoundAtSIntOutOfBounds3()" class="member-name-link">testGetCompoundAtSIntOutOfBounds3()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetCompoundAtSIntOutOfBounds4()" class="member-name-link">testGetCompoundAtSIntOutOfBounds4()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetCompoundAtSIntOutOfBounds5()" class="member-name-link">testGetCompoundAtSIntOutOfBounds5()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetCompoundInQueryAt()" class="member-name-link">testGetCompoundInQueryAt()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetCompoundInQueryAtOutOfBounds()" class="member-name-link">testGetCompoundInQueryAtOutOfBounds()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetCompoundInQueryAtOutOfBounds2()" class="member-name-link">testGetCompoundInQueryAtOutOfBounds2()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetCompoundInQueryAtOutOfBounds3()" class="member-name-link">testGetCompoundInQueryAtOutOfBounds3()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetCompoundInQueryAtOutOfBounds4()" class="member-name-link">testGetCompoundInQueryAtOutOfBounds4()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetCompoundInTargetAt()" class="member-name-link">testGetCompoundInTargetAt()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetCompoundInTargetAtOutOfBounds()" class="member-name-link">testGetCompoundInTargetAtOutOfBounds()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetCompoundInTargetAtOutOfBounds2()" class="member-name-link">testGetCompoundInTargetAtOutOfBounds2()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetCompoundInTargetAtOutOfBounds3()" class="member-name-link">testGetCompoundInTargetAtOutOfBounds3()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetCompoundInTargetAtOutOfBounds4()" class="member-name-link">testGetCompoundInTargetAtOutOfBounds4()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetCompoundsAt()" class="member-name-link">testGetCompoundsAt()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetCompoundsAtOutOfBounds()" class="member-name-link">testGetCompoundsAtOutOfBounds()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetCompoundsAtOutOfBounds2()" class="member-name-link">testGetCompoundsAtOutOfBounds2()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetCompoundsAtOutOfBounds3()" class="member-name-link">testGetCompoundsAtOutOfBounds3()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetCompoundsAtOutOfBounds4()" class="member-name-link">testGetCompoundsAtOutOfBounds4()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetCompoundsAtOutOfBounds5()" class="member-name-link">testGetCompoundsAtOutOfBounds5()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetCompoundsAtOutOfBounds6()" class="member-name-link">testGetCompoundsAtOutOfBounds6()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testGetCompoundSet()" class="member-name-link">testGetCompoundSet()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetCompoundSet()" class="member-name-link">testGetCompoundSet()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testGetCoverage()" class="member-name-link">testGetCoverage()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testGetEnd()" class="member-name-link">testGetEnd()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastTabularParserTest.html#testGetFileExtensions()" class="member-name-link">testGetFileExtensions()</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastTabularParserTest.html" title="class in org.biojava.nbio.core.search.io.blast">BlastTabularParserTest</a></dt>
<dd>
<div class="block">Test of getFileExtensions method, of class BlastTabularParser.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserTest.html#testGetFileExtensions()" class="member-name-link">testGetFileExtensions()</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserTest.html" title="class in org.biojava.nbio.core.search.io.blast">BlastXMLParserTest</a></dt>
<dd>
<div class="block">Test of getFileExtensions method, of class BlastXMLParser.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetIndexInQueryAt()" class="member-name-link">testGetIndexInQueryAt()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetIndexInQueryAtOutOfBounds()" class="member-name-link">testGetIndexInQueryAtOutOfBounds()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetIndexInQueryAtOutOfBounds2()" class="member-name-link">testGetIndexInQueryAtOutOfBounds2()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetIndexInQueryAtOutOfBounds3()" class="member-name-link">testGetIndexInQueryAtOutOfBounds3()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetIndexInQueryAtOutOfBounds4()" class="member-name-link">testGetIndexInQueryAtOutOfBounds4()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetIndexInQueryForTargetAt()" class="member-name-link">testGetIndexInQueryForTargetAt()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetIndexInQueryForTargetAtOutOfBounds()" class="member-name-link">testGetIndexInQueryForTargetAtOutOfBounds()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetIndexInQueryForTargetAtOutOfBounds2()" class="member-name-link">testGetIndexInQueryForTargetAtOutOfBounds2()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetIndexInQueryForTargetAtOutOfBounds3()" class="member-name-link">testGetIndexInQueryForTargetAtOutOfBounds3()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetIndexInQueryForTargetAtOutOfBounds4()" class="member-name-link">testGetIndexInQueryForTargetAtOutOfBounds4()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetIndexInTargetAt()" class="member-name-link">testGetIndexInTargetAt()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetIndexInTargetAtOutOfBounds()" class="member-name-link">testGetIndexInTargetAtOutOfBounds()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetIndexInTargetAtOutOfBounds2()" class="member-name-link">testGetIndexInTargetAtOutOfBounds2()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetIndexInTargetAtOutOfBounds3()" class="member-name-link">testGetIndexInTargetAtOutOfBounds3()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetIndexInTargetAtOutOfBounds4()" class="member-name-link">testGetIndexInTargetAtOutOfBounds4()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetIndexInTargetForQueryAt()" class="member-name-link">testGetIndexInTargetForQueryAt()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetIndexInTargetForQueryAtOutOfBounds()" class="member-name-link">testGetIndexInTargetForQueryAtOutOfBounds()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetIndexInTargetForQueryAtOutOfBounds2()" class="member-name-link">testGetIndexInTargetForQueryAtOutOfBounds2()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetIndexInTargetForQueryAtOutOfBounds3()" class="member-name-link">testGetIndexInTargetForQueryAtOutOfBounds3()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetIndexInTargetForQueryAtOutOfBounds4()" class="member-name-link">testGetIndexInTargetForQueryAtOutOfBounds4()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testGetIndexOf()" class="member-name-link">testGetIndexOf()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetIndexOf()" class="member-name-link">testGetIndexOf()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetIndicesAt()" class="member-name-link">testGetIndicesAt()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetIndicesAtOutOfBounds()" class="member-name-link">testGetIndicesAtOutOfBounds()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetIndicesAtOutOfBounds2()" class="member-name-link">testGetIndicesAtOutOfBounds2()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetIndicesAtOutOfBounds3()" class="member-name-link">testGetIndicesAtOutOfBounds3()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetIndicesAtOutOfBounds4()" class="member-name-link">testGetIndicesAtOutOfBounds4()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetIndicesAtOutOfBounds5()" class="member-name-link">testGetIndicesAtOutOfBounds5()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetIndicesAtOutOfBounds6()" class="member-name-link">testGetIndicesAtOutOfBounds6()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testGetLastIndexOf()" class="member-name-link">testGetLastIndexOf()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetLastIndexOf()" class="member-name-link">testGetLastIndexOf()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testGetLength()" class="member-name-link">testGetLength()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetLength()" class="member-name-link">testGetLength()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testGetLocationInAlignment()" class="member-name-link">testGetLocationInAlignment()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testGetNumGapPositions()" class="member-name-link">testGetNumGapPositions()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testGetNumGaps()" class="member-name-link">testGetNumGaps()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetNumIdenticals()" class="member-name-link">testGetNumIdenticals()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetNumSimilars()" class="member-name-link">testGetNumSimilars()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testGetOriginalSequence()" class="member-name-link">testGetOriginalSequence()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetOriginalSequences()" class="member-name-link">testGetOriginalSequences()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testGetOverlapCount()" class="member-name-link">testGetOverlapCount()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetPercentageOfIdentity()" class="member-name-link">testGetPercentageOfIdentity()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetQuery()" class="member-name-link">testGetQuery()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testGetSequenceAsString()" class="member-name-link">testGetSequenceAsString()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testGetSequenceAsStringIntegerIntegerStrand()" class="member-name-link">testGetSequenceAsStringIntegerIntegerStrand()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testGetSequenceIndexAt()" class="member-name-link">testGetSequenceIndexAt()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testGetSequenceIndexAtOutOfBounds()" class="member-name-link">testGetSequenceIndexAtOutOfBounds()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testGetSequenceIndexAtOutOfBounds2()" class="member-name-link">testGetSequenceIndexAtOutOfBounds2()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testGetSequenceIndexAtOutOfBounds3()" class="member-name-link">testGetSequenceIndexAtOutOfBounds3()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testGetSequenceIndexAtOutOfBounds4()" class="member-name-link">testGetSequenceIndexAtOutOfBounds4()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetSize()" class="member-name-link">testGetSize()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testGetStart()" class="member-name-link">testGetStart()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testGetSubProfile()" class="member-name-link">testGetSubProfile()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testGetSubSequence()" class="member-name-link">testGetSubSequence()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html#testGetTarget()" class="member-name-link">testGetTarget()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankReaderTest.html#testGithub843()" class="member-name-link">testGithub843()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankReaderTest.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReaderTest</a></dt>
<dd>
<div class="block">Biojava fails to parse anticodon and transl_except feature qualifiers when they line wrap.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/HspTest.html#testHashCode()" class="member-name-link">testHashCode()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/HspTest.html" title="class in org.biojava.nbio.core.search.io">HspTest</a></dt>
<dd>
<div class="block">Test of hashCode method, of class Hsp.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranslationTest.html#testHashCollision()" class="member-name-link">testHashCollision()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranslationTest.html" title="class in org.biojava.nbio.core.sequence">TranslationTest</a></dt>
<dd>
<div class="block">test for https://github.com/biojava/biojava/issues/53</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/SequenceViewTest.html#testInverse()" class="member-name-link">testInverse()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/SequenceViewTest.html" title="class in org.biojava.nbio.core.sequence">SequenceViewTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testIsCircular()" class="member-name-link">testIsCircular()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testIsCircular()" class="member-name-link">testIsCircular()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testIterator()" class="member-name-link">testIterator()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testIterator()" class="member-name-link">testIterator()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/SequenceViewTest.html#testLastIndexOf()" class="member-name-link">testLastIndexOf()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/SequenceViewTest.html" title="class in org.biojava.nbio.core.sequence">SequenceViewTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankReaderTest.html#testLegacyLocusCompatable()" class="member-name-link">testLegacyLocusCompatable()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankReaderTest.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReaderTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankReaderTest.html#testNcbiExpandedAccessionFormats()" class="member-name-link">testNcbiExpandedAccessionFormats()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankReaderTest.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReaderTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenericFastaHeaderParserTest.html#testParseHeader()" class="member-name-link">testParseHeader()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenericFastaHeaderParserTest.html" title="class in org.biojava.nbio.core.sequence.io">GenericFastaHeaderParserTest</a></dt>
<dd>
<div class="block">GenBank                           gi|gi-number|gb|accession|locus
 ENA Data Library                 gi|gi-number|emb|accession|locus
 DDBJ, DNA Database of Japan       gi|gi-number|dbj|accession|locus
 NBRF PIR                          pir||entry
 Protein Research Foundation       prf||name
 SWISS-PROT                        sp|accession|name
 Brookhaven Protein Data Bank (1)  pdb|entry|chain
 Brookhaven Protein Data Bank (2)  entry:chain|PDBID|CHAIN|SEQUENCE
 PDB EBI                           PDB:1ECY_A mol:protein length:142  ECOTIN
 Patents                           pat|country|number
 GenInfo Backbone Id               bbs|number
 General database identifier       gnl|database|identifier
 NCBI Reference Sequence           ref|accession|locus
 Local Sequence identifier         lcl|identifier</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankReaderTest.html#testPartialProcess()" class="member-name-link">testPartialProcess()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankReaderTest.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReaderTest</a></dt>
<dd>
<div class="block">Test the process method with a number of sequences to be read at each call.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaReaderTest.html#testProcess()" class="member-name-link">testProcess()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaReaderTest.html" title="class in org.biojava.nbio.core.sequence.io">FastaReaderTest</a></dt>
<dd>
<div class="block">Test of process method, of class FastaReader.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankCookbookTest.html#testProcess()" class="member-name-link">testProcess()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankCookbookTest.html" title="class in org.biojava.nbio.core.sequence.io">GenbankCookbookTest</a></dt>
<dd>
<div class="block">Test of process method, of class GenbankReader.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankReaderTest.html#testProcess()" class="member-name-link">testProcess()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankReaderTest.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReaderTest</a></dt>
<dd>
<div class="block">Test of process method, of class GenbankReader.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankWriterTest.html#testProcess()" class="member-name-link">testProcess()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankWriterTest.html" title="class in org.biojava.nbio.core.sequence.io">GenbankWriterTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/fasta/TestFASTAReader.html#testProcess1()" class="member-name-link">testProcess1()</a> - Method in class org.biojava.nbio.core.fasta.<a href="../org/biojava/nbio/core/fasta/TestFASTAReader.html" title="class in org.biojava.nbio.core.fasta">TestFASTAReader</a></dt>
<dd>
<div class="block">Test file contains one sequence (P02768 from swissprot).</div>
</dd>
<dt><a href="../org/biojava/nbio/core/fasta/TestFASTAReader.html#testProcess1WithBlankLines()" class="member-name-link">testProcess1WithBlankLines()</a> - Method in class org.biojava.nbio.core.fasta.<a href="../org/biojava/nbio/core/fasta/TestFASTAReader.html" title="class in org.biojava.nbio.core.fasta">TestFASTAReader</a></dt>
<dd>
<div class="block">Same as <a href="../org/biojava/nbio/core/fasta/TestFASTAReader.html#testProcess1()"><code>TestFASTAReader.testProcess1()</code></a>, but input contains blank lines.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/fasta/TestFASTAReader.html#testProcess2()" class="member-name-link">testProcess2()</a> - Method in class org.biojava.nbio.core.fasta.<a href="../org/biojava/nbio/core/fasta/TestFASTAReader.html" title="class in org.biojava.nbio.core.fasta">TestFASTAReader</a></dt>
<dd>
<div class="block">Test file contains two sequences.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/fasta/TestFASTAReader.html#testProcess2WithBlankLines()" class="member-name-link">testProcess2WithBlankLines()</a> - Method in class org.biojava.nbio.core.fasta.<a href="../org/biojava/nbio/core/fasta/TestFASTAReader.html" title="class in org.biojava.nbio.core.fasta">TestFASTAReader</a></dt>
<dd>
<div class="block">Sane as <a href="../org/biojava/nbio/core/fasta/TestFASTAReader.html#testProcess2()"><code>TestFASTAReader.testProcess2()</code></a> but input file contain blank lines</div>
</dd>
<dt><a href="../org/biojava/nbio/core/fasta/TestFASTAReader.html#testProcessAll()" class="member-name-link">testProcessAll()</a> - Method in class org.biojava.nbio.core.fasta.<a href="../org/biojava/nbio/core/fasta/TestFASTAReader.html" title="class in org.biojava.nbio.core.fasta">TestFASTAReader</a></dt>
<dd>
<div class="block">Test file contains one sequence (P02768 from swissprot).</div>
</dd>
<dt><a href="../org/biojava/nbio/core/fasta/TestFASTAReader.html#testProcessAllWithBlankLines()" class="member-name-link">testProcessAllWithBlankLines()</a> - Method in class org.biojava.nbio.core.fasta.<a href="../org/biojava/nbio/core/fasta/TestFASTAReader.html" title="class in org.biojava.nbio.core.fasta">TestFASTAReader</a></dt>
<dd>
<div class="block">Same as <a href="../org/biojava/nbio/core/fasta/TestFASTAReader.html#testProcessAll()"><code>TestFASTAReader.testProcessAll()</code></a> but input files contains blank lines</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReaderTest.html#testProteinSequenceFactoring()" class="member-name-link">testProteinSequenceFactoring()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReaderTest.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReaderTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TestSequenceEquals.html#testSameCompounds()" class="member-name-link">testSameCompounds()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TestSequenceEquals.html" title="class in org.biojava.nbio.core.sequence">TestSequenceEquals</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TestSequenceEquals.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence">TestSequenceEquals</a> - Class in <a href="../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">Created by yana on 3/27/17.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TestSequenceEquals.html#%3Cinit%3E()" class="member-name-link">TestSequenceEquals()</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TestSequenceEquals.html" title="class in org.biojava.nbio.core.sequence">TestSequenceEquals</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserTest.html#testSetDatabaseReferences()" class="member-name-link">testSetDatabaseReferences()</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserTest.html" title="class in org.biojava.nbio.core.search.io.blast">BlastXMLParserTest</a></dt>
<dd>
<div class="block">Test of setDatabaseReferences method, of class BlastXMLParser.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastTabularParserTest.html#testSetFile()" class="member-name-link">testSetFile()</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastTabularParserTest.html" title="class in org.biojava.nbio.core.search.io.blast">BlastTabularParserTest</a></dt>
<dd>
<div class="block">Test of setFile method, of class BlastTabularParser.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserTest.html#testSetFile()" class="member-name-link">testSetFile()</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserTest.html" title="class in org.biojava.nbio.core.search.io.blast">BlastXMLParserTest</a></dt>
<dd>
<div class="block">Test of setFile method, of class BlastXMLParser.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserTest.html#testSetQueryReferences()" class="member-name-link">testSetQueryReferences()</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserTest.html" title="class in org.biojava.nbio.core.search.io.blast">BlastXMLParserTest</a></dt>
<dd>
<div class="block">Test of setQueryReferences method, of class BlastXMLParser.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testSimpleAlignedSequenceLocal()" class="member-name-link">testSimpleAlignedSequenceLocal()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testSimpleAlignedSequenceLong()" class="member-name-link">testSimpleAlignedSequenceLong()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testSimpleAlignedSequenceShort()" class="member-name-link">testSimpleAlignedSequenceShort()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testSimpleProfile()" class="member-name-link">testSimpleProfile()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaReaderTest.html#testSmallFasta()" class="member-name-link">testSmallFasta()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaReaderTest.html" title="class in org.biojava.nbio.core.sequence.io">FastaReaderTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaReaderTest.html#testSmallFasta2()" class="member-name-link">testSmallFasta2()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaReaderTest.html" title="class in org.biojava.nbio.core.sequence.io">FastaReaderTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserTest.html#testStoreObjects()" class="member-name-link">testStoreObjects()</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserTest.html" title="class in org.biojava.nbio.core.search.io.blast">BlastXMLParserTest</a></dt>
<dd>
<div class="block">Test of storeObjects method, of class BlastXMLParser.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/LocationTest.html#testStrandFlip()" class="member-name-link">testStrandFlip()</a> - Method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/LocationTest.html" title="class in org.biojava.nbio.core.sequence.location">LocationTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/LocationTest.html#testSubLocations()" class="member-name-link">testSubLocations()</a> - Method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/LocationTest.html" title="class in org.biojava.nbio.core.sequence.location">LocationTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html#testToString()" class="member-name-link">testToString()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testToString()" class="member-name-link">testToString()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testToStringFormatted()" class="member-name-link">testToStringFormatted()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html#testToStringInt()" class="member-name-link">testToStringInt()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/TestUncompressInputStream.html" class="type-name-link" title="class in org.biojava.nbio.core.util">TestUncompressInputStream</a> - Class in <a href="../org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/TestUncompressInputStream.html#%3Cinit%3E()" class="member-name-link">TestUncompressInputStream()</a> - Constructor for class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/TestUncompressInputStream.html" title="class in org.biojava.nbio.core.util">TestUncompressInputStream</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/TestUncompressInputStream.html#testUncompression()" class="member-name-link">testUncompression()</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/TestUncompressInputStream.html" title="class in org.biojava.nbio.core.util">TestUncompressInputStream</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/ABITracerTest.html#testURL()" class="member-name-link">testURL()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/ABITracerTest.html" title="class in org.biojava.nbio.core.sequence.io">ABITracerTest</a></dt>
<dd>
<div class="block">Test of URL method, of class ABITracer.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/LocationTest.html#testWithStrandSwitch()" class="member-name-link">testWithStrandSwitch()</a> - Method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/LocationTest.html" title="class in org.biojava.nbio.core.sequence.location">LocationTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/SimpleGenbankProxySequenceReaderTest.html#testWrongSequence()" class="member-name-link">testWrongSequence()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/SimpleGenbankProxySequenceReaderTest.html" title="class in org.biojava.nbio.core.sequence.loader">SimpleGenbankProxySequenceReaderTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/TextFeature.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.features">TextFeature</a>&lt;<a href="../org/biojava/nbio/core/sequence/features/TextFeature.html" title="type parameter in TextFeature">S</a> extends <a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="../org/biojava/nbio/core/sequence/features/TextFeature.html" title="type parameter in TextFeature">C</a>&gt;,<wbr><a href="../org/biojava/nbio/core/sequence/features/TextFeature.html" title="type parameter in TextFeature">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/features/package-summary.html">org.biojava.nbio.core.sequence.features</a></dt>
<dd>
<div class="block">A implmentation of AbstractFeature</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/TextFeature.html#%3Cinit%3E(java.lang.String,java.lang.String,java.lang.String,java.lang.String)" class="member-name-link">TextFeature(String, String, String, String)</a> - Constructor for class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/TextFeature.html" title="class in org.biojava.nbio.core.sequence.features">TextFeature</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Frame.html#THREE" class="member-name-link">THREE</a> - Enum constant in enum org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Frame.html" title="enum in org.biojava.nbio.core.sequence.transcription">Frame</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html#toList(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">toList(Sequence&lt;C&gt;)</a> - Static method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">For the given <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a> this will return a <a href="https://docs.oracle.com/javase/8/docs/api/java/util/List.html" title="class or interface in java.util" class="external-link"><code>List</code></a> filled with
 the Compounds of that <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a>.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/Location.Tools.html#%3Cinit%3E()" class="member-name-link">Tools()</a> - Constructor for class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/Location.Tools.html" title="class in org.biojava.nbio.core.sequence.location.template">Location.Tools</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html#toString()" class="member-name-link">toString()</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">ScaledSubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns in a format similar to the standard NCBI files.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html#toString()" class="member-name-link">toString()</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">SimpleSubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns in a format similar to the standard NCBI files.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#toString()" class="member-name-link">toString()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>
<div class="block">Provides standard Java language access to results of <a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#getSequenceAsString()"><code>SimpleAlignedSequence.getSequenceAsString()</code></a>.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html#toString()" class="member-name-link">toString()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.html#toString()" class="member-name-link">toString()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns a simple view of the alignment profile.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/AccessionID.html#toString()" class="member-name-link">toString()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/AccessionID.html" title="class in org.biojava.nbio.core.sequence">AccessionID</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#toString()" class="member-name-link">toString()</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">DBReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/Qualifier.html#toString()" class="member-name-link">toString()</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/Qualifier.html" title="class in org.biojava.nbio.core.sequence.features">Qualifier</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#toString()" class="member-name-link">toString()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#toString()" class="member-name-link">toString()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#toString()" class="member-name-link">toString()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SimplePoint.html#toString()" class="member-name-link">toString()</a> - Method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#toString()" class="member-name-link">toString()</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#toString()" class="member-name-link">toString()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava.nbio.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">String representation of the MSA</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html#toString()" class="member-name-link">toString()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#toString()" class="member-name-link">toString()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/LightweightProfile.html#toString()" class="member-name-link">toString()</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/LightweightProfile.html" title="interface in org.biojava.nbio.core.sequence.template">LightweightProfile</a></dt>
<dd>
<div class="block">Returns a simple view of the alignment profile.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#toString()" class="member-name-link">toString()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html#toString()" class="member-name-link">toString()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.CaseInsensitiveTriplet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html#toString()" class="member-name-link">toString()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/CRC64Checksum.html#toString()" class="member-name-link">toString()</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/CRC64Checksum.html" title="class in org.biojava.nbio.core.util">CRC64Checksum</a></dt>
<dd>
<div class="block">Returns a zero-padded 16 character wide string containing the current
 value of this checksum in uppercase hexadecimal format.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/SingleLinkageClusterer.LinkedPair.html#toString()" class="member-name-link">toString()</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SingleLinkageClusterer.LinkedPair.html" title="class in org.biojava.nbio.core.util">SingleLinkageClusterer.LinkedPair</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html#toString(int)" class="member-name-link">toString(int)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.html#toString(int)" class="member-name-link">toString(int)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns a formatted view of the alignment profile.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#toString(int)" class="member-name-link">toString(int)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava.nbio.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">Get a string representation of the MSA with a fixed width</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/LightweightProfile.html#toString(int)" class="member-name-link">toString(int)</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/LightweightProfile.html" title="interface in org.biojava.nbio.core.sequence.template">LightweightProfile</a></dt>
<dd>
<div class="block">Returns a formatted view of the alignment profile.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html#toString(org.biojava.nbio.core.alignment.template.Profile.StringFormat)" class="member-name-link">toString(Profile.StringFormat)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.html#toString(org.biojava.nbio.core.alignment.template.Profile.StringFormat)" class="member-name-link">toString(Profile.StringFormat)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns a formatted view of the alignment profile.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#toString(org.biojava.nbio.core.sequence.template.LightweightProfile.StringFormat)" class="member-name-link">toString(LightweightProfile.StringFormat)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava.nbio.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">Support for different MSA formats</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/LightweightProfile.html#toString(org.biojava.nbio.core.sequence.template.LightweightProfile.StringFormat)" class="member-name-link">toString(LightweightProfile.StringFormat)</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/LightweightProfile.html" title="interface in org.biojava.nbio.core.sequence.template">LightweightProfile</a></dt>
<dd>
<div class="block">Returns a formatted view of the alignment profile.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html#toString(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">toString(Sequence&lt;C&gt;)</a> - Static method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Shortcut to <a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html#toStringBuilder(org.biojava.nbio.core.sequence.template.Sequence)"><code>SequenceMixin.toStringBuilder(org.biojava.nbio.core.sequence.template.Sequence)</code></a>
 which calls toString() on the resulting object.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html#toStringBuilder(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">toStringBuilder(Sequence&lt;C&gt;)</a> - Static method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">For the given Sequence this will return a <a href="https://docs.oracle.com/javase/8/docs/api/java/lang/StringBuilder.html" title="class or interface in java.lang" class="external-link"><code>StringBuilder</code></a> object
 filled with the results of <code>Compound#toString()</code>.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/FileDownloadUtils.html#toUnixPath(java.lang.String)" class="member-name-link">toUnixPath(String)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/FileDownloadUtils.html" title="class in org.biojava.nbio.core.util">FileDownloadUtils</a></dt>
<dd>
<div class="block">Converts path to Unix convention and adds a terminating slash if it was
 omitted.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine</a> - Class in <a href="../org/biojava/nbio/core/sequence/transcription/package-summary.html">org.biojava.nbio.core.sequence.transcription</a></dt>
<dd>
<div class="block">Used as a way of encapsulating the data structures required to parse DNA to a
 Protein sequence.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a> - Class in <a href="../org/biojava/nbio/core/sequence/transcription/package-summary.html">org.biojava.nbio.core.sequence.transcription</a></dt>
<dd>
<div class="block">This class is the way to create a <code>TranslationEngine</code>.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranscriptSequence.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a> - Class in <a href="../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">This is the sequence if you want to go from a gene sequence to a protein sequence.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranscriptSequence.html#%3Cinit%3E(org.biojava.nbio.core.sequence.GeneSequence,int,int)" class="member-name-link">TranscriptSequence(GeneSequence, int, int)</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a></dt>
<dd>
<div class="deprecation-block"><span class="deprecated-label">Deprecated.</span>&nbsp;</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranscriptSequence.html#%3Cinit%3E(org.biojava.nbio.core.sequence.GeneSequence,org.biojava.nbio.core.sequence.AccessionID,int,int)" class="member-name-link">TranscriptSequence(GeneSequence, AccessionID, int, int)</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranscriptSequenceProduct.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence">TranscriptSequenceProduct</a> - Class in <a href="../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranscriptSequenceProduct.html#%3Cinit%3E()" class="member-name-link">TranscriptSequenceProduct()</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranscriptSequenceProduct.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequenceProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#translate(F)" class="member-name-link">translate(F)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/CompoundTranslator.html#translate(F)" class="member-name-link">translate(F)</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/CompoundTranslator.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#translate(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">translate(Sequence&lt;NucleotideCompound&gt;)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>
<div class="block">Quick method to let you go from a CDS to a Peptide quickly.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranslationTest.html#translateBrca2()" class="member-name-link">translateBrca2()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranslationTest.html" title="class in org.biojava.nbio.core.sequence">TranslationTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranslationTest.html#translateBrca2ExonOne()" class="member-name-link">translateBrca2ExonOne()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranslationTest.html" title="class in org.biojava.nbio.core.sequence">TranslationTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranslationTest.html#translateInitMet()" class="member-name-link">translateInitMet()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranslationTest.html" title="class in org.biojava.nbio.core.sequence">TranslationTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranslationTest.html#translateInternalStops()" class="member-name-link">translateInternalStops()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranslationTest.html" title="class in org.biojava.nbio.core.sequence">TranslationTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#translateMany(F)" class="member-name-link">translateMany(F)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/CompoundTranslator.html#translateMany(F)" class="member-name-link">translateMany(F)</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/CompoundTranslator.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranslationTest.html#translateN()" class="member-name-link">translateN()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranslationTest.html" title="class in org.biojava.nbio.core.sequence">TranslationTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html#translateNCodons()" class="member-name-link">translateNCodons()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html" title="class in org.biojava.nbio.core.sequence.transcription">RNAToAminoAcidTranslator</a></dt>
<dd>
<div class="block">Indicates if we want to force exact translation of compounds or not i.e.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#translateNCodons(boolean)" class="member-name-link">translateNCodons(boolean)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranslationTest.html#translateStopAtInternalStops()" class="member-name-link">translateStopAtInternalStops()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranslationTest.html" title="class in org.biojava.nbio.core.sequence">TranslationTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNATest.html#translateToRna()" class="member-name-link">translateToRna()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DNATest.html" title="class in org.biojava.nbio.core.sequence">DNATest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/exceptions/TranslationException.html" class="type-name-link" title="class in org.biojava.nbio.core.exceptions">TranslationException</a> - Exception in <a href="../org/biojava/nbio/core/exceptions/package-summary.html">org.biojava.nbio.core.exceptions</a></dt>
<dd>
<div class="block">Thrown from AbstractCompundTranslator</div>
</dd>
<dt><a href="../org/biojava/nbio/core/exceptions/TranslationException.html#%3Cinit%3E(java.lang.Exception)" class="member-name-link">TranslationException(Exception)</a> - Constructor for exception org.biojava.nbio.core.exceptions.<a href="../org/biojava/nbio/core/exceptions/TranslationException.html" title="class in org.biojava.nbio.core.exceptions">TranslationException</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/exceptions/TranslationException.html#%3Cinit%3E(java.lang.String)" class="member-name-link">TranslationException(String)</a> - Constructor for exception org.biojava.nbio.core.exceptions.<a href="../org/biojava/nbio/core/exceptions/TranslationException.html" title="class in org.biojava.nbio.core.exceptions">TranslationException</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/exceptions/TranslationException.html#%3Cinit%3E(java.lang.String,java.lang.Exception)" class="member-name-link">TranslationException(String, Exception)</a> - Constructor for exception org.biojava.nbio.core.exceptions.<a href="../org/biojava/nbio/core/exceptions/TranslationException.html" title="class in org.biojava.nbio.core.exceptions">TranslationException</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranslationTest.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence">TranslationTest</a> - Class in <a href="../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranslationTest.html#%3Cinit%3E()" class="member-name-link">TranslationTest()</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranslationTest.html" title="class in org.biojava.nbio.core.sequence">TranslationTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#trimStop(boolean)" class="member-name-link">trimStop(boolean)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Frame.html#TWO" class="member-name-link">TWO</a> - Enum constant in enum org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Frame.html" title="enum in org.biojava.nbio.core.sequence.transcription">Frame</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNATest.html#twoBit()" class="member-name-link">twoBit()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DNATest.html" title="class in org.biojava.nbio.core.sequence">DNATest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html#%3Cinit%3E(java.lang.String,org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">TwoBitArrayWorker(String, CompoundSet&lt;C&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">TwoBitSequenceReader.TwoBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html#%3Cinit%3E(org.biojava.nbio.core.sequence.template.CompoundSet,int)" class="member-name-link">TwoBitArrayWorker(CompoundSet&lt;C&gt;, int)</a> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">TwoBitSequenceReader.TwoBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html#%3Cinit%3E(org.biojava.nbio.core.sequence.template.CompoundSet,int%5B%5D)" class="member-name-link">TwoBitArrayWorker(CompoundSet&lt;C&gt;, int[])</a> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">TwoBitSequenceReader.TwoBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html#%3Cinit%3E(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">TwoBitArrayWorker(Sequence&lt;C&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">TwoBitSequenceReader.TwoBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.storage">TwoBitSequenceReader</a>&lt;<a href="../org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html" title="type parameter in TwoBitSequenceReader">C</a> extends <a href="../org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound">NucleotideCompound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/storage/package-summary.html">org.biojava.nbio.core.sequence.storage</a></dt>
<dd>
<div class="block">Implementation of the 2bit encoding.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#%3Cinit%3E(java.lang.String,org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">TwoBitSequenceReader(String, CompoundSet&lt;C&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">TwoBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#%3Cinit%3E(java.lang.String,org.biojava.nbio.core.sequence.template.CompoundSet,org.biojava.nbio.core.sequence.AccessionID)" class="member-name-link">TwoBitSequenceReader(String, CompoundSet&lt;C&gt;, AccessionID)</a> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">TwoBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#%3Cinit%3E(org.biojava.nbio.core.sequence.storage.TwoBitSequenceReader.TwoBitArrayWorker)" class="member-name-link">TwoBitSequenceReader(TwoBitSequenceReader.TwoBitArrayWorker&lt;C&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">TwoBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#%3Cinit%3E(org.biojava.nbio.core.sequence.storage.TwoBitSequenceReader.TwoBitArrayWorker,org.biojava.nbio.core.sequence.AccessionID)" class="member-name-link">TwoBitSequenceReader(TwoBitSequenceReader.TwoBitArrayWorker&lt;C&gt;, AccessionID)</a> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">TwoBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html#%3Cinit%3E(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">TwoBitSequenceReader(Sequence&lt;C&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">TwoBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.storage">TwoBitSequenceReader.TwoBitArrayWorker</a>&lt;<a href="../org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="type parameter in TwoBitSequenceReader.TwoBitArrayWorker">C</a> extends <a href="../org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound">NucleotideCompound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/storage/package-summary.html">org.biojava.nbio.core.sequence.storage</a></dt>
<dd>
<div class="block">Extension of the BitArrayWorker which provides the 2bit implementation
 code.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html#TYPE" class="member-name-link">TYPE</a> - Static variable in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">Sort features by type</div>
</dd>
</dl>
<a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Z</a>&nbsp;<br><a href="../allclasses-index.html">All&nbsp;Classes&nbsp;and&nbsp;Interfaces</a><span class="vertical-separator">|</span><a href="../allpackages-index.html">All&nbsp;Packages</a><span class="vertical-separator">|</span><a href="../constant-values.html">Constant&nbsp;Field&nbsp;Values</a><span class="vertical-separator">|</span><a href="../serialized-form.html">Serialized&nbsp;Form</a></main>
</div>
</div>
</body>
</html>
