<!DOCTYPE HTML>
<html lang="pt">
<head>
<!-- Generated by javadoc (17) on Sun May 15 18:36:12 WEST 2022 -->
<title>I-Index</title>
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta name="dc.created" content="2022-05-15">
<meta name="description" content="index: I">
<meta name="generator" content="javadoc/IndexWriter">
<link rel="stylesheet" type="text/css" href="../stylesheet.css" title="Style">
<link rel="stylesheet" type="text/css" href="../script-dir/jquery-ui.min.css" title="Style">
<link rel="stylesheet" type="text/css" href="../jquery-ui.overrides.css" title="Style">
<script type="text/javascript" src="../script.js"></script>
<script type="text/javascript" src="../script-dir/jquery-3.5.1.min.js"></script>
<script type="text/javascript" src="../script-dir/jquery-ui.min.js"></script>
</head>
<body class="index-page">
<script type="text/javascript">var pathtoroot = "../";
loadScripts(document, 'script');</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<div class="flex-box">
<header role="banner" class="flex-header">
<nav role="navigation">
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="top-nav" id="navbar-top">
<div class="skip-nav"><a href="#skip-navbar-top" title="Skip navigation links">Skip navigation links</a></div>
<ul id="navbar-top-firstrow" class="nav-list" title="Navigation">
<li><a href="../index.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li>Use</li>
<li><a href="../overview-tree.html">Tree</a></li>
<li><a href="../deprecated-list.html">Deprecated</a></li>
<li class="nav-bar-cell1-rev">Index</li>
<li><a href="../help-doc.html#index">Help</a></li>
</ul>
</div>
<div class="sub-nav">
<div class="nav-list-search"><label for="search-input">SEARCH:</label>
<input type="text" id="search-input" value="search" disabled="disabled">
<input type="reset" id="reset-button" value="reset" disabled="disabled">
</div>
</div>
<!-- ========= END OF TOP NAVBAR ========= -->
<span class="skip-nav" id="skip-navbar-top"></span></nav>
</header>
<div class="flex-content">
<main role="main">
<div class="header">
<h1>Index</h1>
</div>
<a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Z</a>&nbsp;<br><a href="../allclasses-index.html">All&nbsp;Classes&nbsp;and&nbsp;Interfaces</a><span class="vertical-separator">|</span><a href="../allpackages-index.html">All&nbsp;Packages</a><span class="vertical-separator">|</span><a href="../constant-values.html">Constant&nbsp;Field&nbsp;Values</a><span class="vertical-separator">|</span><a href="../serialized-form.html">Serialized&nbsp;Form</a>
<h2 class="title" id="I:I">I</h2>
<dl class="index">
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html#indexOf(org.biojava.nbio.core.sequence.template.Sequence,C)" class="member-name-link">indexOf(Sequence&lt;C&gt;, C)</a> - Static method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Performs a linear search of the given Sequence for the given compound.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#initMet(boolean)" class="member-name-link">initMet(boolean)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblReaderTest.html#initObjects()" class="member-name-link">initObjects()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblReaderTest.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReaderTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/InputStreamProvider.html" class="type-name-link" title="class in org.biojava.nbio.core.util">InputStreamProvider</a> - Class in <a href="../org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>
<div class="block">A class that provides an InputStream from a File.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/InputStreamProvider.html#%3Cinit%3E()" class="member-name-link">InputStreamProvider()</a> - Constructor for class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/InputStreamProvider.html" title="class in org.biojava.nbio.core.util">InputStreamProvider</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/XMLHelper.html#inputStreamToDocument(java.io.InputStream)" class="member-name-link">inputStreamToDocument(InputStream)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/XMLHelper.html" title="class in org.biojava.nbio.core.util">XMLHelper</a></dt>
<dd>
<div class="block">Creates an org.w3c.dom.Document from the content of the inputStream</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/InsdcLocations.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations</a> - Class in <a href="../org/biojava/nbio/core/sequence/location/package-summary.html">org.biojava.nbio.core.sequence.location</a></dt>
<dd>
<div class="block">A collection of locations which are used whenever we work with INSDC; some
 of which could be deprecated (from INSDC's point of view) yet appear
 in records.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/InsdcLocations.html#%3Cinit%3E()" class="member-name-link">InsdcLocations()</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/InsdcLocations.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/InsdcLocations.BondLocation.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.BondLocation</a> - Class in <a href="../org/biojava/nbio/core/sequence/location/package-summary.html">org.biojava.nbio.core.sequence.location</a></dt>
<dd>
<div class="block">Used to represent bond locations equivalent to bond(7,8) or bond(7).</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.GroupLocation</a> - Class in <a href="../org/biojava/nbio/core/sequence/location/package-summary.html">org.biojava.nbio.core.sequence.location</a></dt>
<dd>
<div class="block">Deprecated in INSDC yet still appears; equivalent to the order()
 directive except no 5' to 3' ordering is defined.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/InsdcLocations.OneOfLocation.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.OneOfLocation</a> - Class in <a href="../org/biojava/nbio/core/sequence/location/package-summary.html">org.biojava.nbio.core.sequence.location</a></dt>
<dd>
<div class="block">Deprecated in INSDC; refers to a set of locations of which one
 location could be valid e.g. one-of(location, location, location).</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/InsdcLocations.OrderLocation.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.OrderLocation</a> - Class in <a href="../org/biojava/nbio/core/sequence/location/package-summary.html">org.biojava.nbio.core.sequence.location</a></dt>
<dd>
<div class="block">Used to describe a 5' to 3' ordering but no firm assurance it is correct</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/InsdcParser.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.location">InsdcParser</a> - Class in <a href="../org/biojava/nbio/core/sequence/location/package-summary.html">org.biojava.nbio.core.sequence.location</a></dt>
<dd>
<div class="block">Parser for working with INSDC style locations.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/InsdcParser.html#%3Cinit%3E()" class="member-name-link">InsdcParser()</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/InsdcParser.html" title="class in org.biojava.nbio.core.sequence.location">InsdcParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/InsdcParser.html#%3Cinit%3E(org.biojava.nbio.core.sequence.DataSource)" class="member-name-link">InsdcParser(DataSource)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/InsdcParser.html" title="class in org.biojava.nbio.core.sequence.location">InsdcParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/InsdcParserTest.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.location">InsdcParserTest</a> - Class in <a href="../org/biojava/nbio/core/sequence/location/package-summary.html">org.biojava.nbio.core.sequence.location</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/InsdcParserTest.html#%3Cinit%3E()" class="member-name-link">InsdcParserTest()</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/InsdcParserTest.html" title="class in org.biojava.nbio.core.sequence.location">InsdcParserTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/EditSequenceTest.html#insert()" class="member-name-link">insert()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/EditSequenceTest.html" title="class in org.biojava.nbio.core.sequence">EditSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/edits/Edit.Insert.html#%3Cinit%3E(java.lang.String,int)" class="member-name-link">Insert(String, int)</a> - Constructor for class org.biojava.nbio.core.sequence.edits.<a href="../org/biojava/nbio/core/sequence/edits/Edit.Insert.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Insert</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/edits/Edit.Insert.html#%3Cinit%3E(java.lang.String,int,int)" class="member-name-link">Insert(String, int, int)</a> - Constructor for class org.biojava.nbio.core.sequence.edits.<a href="../org/biojava/nbio/core/sequence/edits/Edit.Insert.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Insert</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/edits/Edit.Insert.html#%3Cinit%3E(org.biojava.nbio.core.sequence.template.Sequence,int)" class="member-name-link">Insert(Sequence&lt;C&gt;, int)</a> - Constructor for class org.biojava.nbio.core.sequence.edits.<a href="../org/biojava/nbio/core/sequence/edits/Edit.Insert.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Insert</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/edits/Edit.Insert.html#%3Cinit%3E(org.biojava.nbio.core.sequence.template.Sequence,int,int)" class="member-name-link">Insert(Sequence&lt;C&gt;, int, int)</a> - Constructor for class org.biojava.nbio.core.sequence.edits.<a href="../org/biojava/nbio/core/sequence/edits/Edit.Insert.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Insert</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/IntronSequence.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence">IntronSequence</a> - Class in <a href="../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/IntronSequence.html#%3Cinit%3E(org.biojava.nbio.core.sequence.GeneSequence,int,int)" class="member-name-link">IntronSequence(GeneSequence, int, int)</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/IntronSequence.html" title="class in org.biojava.nbio.core.sequence">IntronSequence</a></dt>
<dd>
<div class="block">Place holder for Intron sequence features</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html#intValue()" class="member-name-link">intValue()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.CaseInsensitiveTriplet</a></dt>
<dd>
<div class="block">Attempts to provide an int version of this codon which multiplies
 each position by</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html#inverse(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">inverse(Sequence&lt;C&gt;)</a> - Static method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">A method which attempts to do the right thing when is comes to a
 reverse/reverse complement</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/util/package-summary.html">org.biojava.nbio.core.sequence.io.util</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html#%3Cinit%3E()" class="member-name-link">IOUtils()</a> - Constructor for class org.biojava.nbio.core.sequence.io.util.<a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.ReaderProcessor.html" class="type-name-link" title="interface in org.biojava.nbio.core.sequence.io.util">IOUtils.ReaderProcessor</a> - Interface in <a href="../org/biojava/nbio/core/sequence/io/util/package-summary.html">org.biojava.nbio.core.sequence.io.util</a></dt>
<dd>
<div class="block">Closure interface used when working with
 <code>IOUtils#processReader(String)</code>.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#isAmbiguous()" class="member-name-link">isAmbiguous()</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound">NucleotideCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#isBetweenCompounds()" class="member-name-link">isBetweenCompounds()</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/Location.html#isBetweenCompounds()" class="member-name-link">isBetweenCompounds()</a> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/Location.html" title="interface in org.biojava.nbio.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Returns true if the position is meant to represent a point between
 two points such as 78^79.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#isCircular()" class="member-name-link">isCircular()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html#isCircular()" class="member-name-link">isCircular()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html#isCircular()" class="member-name-link">isCircular()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns true if this <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a> wraps around from the last alignment column back to the first.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.html#isCircular()" class="member-name-link">isCircular()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns true if any <a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template"><code>AlignedSequence</code></a> is circular.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#isCircular()" class="member-name-link">isCircular()</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/Location.html#isCircular()" class="member-name-link">isCircular()</a> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/Location.html" title="interface in org.biojava.nbio.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Indicates if this location is circular.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankReader.html#isClosed()" class="member-name-link">isClosed()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankReader.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#isComplementable()" class="member-name-link">isComplementable()</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#isComplementable()" class="member-name-link">isComplementable()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#isComplementable()" class="member-name-link">isComplementable()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractNucleotideCompoundSet</a></dt>
<dd>
<div class="block">NucleotideCompounds can always complement</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/CompoundSet.html#isComplementable()" class="member-name-link">isComplementable()</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#isComplex()" class="member-name-link">isComplex()</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/Location.html#isComplex()" class="member-name-link">isComplex()</a> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/Location.html" title="interface in org.biojava.nbio.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Returns true if the location is considered to be complex meaning
 the location is actually composed of sub-locations.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#isCompoundStringLengthEqual()" class="member-name-link">isCompoundStringLengthEqual()</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#isCompoundStringLengthEqual()" class="member-name-link">isCompoundStringLengthEqual()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/CompoundSet.html#isCompoundStringLengthEqual()" class="member-name-link">isCompoundStringLengthEqual()</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundSet</a></dt>
<dd>
<div class="block">Returns true if all String representations of Compounds are of the
 same length.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#isGap(int)" class="member-name-link">isGap(int)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html#isGap(int)" class="member-name-link">isGap(int)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns true if this <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a> has a gap at a particular alignment column.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SimplePoint.html#isHigher(org.biojava.nbio.core.sequence.location.template.Point)" class="member-name-link">isHigher(Point)</a> - Method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/Point.html#isHigher(org.biojava.nbio.core.sequence.location.template.Point)" class="member-name-link">isHigher(Point)</a> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/Point.html" title="interface in org.biojava.nbio.core.sequence.location.template">Point</a></dt>
<dd>
<div class="block">Returns true if the point is higher in value to the current point</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SimplePoint.html#isLower(org.biojava.nbio.core.sequence.location.template.Point)" class="member-name-link">isLower(Point)</a> - Method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/Point.html#isLower(org.biojava.nbio.core.sequence.location.template.Point)" class="member-name-link">isLower(Point)</a> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/Point.html" title="interface in org.biojava.nbio.core.sequence.location.template">Point</a></dt>
<dd>
<div class="block">Returns true if the current point is at a lower position than the
 point given.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/SequenceTools.html#isNucleotideSequence(java.lang.String)" class="member-name-link">isNucleotideSequence(String)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SequenceTools.html" title="class in org.biojava.nbio.core.util">SequenceTools</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#isPartial()" class="member-name-link">isPartial()</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocationProduct.html#isPartial()" class="member-name-link">isPartial()</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocationProduct.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocationProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#isPartialOn3prime()" class="member-name-link">isPartialOn3prime()</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#isPartialOn5prime()" class="member-name-link">isPartialOn5prime()</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html#isStart()" class="member-name-link">isStart()</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html#isStart()" class="member-name-link">isStart()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html#isStart(org.biojava.nbio.core.sequence.compound.AminoAcidCompound)" class="member-name-link">isStart(AminoAcidCompound)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser.IUPACTable</a></dt>
<dd>
<div class="block">Returns true if the given compound was a start codon in this
 codon table.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.html#isStart(org.biojava.nbio.core.sequence.compound.AminoAcidCompound)" class="member-name-link">isStart(AminoAcidCompound)</a> - Method in interface org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.html" title="interface in org.biojava.nbio.core.sequence.transcription">Table</a></dt>
<dd>
<div class="block">Returns true if the given compound could have been a start amino acid;
 this does not assert if the codon that actually coded for the amino
 acid was a start codon.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html#isStop()" class="member-name-link">isStop()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SimplePoint.html#isUncertain()" class="member-name-link">isUncertain()</a> - Method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/Point.html#isUncertain()" class="member-name-link">isUncertain()</a> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/Point.html" title="interface in org.biojava.nbio.core.sequence.location.template">Point</a></dt>
<dd>
<div class="block">Returns a true if the exact point is unknown.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SimplePoint.html#isUnknown()" class="member-name-link">isUnknown()</a> - Method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/Point.html#isUnknown()" class="member-name-link">isUnknown()</a> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/Point.html" title="interface in org.biojava.nbio.core.sequence.location.template">Point</a></dt>
<dd>
<div class="block">Returns true if the current position is unknown but is
 beyond the position encoded for.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#isValidSequence(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">isValidSequence(Sequence&lt;C&gt;)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/CompoundSet.html#isValidSequence(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">isValidSequence(Sequence&lt;C&gt;)</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#isValidSequence(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">isValidSequence(Sequence&lt;AminoAcidCompound&gt;)</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/SingleLinkageClustererProduct.html#isWithinCutoff(int,double,org.biojava.nbio.core.util.SingleLinkageClusterer)" class="member-name-link">isWithinCutoff(int, double, SingleLinkageClusterer)</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SingleLinkageClustererProduct.html" title="class in org.biojava.nbio.core.util">SingleLinkageClustererProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#iterator()" class="member-name-link">iterator()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html#iterator()" class="member-name-link">iterator()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hit.html#iterator()" class="member-name-link">iterator()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hit.html" title="class in org.biojava.nbio.core.search.io">Hit</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Result.html#iterator()" class="member-name-link">iterator()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Result.html" title="class in org.biojava.nbio.core.search.io">Result</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/SearchIO.html#iterator()" class="member-name-link">iterator()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/SearchIO.html" title="class in org.biojava.nbio.core.search.io">SearchIO</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#iterator()" class="member-name-link">iterator()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#iterator()" class="member-name-link">iterator()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#iterator()" class="member-name-link">iterator()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#iterator()" class="member-name-link">iterator()</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>
<div class="block">Iterates through all known sub-locations for this location but does
 not descend</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#iterator()" class="member-name-link">iterator()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#iterator()" class="member-name-link">iterator()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Provides basic iterable access to this class</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#iterator()" class="member-name-link">iterator()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>
<div class="block">Iterator implementation which attempts to move through the 2D structure
 attempting to skip onto the next sequence as &amp; when it is asked to</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#iterator()" class="member-name-link">iterator()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Returns an instance of <a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.SequenceIterator.html" title="class in org.biojava.nbio.core.sequence.template"><code>SequenceMixin.SequenceIterator</code></a></div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#iterator()" class="member-name-link">iterator()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html#iterator()" class="member-name-link">iterator()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/views/WindowedSequence.html#iterator()" class="member-name-link">iterator()</a> - Method in class org.biojava.nbio.core.sequence.views.<a href="../org/biojava/nbio/core/sequence/views/WindowedSequence.html" title="class in org.biojava.nbio.core.sequence.views">WindowedSequence</a></dt>
<dd>
<div class="block">Returns an iterator which will return the windows in a sequence in
 sequential order.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACParser.html#IUPAC_LOCATION" class="member-name-link">IUPAC_LOCATION</a> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/IUPACParser.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACParser.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">IUPACParser</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">Available translations

 
 1 - UNIVERSAL
 2 - VERTEBRATE_MITOCHONDRIAL
 3 - YEAST_MITOCHONDRIAL
 4 - MOLD_MITOCHONDRIAL
 5 - INVERTEBRATE_MITOCHONDRIAL
 6 - CILIATE_NUCLEAR
 9 - ECHINODERM_MITOCHONDRIAL
 10 - EUPLOTID_NUCLEAR
 11 - BACTERIAL
 12 - ALTERNATIVE_YEAST_NUCLEAR
 13 - ASCIDIAN_MITOCHONDRIAL
 14 - FLATWORM_MITOCHONDRIAL
 15 - BLEPHARISMA_MACRONUCLEAR
 16 - 2CHLOROPHYCEAN_MITOCHONDRIAL
 21 - TREMATODE_MITOCHONDRIAL
 23 - SCENEDESMUS_MITOCHONDRIAL
 

 Taken from <a href="https://www.ncbi.nlm.nih.gov/Taxonomy/Utils/wprintgc.cgi?mode=c">NCBI</a> with slight modification and put into the classpath resource.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACParser.html#%3Cinit%3E()" class="member-name-link">IUPACParser()</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/IUPACParser.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser</a></dt>
<dd>
<div class="block">Default version and uses the classpath based IUPAC table</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACParser.html#%3Cinit%3E(java.io.InputStream)" class="member-name-link">IUPACParser(InputStream)</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/IUPACParser.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser</a></dt>
<dd>
<div class="block">Allows you to specify a different IUPAC table.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">IUPACParser.IUPACTable</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">Holds the concept of a codon table from the IUPAC format</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACParserProduct.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">IUPACParserProduct</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACParserProduct.html#%3Cinit%3E()" class="member-name-link">IUPACParserProduct()</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/IUPACParserProduct.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParserProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACParserProduct2.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">IUPACParserProduct2</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACParserProduct2.html#%3Cinit%3E()" class="member-name-link">IUPACParserProduct2()</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/IUPACParserProduct2.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParserProduct2</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html#%3Cinit%3E(java.lang.String,int,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)" class="member-name-link">IUPACTable(String, int, String, String, String, String, String)</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser.IUPACTable</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html#%3Cinit%3E(java.lang.String,java.lang.Integer,java.lang.String,java.lang.String)" class="member-name-link">IUPACTable(String, Integer, String, String)</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser.IUPACTable</a></dt>
<dd>
<div class="block">Constructor which uses the basic IUPAC codon table format.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACTableProduct.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">IUPACTableProduct</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACTableProduct.html#%3Cinit%3E(java.lang.String,java.lang.String,java.lang.String)" class="member-name-link">IUPACTableProduct(String, String, String)</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/IUPACTableProduct.html" title="class in org.biojava.nbio.core.sequence.io">IUPACTableProduct</a></dt>
<dd>&nbsp;</dd>
</dl>
<a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Z</a>&nbsp;<br><a href="../allclasses-index.html">All&nbsp;Classes&nbsp;and&nbsp;Interfaces</a><span class="vertical-separator">|</span><a href="../allpackages-index.html">All&nbsp;Packages</a><span class="vertical-separator">|</span><a href="../constant-values.html">Constant&nbsp;Field&nbsp;Values</a><span class="vertical-separator">|</span><a href="../serialized-form.html">Serialized&nbsp;Form</a></main>
</div>
</div>
</body>
</html>
