<!DOCTYPE HTML>
<html lang="pt">
<head>
<!-- Generated by javadoc (17) on Sun May 15 18:36:12 WEST 2022 -->
<title>D-Index</title>
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta name="dc.created" content="2022-05-15">
<meta name="description" content="index: D">
<meta name="generator" content="javadoc/IndexWriter">
<link rel="stylesheet" type="text/css" href="../stylesheet.css" title="Style">
<link rel="stylesheet" type="text/css" href="../script-dir/jquery-ui.min.css" title="Style">
<link rel="stylesheet" type="text/css" href="../jquery-ui.overrides.css" title="Style">
<script type="text/javascript" src="../script.js"></script>
<script type="text/javascript" src="../script-dir/jquery-3.5.1.min.js"></script>
<script type="text/javascript" src="../script-dir/jquery-ui.min.js"></script>
</head>
<body class="index-page">
<script type="text/javascript">var pathtoroot = "../";
loadScripts(document, 'script');</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<div class="flex-box">
<header role="banner" class="flex-header">
<nav role="navigation">
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="top-nav" id="navbar-top">
<div class="skip-nav"><a href="#skip-navbar-top" title="Skip navigation links">Skip navigation links</a></div>
<ul id="navbar-top-firstrow" class="nav-list" title="Navigation">
<li><a href="../index.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li>Use</li>
<li><a href="../overview-tree.html">Tree</a></li>
<li><a href="../deprecated-list.html">Deprecated</a></li>
<li class="nav-bar-cell1-rev">Index</li>
<li><a href="../help-doc.html#index">Help</a></li>
</ul>
</div>
<div class="sub-nav">
<div class="nav-list-search"><label for="search-input">SEARCH:</label>
<input type="text" id="search-input" value="search" disabled="disabled">
<input type="reset" id="reset-button" value="reset" disabled="disabled">
</div>
</div>
<!-- ========= END OF TOP NAVBAR ========= -->
<span class="skip-nav" id="skip-navbar-top"></span></nav>
</header>
<div class="flex-content">
<main role="main">
<div class="header">
<h1>Index</h1>
</div>
<a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Z</a>&nbsp;<br><a href="../allclasses-index.html">All&nbsp;Classes&nbsp;and&nbsp;Interfaces</a><span class="vertical-separator">|</span><a href="../allpackages-index.html">All&nbsp;Packages</a><span class="vertical-separator">|</span><a href="../constant-values.html">Constant&nbsp;Field&nbsp;Values</a><span class="vertical-separator">|</span><a href="../serialized-form.html">Serialized&nbsp;Form</a>
<h2 class="title" id="I:D">D</h2>
<dl class="index">
<dt><a href="../org/biojava/nbio/core/util/UncompressInputStreamData.html#data" class="member-name-link">data</a> - Variable in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/UncompressInputStreamData.html" title="class in org.biojava.nbio.core.util">UncompressInputStreamData</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/TargetedLocationParserTest.Data.html#%3Cinit%3E(java.lang.String,org.biojava.nbio.core.sequence.template.CompoundSet,java.lang.String,org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">Data(String, CompoundSet&lt;?&gt;, String, CompoundSet&lt;?&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/TargetedLocationParserTest.Data.html" title="class in org.biojava.nbio.core.sequence.location">TargetedLocationParserTest.Data</a></dt>
<dd>
<div class="block">Parser data input.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/DatabaseReferenceInterface.html" class="type-name-link" title="interface in org.biojava.nbio.core.sequence.features">DatabaseReferenceInterface</a> - Interface in <a href="../org/biojava/nbio/core/sequence/features/package-summary.html">org.biojava.nbio.core.sequence.features</a></dt>
<dd>
<div class="block">If a SequenceProxyReader implements this interface then that external source
 has a list of cross reference id(s)</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DataSource.html" class="type-name-link" title="enum in org.biojava.nbio.core.sequence">DataSource</a> - Enum in <a href="../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">
 GenBank                           gi|gi-number|gb|accession|locus
 ENA Data Library                  gi|gi-number|emb|accession|locus
 DDBJ, DNA Database of Japan       gi|gi-number|dbj|accession|locus
 NBRF PIR                          pir||entry
 Protein Research Foundation       prf||name
 SWISS-PROT UNIPROT                sp|accession|name
 Brookhaven Protein Data Bank (1)  pdb|entry|chain
 Brookhaven Protein Data Bank (2)  entry:chain|PDBID|CHAIN|SEQUENCE
 Patents                           pat|country|number
 GenInfo Backbone Id               bbs|number
 General database identifier       gnl|database|identifier
 NCBI Reference Sequence           ref|accession|locus
 Local Sequence identifier         lcl|identifier
 </pr></div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/DBReferenceInfo.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.features">DBReferenceInfo</a> - Class in <a href="../org/biojava/nbio/core/sequence/features/package-summary.html">org.biojava.nbio.core.sequence.features</a></dt>
<dd>
<div class="block">If you have a uniprot ID then it is possible to get a collection
 of other id(s) that the protein is known by.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#%3Cinit%3E(java.lang.String,java.lang.String)" class="member-name-link">DBReferenceInfo(String, String)</a> - Constructor for class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">DBReferenceInfo</a></dt>
<dd>
<div class="block">The source database and id</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DataSource.html#DDBJ" class="member-name-link">DDBJ</a> - Enum constant in enum org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DataSource.html" title="enum in org.biojava.nbio.core.sequence">DataSource</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/PrettyXMLWriter.html#declareNamespace(java.lang.String,java.lang.String)" class="member-name-link">declareNamespace(String, String)</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/PrettyXMLWriter.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriter</a></dt>
<dd>
<div class="block">Declare a namespace for current and following elements
 'prefixHint' is ignored entirely in this implementation</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/XMLWriter.html#declareNamespace(java.lang.String,java.lang.String)" class="member-name-link">declareNamespace(String, String)</a> - Method in interface org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/XMLWriter.html" title="interface in org.biojava.nbio.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Hints that a namespace is going to be used in a sub-tree.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#decorateRna(boolean)" class="member-name-link">decorateRna(boolean)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>
<div class="block">Performs an optimisation where RNASequences are not translated into
 their own objects but are views onto the base DNA sequence.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/SoftHashMap.html#DEFAULT_LIMIT" class="member-name-link">DEFAULT_LIMIT</a> - Static variable in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SoftHashMap.html" title="class in org.biojava.nbio.core.util">SoftHashMap</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#DEFAULT_UNIPROT_BASE_URL" class="member-name-link">DEFAULT_UNIPROT_BASE_URL</a> - Static variable in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/DefaultAAIndexProvider.html" class="type-name-link" title="class in org.biojava.nbio.core.alignment.matrices">DefaultAAIndexProvider</a> - Class in <a href="../org/biojava/nbio/core/alignment/matrices/package-summary.html">org.biojava.nbio.core.alignment.matrices</a></dt>
<dd>
<div class="block">The default provider for AAINDEX loads substitution matrices from the AAINDEX file in the resources directory</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/DefaultAAIndexProvider.html#%3Cinit%3E()" class="member-name-link">DefaultAAIndexProvider()</a> - Constructor for class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/DefaultAAIndexProvider.html" title="class in org.biojava.nbio.core.alignment.matrices">DefaultAAIndexProvider</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/EditSequenceTest.html#delete()" class="member-name-link">delete()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/EditSequenceTest.html" title="class in org.biojava.nbio.core.sequence">EditSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/edits/Edit.Delete.html#%3Cinit%3E(int)" class="member-name-link">Delete(int)</a> - Constructor for class org.biojava.nbio.core.sequence.edits.<a href="../org/biojava/nbio/core/sequence/edits/Edit.Delete.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Delete</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/edits/Edit.Delete.html#%3Cinit%3E(int,int)" class="member-name-link">Delete(int, int)</a> - Constructor for class org.biojava.nbio.core.sequence.edits.<a href="../org/biojava/nbio/core/sequence/edits/Edit.Delete.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Delete</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/FileDownloadUtils.html#deleteDirectory(java.lang.String)" class="member-name-link">deleteDirectory(String)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/FileDownloadUtils.html" title="class in org.biojava.nbio.core.util">FileDownloadUtils</a></dt>
<dd>
<div class="block">Recursively delete a folder &amp; contents</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/FileDownloadUtils.html#deleteDirectory(java.nio.file.Path)" class="member-name-link">deleteDirectory(Path)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/FileDownloadUtils.html" title="class in org.biojava.nbio.core.util">FileDownloadUtils</a></dt>
<dd>
<div class="block">Recursively delete a folder &amp; contents</div>
</dd>
<dt><a href="../demo/package-summary.html">demo</a> - package demo</dt>
<dd>&nbsp;</dd>
<dt><a href="../demo/DemoSixFrameTranslation.html" class="type-name-link" title="class in demo">DemoSixFrameTranslation</a> - Class in <a href="../demo/package-summary.html">demo</a></dt>
<dd>
<div class="block">Created by andreas on 8/10/15.</div>
</dd>
<dt><a href="../demo/DemoSixFrameTranslation.html#%3Cinit%3E()" class="member-name-link">DemoSixFrameTranslation()</a> - Constructor for class demo.<a href="../demo/DemoSixFrameTranslation.html" title="class in demo">DemoSixFrameTranslation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/LocationHelper.html#detectCicular(java.util.List)" class="member-name-link">detectCicular(List&lt;Location&gt;)</a> - Static method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/LocationHelper.html" title="class in org.biojava.nbio.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Loops through the given list of locations and returns true if it looks
 like they represent a circular location.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/LocationHelper.html#detectEnd(java.util.List,boolean)" class="member-name-link">detectEnd(List&lt;Location&gt;, boolean)</a> - Static method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/LocationHelper.html" title="class in org.biojava.nbio.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">This will attempt to find what the last point is and returns that
 position.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/LocationHelper.html#detectStart(java.util.List)" class="member-name-link">detectStart(List&lt;Location&gt;)</a> - Static method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/LocationHelper.html" title="class in org.biojava.nbio.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Assumes that the first element is the start &amp; clones it</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/LocationHelper.html#detectStrand(java.util.List)" class="member-name-link">detectStrand(List&lt;Location&gt;)</a> - Static method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/LocationHelper.html" title="class in org.biojava.nbio.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Loops through the given list of locations and returns the consensus
 Strand class.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#dnaCompounds(org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">dnaCompounds(CompoundSet&lt;NucleotideCompound&gt;)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/DNACompoundSet.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.compound">DNACompoundSet</a> - Class in <a href="../org/biojava/nbio/core/sequence/compound/package-summary.html">org.biojava.nbio.core.sequence.compound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/DNACompoundSet.html#%3Cinit%3E()" class="member-name-link">DNACompoundSet()</a> - Constructor for class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/DNACompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">DNACompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#dnaRnaTranslator(org.biojava.nbio.core.sequence.transcription.DNAToRNATranslator)" class="member-name-link">dnaRnaTranslator(DNAToRNATranslator)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNASequence.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence">DNASequence</a> - Class in <a href="../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">This is class should model the attributes associated with a DNA sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNASequence.html#%3Cinit%3E()" class="member-name-link">DNASequence()</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Shouldn't be used but makes it bean happy</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNASequence.html#%3Cinit%3E(java.lang.String)" class="member-name-link">DNASequence(String)</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">String is king and create a sequence from DNA with default DNA compound set</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNASequence.html#%3Cinit%3E(java.lang.String,org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">DNASequence(String, CompoundSet&lt;NucleotideCompound&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Create a sequence from a string with user defined compound set</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNASequence.html#%3Cinit%3E(org.biojava.nbio.core.sequence.template.SequenceReader)" class="member-name-link">DNASequence(SequenceReader&lt;NucleotideCompound&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Create a sequence where the actual storage of the sequence data is somewhere else</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNASequence.html#%3Cinit%3E(org.biojava.nbio.core.sequence.template.SequenceReader,org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">DNASequence(SequenceReader&lt;NucleotideCompound&gt;, CompoundSet&lt;NucleotideCompound&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Create a sequence from a ProxySequencereader and user defined compound set</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNASequence.DNAType.html" class="type-name-link" title="enum in org.biojava.nbio.core.sequence">DNASequence.DNAType</a> - Enum in <a href="../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">The type of DNA sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/DNASequenceCreator.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">DNASequenceCreator</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">A helper class that allows different ways to read a string and create a DNA sequence.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/DNASequenceCreator.html#%3Cinit%3E(org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">DNASequenceCreator(CompoundSet&lt;NucleotideCompound&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/DNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">DNASequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNATest.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence">DNATest</a> - Class in <a href="../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNATest.html#%3Cinit%3E()" class="member-name-link">DNATest()</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DNATest.html" title="class in org.biojava.nbio.core.sequence">DNATest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.transcription">DNAToRNATranslator</a> - Class in <a href="../org/biojava/nbio/core/sequence/transcription/package-summary.html">org.biojava.nbio.core.sequence.transcription</a></dt>
<dd>
<div class="block">Performs the first stage of transcription by going from DNA to RNA.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html#%3Cinit%3E(org.biojava.nbio.core.sequence.io.template.SequenceCreatorInterface,org.biojava.nbio.core.sequence.template.CompoundSet,org.biojava.nbio.core.sequence.template.CompoundSet,boolean)" class="member-name-link">DNAToRNATranslator(SequenceCreatorInterface&lt;NucleotideCompound&gt;, CompoundSet&lt;NucleotideCompound&gt;, CompoundSet&lt;NucleotideCompound&gt;, boolean)</a> - Constructor for class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html" title="class in org.biojava.nbio.core.sequence.transcription">DNAToRNATranslator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/FileDownloadUtils.html#downloadFile(java.net.URL,java.io.File)" class="member-name-link">downloadFile(URL, File)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/FileDownloadUtils.html" title="class in org.biojava.nbio.core.util">FileDownloadUtils</a></dt>
<dd>
<div class="block">Download the content provided at URL url and store the result to a local
 file, using a temp file to cache the content in case something goes wrong
 in download.</div>
</dd>
</dl>
<a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Z</a>&nbsp;<br><a href="../allclasses-index.html">All&nbsp;Classes&nbsp;and&nbsp;Interfaces</a><span class="vertical-separator">|</span><a href="../allpackages-index.html">All&nbsp;Packages</a><span class="vertical-separator">|</span><a href="../constant-values.html">Constant&nbsp;Field&nbsp;Values</a><span class="vertical-separator">|</span><a href="../serialized-form.html">Serialized&nbsp;Form</a></main>
</div>
</div>
</body>
</html>
