<!DOCTYPE HTML>
<html lang="pt">
<head>
<!-- Generated by javadoc (17) on Sun May 15 18:36:12 WEST 2022 -->
<title>A-Index</title>
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta name="dc.created" content="2022-05-15">
<meta name="description" content="index: A">
<meta name="generator" content="javadoc/IndexWriter">
<link rel="stylesheet" type="text/css" href="../stylesheet.css" title="Style">
<link rel="stylesheet" type="text/css" href="../script-dir/jquery-ui.min.css" title="Style">
<link rel="stylesheet" type="text/css" href="../jquery-ui.overrides.css" title="Style">
<script type="text/javascript" src="../script.js"></script>
<script type="text/javascript" src="../script-dir/jquery-3.5.1.min.js"></script>
<script type="text/javascript" src="../script-dir/jquery-ui.min.js"></script>
</head>
<body class="index-page">
<script type="text/javascript">var pathtoroot = "../";
loadScripts(document, 'script');</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<div class="flex-box">
<header role="banner" class="flex-header">
<nav role="navigation">
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="top-nav" id="navbar-top">
<div class="skip-nav"><a href="#skip-navbar-top" title="Skip navigation links">Skip navigation links</a></div>
<ul id="navbar-top-firstrow" class="nav-list" title="Navigation">
<li><a href="../index.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li>Use</li>
<li><a href="../overview-tree.html">Tree</a></li>
<li><a href="../deprecated-list.html">Deprecated</a></li>
<li class="nav-bar-cell1-rev">Index</li>
<li><a href="../help-doc.html#index">Help</a></li>
</ul>
</div>
<div class="sub-nav">
<div class="nav-list-search"><label for="search-input">SEARCH:</label>
<input type="text" id="search-input" value="search" disabled="disabled">
<input type="reset" id="reset-button" value="reset" disabled="disabled">
</div>
</div>
<!-- ========= END OF TOP NAVBAR ========= -->
<span class="skip-nav" id="skip-navbar-top"></span></nav>
</header>
<div class="flex-content">
<main role="main">
<div class="header">
<h1>Index</h1>
</div>
<a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Z</a>&nbsp;<br><a href="../allclasses-index.html">All&nbsp;Classes&nbsp;and&nbsp;Interfaces</a><span class="vertical-separator">|</span><a href="../allpackages-index.html">All&nbsp;Packages</a><span class="vertical-separator">|</span><a href="../constant-values.html">Constant&nbsp;Field&nbsp;Values</a><span class="vertical-separator">|</span><a href="../serialized-form.html">Serialized&nbsp;Form</a>
<h2 class="title" id="I:A">A</h2>
<dl class="index">
<dt><a href="../org/biojava/nbio/core/alignment/matrices/AAindexFactory.html" class="type-name-link" title="class in org.biojava.nbio.core.alignment.matrices">AAindexFactory</a> - Class in <a href="../org/biojava/nbio/core/alignment/matrices/package-summary.html">org.biojava.nbio.core.alignment.matrices</a></dt>
<dd>
<div class="block">Factory class to get Providers for substitution matrices that are provided by
 the AAINDEX database.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/AAindexFactory.html#%3Cinit%3E()" class="member-name-link">AAindexFactory()</a> - Constructor for class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/AAindexFactory.html" title="class in org.biojava.nbio.core.alignment.matrices">AAindexFactory</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/AAIndexFileParser.html" class="type-name-link" title="class in org.biojava.nbio.core.alignment.matrices">AAIndexFileParser</a> - Class in <a href="../org/biojava/nbio/core/alignment/matrices/package-summary.html">org.biojava.nbio.core.alignment.matrices</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/AAIndexFileParser.html#%3Cinit%3E()" class="member-name-link">AAIndexFileParser()</a> - Constructor for class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/AAIndexFileParser.html" title="class in org.biojava.nbio.core.alignment.matrices">AAIndexFileParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/AAIndexProvider.html" class="type-name-link" title="interface in org.biojava.nbio.core.alignment.matrices">AAIndexProvider</a> - Interface in <a href="../org/biojava/nbio/core/alignment/matrices/package-summary.html">org.biojava.nbio.core.alignment.matrices</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/ABITrace.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">ABITrace</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">Title: ABITrace</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/ABITrace.html#%3Cinit%3E(byte%5B%5D)" class="member-name-link">ABITrace(byte[])</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/ABITrace.html" title="class in org.biojava.nbio.core.sequence.io">ABITrace</a></dt>
<dd>
<div class="block">The <code>byte[]</code> constructor parses an ABI file represented as a byte array.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/ABITrace.html#%3Cinit%3E(java.io.File)" class="member-name-link">ABITrace(File)</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/ABITrace.html" title="class in org.biojava.nbio.core.sequence.io">ABITrace</a></dt>
<dd>
<div class="block">The File constructor opens a local ABI file and parses the content.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/ABITrace.html#%3Cinit%3E(java.net.URL)" class="member-name-link">ABITrace(URL)</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/ABITrace.html" title="class in org.biojava.nbio.core.sequence.io">ABITrace</a></dt>
<dd>
<div class="block">The URL constructor opens an ABI file from any URL.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/ABITracerCompoundSet.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.compound">ABITracerCompoundSet</a> - Class in <a href="../org/biojava/nbio/core/sequence/compound/package-summary.html">org.biojava.nbio.core.sequence.compound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/ABITracerCompoundSet.html#%3Cinit%3E()" class="member-name-link">ABITracerCompoundSet()</a> - Constructor for class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/ABITracerCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">ABITracerCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/ABITracerTest.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">ABITracerTest</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">Test file 3730.ab1 is from https://github.com/biopython/biopython/blob/master/Tests/Abi/3730.ab1 The test data for
 comparing the results from ABITrace.java for file 3730.ab1 is from
 https://github.com/biopython/biopython/blob/master/Tests/Abi/test_data</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/ABITracerTest.html#%3Cinit%3E()" class="member-name-link">ABITracerTest()</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/ABITracerTest.html" title="class in org.biojava.nbio.core.sequence.io">ABITracerTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a> - Class in <a href="../org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>
<div class="block">The details of a Compound</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html#%3Cinit%3E()" class="member-name-link">AbstractCompound()</a> - Constructor for class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html#%3Cinit%3E(java.lang.String)" class="member-name-link">AbstractCompound(String)</a> - Constructor for class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a>&lt;<a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="type parameter in AbstractCompoundSet">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#%3Cinit%3E()" class="member-name-link">AbstractCompoundSet()</a> - Constructor for class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundTranslator</a>&lt;<a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="type parameter in AbstractCompoundTranslator">F</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>,<wbr><a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="type parameter in AbstractCompoundTranslator">T</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#%3Cinit%3E(org.biojava.nbio.core.sequence.io.template.SequenceCreatorInterface,org.biojava.nbio.core.sequence.template.CompoundSet,org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">AbstractCompoundTranslator(SequenceCreatorInterface&lt;T&gt;, CompoundSet&lt;F&gt;, CompoundSet&lt;T&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html#%3Cinit%3E(int)" class="member-name-link">AbstractEdit(int)</a> - Constructor for class org.biojava.nbio.core.sequence.edits.<a href="../org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.AbstractEdit</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html#%3Cinit%3E(int,int)" class="member-name-link">AbstractEdit(int, int)</a> - Constructor for class org.biojava.nbio.core.sequence.edits.<a href="../org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.AbstractEdit</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a>&lt;<a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="type parameter in AbstractFeature">S</a> extends <a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="type parameter in AbstractFeature">C</a>&gt;,<wbr><a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="type parameter in AbstractFeature">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/features/package-summary.html">org.biojava.nbio.core.sequence.features</a></dt>
<dd>
<div class="block">A feature is currently any descriptive item that can be associated with a sequence position(s)
 A feature has a type and a source which is currently a string to allow flexibility for the user
 Ideally well defined features should have a class to describe attributes of that feature</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html#%3Cinit%3E(java.lang.String,java.lang.String)" class="member-name-link">AbstractFeature(String, String)</a> - Constructor for class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">A feature has a type and a source</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a> - Class in <a href="../org/biojava/nbio/core/sequence/location/template/package-summary.html">org.biojava.nbio.core.sequence.location.template</a></dt>
<dd>
<div class="block">Base abstraction of a location which encodes for the majority of important
 features about a location such as the start, end and strand</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#%3Cinit%3E(org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.Strand,boolean,boolean,java.util.List)" class="member-name-link">AbstractLocation(Point, Point, Strand, boolean, boolean, List&lt;Location&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>
<div class="block">Default constructor</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#%3Cinit%3E(org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.Strand,boolean,boolean,org.biojava.nbio.core.sequence.AccessionID,java.util.List)" class="member-name-link">AbstractLocation(Point, Point, Strand, boolean, boolean, AccessionID, List&lt;Location&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>
<div class="block">Default constructor</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocationProduct.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocationProduct</a> - Class in <a href="../org/biojava/nbio/core/sequence/location/template/package-summary.html">org.biojava.nbio.core.sequence.location.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocationProduct.html#%3Cinit%3E()" class="member-name-link">AbstractLocationProduct()</a> - Constructor for class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocationProduct.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocationProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.template">AbstractNucleotideCompoundSet</a>&lt;<a href="../org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html" title="type parameter in AbstractNucleotideCompoundSet">C</a> extends <a href="../org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound">NucleotideCompound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#%3Cinit%3E()" class="member-name-link">AbstractNucleotideCompoundSet()</a> - Constructor for class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractNucleotideCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/reference/AbstractReference.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.reference">AbstractReference</a> - Class in <a href="../org/biojava/nbio/core/sequence/reference/package-summary.html">org.biojava.nbio.core.sequence.reference</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/reference/AbstractReference.html#%3Cinit%3E()" class="member-name-link">AbstractReference()</a> - Constructor for class org.biojava.nbio.core.sequence.reference.<a href="../org/biojava/nbio/core/sequence/reference/AbstractReference.html" title="class in org.biojava.nbio.core.sequence.reference">AbstractReference</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="type parameter in AbstractSequence">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>
<div class="block">The base class for DNA, RNA and Protein sequences.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#%3Cinit%3E()" class="member-name-link">AbstractSequence()</a> - Constructor for class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#%3Cinit%3E(java.lang.String,org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">AbstractSequence(String, CompoundSet&lt;C&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Create a Sequence from a simple string where the values should be found in compoundSet</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#%3Cinit%3E(org.biojava.nbio.core.sequence.template.SequenceReader,org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">AbstractSequence(SequenceReader&lt;C&gt;, CompoundSet&lt;C&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">A ProxySequenceReader allows abstraction of both the storage of the sequence data and the location
 of the sequence data.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.AnnotationType.html" class="type-name-link" title="enum in org.biojava.nbio.core.sequence.template">AbstractSequence.AnnotationType</a> - Enum in <a href="../org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AccesionedLocation.html" class="type-name-link" title="interface in org.biojava.nbio.core.sequence.location.template">AccesionedLocation</a> - Interface in <a href="../org/biojava/nbio/core/sequence/location/template/package-summary.html">org.biojava.nbio.core.sequence.location.template</a></dt>
<dd>
<div class="block">A location which is bound to an AccessionID.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/Accessioned.html" class="type-name-link" title="interface in org.biojava.nbio.core.sequence.template">Accessioned</a> - Interface in <a href="../org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>
<div class="block">Indicates an entity is accessioned</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/AccessionID.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence">AccessionID</a> - Class in <a href="../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">Used in Sequences as the unique identifier.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/AccessionID.html#%3Cinit%3E()" class="member-name-link">AccessionID()</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/AccessionID.html" title="class in org.biojava.nbio.core.sequence">AccessionID</a></dt>
<dd>
<div class="block">Default constructor sets id t empty string</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/AccessionID.html#%3Cinit%3E(java.lang.String)" class="member-name-link">AccessionID(String)</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/AccessionID.html" title="class in org.biojava.nbio.core.sequence">AccessionID</a></dt>
<dd>
<div class="block">Creates an id with default DataSource.LOCAL source</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/AccessionID.html#%3Cinit%3E(java.lang.String,org.biojava.nbio.core.sequence.DataSource)" class="member-name-link">AccessionID(String, DataSource)</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/AccessionID.html" title="class in org.biojava.nbio.core.sequence">AccessionID</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/AccessionID.html#%3Cinit%3E(java.lang.String,org.biojava.nbio.core.sequence.DataSource,java.lang.Integer,java.lang.String)" class="member-name-link">AccessionID(String, DataSource, Integer, String)</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/AccessionID.html" title="class in org.biojava.nbio.core.sequence">AccessionID</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#addAlignedSequence(S)" class="member-name-link">addAlignedSequence(S)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava.nbio.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">A sequence that has been aligned to other sequences will have inserts.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranscriptSequence.html#addCDS(org.biojava.nbio.core.sequence.AccessionID,int,int,int)" class="member-name-link">addCDS(AccessionID, int, int, int)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a></dt>
<dd>
<div class="block">Add a Coding Sequence region with phase to the transcript sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranscriptSequenceProduct.html#addCDS(org.biojava.nbio.core.sequence.AccessionID,int,int,int,org.biojava.nbio.core.sequence.TranscriptSequence)" class="member-name-link">addCDS(AccessionID, int, int, int, TranscriptSequence)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranscriptSequenceProduct.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequenceProduct</a></dt>
<dd>
<div class="block">Add a Coding Sequence region with phase to the transcript sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/XMLHelper.html#addChildElement(org.w3c.dom.Element,java.lang.String)" class="member-name-link">addChildElement(Element, String)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/XMLHelper.html" title="class in org.biojava.nbio.core.util">XMLHelper</a></dt>
<dd>
<div class="block">Creates a new element called elementName and adds it to parentElement</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/GeneSequence.html#addExon(org.biojava.nbio.core.sequence.AccessionID,int,int)" class="member-name-link">addExon(AccessionID, int, int)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Add an ExonSequence mainly used to mark as a feature</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#addFeature(int,int,org.biojava.nbio.core.sequence.features.FeatureInterface)" class="member-name-link">addFeature(int, int, FeatureInterface&lt;AbstractSequence&lt;C&gt;, C&gt;)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Method to help set the proper details for a feature as it relates to a sequence
 where the feature needs to have a location on the sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#addFeature(org.biojava.nbio.core.sequence.features.FeatureInterface)" class="member-name-link">addFeature(FeatureInterface&lt;AbstractSequence&lt;C&gt;, C&gt;)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Add a feature to this sequence.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/ChromosomeSequence.html#addGene(org.biojava.nbio.core.sequence.AccessionID,int,int,org.biojava.nbio.core.sequence.Strand)" class="member-name-link">addGene(AccessionID, int, int, Strand)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/ChromosomeSequence.html" title="class in org.biojava.nbio.core.sequence">ChromosomeSequence</a></dt>
<dd>
<div class="block">Add a gene to the chromosome sequence using bioIndexing starts at 1 instead of 0.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/GeneSequence.html#addIntron(org.biojava.nbio.core.sequence.AccessionID,int,int)" class="member-name-link">addIntron(AccessionID, int, int)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Add an Intron Currently used to mark an IntronSequence as a feature</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/GeneSequence.html#addIntronsUsingExons()" class="member-name-link">addIntronsUsingExons()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Once everything has been added to the gene sequence where you might have added exon sequences only then you
 can infer the intron sequences and add them.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#addNote(java.lang.String)" class="member-name-link">addNote(String)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Add notes about this sequence that will get exported for GFF3</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#addProperty(java.lang.String,java.lang.String)" class="member-name-link">addProperty(String, String)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">DBReferenceInfo</a></dt>
<dd>
<div class="block">Add a property and type to associate with this DBReferenceInfo</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html#addQualifier(java.lang.String,org.biojava.nbio.core.sequence.features.Qualifier)" class="member-name-link">addQualifier(String, Qualifier)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#addQualifier(java.lang.String,org.biojava.nbio.core.sequence.features.Qualifier)" class="member-name-link">addQualifier(String, Qualifier)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html#addQualifier(java.lang.String,org.biojava.nbio.core.sequence.features.Qualifier)" class="member-name-link">addQualifier(String, Qualifier)</a> - Method in interface org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Add a qualifier</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/QuantityFeature.html#addQuantity(java.lang.Number)" class="member-name-link">addQuantity(Number)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/QuantityFeature.html" title="class in org.biojava.nbio.core.sequence.features">QuantityFeature</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#addReference(org.biojava.nbio.core.sequence.reference.AbstractReference)" class="member-name-link">addReference(AbstractReference)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">GenericGenbankHeaderParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranscriptSequence.html#addStartCodonSequence(org.biojava.nbio.core.sequence.AccessionID,int,int)" class="member-name-link">addStartCodonSequence(AccessionID, int, int)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a></dt>
<dd>
<div class="block">Sets the start codon sequence at given begin /  end location.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranscriptSequenceProduct.html#addStartCodonSequence(org.biojava.nbio.core.sequence.AccessionID,int,int,org.biojava.nbio.core.sequence.TranscriptSequence)" class="member-name-link">addStartCodonSequence(AccessionID, int, int, TranscriptSequence)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranscriptSequenceProduct.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequenceProduct</a></dt>
<dd>
<div class="block">Sets the start codon sequence at given begin /  end location.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranscriptSequence.html#addStopCodonSequence(org.biojava.nbio.core.sequence.AccessionID,int,int)" class="member-name-link">addStopCodonSequence(AccessionID, int, int)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a></dt>
<dd>
<div class="block">Sets the stop codon sequence at given begin /  end location.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranscriptSequenceProduct.html#addStopCodonSequence(org.biojava.nbio.core.sequence.AccessionID,int,int,org.biojava.nbio.core.sequence.TranscriptSequence)" class="member-name-link">addStopCodonSequence(AccessionID, int, int, TranscriptSequence)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranscriptSequenceProduct.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequenceProduct</a></dt>
<dd>
<div class="block">Sets the stop codon sequence at given begin /  end location.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/FlatFileCache.html#addToCache(java.lang.String,java.io.File)" class="member-name-link">addToCache(String, File)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/FlatFileCache.html" title="class in org.biojava.nbio.core.util">FlatFileCache</a></dt>
<dd>
<div class="block">The file is read and the bytes stored immediately.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/GeneSequence.html#addTranscript(org.biojava.nbio.core.sequence.AccessionID,int,int)" class="member-name-link">addTranscript(AccessionID, int, int)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Add a transcription sequence to a gene which describes a ProteinSequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" class="type-name-link" title="interface in org.biojava.nbio.core.alignment.template">AlignedSequence</a>&lt;<a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="type parameter in AlignedSequence">S</a> extends <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="type parameter in AlignedSequence">C</a>&gt;,<wbr><a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="type parameter in AlignedSequence">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="../org/biojava/nbio/core/alignment/template/package-summary.html">org.biojava.nbio.core.alignment.template</a></dt>
<dd>
<div class="block">Defines a data structure for a <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a> within an alignment.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.Step.html" class="type-name-link" title="enum in org.biojava.nbio.core.alignment.template">AlignedSequence.Step</a> - Enum in <a href="../org/biojava/nbio/core/alignment/template/package-summary.html">org.biojava.nbio.core.alignment.template</a></dt>
<dd>
<div class="block">Defines an alignment step in order to pass alignment information from an <code>Aligner</code> to a constructor.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/SequenceOptimizationHints.SequenceCollection.html#ALL_SEQUENCES" class="member-name-link">ALL_SEQUENCES</a> - Enum constant in enum org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/SequenceOptimizationHints.SequenceCollection.html" title="enum in org.biojava.nbio.core.sequence">SequenceOptimizationHints.SequenceCollection</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/PrettyXMLWriterProduct2.html#allocPrefix(java.lang.String,java.util.Map)" class="member-name-link">allocPrefix(String, Map&lt;String, String&gt;)</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/PrettyXMLWriterProduct2.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriterProduct2</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.StringFormat.html#ALN" class="member-name-link">ALN</a> - Enum constant in enum org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.StringFormat.html" title="enum in org.biojava.nbio.core.alignment.template">Profile.StringFormat</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/LightweightProfile.StringFormat.html#ALN" class="member-name-link">ALN</a> - Enum constant in enum org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/LightweightProfile.StringFormat.html" title="enum in org.biojava.nbio.core.sequence.template">LightweightProfile.StringFormat</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AmbiguityDNACompoundSet.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.compound">AmbiguityDNACompoundSet</a> - Class in <a href="../org/biojava/nbio/core/sequence/compound/package-summary.html">org.biojava.nbio.core.sequence.compound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AmbiguityDNACompoundSet.html#%3Cinit%3E()" class="member-name-link">AmbiguityDNACompoundSet()</a> - Constructor for class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/AmbiguityDNACompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AmbiguityDNACompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AmbiguityDNACompoundTest.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.compound">AmbiguityDNACompoundTest</a> - Class in <a href="../org/biojava/nbio/core/sequence/compound/package-summary.html">org.biojava.nbio.core.sequence.compound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AmbiguityDNACompoundTest.html#%3Cinit%3E()" class="member-name-link">AmbiguityDNACompoundTest()</a> - Constructor for class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/AmbiguityDNACompoundTest.html" title="class in org.biojava.nbio.core.sequence.compound">AmbiguityDNACompoundTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AmbiguityDNARNAHybridCompoundSet.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.compound">AmbiguityDNARNAHybridCompoundSet</a> - Class in <a href="../org/biojava/nbio/core/sequence/compound/package-summary.html">org.biojava.nbio.core.sequence.compound</a></dt>
<dd>
<div class="block">Ambiguity set for hybrid DNA/RNA sequences.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AmbiguityDNARNAHybridCompoundSet.html#%3Cinit%3E()" class="member-name-link">AmbiguityDNARNAHybridCompoundSet()</a> - Constructor for class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/AmbiguityDNARNAHybridCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AmbiguityDNARNAHybridCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AmbiguityRNACompoundSet.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.compound">AmbiguityRNACompoundSet</a> - Class in <a href="../org/biojava/nbio/core/sequence/compound/package-summary.html">org.biojava.nbio.core.sequence.compound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AmbiguityRNACompoundSet.html#%3Cinit%3E()" class="member-name-link">AmbiguityRNACompoundSet()</a> - Constructor for class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/AmbiguityRNACompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AmbiguityRNACompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompound</a> - Class in <a href="../org/biojava/nbio/core/sequence/compound/package-summary.html">org.biojava.nbio.core.sequence.compound</a></dt>
<dd>
<div class="block">Used to describe an Amino Acid.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html#%3Cinit%3E(org.biojava.nbio.core.sequence.compound.AminoAcidCompoundSet,java.lang.String,java.lang.String,java.lang.String,java.lang.Float)" class="member-name-link">AminoAcidCompound(AminoAcidCompoundSet, String, String, String, Float)</a> - Constructor for class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompoundSet</a> - Class in <a href="../org/biojava/nbio/core/sequence/compound/package-summary.html">org.biojava.nbio.core.sequence.compound</a></dt>
<dd>
<div class="block">Set of proteinogenic amino acids.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#%3Cinit%3E()" class="member-name-link">AminoAcidCompoundSet()</a> - Constructor for class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#aminoAcidsCompounds(org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">aminoAcidsCompounds(CompoundSet&lt;AminoAcidCompound&gt;)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/ArrayListProxySequenceReader.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.loader">ArrayListProxySequenceReader</a>&lt;<a href="../org/biojava/nbio/core/sequence/loader/ArrayListProxySequenceReader.html" title="type parameter in ArrayListProxySequenceReader">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/loader/package-summary.html">org.biojava.nbio.core.sequence.loader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/ArrayListProxySequenceReader.html#%3Cinit%3E()" class="member-name-link">ArrayListProxySequenceReader()</a> - Constructor for class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/ArrayListProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">ArrayListProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a>&lt;<a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="type parameter in ArrayListSequenceReader">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/storage/package-summary.html">org.biojava.nbio.core.sequence.storage</a></dt>
<dd>
<div class="block">Stores a Sequence as a collection of compounds in an ArrayList</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#%3Cinit%3E()" class="member-name-link">ArrayListSequenceReader()</a> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#%3Cinit%3E(java.lang.String,org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">ArrayListSequenceReader(String, CompoundSet&lt;C&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#%3Cinit%3E(java.util.List,org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">ArrayListSequenceReader(List&lt;C&gt;, CompoundSet&lt;C&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/LocationParserTest.html#assertInsdcLoc(java.lang.String,org.biojava.nbio.core.sequence.location.template.Location)" class="member-name-link">assertInsdcLoc(String, Location)</a> - Method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/LocationParserTest.html" title="class in org.biojava.nbio.core.sequence.location">LocationParserTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNATest.html#at()" class="member-name-link">at()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DNATest.html" title="class in org.biojava.nbio.core.sequence">DNATest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/PrettyXMLWriter.html#attribute(java.lang.String,java.lang.String)" class="member-name-link">attribute(String, String)</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/PrettyXMLWriter.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/XMLWriter.html#attribute(java.lang.String,java.lang.String)" class="member-name-link">attribute(String, String)</a> - Method in interface org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/XMLWriter.html" title="interface in org.biojava.nbio.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Add an un-qualified attribute to an element.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/PrettyXMLWriter.html#attribute(java.lang.String,java.lang.String,java.lang.String)" class="member-name-link">attribute(String, String, String)</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/PrettyXMLWriter.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/XMLWriter.html#attribute(java.lang.String,java.lang.String,java.lang.String)" class="member-name-link">attribute(String, String, String)</a> - Method in interface org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/XMLWriter.html" title="interface in org.biojava.nbio.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Add an attribute to an element.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/UncompressInputStream.html#available()" class="member-name-link">available()</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/UncompressInputStream.html" title="class in org.biojava.nbio.core.util">UncompressInputStream</a></dt>
<dd>&nbsp;</dd>
</dl>
<a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Z</a>&nbsp;<br><a href="../allclasses-index.html">All&nbsp;Classes&nbsp;and&nbsp;Interfaces</a><span class="vertical-separator">|</span><a href="../allpackages-index.html">All&nbsp;Packages</a><span class="vertical-separator">|</span><a href="../constant-values.html">Constant&nbsp;Field&nbsp;Values</a><span class="vertical-separator">|</span><a href="../serialized-form.html">Serialized&nbsp;Form</a></main>
</div>
</div>
</body>
</html>
