<!DOCTYPE HTML>
<html lang="pt">
<head>
<!-- Generated by javadoc (17) on Sun May 15 18:36:12 WEST 2022 -->
<title>P-Index</title>
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta name="dc.created" content="2022-05-15">
<meta name="description" content="index: P">
<meta name="generator" content="javadoc/IndexWriter">
<link rel="stylesheet" type="text/css" href="../stylesheet.css" title="Style">
<link rel="stylesheet" type="text/css" href="../script-dir/jquery-ui.min.css" title="Style">
<link rel="stylesheet" type="text/css" href="../jquery-ui.overrides.css" title="Style">
<script type="text/javascript" src="../script.js"></script>
<script type="text/javascript" src="../script-dir/jquery-3.5.1.min.js"></script>
<script type="text/javascript" src="../script-dir/jquery-ui.min.js"></script>
</head>
<body class="index-page">
<script type="text/javascript">var pathtoroot = "../";
loadScripts(document, 'script');</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<div class="flex-box">
<header role="banner" class="flex-header">
<nav role="navigation">
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="top-nav" id="navbar-top">
<div class="skip-nav"><a href="#skip-navbar-top" title="Skip navigation links">Skip navigation links</a></div>
<ul id="navbar-top-firstrow" class="nav-list" title="Navigation">
<li><a href="../index.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li>Use</li>
<li><a href="../overview-tree.html">Tree</a></li>
<li><a href="../deprecated-list.html">Deprecated</a></li>
<li class="nav-bar-cell1-rev">Index</li>
<li><a href="../help-doc.html#index">Help</a></li>
</ul>
</div>
<div class="sub-nav">
<div class="nav-list-search"><label for="search-input">SEARCH:</label>
<input type="text" id="search-input" value="search" disabled="disabled">
<input type="reset" id="reset-button" value="reset" disabled="disabled">
</div>
</div>
<!-- ========= END OF TOP NAVBAR ========= -->
<span class="skip-nav" id="skip-navbar-top"></span></nav>
</header>
<div class="flex-content">
<main role="main">
<div class="header">
<h1>Index</h1>
</div>
<a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Z</a>&nbsp;<br><a href="../allclasses-index.html">All&nbsp;Classes&nbsp;and&nbsp;Interfaces</a><span class="vertical-separator">|</span><a href="../allpackages-index.html">All&nbsp;Packages</a><span class="vertical-separator">|</span><a href="../constant-values.html">Constant&nbsp;Field&nbsp;Values</a><span class="vertical-separator">|</span><a href="../serialized-form.html">Serialized&nbsp;Form</a>
<h2 class="title" id="I:P">P</h2>
<dl class="index">
<dt><a href="../org/biojava/nbio/core/util/StringManipulationHelper.html#padLeft(java.lang.String,int)" class="member-name-link">padLeft(String, int)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/StringManipulationHelper.html" title="class in org.biojava.nbio.core.util">StringManipulationHelper</a></dt>
<dd>
<div class="block">Adds padding to left of supplied string</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/StringManipulationHelper.html#padRight(java.lang.String,int)" class="member-name-link">padRight(String, int)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/StringManipulationHelper.html" title="class in org.biojava.nbio.core.util">StringManipulationHelper</a></dt>
<dd>
<div class="block">Adds padding to right of supplied string</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/StartCodonSequence.html#parentGeneSequence" class="member-name-link">parentGeneSequence</a> - Variable in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/StartCodonSequence.html" title="class in org.biojava.nbio.core.sequence">StartCodonSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/StopCodonSequence.html#parentGeneSequence" class="member-name-link">parentGeneSequence</a> - Variable in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/StopCodonSequence.html" title="class in org.biojava.nbio.core.sequence">StopCodonSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/AAIndexFileParser.html#parse(java.io.InputStream)" class="member-name-link">parse(InputStream)</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/AAIndexFileParser.html" title="class in org.biojava.nbio.core.alignment.matrices">AAIndexFileParser</a></dt>
<dd>
<div class="block">Parse an inputStream that points to an AAINDEX database file</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/InsdcParser.html#parse(java.lang.String)" class="member-name-link">parse(String)</a> - Method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/InsdcParser.html" title="class in org.biojava.nbio.core.sequence.location">InsdcParser</a></dt>
<dd>
<div class="block">Main method for parsing a location from a String instance</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#parsedCompounds" class="member-name-link">parsedCompounds</a> - Variable in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../demo/ParseFastaFileDemo.html" class="type-name-link" title="class in demo">ParseFastaFileDemo</a> - Class in <a href="../demo/package-summary.html">demo</a></dt>
<dd>
<div class="block">Created by andreas on 6/17/15.</div>
</dd>
<dt><a href="../demo/ParseFastaFileDemo.html#%3Cinit%3E()" class="member-name-link">ParseFastaFileDemo()</a> - Constructor for class demo.<a href="../demo/ParseFastaFileDemo.html" title="class in demo">ParseFastaFileDemo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#parseFeatures(org.biojava.nbio.core.sequence.template.AbstractSequence)" class="member-name-link">parseFeatures(AbstractSequence&lt;C&gt;)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#parseHeader(java.lang.String,S)" class="member-name-link">parseHeader(String, S)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">GenericFastaHeaderParser</a></dt>
<dd>
<div class="block">Parse the header and set the values in the sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#parseHeader(java.lang.String,S)" class="member-name-link">parseHeader(String, S)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">GenericGenbankHeaderParser</a></dt>
<dd>
<div class="block">Parse the header and set the values in the sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#parseHeader(java.lang.String,S)" class="member-name-link">parseHeader(String, S)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">PlainFastaHeaderParser</a></dt>
<dd>
<div class="block">Parse the header and set the values in the sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/template/SequenceHeaderParserInterface.html#parseHeader(java.lang.String,S)" class="member-name-link">parseHeader(String, S)</a> - Method in interface org.biojava.nbio.core.sequence.io.template.<a href="../org/biojava/nbio/core/sequence/io/template/SequenceHeaderParserInterface.html" title="interface in org.biojava.nbio.core.sequence.io.template">SequenceHeaderParserInterface</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/LocationParserTest.html#PARSER" class="member-name-link">PARSER</a> - Static variable in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/LocationParserTest.html" title="class in org.biojava.nbio.core.sequence.location">LocationParserTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/exceptions/ParserException.html" class="type-name-link" title="class in org.biojava.nbio.core.exceptions">ParserException</a> - Exception in <a href="../org/biojava/nbio/core/exceptions/package-summary.html">org.biojava.nbio.core.exceptions</a></dt>
<dd>
<div class="block">General abstraction of different parsing errors</div>
</dd>
<dt><a href="../org/biojava/nbio/core/exceptions/ParserException.html#%3Cinit%3E(java.lang.Exception)" class="member-name-link">ParserException(Exception)</a> - Constructor for exception org.biojava.nbio.core.exceptions.<a href="../org/biojava/nbio/core/exceptions/ParserException.html" title="class in org.biojava.nbio.core.exceptions">ParserException</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/exceptions/ParserException.html#%3Cinit%3E(java.lang.String)" class="member-name-link">ParserException(String)</a> - Constructor for exception org.biojava.nbio.core.exceptions.<a href="../org/biojava/nbio/core/exceptions/ParserException.html" title="class in org.biojava.nbio.core.exceptions">ParserException</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/exceptions/ParserException.html#%3Cinit%3E(java.lang.String,java.lang.Exception)" class="member-name-link">ParserException(String, Exception)</a> - Constructor for exception org.biojava.nbio.core.exceptions.<a href="../org/biojava/nbio/core/exceptions/ParserException.html" title="class in org.biojava.nbio.core.exceptions">ParserException</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranslationTest.html#parseSequences()" class="member-name-link">parseSequences()</a> - Static method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranslationTest.html" title="class in org.biojava.nbio.core.sequence">TranslationTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACParserProduct.html#parseTables()" class="member-name-link">parseTables()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/IUPACParserProduct.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParserProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#parseUniprotXMLString(java.lang.String,org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">parseUniprotXMLString(String, CompoundSet&lt;C&gt;)</a> - Static method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">The passed in xml is parsed as a DOM object so we know everything about the protein.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DataSource.html#PATENTS" class="member-name-link">PATENTS</a> - Enum constant in enum org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DataSource.html" title="enum in org.biojava.nbio.core.sequence">DataSource</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DataSource.html#PDB1" class="member-name-link">PDB1</a> - Enum constant in enum org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DataSource.html" title="enum in org.biojava.nbio.core.sequence">DataSource</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DataSource.html#PDB2" class="member-name-link">PDB2</a> - Enum constant in enum org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DataSource.html" title="enum in org.biojava.nbio.core.sequence">DataSource</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DataSource.html#PDBe" class="member-name-link">PDBe</a> - Enum constant in enum org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DataSource.html" title="enum in org.biojava.nbio.core.sequence">DataSource</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.StringFormat.html#PDBWEB" class="member-name-link">PDBWEB</a> - Enum constant in enum org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.StringFormat.html" title="enum in org.biojava.nbio.core.alignment.template">Profile.StringFormat</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/LightweightProfile.StringFormat.html#PDBWEB" class="member-name-link">PDBWEB</a> - Enum constant in enum org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/LightweightProfile.StringFormat.html" title="enum in org.biojava.nbio.core.sequence.template">LightweightProfile.StringFormat</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/SequenceTools.html#percentNucleotideSequence(java.lang.String)" class="member-name-link">percentNucleotideSequence(String)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SequenceTools.html" title="class in org.biojava.nbio.core.util">SequenceTools</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/SequenceTools.html#permuteCyclic(java.lang.String,int)" class="member-name-link">permuteCyclic(String, int)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SequenceTools.html" title="class in org.biojava.nbio.core.util">SequenceTools</a></dt>
<dd>
<div class="block">Cyclically permute the characters in <code>string</code> <em>forward</em> by <code>n</code> elements.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/SequenceTools.html#permuteCyclic(T%5B%5D,T%5B%5D,int)" class="member-name-link">permuteCyclic(T[], T[], int)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SequenceTools.html" title="class in org.biojava.nbio.core.util">SequenceTools</a></dt>
<dd>
<div class="block">Cyclically permute <code>array</code> <em>forward</em> by <code>n</code> elements.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DataSource.html#PFAM" class="member-name-link">PFAM</a> - Enum constant in enum org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DataSource.html" title="enum in org.biojava.nbio.core.sequence">DataSource</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/FileDownloadUtils.html#ping(java.lang.String,int)" class="member-name-link">ping(String, int)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/FileDownloadUtils.html" title="class in org.biojava.nbio.core.util">FileDownloadUtils</a></dt>
<dd>
<div class="block">Pings a HTTP URL.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">PlainFastaHeaderParser</a>&lt;<a href="../org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html" title="type parameter in PlainFastaHeaderParser">S</a> extends <a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="../org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html" title="type parameter in PlainFastaHeaderParser">C</a>&gt;,<wbr><a href="../org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html" title="type parameter in PlainFastaHeaderParser">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">The plain fasta header takes everything in the header as a single entity.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html#%3Cinit%3E()" class="member-name-link">PlainFastaHeaderParser()</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">PlainFastaHeaderParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNASequence.DNAType.html#PLASMID" class="member-name-link">PLASMID</a> - Enum constant in enum org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DNASequence.DNAType.html" title="enum in org.biojava.nbio.core.sequence">DNASequence.DNAType</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNASequence.DNAType.html#PLASTID" class="member-name-link">PLASTID</a> - Enum constant in enum org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DNASequence.DNAType.html" title="enum in org.biojava.nbio.core.sequence">DNASequence.DNAType</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/Point.html" class="type-name-link" title="interface in org.biojava.nbio.core.sequence.location.template">Point</a> - Interface in <a href="../org/biojava/nbio/core/sequence/location/template/package-summary.html">org.biojava.nbio.core.sequence.location.template</a></dt>
<dd>
<div class="block">Holds a single point part of a location</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/Point.Resolver.html" class="type-name-link" title="interface in org.biojava.nbio.core.sequence.location.template">Point.Resolver</a>&lt;<a href="../org/biojava/nbio/core/sequence/location/template/Point.Resolver.html" title="type parameter in Point.Resolver">T</a> extends <a href="../org/biojava/nbio/core/sequence/location/template/Point.html" title="interface in org.biojava.nbio.core.sequence.location.template">Point</a>&gt; - Interface in <a href="../org/biojava/nbio/core/sequence/location/template/package-summary.html">org.biojava.nbio.core.sequence.location.template</a></dt>
<dd>
<div class="block">Used to resolve a position about a point</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#populate(java.lang.String)" class="member-name-link">populate(String)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Loops through the chars in a String and passes them onto
 <a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#setCompoundAt(char,int)"><code>BitSequenceReader.BitArrayWorker.setCompoundAt(char, int)</code></a></div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#populate(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">populate(Sequence&lt;C&gt;)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Loops through the Compounds in a Sequence and passes them onto
 <a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#setCompoundAt(C,int)"><code>BitSequenceReader.BitArrayWorker.setCompoundAt(Compound, int)</code></a></div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACParserProduct2.html#populateLookups(org.biojava.nbio.core.sequence.io.IUPACParser)" class="member-name-link">populateLookups(IUPACParser)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/IUPACParserProduct2.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParserProduct2</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/Strand.html#POSITIVE" class="member-name-link">POSITIVE</a> - Enum constant in enum org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/Strand.html" title="enum in org.biojava.nbio.core.sequence">Strand</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.AnnotationType.html#PREDICTED" class="member-name-link">PREDICTED</a> - Enum constant in enum org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.AnnotationType.html" title="enum in org.biojava.nbio.core.sequence.template">AbstractSequence.AnnotationType</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/FileDownloadUtils.html#prepareURLConnection(java.lang.String,int)" class="member-name-link">prepareURLConnection(String, int)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/FileDownloadUtils.html" title="class in org.biojava.nbio.core.util">FileDownloadUtils</a></dt>
<dd>
<div class="block">Prepare <a href="https://docs.oracle.com/javase/8/docs/api/java/net/URLConnection.html" title="class or interface in java.net" class="external-link"><code>URLConnection</code></a> with customised timeouts.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/PrettyXMLWriter.html" class="type-name-link" title="class in org.biojava.nbio.core.util">PrettyXMLWriter</a> - Class in <a href="../org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>
<div class="block">Implementation of XMLWriter which emits nicely formatted documents
 to a PrintWriter.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/PrettyXMLWriter.html#%3Cinit%3E(java.io.PrintWriter)" class="member-name-link">PrettyXMLWriter(PrintWriter)</a> - Constructor for class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/PrettyXMLWriter.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/PrettyXMLWriterProduct.html" class="type-name-link" title="class in org.biojava.nbio.core.util">PrettyXMLWriterProduct</a> - Class in <a href="../org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/PrettyXMLWriterProduct.html#%3Cinit%3E()" class="member-name-link">PrettyXMLWriterProduct()</a> - Constructor for class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/PrettyXMLWriterProduct.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriterProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/PrettyXMLWriterProduct2.html" class="type-name-link" title="class in org.biojava.nbio.core.util">PrettyXMLWriterProduct2</a> - Class in <a href="../org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/PrettyXMLWriterProduct2.html#%3Cinit%3E()" class="member-name-link">PrettyXMLWriterProduct2()</a> - Constructor for class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/PrettyXMLWriterProduct2.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriterProduct2</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DataSource.html#PRF" class="member-name-link">PRF</a> - Enum constant in enum org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DataSource.html" title="enum in org.biojava.nbio.core.sequence">DataSource</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/Hashcoder.html#PRIME" class="member-name-link">PRIME</a> - Static variable in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/Hashcoder.html" title="class in org.biojava.nbio.core.util">Hashcoder</a></dt>
<dd>
<div class="block">The prime number used to multiply any calculated hashcode seed by

 i.e. result = PRIME*result + c

 Where result is the result of the previous calculation (at first this will
 be seed) and c is the calculated int to add to result</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/PrettyXMLWriter.html#print(java.lang.String)" class="member-name-link">print(String)</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/PrettyXMLWriter.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/XMLWriter.html#print(java.lang.String)" class="member-name-link">print(String)</a> - Method in interface org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/XMLWriter.html" title="interface in org.biojava.nbio.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Prints some textual content in an element.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html#print(java.lang.String,java.io.File)" class="member-name-link">print(String, File)</a> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Prints <code>string</code> to <code>file</code>.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/PrettyXMLWriterProduct.html#printAttributeValue(java.lang.String,java.io.PrintWriter)" class="member-name-link">printAttributeValue(String, PrintWriter)</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/PrettyXMLWriterProduct.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriterProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/PrettyXMLWriterProduct.html#printChars(java.lang.String,java.io.PrintWriter)" class="member-name-link">printChars(String, PrintWriter)</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/PrettyXMLWriterProduct.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriterProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/PrettyXMLWriter.html#println(java.lang.String)" class="member-name-link">println(String)</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/PrettyXMLWriter.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/XMLWriter.html#println(java.lang.String)" class="member-name-link">println(String)</a> - Method in interface org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/XMLWriter.html" title="interface in org.biojava.nbio.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Prints some textual content, terminated with a newline character.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/PrettyXMLWriter.html#printRaw(java.lang.String)" class="member-name-link">printRaw(String)</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/PrettyXMLWriter.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/XMLWriter.html#printRaw(java.lang.String)" class="member-name-link">printRaw(String)</a> - Method in interface org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/XMLWriter.html" title="interface in org.biojava.nbio.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Send raw data to the stream.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#process()" class="member-name-link">process()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaGeneWriter.html" title="class in org.biojava.nbio.core.sequence.io">FastaGeneWriter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaReader.html#process()" class="member-name-link">process()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaReader.html" title="class in org.biojava.nbio.core.sequence.io">FastaReader</a></dt>
<dd>
<div class="block">The parsing is done in this method.<br>
 This method tries to process all the available fasta records
 in the File or InputStream, closes the underlying resource,
 and return the results in <a href="https://docs.oracle.com/javase/8/docs/api/java/util/LinkedHashMap.html" title="class or interface in java.util" class="external-link"><code>LinkedHashMap</code></a>.<br>
 You don't need to call <a href="../org/biojava/nbio/core/sequence/io/FastaReader.html#close()"><code>FastaReader.close()</code></a> after calling this method.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaWriter.html#process()" class="member-name-link">process()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaWriter.html" title="class in org.biojava.nbio.core.sequence.io">FastaWriter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankReader.html#process()" class="member-name-link">process()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankReader.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReader</a></dt>
<dd>
<div class="block">The parsing is done in this method.<br>
 This method will return all the available Genbank records
 in the File or InputStream, closes the underlying resource,
 and return the results in <a href="https://docs.oracle.com/javase/8/docs/api/java/util/LinkedHashMap.html" title="class or interface in java.util" class="external-link"><code>LinkedHashMap</code></a>.<br>
 You don't need to call <a href="../org/biojava/nbio/core/sequence/io/GenbankReader.html#close()"><code>GenbankReader.close()</code></a> after calling this method.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankWriter.html#process()" class="member-name-link">process()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankWriter.html" title="class in org.biojava.nbio.core.sequence.io">GenbankWriter</a></dt>
<dd>
<div class="block">Allow an override of operating system line separator for programs that
 needs a specific CRLF or CR or LF option</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaReader.html#process(int)" class="member-name-link">process(int)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaReader.html" title="class in org.biojava.nbio.core.sequence.io">FastaReader</a></dt>
<dd>
<div class="block">This method tries to parse maximum <code>max</code> records from
 the open File or InputStream, and leaves the underlying resource open.<br>
 Subsequent calls to the same method continue parsing the rest of the file.<br>
 This is particularly useful when dealing with very big data files,
 (e.g.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankReader.html#process(int)" class="member-name-link">process(int)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankReader.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReader</a></dt>
<dd>
<div class="block">This method tries to parse maximum <code>max</code> records from
 the open File or InputStream, and leaves the underlying resource open.<br>

 Subsequent calls to the same method continue parsing the rest of the file.<br>
 This is particularly useful when dealing with very big data files,
 (e.g.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblReader.html#process(java.io.File)" class="member-name-link">process(File)</a> - Static method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblReader.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReader</a></dt>
<dd>
<div class="block">The parsing is done in this method.<br>
 This method tries to process all the Embl records
 in the File , closes the underlying resource,
 and return the results in object of EmblRecord.<br></div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.ReaderProcessor.html#process(java.lang.String)" class="member-name-link">process(String)</a> - Method in interface org.biojava.nbio.core.sequence.io.util.<a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.ReaderProcessor.html" title="interface in org.biojava.nbio.core.sequence.io.util">IOUtils.ReaderProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaReaderTest.html#processIntTest()" class="member-name-link">processIntTest()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaReaderTest.html" title="class in org.biojava.nbio.core.sequence.io">FastaReaderTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html#processReader(java.io.BufferedReader,org.biojava.nbio.core.sequence.io.util.IOUtils.ReaderProcessor)" class="member-name-link">processReader(BufferedReader, IOUtils.ReaderProcessor)</a> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Takes in a reader and a processor, reads every line from the given
 file and then invokes the processor.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.html" class="type-name-link" title="interface in org.biojava.nbio.core.alignment.template">Profile</a>&lt;<a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="type parameter in Profile">S</a> extends <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="type parameter in Profile">C</a>&gt;,<wbr><a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="type parameter in Profile">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="../org/biojava/nbio/core/alignment/template/package-summary.html">org.biojava.nbio.core.alignment.template</a></dt>
<dd>
<div class="block">Defines a data structure for the results of sequence alignment.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.StringFormat.html" class="type-name-link" title="enum in org.biojava.nbio.core.alignment.template">Profile.StringFormat</a> - Enum in <a href="../org/biojava/nbio/core/alignment/template/package-summary.html">org.biojava.nbio.core.alignment.template</a></dt>
<dd>
<div class="block">List of output formats.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/ProfilePair.html" class="type-name-link" title="interface in org.biojava.nbio.core.alignment.template">ProfilePair</a>&lt;<a href="../org/biojava/nbio/core/alignment/template/ProfilePair.html" title="type parameter in ProfilePair">S</a> extends <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="../org/biojava/nbio/core/alignment/template/ProfilePair.html" title="type parameter in ProfilePair">C</a>&gt;,<wbr><a href="../org/biojava/nbio/core/alignment/template/ProfilePair.html" title="type parameter in ProfilePair">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="../org/biojava/nbio/core/alignment/template/package-summary.html">org.biojava.nbio.core.alignment.template</a></dt>
<dd>
<div class="block">Defines a data structure for the results of the alignment of a pair of <a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template"><code>Profile</code></a>s.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/ProfileView.html" class="type-name-link" title="interface in org.biojava.nbio.core.alignment.template">ProfileView</a>&lt;<a href="../org/biojava/nbio/core/alignment/template/ProfileView.html" title="type parameter in ProfileView">S</a> extends <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="../org/biojava/nbio/core/alignment/template/ProfileView.html" title="type parameter in ProfileView">C</a>&gt;,<wbr><a href="../org/biojava/nbio/core/alignment/template/ProfileView.html" title="type parameter in ProfileView">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="../org/biojava/nbio/core/alignment/template/package-summary.html">org.biojava.nbio.core.alignment.template</a></dt>
<dd>
<div class="block">Defines a data structure for a view of sequence alignment.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html#proteinCreator(org.biojava.nbio.core.sequence.io.template.SequenceCreatorInterface)" class="member-name-link">proteinCreator(SequenceCreatorInterface&lt;AminoAcidCompound&gt;)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/ProteinSequence.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence">ProteinSequence</a> - Class in <a href="../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">The representation of a ProteinSequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/ProteinSequence.html#%3Cinit%3E(java.lang.String)" class="member-name-link">ProteinSequence(String)</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/ProteinSequence.html" title="class in org.biojava.nbio.core.sequence">ProteinSequence</a></dt>
<dd>
<div class="block">Create a protein from a string</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/ProteinSequence.html#%3Cinit%3E(java.lang.String,org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">ProteinSequence(String, CompoundSet&lt;AminoAcidCompound&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/ProteinSequence.html" title="class in org.biojava.nbio.core.sequence">ProteinSequence</a></dt>
<dd>
<div class="block">Create a protein from a string with a user defined set of amino acids</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/ProteinSequence.html#%3Cinit%3E(org.biojava.nbio.core.sequence.template.ProxySequenceReader)" class="member-name-link">ProteinSequence(ProxySequenceReader&lt;AminoAcidCompound&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/ProteinSequence.html" title="class in org.biojava.nbio.core.sequence">ProteinSequence</a></dt>
<dd>
<div class="block">A protein sequence where the storage of the sequence is somewhere else.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/ProteinSequence.html#%3Cinit%3E(org.biojava.nbio.core.sequence.template.ProxySequenceReader,org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">ProteinSequence(ProxySequenceReader&lt;AminoAcidCompound&gt;, CompoundSet&lt;AminoAcidCompound&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/ProteinSequence.html" title="class in org.biojava.nbio.core.sequence">ProteinSequence</a></dt>
<dd>
<div class="block">A protein sequence where the storage of the sequence is somewhere else
 with user defined set of amino acids.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">ProteinSequenceCreator</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">Used to create a ProteinSequence from a String to allow for details
 about the location of the sequence etc.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html#%3Cinit%3E(org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">ProteinSequenceCreator(CompoundSet&lt;AminoAcidCompound&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">ProteinSequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/ProteinSequenceTest.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence">ProteinSequenceTest</a> - Class in <a href="../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/ProteinSequenceTest.html#%3Cinit%3E()" class="member-name-link">ProteinSequenceTest()</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/ProteinSequenceTest.html" title="class in org.biojava.nbio.core.sequence">ProteinSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/ProxySequenceReader.html" class="type-name-link" title="interface in org.biojava.nbio.core.sequence.template">ProxySequenceReader</a>&lt;<a href="../org/biojava/nbio/core/sequence/template/ProxySequenceReader.html" title="type parameter in ProxySequenceReader">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="../org/biojava/nbio/core/sequence/template/package-summary.html">org.biojava.nbio.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/SoftHashMap.html#put(K,V)" class="member-name-link">put(K, V)</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SoftHashMap.html" title="class in org.biojava.nbio.core.util">SoftHashMap</a></dt>
<dd>
<div class="block">Here we put the key, value pair into the HashMap using a SoftValue
 object.</div>
</dd>
</dl>
<a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Z</a>&nbsp;<br><a href="../allclasses-index.html">All&nbsp;Classes&nbsp;and&nbsp;Interfaces</a><span class="vertical-separator">|</span><a href="../allpackages-index.html">All&nbsp;Packages</a><span class="vertical-separator">|</span><a href="../constant-values.html">Constant&nbsp;Field&nbsp;Values</a><span class="vertical-separator">|</span><a href="../serialized-form.html">Serialized&nbsp;Form</a></main>
</div>
</div>
</body>
</html>
