<!DOCTYPE HTML>
<html lang="pt">
<head>
<!-- Generated by javadoc (17) on Sun May 15 18:36:12 WEST 2022 -->
<title>E-Index</title>
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta name="dc.created" content="2022-05-15">
<meta name="description" content="index: E">
<meta name="generator" content="javadoc/IndexWriter">
<link rel="stylesheet" type="text/css" href="../stylesheet.css" title="Style">
<link rel="stylesheet" type="text/css" href="../script-dir/jquery-ui.min.css" title="Style">
<link rel="stylesheet" type="text/css" href="../jquery-ui.overrides.css" title="Style">
<script type="text/javascript" src="../script.js"></script>
<script type="text/javascript" src="../script-dir/jquery-3.5.1.min.js"></script>
<script type="text/javascript" src="../script-dir/jquery-ui.min.js"></script>
</head>
<body class="index-page">
<script type="text/javascript">var pathtoroot = "../";
loadScripts(document, 'script');</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<div class="flex-box">
<header role="banner" class="flex-header">
<nav role="navigation">
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="top-nav" id="navbar-top">
<div class="skip-nav"><a href="#skip-navbar-top" title="Skip navigation links">Skip navigation links</a></div>
<ul id="navbar-top-firstrow" class="nav-list" title="Navigation">
<li><a href="../index.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li>Use</li>
<li><a href="../overview-tree.html">Tree</a></li>
<li><a href="../deprecated-list.html">Deprecated</a></li>
<li class="nav-bar-cell1-rev">Index</li>
<li><a href="../help-doc.html#index">Help</a></li>
</ul>
</div>
<div class="sub-nav">
<div class="nav-list-search"><label for="search-input">SEARCH:</label>
<input type="text" id="search-input" value="search" disabled="disabled">
<input type="reset" id="reset-button" value="reset" disabled="disabled">
</div>
</div>
<!-- ========= END OF TOP NAVBAR ========= -->
<span class="skip-nav" id="skip-navbar-top"></span></nav>
</header>
<div class="flex-content">
<main role="main">
<div class="header">
<h1>Index</h1>
</div>
<a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Z</a>&nbsp;<br><a href="../allclasses-index.html">All&nbsp;Classes&nbsp;and&nbsp;Interfaces</a><span class="vertical-separator">|</span><a href="../allpackages-index.html">All&nbsp;Packages</a><span class="vertical-separator">|</span><a href="../constant-values.html">Constant&nbsp;Field&nbsp;Values</a><span class="vertical-separator">|</span><a href="../serialized-form.html">Serialized&nbsp;Form</a>
<h2 class="title" id="I:E">E</h2>
<dl class="index">
<dt><a href="../org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html#edit(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">edit(Sequence&lt;C&gt;)</a> - Method in class org.biojava.nbio.core.sequence.edits.<a href="../org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.AbstractEdit</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/edits/Edit.html#edit(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">edit(Sequence&lt;C&gt;)</a> - Method in interface org.biojava.nbio.core.sequence.edits.<a href="../org/biojava/nbio/core/sequence/edits/Edit.html" title="interface in org.biojava.nbio.core.sequence.edits">Edit</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/edits/Edit.html" class="type-name-link" title="interface in org.biojava.nbio.core.sequence.edits">Edit</a>&lt;<a href="../org/biojava/nbio/core/sequence/edits/Edit.html" title="type parameter in Edit">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="../org/biojava/nbio/core/sequence/edits/package-summary.html">org.biojava.nbio.core.sequence.edits</a></dt>
<dd>
<div class="block">Interface for carrying out edit operations on a Sequence.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.edits">Edit.AbstractEdit</a>&lt;<a href="../org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html" title="type parameter in Edit.AbstractEdit">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/edits/package-summary.html">org.biojava.nbio.core.sequence.edits</a></dt>
<dd>
<div class="block">Abstract class which defines all edit operations as a call to discover
 what 5' and 3' ends of an editing Sequence should be joined together
 with a target Sequence.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/edits/Edit.Delete.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.edits">Edit.Delete</a>&lt;<a href="../org/biojava/nbio/core/sequence/edits/Edit.Delete.html" title="type parameter in Edit.Delete">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/edits/package-summary.html">org.biojava.nbio.core.sequence.edits</a></dt>
<dd>
<div class="block">Implementation which allows for the deletion of bases from a Sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/edits/Edit.Insert.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.edits">Edit.Insert</a>&lt;<a href="../org/biojava/nbio/core/sequence/edits/Edit.Insert.html" title="type parameter in Edit.Insert">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/edits/package-summary.html">org.biojava.nbio.core.sequence.edits</a></dt>
<dd>
<div class="block">Edit implementation which allows us to insert a base at any position
 in a Sequence.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/edits/Edit.Substitute.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.edits">Edit.Substitute</a>&lt;<a href="../org/biojava/nbio/core/sequence/edits/Edit.Substitute.html" title="type parameter in Edit.Substitute">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/edits/package-summary.html">org.biojava.nbio.core.sequence.edits</a></dt>
<dd>
<div class="block">Allows for the substitution of bases into an existing Sequence.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/EditSequenceTest.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence">EditSequenceTest</a> - Class in <a href="../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/EditSequenceTest.html#%3Cinit%3E()" class="member-name-link">EditSequenceTest()</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/EditSequenceTest.html" title="class in org.biojava.nbio.core.sequence">EditSequenceTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblId.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io.embl">EmblId</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/embl/package-summary.html">org.biojava.nbio.core.sequence.io.embl</a></dt>
<dd>
<div class="block">This class contains the processed data of embl file
 Primary accession number
 Sequence version number
 Topology: 'circular' or 'linear'
 Molecule type
 Data class
 Taxonomic division
 Sequence length</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblId.html#%3Cinit%3E(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)" class="member-name-link">EmblId(String, String, String, String, String, String, String)</a> - Constructor for class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblId.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblId</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblReader.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReader</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/embl/package-summary.html">org.biojava.nbio.core.sequence.io.embl</a></dt>
<dd>
<div class="block">This class should process the data of embl file</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblReader.html#%3Cinit%3E()" class="member-name-link">EmblReader()</a> - Constructor for class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblReader.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblReaderTest.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReaderTest</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/embl/package-summary.html">org.biojava.nbio.core.sequence.io.embl</a></dt>
<dd>
<div class="block">This class should test the parser of EmblReader class</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblReaderTest.html#%3Cinit%3E()" class="member-name-link">EmblReaderTest()</a> - Constructor for class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblReaderTest.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReaderTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/embl/package-summary.html">org.biojava.nbio.core.sequence.io.embl</a></dt>
<dd>
<div class="block">this class contains the parsed data of embl file</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#%3Cinit%3E()" class="member-name-link">EmblRecord()</a> - Constructor for class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReference</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/embl/package-summary.html">org.biojava.nbio.core.sequence.io.embl</a></dt>
<dd>
<div class="block">This class contains the processed data of embl file that
 contains the referenceNumber, referenceComment, referencePosition
 referenceCrossReference, referenceGroup, referenceAuthor
 referenceTitle, referenceLocation</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html#%3Cinit%3E()" class="member-name-link">EmblReference()</a> - Constructor for class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReference</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/JoiningSequenceReaderTest.html#empty()" class="member-name-link">empty()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/JoiningSequenceReaderTest.html" title="class in org.biojava.nbio.core.sequence">JoiningSequenceReaderTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/Location.html#EMPTY" class="member-name-link">EMPTY</a> - Static variable in interface org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/Location.html" title="interface in org.biojava.nbio.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Basic location which is set to the minimum and maximum bounds of
 <a href="https://docs.oracle.com/javase/8/docs/api/java/lang/Integer.html" title="class or interface in java.lang" class="external-link"><code>Integer</code></a>.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DataSource.html#ENA" class="member-name-link">ENA</a> - Enum constant in enum org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DataSource.html" title="enum in org.biojava.nbio.core.sequence">DataSource</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/UncompressInputStreamData.html#end" class="member-name-link">end</a> - Variable in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/UncompressInputStreamData.html" title="class in org.biojava.nbio.core.util">UncompressInputStreamData</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/exceptions/Messages.html#ENDOFFILE" class="member-name-link">ENDOFFILE</a> - Static variable in class org.biojava.nbio.core.exceptions.<a href="../org/biojava/nbio/core/exceptions/Messages.html" title="class in org.biojava.nbio.core.exceptions">Messages</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/SoftHashMap.html#entrySet()" class="member-name-link">entrySet()</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SoftHashMap.html" title="class in org.biojava.nbio.core.util">SoftHashMap</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/UncompressInputStreamData.html#eof" class="member-name-link">eof</a> - Variable in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/UncompressInputStreamData.html" title="class in org.biojava.nbio.core.util">UncompressInputStreamData</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/Equals.html#equal(boolean,boolean)" class="member-name-link">equal(boolean, boolean)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/Equals.html" title="class in org.biojava.nbio.core.util">Equals</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/Equals.html#equal(int,int)" class="member-name-link">equal(int, int)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/Equals.html" title="class in org.biojava.nbio.core.util">Equals</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/Equals.html#equal(long,long)" class="member-name-link">equal(long, long)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/Equals.html" title="class in org.biojava.nbio.core.util">Equals</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/Equals.html#equal(java.lang.Object,java.lang.Object)" class="member-name-link">equal(Object, Object)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/Equals.html" title="class in org.biojava.nbio.core.util">Equals</a></dt>
<dd>
<div class="block">Does not compare class types.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/SequenceTools.html#equalLengthSequences(org.biojava.nbio.core.sequence.ProteinSequence%5B%5D)" class="member-name-link">equalLengthSequences(ProteinSequence[])</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SequenceTools.html" title="class in org.biojava.nbio.core.util">SequenceTools</a></dt>
<dd>
<div class="block">A method to check whether an array of sequences contains at least two sequences having an equal length.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#equals(java.lang.Object)" class="member-name-link">equals(Object)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hit.html#equals(java.lang.Object)" class="member-name-link">equals(Object)</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hit.html" title="class in org.biojava.nbio.core.search.io">Hit</a></dt>
<dd>
<div class="block">Implements conceptual comparisons of search results.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hsp.html#equals(java.lang.Object)" class="member-name-link">equals(Object)</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hsp.html" title="class in org.biojava.nbio.core.search.io">Hsp</a></dt>
<dd>
<div class="block">Experimental.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Result.html#equals(java.lang.Object)" class="member-name-link">equals(Object)</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Result.html" title="class in org.biojava.nbio.core.search.io">Result</a></dt>
<dd>
<div class="block">Experimental.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/AccessionID.html#equals(java.lang.Object)" class="member-name-link">equals(Object)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/AccessionID.html" title="class in org.biojava.nbio.core.sequence">AccessionID</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html#equals(java.lang.Object)" class="member-name-link">equals(Object)</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html#equals(java.lang.Object)" class="member-name-link">equals(Object)</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#equals(java.lang.Object)" class="member-name-link">equals(Object)</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound">NucleotideCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#equals(java.lang.Object)" class="member-name-link">equals(Object)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#equals(java.lang.Object)" class="member-name-link">equals(Object)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#equals(java.lang.Object)" class="member-name-link">equals(Object)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/FuzzyPoint.html#equals(java.lang.Object)" class="member-name-link">equals(Object)</a> - Method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/FuzzyPoint.html" title="class in org.biojava.nbio.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SimplePoint.html#equals(java.lang.Object)" class="member-name-link">equals(Object)</a> - Method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#equals(java.lang.Object)" class="member-name-link">equals(Object)</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#equals(java.lang.Object)" class="member-name-link">equals(Object)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#equals(java.lang.Object)" class="member-name-link">equals(Object)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#equals(java.lang.Object)" class="member-name-link">equals(Object)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#equals(java.lang.Object)" class="member-name-link">equals(Object)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#equals(java.lang.Object)" class="member-name-link">equals(Object)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html#equals(java.lang.Object)" class="member-name-link">equals(Object)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#equals(java.lang.Object)" class="member-name-link">equals(Object)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#equals(java.lang.Object)" class="member-name-link">equals(Object)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html#equals(java.lang.Object)" class="member-name-link">equals(Object)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#equals(java.lang.Object)" class="member-name-link">equals(Object)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html#equals(java.lang.Object)" class="member-name-link">equals(Object)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.CaseInsensitiveTriplet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html#equals(java.lang.Object)" class="member-name-link">equals(Object)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#equals(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">equals(Sequence&lt;C&gt;)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/Equals.html" class="type-name-link" title="class in org.biojava.nbio.core.util">Equals</a> - Class in <a href="../org/biojava/nbio/core/util/package-summary.html">org.biojava.nbio.core.util</a></dt>
<dd>
<div class="block">A set of helper methods which return true if the two parameters are
 equal to each other.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/Equals.html#%3Cinit%3E()" class="member-name-link">Equals()</a> - Constructor for class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/Equals.html" title="class in org.biojava.nbio.core.util">Equals</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html#equalsIgnoreCase(org.biojava.nbio.core.sequence.template.Compound)" class="member-name-link">equalsIgnoreCase(Compound)</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html#equalsIgnoreCase(org.biojava.nbio.core.sequence.template.Compound)" class="member-name-link">equalsIgnoreCase(Compound)</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#equalsIgnoreCase(org.biojava.nbio.core.sequence.template.Compound)" class="member-name-link">equalsIgnoreCase(Compound)</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound">NucleotideCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html#equalsIgnoreCase(org.biojava.nbio.core.sequence.template.Compound)" class="member-name-link">equalsIgnoreCase(Compound)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/Compound.html#equalsIgnoreCase(org.biojava.nbio.core.sequence.template.Compound)" class="member-name-link">equalsIgnoreCase(Compound)</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#equalsIgnoreCase(org.biojava.nbio.core.sequence.template.Compound)" class="member-name-link">equalsIgnoreCase(Compound)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html#equalsIgnoreCase(org.biojava.nbio.core.sequence.template.Compound)" class="member-name-link">equalsIgnoreCase(Compound)</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/StringManipulationHelper.html#equalsToIgnoreEndline(java.lang.String,java.lang.String)" class="member-name-link">equalsToIgnoreEndline(String, String)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/StringManipulationHelper.html" title="class in org.biojava.nbio.core.util">StringManipulationHelper</a></dt>
<dd>
<div class="block">Compares two strings in a case-sensitive manner for equality, line by line, ignoring any difference
 of end line delimiters contained within the 2 Strings.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/StringManipulationHelper.html#equalsToXml(java.lang.String,java.lang.String)" class="member-name-link">equalsToXml(String, String)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/StringManipulationHelper.html" title="class in org.biojava.nbio.core.util">StringManipulationHelper</a></dt>
<dd>
<div class="block">This method is not implemented or used, never returns true
 and should probably be removed.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#eutilBaseURL" class="member-name-link">eutilBaseURL</a> - Static variable in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/ExonComparator.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence">ExonComparator</a> - Class in <a href="../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">Sort Exon where it is a little confusing if exons should always be ordered left to right
 where a negative stranded gene should go the other direction.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/ExonComparator.html#%3Cinit%3E()" class="member-name-link">ExonComparator()</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/ExonComparator.html" title="class in org.biojava.nbio.core.sequence">ExonComparator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/ExonSequence.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence">ExonSequence</a> - Class in <a href="../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>
<div class="block">A gene contains a collection of Exon sequences</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/ExonSequence.html#%3Cinit%3E(org.biojava.nbio.core.sequence.GeneSequence,int,int)" class="member-name-link">ExonSequence(GeneSequence, int, int)</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/ExonSequence.html" title="class in org.biojava.nbio.core.sequence">ExonSequence</a></dt>
<dd>
<div class="block">Need a parent gene sequence and the bioBegin and bioEnd.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/FileDownloadUtils.html#expandUserHome(java.lang.String)" class="member-name-link">expandUserHome(String)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/FileDownloadUtils.html" title="class in org.biojava.nbio.core.util">FileDownloadUtils</a></dt>
<dd>
<div class="block">Expands ~ in paths to the user's home directory.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/InsdcParserTest.html#extractAccessionTest(java.lang.String,java.lang.String)" class="member-name-link">extractAccessionTest(String, String)</a> - Method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/InsdcParserTest.html" title="class in org.biojava.nbio.core.sequence.location">InsdcParserTest</a></dt>
<dd>
<div class="block">Test for issue #254</div>
</dd>
</dl>
<a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Z</a>&nbsp;<br><a href="../allclasses-index.html">All&nbsp;Classes&nbsp;and&nbsp;Interfaces</a><span class="vertical-separator">|</span><a href="../allpackages-index.html">All&nbsp;Packages</a><span class="vertical-separator">|</span><a href="../constant-values.html">Constant&nbsp;Field&nbsp;Values</a><span class="vertical-separator">|</span><a href="../serialized-form.html">Serialized&nbsp;Form</a></main>
</div>
</div>
</body>
</html>
