<!DOCTYPE HTML>
<html lang="pt">
<head>
<!-- Generated by javadoc (17) on Sun May 15 18:36:12 WEST 2022 -->
<title>G-Index</title>
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta name="dc.created" content="2022-05-15">
<meta name="description" content="index: G">
<meta name="generator" content="javadoc/IndexWriter">
<link rel="stylesheet" type="text/css" href="../stylesheet.css" title="Style">
<link rel="stylesheet" type="text/css" href="../script-dir/jquery-ui.min.css" title="Style">
<link rel="stylesheet" type="text/css" href="../jquery-ui.overrides.css" title="Style">
<script type="text/javascript" src="../script.js"></script>
<script type="text/javascript" src="../script-dir/jquery-3.5.1.min.js"></script>
<script type="text/javascript" src="../script-dir/jquery-ui.min.js"></script>
</head>
<body class="index-page">
<script type="text/javascript">var pathtoroot = "../";
loadScripts(document, 'script');</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<div class="flex-box">
<header role="banner" class="flex-header">
<nav role="navigation">
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="top-nav" id="navbar-top">
<div class="skip-nav"><a href="#skip-navbar-top" title="Skip navigation links">Skip navigation links</a></div>
<ul id="navbar-top-firstrow" class="nav-list" title="Navigation">
<li><a href="../index.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li>Use</li>
<li><a href="../overview-tree.html">Tree</a></li>
<li><a href="../deprecated-list.html">Deprecated</a></li>
<li class="nav-bar-cell1-rev">Index</li>
<li><a href="../help-doc.html#index">Help</a></li>
</ul>
</div>
<div class="sub-nav">
<div class="nav-list-search"><label for="search-input">SEARCH:</label>
<input type="text" id="search-input" value="search" disabled="disabled">
<input type="reset" id="reset-button" value="reset" disabled="disabled">
</div>
</div>
<!-- ========= END OF TOP NAVBAR ========= -->
<span class="skip-nav" id="skip-navbar-top"></span></nav>
</header>
<div class="flex-content">
<main role="main">
<div class="header">
<h1>Index</h1>
</div>
<a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Z</a>&nbsp;<br><a href="../allclasses-index.html">All&nbsp;Classes&nbsp;and&nbsp;Interfaces</a><span class="vertical-separator">|</span><a href="../allpackages-index.html">All&nbsp;Packages</a><span class="vertical-separator">|</span><a href="../constant-values.html">Constant&nbsp;Field&nbsp;Values</a><span class="vertical-separator">|</span><a href="../serialized-form.html">Serialized&nbsp;Form</a>
<h2 class="title" id="I:G">G</h2>
<dl class="index">
<dt><a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.Step.html#GAP" class="member-name-link">GAP</a> - Enum constant in enum org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.Step.html" title="enum in org.biojava.nbio.core.alignment.template">AlignedSequence.Step</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNATest.html#gc()" class="member-name-link">gc()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DNATest.html" title="class in org.biojava.nbio.core.sequence">DNATest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.StringFormat.html#GCG" class="member-name-link">GCG</a> - Enum constant in enum org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.StringFormat.html" title="enum in org.biojava.nbio.core.alignment.template">Profile.StringFormat</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/LightweightProfile.StringFormat.html#GCG" class="member-name-link">GCG</a> - Enum constant in enum org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/LightweightProfile.StringFormat.html" title="enum in org.biojava.nbio.core.sequence.template">LightweightProfile.StringFormat</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DataSource.html#GENBANK" class="member-name-link">GENBANK</a> - Enum constant in enum org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DataSource.html" title="enum in org.biojava.nbio.core.sequence">DataSource</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#GENBANK_FORMAT" class="member-name-link">GENBANK_FORMAT</a> - Static variable in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>
<div class="block">The name of this format</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankCookbookTest.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">GenbankCookbookTest</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankCookbookTest.html#%3Cinit%3E()" class="member-name-link">GenbankCookbookTest()</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankCookbookTest.html" title="class in org.biojava.nbio.core.sequence.io">GenbankCookbookTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#genbankDirectoryCache" class="member-name-link">genbankDirectoryCache</a> - Static variable in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html" class="type-name-link" title="interface in org.biojava.nbio.core.sequence.io.template">GenbankHeaderFormatInterface</a>&lt;<a href="../org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html" title="type parameter in GenbankHeaderFormatInterface">S</a> extends <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;?&gt;,<wbr><a href="../org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html" title="type parameter in GenbankHeaderFormatInterface">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Interface in <a href="../org/biojava/nbio/core/sequence/io/template/package-summary.html">org.biojava.nbio.core.sequence.io.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#genbankParser" class="member-name-link">genbankParser</a> - Variable in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReader</a>&lt;<a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="type parameter in GenbankProxySequenceReader">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/loader/package-summary.html">org.biojava.nbio.core.sequence.loader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#%3Cinit%3E(java.lang.String,java.lang.String,org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">GenbankProxySequenceReader(String, String, CompoundSet&lt;C&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#genbankProxySequenceReaderProduct" class="member-name-link">genbankProxySequenceReaderProduct</a> - Variable in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReaderProduct.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReaderProduct</a> - Class in <a href="../org/biojava/nbio/core/sequence/loader/package-summary.html">org.biojava.nbio.core.sequence.loader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReaderProduct.html#%3Cinit%3E()" class="member-name-link">GenbankProxySequenceReaderProduct()</a> - Constructor for class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReaderProduct.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReaderProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReaderTest.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReaderTest</a> - Class in <a href="../org/biojava/nbio/core/sequence/loader/package-summary.html">org.biojava.nbio.core.sequence.loader</a></dt>
<dd>
<div class="block">Testing example for issue #834</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReaderTest.html#%3Cinit%3E(java.lang.String)" class="member-name-link">GenbankProxySequenceReaderTest(String)</a> - Constructor for class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReaderTest.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReaderTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankReader.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">GenbankReader</a>&lt;<a href="../org/biojava/nbio/core/sequence/io/GenbankReader.html" title="type parameter in GenbankReader">S</a> extends <a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="../org/biojava/nbio/core/sequence/io/GenbankReader.html" title="type parameter in GenbankReader">C</a>&gt;,<wbr><a href="../org/biojava/nbio/core/sequence/io/GenbankReader.html" title="type parameter in GenbankReader">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">Use <a href="../org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html" title="class in org.biojava.nbio.core.sequence.io"><code>GenbankReaderHelper</code></a> as an example of how to use this class where <a href="../org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html" title="class in org.biojava.nbio.core.sequence.io"><code>GenbankReaderHelper</code></a> should be the
 primary class used to read Genbank files</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankReader.html#%3Cinit%3E(java.io.File,org.biojava.nbio.core.sequence.io.template.SequenceHeaderParserInterface,org.biojava.nbio.core.sequence.io.template.SequenceCreatorInterface)" class="member-name-link">GenbankReader(File, SequenceHeaderParserInterface&lt;S, C&gt;, SequenceCreatorInterface&lt;C&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankReader.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReader</a></dt>
<dd>
<div class="block">If you are going to use the FileProxyProteinSequenceCreator then you
 need to use this constructor because we need details about
 the location of the file.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankReader.html#%3Cinit%3E(java.io.InputStream,org.biojava.nbio.core.sequence.io.template.SequenceHeaderParserInterface,org.biojava.nbio.core.sequence.io.template.SequenceCreatorInterface)" class="member-name-link">GenbankReader(InputStream, SequenceHeaderParserInterface&lt;S, C&gt;, SequenceCreatorInterface&lt;C&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankReader.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReader</a></dt>
<dd>
<div class="block">If you are going to use <a href="../org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io"><code>FileProxyProteinSequenceCreator</code></a> then do not use this constructor because we need details about
 local file offsets for quick reads.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">GenbankReaderHelper</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html#%3Cinit%3E()" class="member-name-link">GenbankReaderHelper()</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReaderHelper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankReaderTest.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">GenbankReaderTest</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankReaderTest.html#%3Cinit%3E()" class="member-name-link">GenbankReaderTest()</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankReaderTest.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReaderTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/reference/GenbankReference.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.reference">GenbankReference</a> - Class in <a href="../org/biojava/nbio/core/sequence/reference/package-summary.html">org.biojava.nbio.core.sequence.reference</a></dt>
<dd>
<div class="block">For Genbank format file only.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/reference/GenbankReference.html#%3Cinit%3E()" class="member-name-link">GenbankReference()</a> - Constructor for class org.biojava.nbio.core.sequence.reference.<a href="../org/biojava/nbio/core/sequence/reference/GenbankReference.html" title="class in org.biojava.nbio.core.sequence.reference">GenbankReference</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a>&lt;<a href="../org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="type parameter in GenbankSequenceParser">S</a> extends <a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="../org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="type parameter in GenbankSequenceParser">C</a>&gt;,<wbr><a href="../org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="type parameter in GenbankSequenceParser">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#%3Cinit%3E()" class="member-name-link">GenbankSequenceParser()</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankWriter.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">GenbankWriter</a>&lt;<a href="../org/biojava/nbio/core/sequence/io/GenbankWriter.html" title="type parameter in GenbankWriter">S</a> extends <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;?&gt;,<wbr><a href="../org/biojava/nbio/core/sequence/io/GenbankWriter.html" title="type parameter in GenbankWriter">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankWriter.html#%3Cinit%3E(java.io.OutputStream,java.util.Collection,org.biojava.nbio.core.sequence.io.template.GenbankHeaderFormatInterface)" class="member-name-link">GenbankWriter(OutputStream, Collection&lt;S&gt;, GenbankHeaderFormatInterface&lt;S, C&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankWriter.html" title="class in org.biojava.nbio.core.sequence.io">GenbankWriter</a></dt>
<dd>
<div class="block">Use default line length of 60</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankWriter.html#%3Cinit%3E(java.io.OutputStream,java.util.Collection,org.biojava.nbio.core.sequence.io.template.GenbankHeaderFormatInterface,int)" class="member-name-link">GenbankWriter(OutputStream, Collection&lt;S&gt;, GenbankHeaderFormatInterface&lt;S, C&gt;, int)</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankWriter.html" title="class in org.biojava.nbio.core.sequence.io">GenbankWriter</a></dt>
<dd>
<div class="block">Set custom lineLength</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">GenbankWriterHelper</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">The class that should be used to write out genbank file of a sequence
 collection</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html#%3Cinit%3E()" class="member-name-link">GenbankWriterHelper()</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html" title="class in org.biojava.nbio.core.sequence.io">GenbankWriterHelper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankWriterTest.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">GenbankWriterTest</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankWriterTest.html#%3Cinit%3E()" class="member-name-link">GenbankWriterTest()</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankWriterTest.html" title="class in org.biojava.nbio.core.sequence.io">GenbankWriterTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DataSource.html#GENERAL" class="member-name-link">GENERAL</a> - Enum constant in enum org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DataSource.html" title="enum in org.biojava.nbio.core.sequence">DataSource</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">GenericFastaHeaderFormat</a>&lt;<a href="../org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.html" title="type parameter in GenericFastaHeaderFormat">S</a> extends <a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;?&gt;,<wbr><a href="../org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.html" title="type parameter in GenericFastaHeaderFormat">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">We store the original header if the sequence is parsed from a fasta file and will use that exact
 sequence if we write out the sequences to a fasta file.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.html#%3Cinit%3E()" class="member-name-link">GenericFastaHeaderFormat()</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.html" title="class in org.biojava.nbio.core.sequence.io">GenericFastaHeaderFormat</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">GenericFastaHeaderParser</a>&lt;<a href="../org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html" title="type parameter in GenericFastaHeaderParser">S</a> extends <a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="../org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html" title="type parameter in GenericFastaHeaderParser">C</a>&gt;,<wbr><a href="../org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html" title="type parameter in GenericFastaHeaderParser">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>
<div class="block">The default fasta header parser where some headers are well defined based on the source
 database which allows us to set the source of the protein sequence and the identifier
 that can be used in future implementations to load features from external sources

 If the user has a custom header with local data then they can create their own implementation
 of a FastaHeaderParserInterface</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html#%3Cinit%3E()" class="member-name-link">GenericFastaHeaderParser()</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">GenericFastaHeaderParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenericFastaHeaderParserTest.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">GenericFastaHeaderParserTest</a> - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenericFastaHeaderParserTest.html#%3Cinit%3E()" class="member-name-link">GenericFastaHeaderParserTest()</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenericFastaHeaderParserTest.html" title="class in org.biojava.nbio.core.sequence.io">GenericFastaHeaderParserTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">GenericGenbankHeaderFormat</a>&lt;<a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html" title="type parameter in GenericGenbankHeaderFormat">S</a> extends <a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html" title="type parameter in GenericGenbankHeaderFormat">C</a>&gt;,<wbr><a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html" title="type parameter in GenericGenbankHeaderFormat">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#%3Cinit%3E()" class="member-name-link">GenericGenbankHeaderFormat()</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html" title="class in org.biojava.nbio.core.sequence.io">GenericGenbankHeaderFormat</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#%3Cinit%3E(java.lang.String)" class="member-name-link">GenericGenbankHeaderFormat(String)</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html" title="class in org.biojava.nbio.core.sequence.io">GenericGenbankHeaderFormat</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">GenericGenbankHeaderParser</a>&lt;<a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" title="type parameter in GenericGenbankHeaderParser">S</a> extends <a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" title="type parameter in GenericGenbankHeaderParser">C</a>&gt;,<wbr><a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" title="type parameter in GenericGenbankHeaderParser">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html#%3Cinit%3E()" class="member-name-link">GenericGenbankHeaderParser()</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">GenericGenbankHeaderParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence.io">GenericInsdcHeaderFormat</a>&lt;<a href="../org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html" title="type parameter in GenericInsdcHeaderFormat">S</a> extends <a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="../org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html" title="type parameter in GenericInsdcHeaderFormat">C</a>&gt;,<wbr><a href="../org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html" title="type parameter in GenericInsdcHeaderFormat">C</a> extends <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt; - Class in <a href="../org/biojava/nbio/core/sequence/io/package-summary.html">org.biojava.nbio.core.sequence.io</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html#%3Cinit%3E()" class="member-name-link">GenericInsdcHeaderFormat()</a> - Constructor for class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html" title="class in org.biojava.nbio.core.sequence.io">GenericInsdcHeaderFormat</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/GeneSequence.html" class="type-name-link" title="class in org.biojava.nbio.core.sequence">GeneSequence</a> - Class in <a href="../org/biojava/nbio/core/sequence/package-summary.html">org.biojava.nbio.core.sequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/GeneSequence.html#%3Cinit%3E(org.biojava.nbio.core.sequence.ChromosomeSequence,int,int,org.biojava.nbio.core.sequence.Strand)" class="member-name-link">GeneSequence(ChromosomeSequence, int, int, Strand)</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>
<div class="deprecation-block"><span class="deprecated-label">Deprecated.</span>&nbsp;</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/GeneSequence.html#%3Cinit%3E(org.biojava.nbio.core.sequence.ChromosomeSequence,org.biojava.nbio.core.sequence.AccessionID,int,int,org.biojava.nbio.core.sequence.Strand)" class="member-name-link">GeneSequence(ChromosomeSequence, AccessionID, int, int, Strand)</a> - Constructor for class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">A class that keeps track of the details of a GeneSequence which is difficult to properly model.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DataSource.html#GENINFO" class="member-name-link">GENINFO</a> - Enum constant in enum org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DataSource.html" title="enum in org.biojava.nbio.core.sequence">DataSource</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/views/WindowedSequence.html#get(int)" class="member-name-link">get(int)</a> - Method in class org.biojava.nbio.core.sequence.views.<a href="../org/biojava/nbio/core/sequence/views/WindowedSequence.html" title="class in org.biojava.nbio.core.sequence.views">WindowedSequence</a></dt>
<dd>
<div class="block">Returns the window specified at the given index in offsets i.e. asking
 for position 2 in a moving window sequence of size 3 will get you
 the window starting at position 4.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/SoftHashMap.html#get(java.lang.Object)" class="member-name-link">get(Object)</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SoftHashMap.html" title="class in org.biojava.nbio.core.util">SoftHashMap</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/AAindexFactory.html#getAAIndexProvider()" class="member-name-link">getAAIndexProvider()</a> - Static method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/AAindexFactory.html" title="class in org.biojava.nbio.core.alignment.matrices">AAindexFactory</a></dt>
<dd>
<div class="block">Gets singleton instance of an <code>AAIndexProvider</code>, always non-null</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/ABITracerCompoundSet.html#getABITracerCompoundSet()" class="member-name-link">getABITracerCompoundSet()</a> - Static method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/ABITracerCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">ABITracerCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#getAccession()" class="member-name-link">getAccession()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#getAccession()" class="member-name-link">getAccession()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#getAccession()" class="member-name-link">getAccession()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getAccession()" class="member-name-link">getAccession()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#getAccession()" class="member-name-link">getAccession()</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#getAccession()" class="member-name-link">getAccession()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#getAccession()" class="member-name-link">getAccession()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#getAccession()" class="member-name-link">getAccession()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#getAccession()" class="member-name-link">getAccession()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Unsupoorted</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getAccession()" class="member-name-link">getAccession()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/Accessioned.html#getAccession()" class="member-name-link">getAccession()</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/Accessioned.html" title="interface in org.biojava.nbio.core.sequence.template">Accessioned</a></dt>
<dd>
<div class="block">Returns the AccessionID this location is currently bound with</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html#getAccession()" class="member-name-link">getAccession()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#getAccessionNumber()" class="member-name-link">getAccessionNumber()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>
<div class="block">The AC (Accession number) line lists the accession numbers associated with
 the entry.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getAccessions()" class="member-name-link">getAccessions()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">Pull uniprot accessions associated with this sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getAliases()" class="member-name-link">getAliases()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">Pull uniprot protein aliases associated with this sequence
 Provided for backwards compatibility now that we support both
 gene and protein aliases via separate methods.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html#getAlignedSequence(int)" class="member-name-link">getAlignedSequence(int)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.html#getAlignedSequence(int)" class="member-name-link">getAlignedSequence(int)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns <a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template"><code>AlignedSequence</code></a> at given index.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#getAlignedSequence(int)" class="member-name-link">getAlignedSequence(int)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava.nbio.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">Uses bioIndex starting at 1 instead of 0</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/LightweightProfile.html#getAlignedSequence(int)" class="member-name-link">getAlignedSequence(int)</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/LightweightProfile.html" title="interface in org.biojava.nbio.core.sequence.template">LightweightProfile</a></dt>
<dd>
<div class="block">Returns <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a> at given index.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html#getAlignedSequence(S)" class="member-name-link">getAlignedSequence(S)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.html#getAlignedSequence(S)" class="member-name-link">getAlignedSequence(S)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template">Profile</a></dt>
<dd>
<div class="block">Searches for the given <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a> within this alignment profile.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html#getAlignedSequences()" class="member-name-link">getAlignedSequences()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.html#getAlignedSequences()" class="member-name-link">getAlignedSequences()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns a <a href="https://docs.oracle.com/javase/8/docs/api/java/util/List.html" title="class or interface in java.util" class="external-link"><code>List</code></a> containing the individual <a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template"><code>AlignedSequence</code></a>s of this alignment.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#getAlignedSequences()" class="member-name-link">getAlignedSequences()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava.nbio.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">Get the list of sequences</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/LightweightProfile.html#getAlignedSequences()" class="member-name-link">getAlignedSequences()</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/LightweightProfile.html" title="interface in org.biojava.nbio.core.sequence.template">LightweightProfile</a></dt>
<dd>
<div class="block">Returns a <a href="https://docs.oracle.com/javase/8/docs/api/java/util/List.html" title="class or interface in java.util" class="external-link"><code>List</code></a> containing the individual <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a>s of this alignment.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html#getAlignedSequences(int...)" class="member-name-link">getAlignedSequences(int...)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.html#getAlignedSequences(int...)" class="member-name-link">getAlignedSequences(int...)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns a <a href="https://docs.oracle.com/javase/8/docs/api/java/util/List.html" title="class or interface in java.util" class="external-link"><code>List</code></a> containing some of the individual <a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template"><code>AlignedSequence</code></a>s of this alignment.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html#getAlignedSequences(S...)" class="member-name-link">getAlignedSequences(S...)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.html#getAlignedSequences(S...)" class="member-name-link">getAlignedSequences(S...)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns a <a href="https://docs.oracle.com/javase/8/docs/api/java/util/List.html" title="class or interface in java.util" class="external-link"><code>List</code></a> containing some of the individual <a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template"><code>AlignedSequence</code></a>s of this alignment.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hsp.html#getAlignment()" class="member-name-link">getAlignment()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hsp.html" title="class in org.biojava.nbio.core.search.io">Hsp</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#getAlignmentFromSequence()" class="member-name-link">getAlignmentFromSequence()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html#getAlignmentFromSequence()" class="member-name-link">getAlignmentFromSequence()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns the alignment.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#getAlignmentIndexAt(int)" class="member-name-link">getAlignmentIndexAt(int)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html#getAlignmentIndexAt(int)" class="member-name-link">getAlignmentIndexAt(int)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns the column index within an alignment corresponding to the given index in the original <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a>.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#getAllCompounds()" class="member-name-link">getAllCompounds()</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#getAllCompounds()" class="member-name-link">getAllCompounds()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/CompoundSet.html#getAllCompounds()" class="member-name-link">getAllCompounds()</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Frame.html#getAllFrames()" class="member-name-link">getAllFrames()</a> - Static method in enum org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Frame.html" title="enum in org.biojava.nbio.core.sequence.transcription">Frame</a></dt>
<dd>
<div class="block">Delegates to <a href="../org/biojava/nbio/core/sequence/transcription/Frame.html#values()"><code>Frame.values()</code></a></div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html#getAmbiguity(org.biojava.nbio.core.sequence.compound.NucleotideCompound...)" class="member-name-link">getAmbiguity(NucleotideCompound...)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractNucleotideCompoundSet</a></dt>
<dd>
<div class="block">Calculates the best symbol for a collection of compounds.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html#getAminoAcid()" class="member-name-link">getAminoAcid()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#getAminoAcidCompounds()" class="member-name-link">getAminoAcidCompounds()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#getAminoAcidCompoundSet()" class="member-name-link">getAminoAcidCompoundSet()</a> - Static method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html#getAminoAcidSubstitutionMatrix(java.lang.String)" class="member-name-link">getAminoAcidSubstitutionMatrix(String)</a> - Static method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.core.alignment.matrices">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns a substitution matrix for <a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava.nbio.core.sequence.compound"><code>amino acids</code></a> given by the name <code>name</code>.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getAnnotationType()" class="member-name-link">getAnnotationType()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#getAsList()" class="member-name-link">getAsList()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#getAsList()" class="member-name-link">getAsList()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#getAsList()" class="member-name-link">getAsList()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getAsList()" class="member-name-link">getAsList()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#getAsList()" class="member-name-link">getAsList()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#getAsList()" class="member-name-link">getAsList()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Returns this Sequence store as a List</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#getAsList()" class="member-name-link">getAsList()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#getAsList()" class="member-name-link">getAsList()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Delegates to <a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html#toList(org.biojava.nbio.core.sequence.template.Sequence)"><code>SequenceMixin.toList(org.biojava.nbio.core.sequence.template.Sequence)</code></a></div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getAsList()" class="member-name-link">getAsList()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/Sequence.html#getAsList()" class="member-name-link">getAsList()</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Returns the Sequence as a List of compounds</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html#getAsList()" class="member-name-link">getAsList()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#getAssemblyHeader()" class="member-name-link">getAssemblyHeader()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>
<div class="block">The AH (Assembly Header) line provides column headings for the assembly information.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#getAssemblyInformation()" class="member-name-link">getAssemblyInformation()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>
<div class="block">The AS (Assembly Information) lines provide information on the composition of
 a TPA or TSA sequence.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/reference/AbstractReference.html#getAuthors()" class="member-name-link">getAuthors()</a> - Method in class org.biojava.nbio.core.sequence.reference.<a href="../org/biojava/nbio/core/sequence/reference/AbstractReference.html" title="class in org.biojava.nbio.core.sequence.reference">AbstractReference</a></dt>
<dd>
<div class="block">The authors are a list of Inventors that retrieved from the Reference section.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/reference/GenbankReference.html#getAuthors()" class="member-name-link">getAuthors()</a> - Method in class org.biojava.nbio.core.sequence.reference.<a href="../org/biojava/nbio/core/sequence/reference/GenbankReference.html" title="class in org.biojava.nbio.core.sequence.reference">GenbankReference</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/reference/ReferenceInterface.html#getAuthors()" class="member-name-link">getAuthors()</a> - Method in interface org.biojava.nbio.core.sequence.reference.<a href="../org/biojava/nbio/core/sequence/reference/ReferenceInterface.html" title="interface in org.biojava.nbio.core.sequence.reference">ReferenceInterface</a></dt>
<dd>
<div class="block">Get the authors that retrieved from Reference section.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/views/WindowedSequence.html#getBackingSequence()" class="member-name-link">getBackingSequence()</a> - Method in class org.biojava.nbio.core.sequence.views.<a href="../org/biojava/nbio/core/sequence/views/WindowedSequence.html" title="class in org.biojava.nbio.core.sequence.views">WindowedSequence</a></dt>
<dd>
<div class="block">Access the sequence which backs this window</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html#getBase()" class="member-name-link">getBase()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/ABITrace.html#getBasecalls()" class="member-name-link">getBasecalls()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/ABITrace.html" title="class in org.biojava.nbio.core.sequence.io">ABITrace</a></dt>
<dd>
<div class="block">Returns an <code>int[]</code> array that represents the basecalls - each int in the
 array corresponds to an x-coordinate point in the graph that is a peak (a base location).</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getBioBegin()" class="member-name-link">getBioBegin()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getBioEnd()" class="member-name-link">getBioEnd()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html#getBioEnd()" class="member-name-link">getBioEnd()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceView.html#getBioEnd()" class="member-name-link">getBioEnd()</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceView.html" title="interface in org.biojava.nbio.core.sequence.template">SequenceView</a></dt>
<dd>
<div class="block">1-indexed, inclusive.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html#getBioStart()" class="member-name-link">getBioStart()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceView.html#getBioStart()" class="member-name-link">getBioStart()</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceView.html" title="interface in org.biojava.nbio.core.sequence.template">SequenceView</a></dt>
<dd>
<div class="block">1-indexed, inclusive.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html#getBlosum100()" class="member-name-link">getBlosum100()</a> - Static method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.core.alignment.matrices">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 100 matrix by Henikoff &amp; Henikoff</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html#getBlosum30()" class="member-name-link">getBlosum30()</a> - Static method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.core.alignment.matrices">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 30 matrix by Henikoff &amp; Henikoff</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html#getBlosum35()" class="member-name-link">getBlosum35()</a> - Static method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.core.alignment.matrices">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 35 matrix by Henikoff &amp; Henikoff</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html#getBlosum40()" class="member-name-link">getBlosum40()</a> - Static method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.core.alignment.matrices">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 40 matrix by Henikoff &amp; Henikoff</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html#getBlosum45()" class="member-name-link">getBlosum45()</a> - Static method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.core.alignment.matrices">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 45 matrix by Henikoff &amp; Henikoff</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html#getBlosum50()" class="member-name-link">getBlosum50()</a> - Static method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.core.alignment.matrices">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 50 matrix by Henikoff &amp; Henikoff</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html#getBlosum55()" class="member-name-link">getBlosum55()</a> - Static method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.core.alignment.matrices">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 55 matrix by Henikoff &amp; Henikoff</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html#getBlosum60()" class="member-name-link">getBlosum60()</a> - Static method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.core.alignment.matrices">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 60 matrix by Henikoff &amp; Henikoff</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html#getBlosum62()" class="member-name-link">getBlosum62()</a> - Static method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html#getBlosum62()" class="member-name-link">getBlosum62()</a> - Static method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.core.alignment.matrices">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 62 matrix by Henikoff &amp; Henikoff</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html#getBlosum65()" class="member-name-link">getBlosum65()</a> - Static method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.core.alignment.matrices">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 65 matrix by Henikoff &amp; Henikoff</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html#getBlosum70()" class="member-name-link">getBlosum70()</a> - Static method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.core.alignment.matrices">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 70 matrix by Henikoff &amp; Henikoff</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html#getBlosum75()" class="member-name-link">getBlosum75()</a> - Static method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.core.alignment.matrices">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 75 matrix by Henikoff &amp; Henikoff</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html#getBlosum80()" class="member-name-link">getBlosum80()</a> - Static method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.core.alignment.matrices">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 80 matrix by Henikoff &amp; Henikoff</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html#getBlosum85()" class="member-name-link">getBlosum85()</a> - Static method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.core.alignment.matrices">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 85 matrix by Henikoff &amp; Henikoff</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html#getBlosum90()" class="member-name-link">getBlosum90()</a> - Static method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.core.alignment.matrices">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 90 matrix by Henikoff &amp; Henikoff</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReaderProduct.html#getBufferedInputStream(java.lang.String,java.lang.String)" class="member-name-link">getBufferedInputStream(String, String)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReaderProduct.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReaderProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#getBufferedReader()" class="member-name-link">getBufferedReader()</a> - Method in class org.biojava.nbio.core.sequence.io.util.<a href="../org/biojava/nbio/core/sequence/io/util/ClasspathResource.html" title="class in org.biojava.nbio.core.sequence.io.util">ClasspathResource</a></dt>
<dd>
<div class="block">Returns the reader representation of this classpath resource</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html#getBytesRead()" class="member-name-link">getBytesRead()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava.nbio.core.sequence.io">BufferedReaderBytesRead</a></dt>
<dd>
<div class="block">Keep track of bytesread via ReadLine to account for CR-LF in the stream.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranscriptSequenceProduct.html#getCdsSequenceHashMap()" class="member-name-link">getCdsSequenceHashMap()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranscriptSequenceProduct.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequenceProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranscriptSequence.html#getCDSSequences()" class="member-name-link">getCDSSequences()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a></dt>
<dd>
<div class="block">Get the CDS sequences that have been added to the TranscriptSequences</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html#getChildrenFeatures()" class="member-name-link">getChildrenFeatures()</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">Get the children features</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#getChildrenFeatures()" class="member-name-link">getChildrenFeatures()</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html#getChildrenFeatures()" class="member-name-link">getChildrenFeatures()</a> - Method in interface org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Get the features contained by this feature</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/ChromosomeSequence.html#getChromosomeNumber()" class="member-name-link">getChromosomeNumber()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/ChromosomeSequence.html" title="class in org.biojava.nbio.core.sequence">ChromosomeSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/SingleLinkageClusterer.LinkedPair.html#getClosestDistance()" class="member-name-link">getClosestDistance()</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SingleLinkageClusterer.LinkedPair.html" title="class in org.biojava.nbio.core.util">SingleLinkageClusterer.LinkedPair</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/SingleLinkageClustererProduct.html#getClosestPair(org.biojava.nbio.core.util.SingleLinkageClusterer)" class="member-name-link">getClosestPair(SingleLinkageClusterer)</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SingleLinkageClustererProduct.html" title="class in org.biojava.nbio.core.util">SingleLinkageClustererProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/SingleLinkageClustererProductProduct.html#getClosestPair(org.biojava.nbio.core.util.SingleLinkageClusterer,double%5B%5D%5B%5D)" class="member-name-link">getClosestPair(SingleLinkageClusterer, double[][])</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SingleLinkageClustererProductProduct.html" title="class in org.biojava.nbio.core.util">SingleLinkageClustererProductProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/SingleLinkageClusterer.html#getClusters(double)" class="member-name-link">getClusters(double)</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SingleLinkageClusterer.html" title="class in org.biojava.nbio.core.util">SingleLinkageClusterer</a></dt>
<dd>
<div class="block">Get the clusters by cutting the dendrogram at given cutoff</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/CDSSequence.html#getCodingSequence()" class="member-name-link">getCodingSequence()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/CDSSequence.html" title="class in org.biojava.nbio.core.sequence">CDSSequence</a></dt>
<dd>
<div class="block">A CDS sequence if negative stranded needs to be reverse complement
 to represent the actual coding sequence.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html#getCodonCompoundSet(org.biojava.nbio.core.sequence.template.CompoundSet,org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">getCodonCompoundSet(CompoundSet&lt;NucleotideCompound&gt;, CompoundSet&lt;AminoAcidCompound&gt;)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser.IUPACTable</a></dt>
<dd>
<div class="block">Returns the compound set of codons</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.html#getCodonCompoundSet(org.biojava.nbio.core.sequence.template.CompoundSet,org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">getCodonCompoundSet(CompoundSet&lt;NucleotideCompound&gt;, CompoundSet&lt;AminoAcidCompound&gt;)</a> - Method in interface org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.html" title="interface in org.biojava.nbio.core.sequence.transcription">Table</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html#getCodons(org.biojava.nbio.core.sequence.template.CompoundSet,org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">getCodons(CompoundSet&lt;NucleotideCompound&gt;, CompoundSet&lt;AminoAcidCompound&gt;)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser.IUPACTable</a></dt>
<dd>
<div class="block">Returns a list of codons where the source and target compounds
 are the same as those given by the parameters.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.html#getCodons(org.biojava.nbio.core.sequence.template.CompoundSet,org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">getCodons(CompoundSet&lt;NucleotideCompound&gt;, CompoundSet&lt;AminoAcidCompound&gt;)</a> - Method in interface org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.html" title="interface in org.biojava.nbio.core.sequence.transcription">Table</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html#getCols()" class="member-name-link">getCols()</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html#getColumn(C)" class="member-name-link">getColumn(C)</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/SubstitutionMatrix.html#getColumn(C)" class="member-name-link">getColumn(C)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava.nbio.core.alignment.template">SubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html#getColumn(org.biojava.nbio.core.sequence.compound.AminoAcidCompound)" class="member-name-link">getColumn(AminoAcidCompound)</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html#getComment()" class="member-name-link">getComment()</a> - Static method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getComments()" class="member-name-link">getComments()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#getComplement()" class="member-name-link">getComplement()</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound">NucleotideCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNASequence.html#getComplement()" class="member-name-link">getComplement()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Returns a Sequence which will complement every base</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/RNASequence.html#getComplement()" class="member-name-link">getComplement()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/RNASequence.html" title="class in org.biojava.nbio.core.sequence">RNASequence</a></dt>
<dd>
<div class="block">Get the complement view of the RNA sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/ComplementCompound.html#getComplement()" class="member-name-link">getComplement()</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/ComplementCompound.html" title="interface in org.biojava.nbio.core.sequence.template">ComplementCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html#getComposition(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">getComposition(Sequence&lt;C&gt;)</a> - Static method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Does a linear scan over the given Sequence and records the number of
 times each base appears.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#getCompoundAt(int)" class="member-name-link">getCompoundAt(int)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#getCompoundAt(int)" class="member-name-link">getCompoundAt(int)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#getCompoundAt(int)" class="member-name-link">getCompoundAt(int)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getCompoundAt(int)" class="member-name-link">getCompoundAt(int)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#getCompoundAt(int)" class="member-name-link">getCompoundAt(int)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#getCompoundAt(int)" class="member-name-link">getCompoundAt(int)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Returns the compound at the specified biological index</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#getCompoundAt(int)" class="member-name-link">getCompoundAt(int)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Returns the compound at the specified biological index</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#getCompoundAt(int)" class="member-name-link">getCompoundAt(int)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#getCompoundAt(int)" class="member-name-link">getCompoundAt(int)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Always returns the compound given at construction</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getCompoundAt(int)" class="member-name-link">getCompoundAt(int)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/Sequence.html#getCompoundAt(int)" class="member-name-link">getCompoundAt(int)</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Returns the Compound at the given biological index</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html#getCompoundAt(int)" class="member-name-link">getCompoundAt(int)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/views/ComplementSequenceView.html#getCompoundAt(int)" class="member-name-link">getCompoundAt(int)</a> - Method in class org.biojava.nbio.core.sequence.views.<a href="../org/biojava/nbio/core/sequence/views/ComplementSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">ComplementSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/views/ReversedSequenceView.html#getCompoundAt(int)" class="member-name-link">getCompoundAt(int)</a> - Method in class org.biojava.nbio.core.sequence.views.<a href="../org/biojava/nbio/core/sequence/views/ReversedSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">ReversedSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/views/RnaSequenceView.html#getCompoundAt(int)" class="member-name-link">getCompoundAt(int)</a> - Method in class org.biojava.nbio.core.sequence.views.<a href="../org/biojava/nbio/core/sequence/views/RnaSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html#getCompoundAt(int,int)" class="member-name-link">getCompoundAt(int, int)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.html#getCompoundAt(int,int)" class="member-name-link">getCompoundAt(int, int)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Compound</code></a> at row of given sequence and column of alignment index.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html#getCompoundAt(S,int)" class="member-name-link">getCompoundAt(S, int)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.html#getCompoundAt(S,int)" class="member-name-link">getCompoundAt(S, int)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Compound</code></a> at row of given sequence and column of alignment index.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html#getCompoundCountsAt(int)" class="member-name-link">getCompoundCountsAt(int)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.html#getCompoundCountsAt(int)" class="member-name-link">getCompoundCountsAt(int)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the number of each <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Compound</code></a> in the given column for all compounds in <a href="../org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template"><code>CompoundSet</code></a>.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html#getCompoundCountsAt(int,java.util.List)" class="member-name-link">getCompoundCountsAt(int, List&lt;C&gt;)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.html#getCompoundCountsAt(int,java.util.List)" class="member-name-link">getCompoundCountsAt(int, List&lt;C&gt;)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the number of each <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Compound</code></a> in the given column only for compounds in the given list.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#getCompoundForString(java.lang.String)" class="member-name-link">getCompoundForString(String)</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#getCompoundForString(java.lang.String)" class="member-name-link">getCompoundForString(String)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/CompoundSet.html#getCompoundForString(java.lang.String)" class="member-name-link">getCompoundForString(String)</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundSet</a></dt>
<dd>
<div class="block">Return null if not recognised.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html#getCompoundInQueryAt(int)" class="member-name-link">getCompoundInQueryAt(int)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/SequencePair.html#getCompoundInQueryAt(int)" class="member-name-link">getCompoundInQueryAt(int)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/SequencePair.html" title="interface in org.biojava.nbio.core.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Compound</code></a> in query sequence at given column index in alignment.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html#getCompoundInTargetAt(int)" class="member-name-link">getCompoundInTargetAt(int)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/SequencePair.html#getCompoundInTargetAt(int)" class="member-name-link">getCompoundInTargetAt(int)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/SequencePair.html" title="interface in org.biojava.nbio.core.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Compound</code></a> in target sequence at given column index in alignment.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/views/WindowedSequence.html#getCompounds(int)" class="member-name-link">getCompounds(int)</a> - Method in class org.biojava.nbio.core.sequence.views.<a href="../org/biojava/nbio/core/sequence/views/WindowedSequence.html" title="class in org.biojava.nbio.core.sequence.views">WindowedSequence</a></dt>
<dd>
<div class="block">For a given position into the windowed view this will return those
 compounds we can see in the window. i.e. in the sequence AGGCCT requesting
 index 1 returns AGG and requesting index 2 return CCT.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html#getCompoundsAt(int)" class="member-name-link">getCompoundsAt(int)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.html#getCompoundsAt(int)" class="member-name-link">getCompoundsAt(int)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Compound</code></a> elements of the original <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a>s at the given column.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#getCompoundsAt(int)" class="member-name-link">getCompoundsAt(int)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava.nbio.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">Get a list of compounds at a sequence position</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/LightweightProfile.html#getCompoundsAt(int)" class="member-name-link">getCompoundsAt(int)</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/LightweightProfile.html" title="interface in org.biojava.nbio.core.sequence.template">LightweightProfile</a></dt>
<dd>
<div class="block">Returns the <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Compound</code></a> elements of the original <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a>s at the given column.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html#getCompoundSet()" class="member-name-link">getCompoundSet()</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html#getCompoundSet()" class="member-name-link">getCompoundSet()</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#getCompoundSet()" class="member-name-link">getCompoundSet()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html#getCompoundSet()" class="member-name-link">getCompoundSet()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.html#getCompoundSet()" class="member-name-link">getCompoundSet()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns <a href="../org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template"><code>CompoundSet</code></a> of all <a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template"><code>AlignedSequence</code></a>s</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/SubstitutionMatrix.html#getCompoundSet()" class="member-name-link">getCompoundSet()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava.nbio.core.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns the <a href="../org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template"><code>CompoundSet</code></a> on which the matrix is defined.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html#getCompoundSet()" class="member-name-link">getCompoundSet()</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#getCompoundSet()" class="member-name-link">getCompoundSet()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#getCompoundSet()" class="member-name-link">getCompoundSet()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getCompoundSet()" class="member-name-link">getCompoundSet()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#getCompoundSet()" class="member-name-link">getCompoundSet()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava.nbio.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">Get the Compounds defined in the first sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#getCompoundSet()" class="member-name-link">getCompoundSet()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#getCompoundSet()" class="member-name-link">getCompoundSet()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Returns the compound set backing this store</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#getCompoundSet()" class="member-name-link">getCompoundSet()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Returns the compound set backing this store</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#getCompoundSet()" class="member-name-link">getCompoundSet()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#getCompoundSet()" class="member-name-link">getCompoundSet()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Returns the compound set given at construction</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getCompoundSet()" class="member-name-link">getCompoundSet()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/LightweightProfile.html#getCompoundSet()" class="member-name-link">getCompoundSet()</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/LightweightProfile.html" title="interface in org.biojava.nbio.core.sequence.template">LightweightProfile</a></dt>
<dd>
<div class="block">Returns <a href="../org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template"><code>CompoundSet</code></a> of all <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a>s</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/Sequence.html#getCompoundSet()" class="member-name-link">getCompoundSet()</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Gets the compound set used to back this Sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html#getCompoundSet()" class="member-name-link">getCompoundSet()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#getCompoundType()" class="member-name-link">getCompoundType()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html#getCompoundWeightsAt(int)" class="member-name-link">getCompoundWeightsAt(int)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.html#getCompoundWeightsAt(int)" class="member-name-link">getCompoundWeightsAt(int)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the fraction of each <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Compound</code></a> in the given column for all compounds in <a href="../org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template"><code>CompoundSet</code></a>.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html#getCompoundWeightsAt(int,java.util.List)" class="member-name-link">getCompoundWeightsAt(int, List&lt;C&gt;)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.html#getCompoundWeightsAt(int,java.util.List)" class="member-name-link">getCompoundWeightsAt(int, List&lt;C&gt;)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the fraction of each <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Compound</code></a> in the given column only for compounds in the given list.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#getConstituents()" class="member-name-link">getConstituents()</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound">NucleotideCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#getConstructedSequence()" class="member-name-link">getConstructedSequence()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>
<div class="block">Con(structed) sequences in the CON data classes represent complete
 chromosomes, genomes and other long sequences constructed from segment entries.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#getCoverage()" class="member-name-link">getCoverage()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html#getCoverage()" class="member-name-link">getCoverage()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns the coverage, as a fraction between 0 and 1, of this <a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template"><code>AlignedSequence</code></a> with respect to the original sequence.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#getCreatedDate()" class="member-name-link">getCreatedDate()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>
<div class="block">The DT  line shows when an entry first appeared in the database</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#getCreator()" class="member-name-link">getCreator()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#getDatabase()" class="member-name-link">getDatabase()</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">DBReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#getDatabaseCrossReference()" class="member-name-link">getDatabaseCrossReference()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>
<div class="block">The DR (Database Cross-reference) line cross-references other databases which
 contain information related to the entry in which the DR line appears.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserProduct.html#getDatabaseReferences()" class="member-name-link">getDatabaseReferences()</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserProduct.html" title="class in org.biojava.nbio.core.search.io.blast">BlastXMLParserProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/DatabaseReferenceInterface.html#getDatabaseReferences()" class="member-name-link">getDatabaseReferences()</a> - Method in interface org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/DatabaseReferenceInterface.html" title="interface in org.biojava.nbio.core.sequence.features">DatabaseReferenceInterface</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#getDatabaseReferences()" class="member-name-link">getDatabaseReferences()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#getDatabaseReferences()" class="member-name-link">getDatabaseReferences()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getDatabaseReferences()" class="member-name-link">getDatabaseReferences()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">The Uniprot mappings to other database identifiers for this sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getDatabaseReferences()" class="member-name-link">getDatabaseReferences()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserProduct.html#getDatabaseReferencesMap()" class="member-name-link">getDatabaseReferencesMap()</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserProduct.html" title="class in org.biojava.nbio.core.search.io.blast">BlastXMLParserProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblId.html#getDataClass()" class="member-name-link">getDataClass()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblId.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblId</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/AccessionID.html#getDataSource()" class="member-name-link">getDataSource()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/AccessionID.html" title="class in org.biojava.nbio.core.sequence">AccessionID</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/InsdcParser.html#getDataSource()" class="member-name-link">getDataSource()</a> - Method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/InsdcParser.html" title="class in org.biojava.nbio.core.sequence.location">InsdcParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TaxonomyID.html#getDataSource()" class="member-name-link">getDataSource()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TaxonomyID.html" title="class in org.biojava.nbio.core.sequence">TaxonomyID</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Result.html#getDbFile()" class="member-name-link">getDbFile()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Result.html" title="class in org.biojava.nbio.core.search.io">Result</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#getDefault()" class="member-name-link">getDefault()</a> - Static method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>
<div class="block">Default instance to use when Transcribing from DNA -&gt; RNA -&gt;
 Protein.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Frame.html#getDefaultFrame()" class="member-name-link">getDefaultFrame()</a> - Static method in enum org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Frame.html" title="enum in org.biojava.nbio.core.sequence.transcription">Frame</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/SingleLinkageClusterer.html#getDendrogram()" class="member-name-link">getDendrogram()</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SingleLinkageClusterer.html" title="class in org.biojava.nbio.core.util">SingleLinkageClusterer</a></dt>
<dd>
<div class="block">Get the full dendrogram (size n-1) result of the hierarchical clustering</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/SingleLinkageClusterer.html#getDendrogram2()" class="member-name-link">getDendrogram2()</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SingleLinkageClusterer.html" title="class in org.biojava.nbio.core.util">SingleLinkageClusterer</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html#getDescription()" class="member-name-link">getDescription()</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html#getDescription()" class="member-name-link">getDescription()</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/SubstitutionMatrix.html#getDescription()" class="member-name-link">getDescription()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava.nbio.core.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns the description of this matrix.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html#getDescription()" class="member-name-link">getDescription()</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html#getDescription()" class="member-name-link">getDescription()</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#getDescription()" class="member-name-link">getDescription()</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html#getDescription()" class="member-name-link">getDescription()</a> - Method in interface org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Get the description that can be used to describe the feature</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html#getDescription()" class="member-name-link">getDescription()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getDescription()" class="member-name-link">getDescription()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/Compound.html#getDescription()" class="member-name-link">getDescription()</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#getDescription()" class="member-name-link">getDescription()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html#getDescription()" class="member-name-link">getDescription()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/SingleLinkageClustererProduct.html#getDistance(int,int)" class="member-name-link">getDistance(int, int)</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SingleLinkageClustererProduct.html" title="class in org.biojava.nbio.core.util">SingleLinkageClustererProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html#getDistribution(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">getDistribution(Sequence&lt;C&gt;)</a> - Static method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Analogous to <a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html#getComposition(org.biojava.nbio.core.sequence.template.Sequence)"><code>SequenceMixin.getComposition(Sequence)</code></a> but returns the
 distribution of that <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Compound</code></a> over the given sequence.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranscriptSequence.html#getDNACodingSequence()" class="member-name-link">getDNACodingSequence()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a></dt>
<dd>
<div class="block">Get the stitched together CDS sequences then maps to the cDNA</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranscriptSequenceProduct.html#getDNACodingSequence(org.biojava.nbio.core.sequence.TranscriptSequence)" class="member-name-link">getDNACodingSequence(TranscriptSequence)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranscriptSequenceProduct.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequenceProduct</a></dt>
<dd>
<div class="block">Get the stitched together CDS sequences then maps to the cDNA</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#getDnaCompounds()" class="member-name-link">getDnaCompounds()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AmbiguityDNACompoundSet.html#getDNACompoundSet()" class="member-name-link">getDNACompoundSet()</a> - Static method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/AmbiguityDNACompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AmbiguityDNACompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/DNACompoundSet.html#getDNACompoundSet()" class="member-name-link">getDNACompoundSet()</a> - Static method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/DNACompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">DNACompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AmbiguityDNARNAHybridCompoundSet.html#getDNARNAHybridCompoundSet()" class="member-name-link">getDNARNAHybridCompoundSet()</a> - Static method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/AmbiguityDNARNAHybridCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AmbiguityDNARNAHybridCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#getDnaRnaTranslator()" class="member-name-link">getDnaRnaTranslator()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/views/RnaSequenceView.html#getDnaToRna()" class="member-name-link">getDnaToRna()</a> - Method in class org.biojava.nbio.core.sequence.views.<a href="../org/biojava/nbio/core/sequence/views/RnaSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNASequence.html#getDNAType()" class="member-name-link">getDNAType()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#getEmblId()" class="member-name-link">getEmblId()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>
<div class="block">The ID (IDentification) line
 The tokens represent:
 1.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#getEmblReference()" class="member-name-link">getEmblReference()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>
<div class="block">The Reference (RN, RC, RP, RX, RG, RA, RT, RL) Lines
 These lines comprise the literature citations within the database.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#getEnd()" class="member-name-link">getEnd()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html#getEnd()" class="member-name-link">getEnd()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns the <a href="../org/biojava/nbio/core/sequence/location/template/Point.html" title="interface in org.biojava.nbio.core.sequence.location.template"><code>Point</code></a> within an alignment of the last element of the original <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a>.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/ProfileView.html#getEnd()" class="member-name-link">getEnd()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/ProfileView.html" title="interface in org.biojava.nbio.core.alignment.template">ProfileView</a></dt>
<dd>
<div class="block">Returns the column index of the viewed <a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template"><code>Profile</code></a> corresponding to the final element in this view</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html#getEnd()" class="member-name-link">getEnd()</a> - Method in class org.biojava.nbio.core.sequence.edits.<a href="../org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.AbstractEdit</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#getEnd()" class="member-name-link">getEnd()</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/Location.html#getEnd()" class="member-name-link">getEnd()</a> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/Location.html" title="interface in org.biojava.nbio.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">End of the location</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/edits/Edit.Substitute.html#getEnd(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">getEnd(Sequence&lt;C&gt;)</a> - Method in class org.biojava.nbio.core.sequence.edits.<a href="../org/biojava/nbio/core/sequence/edits/Edit.Substitute.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Substitute</a></dt>
<dd>
<div class="block">Must use this rather than the no-args getEnd as this can return
 -1 and the length of a sub is dependent on the length of the
 Sequence; we cannot assume 1:1 mapping between characters in a
 String and the number of compounds we will have to insert.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#getEquivalentCompounds(C)" class="member-name-link">getEquivalentCompounds(C)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/CompoundSet.html#getEquivalentCompounds(C)" class="member-name-link">getEquivalentCompounds(C)</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#getEquivalentCompounds(org.biojava.nbio.core.sequence.compound.AminoAcidCompound)" class="member-name-link">getEquivalentCompounds(AminoAcidCompound)</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReaderProduct.html#getEutilsInputStream(java.lang.String,java.lang.String)" class="member-name-link">getEutilsInputStream(String, String)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReaderProduct.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReaderProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/SearchIO.html#getEvalueThreshold()" class="member-name-link">getEvalueThreshold()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/SearchIO.html" title="class in org.biojava.nbio.core.search.io">SearchIO</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReaderTest.html#getExamples()" class="member-name-link">getExamples()</a> - Static method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReaderTest.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReaderTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/GeneSequence.html#getExonSequences()" class="member-name-link">getExonSequences()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Get the exons as an ArrayList.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#getFeatureHeader()" class="member-name-link">getFeatureHeader()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>
<div class="block">The FH (Feature Header) lines are present only to improve readability of
 an entry when it is printed or displayed on a terminal screen.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getFeatureRetriever()" class="member-name-link">getFeatureRetriever()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureRetriever.html#getFeatures()" class="member-name-link">getFeatures()</a> - Method in interface org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureRetriever.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureRetriever</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#getFeatures()" class="member-name-link">getFeatures()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#getFeatures()" class="member-name-link">getFeatures()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getFeatures()" class="member-name-link">getFeatures()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getFeatures(int)" class="member-name-link">getFeatures(int)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Return features at a sequence position</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#getFeatures(java.lang.String)" class="member-name-link">getFeatures(String)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getFeatures(java.lang.String,int)" class="member-name-link">getFeatures(String, int)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Return features at a sequence position by type</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getFeaturesByType(java.lang.String)" class="member-name-link">getFeaturesByType(String)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getFeaturesKeyWord()" class="member-name-link">getFeaturesKeyWord()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#getFeatureTable()" class="member-name-link">getFeatureTable()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>
<div class="block">The FT (Feature Table) lines provide a mechanism for the annotation of the
 sequence data.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/FileDownloadUtils.html#getFileExtension(java.io.File)" class="member-name-link">getFileExtension(File)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/FileDownloadUtils.html" title="class in org.biojava.nbio.core.util">FileDownloadUtils</a></dt>
<dd>
<div class="block">Gets the file extension of a file, excluding '.'.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastTabularParser.html#getFileExtensions()" class="member-name-link">getFileExtensions()</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastTabularParser.html" title="class in org.biojava.nbio.core.search.io.blast">BlastTabularParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParser.html#getFileExtensions()" class="member-name-link">getFileExtensions()</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParser.html" title="class in org.biojava.nbio.core.search.io.blast">BlastXMLParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/ResultFactory.html#getFileExtensions()" class="member-name-link">getFileExtensions()</a> - Method in interface org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/ResultFactory.html" title="interface in org.biojava.nbio.core.search.io">ResultFactory</a></dt>
<dd>
<div class="block">returns a list of file extensions associated to this ResultFactory</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/FileDownloadUtils.html#getFilePrefix(java.io.File)" class="member-name-link">getFilePrefix(File)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/FileDownloadUtils.html" title="class in org.biojava.nbio.core.util">FileDownloadUtils</a></dt>
<dd>
<div class="block">Gets the file name up to and excluding the first
 '.' character.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/SingleLinkageClusterer.LinkedPair.html#getFirst()" class="member-name-link">getFirst()</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SingleLinkageClusterer.LinkedPair.html" title="class in org.biojava.nbio.core.util">SingleLinkageClusterer.LinkedPair</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Frame.html#getForwardFrames()" class="member-name-link">getForwardFrames()</a> - Static method in enum org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Frame.html" title="enum in org.biojava.nbio.core.sequence.transcription">Frame</a></dt>
<dd>
<div class="block">Returns all frames in the forward orientation</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#getFromCompoundSet()" class="member-name-link">getFromCompoundSet()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/RNASequence.html#getGC()" class="member-name-link">getGC()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/RNASequence.html" title="class in org.biojava.nbio.core.sequence">RNASequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNASequence.html#getGCCount()" class="member-name-link">getGCCount()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Get the GC count in the DNA Sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html#getGCGChecksum(java.util.List)" class="member-name-link">getGCGChecksum(List&lt;S&gt;)</a> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Calculates GCG checksum for entire list of sequences</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html#getGCGChecksum(S)" class="member-name-link">getGCGChecksum(S)</a> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Calculates GCG checksum for a given sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html#getGCGHeader(java.util.List)" class="member-name-link">getGCGHeader(List&lt;S&gt;)</a> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Assembles a GCG file header</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html#getGCGType(org.biojava.nbio.core.sequence.template.CompoundSet)" class="member-name-link">getGCGType(CompoundSet&lt;C&gt;)</a> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Determines GCG type</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#getGenbankDirectoryCache()" class="member-name-link">getGenbankDirectoryCache()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReader</a></dt>
<dd>
<div class="block">Local directory cache of Genbank that can be downloaded</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/ChromosomeSequence.html#getGene(java.lang.String)" class="member-name-link">getGene(String)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/ChromosomeSequence.html" title="class in org.biojava.nbio.core.sequence">ChromosomeSequence</a></dt>
<dd>
<div class="block">Get the gene based on accession.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getGeneAliases()" class="member-name-link">getGeneAliases()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">Pull uniprot gene aliases associated with this sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getGeneName()" class="member-name-link">getGeneName()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">Get the gene name associated with this sequence.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/ChromosomeSequence.html#getGeneSequences()" class="member-name-link">getGeneSequences()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/ChromosomeSequence.html" title="class in org.biojava.nbio.core.sequence">ChromosomeSequence</a></dt>
<dd>
<div class="block">Get the list of genes that have been added to the ChromosomeSequence where accession.toString is the key.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html#getGonnet250()" class="member-name-link">getGonnet250()</a> - Static method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.core.alignment.matrices">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns PAM 250 matrix by Gonnet, Cohen &amp; Benner</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#getHeader()" class="member-name-link">getHeader()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#getHeader()" class="member-name-link">getHeader()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.html#getHeader(S)" class="member-name-link">getHeader(S)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.html" title="class in org.biojava.nbio.core.sequence.io">GenericFastaHeaderFormat</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html#getHeader(S)" class="member-name-link">getHeader(S)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html" title="class in org.biojava.nbio.core.sequence.io">GenericGenbankHeaderFormat</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/template/FastaHeaderFormatInterface.html#getHeader(S)" class="member-name-link">getHeader(S)</a> - Method in interface org.biojava.nbio.core.sequence.io.template.<a href="../org/biojava/nbio/core/sequence/io/template/FastaHeaderFormatInterface.html" title="interface in org.biojava.nbio.core.sequence.io.template">FastaHeaderFormatInterface</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html#getHeader(S)" class="member-name-link">getHeader(S)</a> - Method in interface org.biojava.nbio.core.sequence.io.template.<a href="../org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html" title="interface in org.biojava.nbio.core.sequence.io.template">GenbankHeaderFormatInterface</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#getHeaderParser()" class="member-name-link">getHeaderParser()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hit.html#getHitAccession()" class="member-name-link">getHitAccession()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hit.html" title="class in org.biojava.nbio.core.search.io">Hit</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Result.html#getHitCounter()" class="member-name-link">getHitCounter()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Result.html" title="class in org.biojava.nbio.core.search.io">Result</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hit.html#getHitDef()" class="member-name-link">getHitDef()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hit.html" title="class in org.biojava.nbio.core.search.io">Hit</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hit.html#getHitId()" class="member-name-link">getHitId()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hit.html" title="class in org.biojava.nbio.core.search.io">Hit</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hit.html#getHitLen()" class="member-name-link">getHitLen()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hit.html" title="class in org.biojava.nbio.core.search.io">Hit</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hit.html#getHitNum()" class="member-name-link">getHitNum()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hit.html" title="class in org.biojava.nbio.core.search.io">Hit</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hit.html#getHitSequence()" class="member-name-link">getHitSequence()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hit.html" title="class in org.biojava.nbio.core.search.io">Hit</a></dt>
<dd>
<div class="block">returns the reference to the original and whole sequence hit in the database.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hsp.html#getHspAlignLen()" class="member-name-link">getHspAlignLen()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hsp.html" title="class in org.biojava.nbio.core.search.io">Hsp</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hsp.html#getHspBitScore()" class="member-name-link">getHspBitScore()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hsp.html" title="class in org.biojava.nbio.core.search.io">Hsp</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hsp.html#getHspEvalue()" class="member-name-link">getHspEvalue()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hsp.html" title="class in org.biojava.nbio.core.search.io">Hsp</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hsp.html#getHspGaps()" class="member-name-link">getHspGaps()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hsp.html" title="class in org.biojava.nbio.core.search.io">Hsp</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hsp.html#getHspHitFrame()" class="member-name-link">getHspHitFrame()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hsp.html" title="class in org.biojava.nbio.core.search.io">Hsp</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hsp.html#getHspHitFrom()" class="member-name-link">getHspHitFrom()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hsp.html" title="class in org.biojava.nbio.core.search.io">Hsp</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hsp.html#getHspHitTo()" class="member-name-link">getHspHitTo()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hsp.html" title="class in org.biojava.nbio.core.search.io">Hsp</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hsp.html#getHspHseq()" class="member-name-link">getHspHseq()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hsp.html" title="class in org.biojava.nbio.core.search.io">Hsp</a></dt>
<dd>
<div class="block">HSP aligned hit sequence string</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hsp.html#getHspIdentity()" class="member-name-link">getHspIdentity()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hsp.html" title="class in org.biojava.nbio.core.search.io">Hsp</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hsp.html#getHspIdentityString()" class="member-name-link">getHspIdentityString()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hsp.html" title="class in org.biojava.nbio.core.search.io">Hsp</a></dt>
<dd>
<div class="block">Identity string representing correspondence between aligned residues</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hsp.html#getHspNum()" class="member-name-link">getHspNum()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hsp.html" title="class in org.biojava.nbio.core.search.io">Hsp</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hsp.html#getHspPositive()" class="member-name-link">getHspPositive()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hsp.html" title="class in org.biojava.nbio.core.search.io">Hsp</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hsp.html#getHspQseq()" class="member-name-link">getHspQseq()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hsp.html" title="class in org.biojava.nbio.core.search.io">Hsp</a></dt>
<dd>
<div class="block">HSP aligned query sequence string</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hsp.html#getHspQueryFrame()" class="member-name-link">getHspQueryFrame()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hsp.html" title="class in org.biojava.nbio.core.search.io">Hsp</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hsp.html#getHspQueryFrom()" class="member-name-link">getHspQueryFrom()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hsp.html" title="class in org.biojava.nbio.core.search.io">Hsp</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hsp.html#getHspQueryTo()" class="member-name-link">getHspQueryTo()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hsp.html" title="class in org.biojava.nbio.core.search.io">Hsp</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hsp.html#getHspScore()" class="member-name-link">getHspScore()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hsp.html" title="class in org.biojava.nbio.core.search.io">Hsp</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#getId()" class="member-name-link">getId()</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">DBReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html#getId()" class="member-name-link">getId()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser.IUPACTable</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/AccessionID.html#getID()" class="member-name-link">getID()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/AccessionID.html" title="class in org.biojava.nbio.core.sequence">AccessionID</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TaxonomyID.html#getID()" class="member-name-link">getID()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TaxonomyID.html" title="class in org.biojava.nbio.core.sequence">TaxonomyID</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/AccessionID.html#getIdentifier()" class="member-name-link">getIdentifier()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/AccessionID.html" title="class in org.biojava.nbio.core.sequence">AccessionID</a></dt>
<dd>
<div class="block">In case if <a href="../org/biojava/nbio/core/sequence/AccessionID.html#getID()"><code>AccessionID.getID()</code></a> is not unique, keeps the alternative id, e.g.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html#getIdentity()" class="member-name-link">getIdentity()</a> - Static method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.core.alignment.matrices">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Gets identity matrix where matches score 1 and mismatches score -10000</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html#getIDFormat(java.util.List)" class="member-name-link">getIDFormat(List&lt;S&gt;)</a> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Creates format String for accession IDs</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACParser.html#getIdLookup()" class="member-name-link">getIdLookup()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/IUPACParser.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/ABITrace.html#getImage(int,int)" class="member-name-link">getImage(int, int)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/ABITrace.html" title="class in org.biojava.nbio.core.sequence.io">ABITrace</a></dt>
<dd>
<div class="block">Returns a BufferedImage that represents the entire trace.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html#getIndexInQueryAt(int)" class="member-name-link">getIndexInQueryAt(int)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/SequencePair.html#getIndexInQueryAt(int)" class="member-name-link">getIndexInQueryAt(int)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/SequencePair.html" title="interface in org.biojava.nbio.core.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the query index corresponding to a given alignment column.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html#getIndexInQueryForTargetAt(int)" class="member-name-link">getIndexInQueryForTargetAt(int)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/SequencePair.html#getIndexInQueryForTargetAt(int)" class="member-name-link">getIndexInQueryForTargetAt(int)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/SequencePair.html" title="interface in org.biojava.nbio.core.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the query index corresponding to a given target index.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html#getIndexInTargetAt(int)" class="member-name-link">getIndexInTargetAt(int)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/SequencePair.html#getIndexInTargetAt(int)" class="member-name-link">getIndexInTargetAt(int)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/SequencePair.html" title="interface in org.biojava.nbio.core.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the target index corresponding to a given alignment column.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html#getIndexInTargetForQueryAt(int)" class="member-name-link">getIndexInTargetForQueryAt(int)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/SequencePair.html#getIndexInTargetForQueryAt(int)" class="member-name-link">getIndexInTargetForQueryAt(int)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/SequencePair.html" title="interface in org.biojava.nbio.core.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the target index corresponding to a given query index.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#getIndexOf(C)" class="member-name-link">getIndexOf(C)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html#getIndexOf(C)" class="member-name-link">getIndexOf(C)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.html#getIndexOf(C)" class="member-name-link">getIndexOf(C)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template">Profile</a></dt>
<dd>
<div class="block">Searches for the given <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Compound</code></a> within this alignment profile.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#getIndexOf(C)" class="member-name-link">getIndexOf(C)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#getIndexOf(C)" class="member-name-link">getIndexOf(C)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getIndexOf(C)" class="member-name-link">getIndexOf(C)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#getIndexOf(C)" class="member-name-link">getIndexOf(C)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#getIndexOf(C)" class="member-name-link">getIndexOf(C)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Returns the first occurrence of the given compound in this store; performs
 a linear search</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#getIndexOf(C)" class="member-name-link">getIndexOf(C)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#getIndexOf(C)" class="member-name-link">getIndexOf(C)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Returns 1 if the given compound is equal to the one given during
 construction; otherwise will return -1.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getIndexOf(C)" class="member-name-link">getIndexOf(C)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/Sequence.html#getIndexOf(C)" class="member-name-link">getIndexOf(C)</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Scans through the Sequence looking for the first occurrence of the given
 compound</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html#getIndexOf(C)" class="member-name-link">getIndexOf(C)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/views/ComplementSequenceView.html#getIndexOf(C)" class="member-name-link">getIndexOf(C)</a> - Method in class org.biojava.nbio.core.sequence.views.<a href="../org/biojava/nbio/core/sequence/views/ComplementSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">ComplementSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/views/RnaSequenceView.html#getIndexOf(org.biojava.nbio.core.sequence.compound.NucleotideCompound)" class="member-name-link">getIndexOf(NucleotideCompound)</a> - Method in class org.biojava.nbio.core.sequence.views.<a href="../org/biojava/nbio/core/sequence/views/RnaSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html#getIndicesAt(int)" class="member-name-link">getIndicesAt(int)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.html#getIndicesAt(int)" class="member-name-link">getIndicesAt(int)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the indices in the original <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a>s corresponding to the given column.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/SingleLinkageClustererProductProduct.html#getIndicesToCheck()" class="member-name-link">getIndicesToCheck()</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SingleLinkageClustererProductProduct.html" title="class in org.biojava.nbio.core.util">SingleLinkageClustererProductProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#getInputStream()" class="member-name-link">getInputStream()</a> - Method in class org.biojava.nbio.core.sequence.io.util.<a href="../org/biojava/nbio/core/sequence/io/util/ClasspathResource.html" title="class in org.biojava.nbio.core.sequence.io.util">ClasspathResource</a></dt>
<dd>
<div class="block">Returns the InputStream instance of this classpath resource</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/InputStreamProvider.html#getInputStream(java.io.File)" class="member-name-link">getInputStream(File)</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/InputStreamProvider.html" title="class in org.biojava.nbio.core.util">InputStreamProvider</a></dt>
<dd>
<div class="block">Get an InputStream for the file.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/FlatFileCache.html#getInputStream(java.lang.String)" class="member-name-link">getInputStream(String)</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/FlatFileCache.html" title="class in org.biojava.nbio.core.util">FlatFileCache</a></dt>
<dd>
<div class="block">Gets the cached file as an InputStream.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/InputStreamProvider.html#getInputStream(java.lang.String)" class="member-name-link">getInputStream(String)</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/InputStreamProvider.html" title="class in org.biojava.nbio.core.util">InputStreamProvider</a></dt>
<dd>
<div class="block">Get an InputStream for given file path.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/InputStreamProvider.html#getInputStream(java.net.URL)" class="member-name-link">getInputStream(URL)</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/InputStreamProvider.html" title="class in org.biojava.nbio.core.util">InputStreamProvider</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/DefaultAAIndexProvider.html#getInputStreamToAAindexFile()" class="member-name-link">getInputStreamToAAindexFile()</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/DefaultAAIndexProvider.html" title="class in org.biojava.nbio.core.alignment.matrices">DefaultAAIndexProvider</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACParser.html#getInstance()" class="member-name-link">getInstance()</a> - Static method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/IUPACParser.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/GeneSequence.html#getIntronSequences()" class="member-name-link">getIntronSequences()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Get the introns as an ArrayList.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#getInverse()" class="member-name-link">getInverse()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#getInverse()" class="member-name-link">getInverse()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#getInverse()" class="member-name-link">getInverse()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getInverse()" class="member-name-link">getInverse()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/RNASequence.html#getInverse()" class="member-name-link">getInverse()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/RNASequence.html" title="class in org.biojava.nbio.core.sequence">RNASequence</a></dt>
<dd>
<div class="block">Get the inverse view of the sequence.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#getInverse()" class="member-name-link">getInverse()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#getInverse()" class="member-name-link">getInverse()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#getInverse()" class="member-name-link">getInverse()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#getInverse()" class="member-name-link">getInverse()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getInverse()" class="member-name-link">getInverse()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/Sequence.html#getInverse()" class="member-name-link">getInverse()</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Does the <em>right thing</em> to get the inverse of the current
 Sequence.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html#getInverse()" class="member-name-link">getInverse()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/SingleLinkageClustererProductProduct.html#getIsScoreMatrix()" class="member-name-link">getIsScoreMatrix()</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SingleLinkageClustererProductProduct.html" title="class in org.biojava.nbio.core.util">SingleLinkageClustererProductProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Result.html#getIterationNumber()" class="member-name-link">getIterationNumber()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Result.html" title="class in org.biojava.nbio.core.search.io">Result</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACParserProduct2.html#getIUPACParserProduct()" class="member-name-link">getIUPACParserProduct()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/IUPACParserProduct2.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParserProduct2</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/reference/AbstractReference.html#getJournal()" class="member-name-link">getJournal()</a> - Method in class org.biojava.nbio.core.sequence.reference.<a href="../org/biojava/nbio/core/sequence/reference/AbstractReference.html" title="class in org.biojava.nbio.core.sequence.reference">AbstractReference</a></dt>
<dd>
<div class="block">The journal usually contains the Publication Number, Publication Date and Assignee</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/reference/GenbankReference.html#getJournal()" class="member-name-link">getJournal()</a> - Method in class org.biojava.nbio.core.sequence.reference.<a href="../org/biojava/nbio/core/sequence/reference/GenbankReference.html" title="class in org.biojava.nbio.core.sequence.reference">GenbankReference</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/reference/ReferenceInterface.html#getJournal()" class="member-name-link">getJournal()</a> - Method in interface org.biojava.nbio.core.sequence.reference.<a href="../org/biojava/nbio/core/sequence/reference/ReferenceInterface.html" title="interface in org.biojava.nbio.core.sequence.reference">ReferenceInterface</a></dt>
<dd>
<div class="block">Get the journal that retrieved from Reference section.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#getKeyword()" class="member-name-link">getKeyword()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>
<div class="block">The KW (KeyWord) lines provide information which can be used to generate
 cross-reference indexes of the sequence entries based on functional,
 structural, or other categories deemed important.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeaturesKeyWordInterface.html#getKeyWords()" class="member-name-link">getKeyWords()</a> - Method in interface org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeaturesKeyWordInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeaturesKeyWordInterface</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#getKeyWords()" class="member-name-link">getKeyWords()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html#getKeyWords()" class="member-name-link">getKeyWords()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getKeyWords()" class="member-name-link">getKeyWords()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">Pull UniProt key words which is a mixed bag of words associated with this sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#getLastIndexOf(C)" class="member-name-link">getLastIndexOf(C)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html#getLastIndexOf(C)" class="member-name-link">getLastIndexOf(C)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.html#getLastIndexOf(C)" class="member-name-link">getLastIndexOf(C)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template">Profile</a></dt>
<dd>
<div class="block">Searches for the given <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Compound</code></a> within this alignment profile.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#getLastIndexOf(C)" class="member-name-link">getLastIndexOf(C)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#getLastIndexOf(C)" class="member-name-link">getLastIndexOf(C)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getLastIndexOf(C)" class="member-name-link">getLastIndexOf(C)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#getLastIndexOf(C)" class="member-name-link">getLastIndexOf(C)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#getLastIndexOf(C)" class="member-name-link">getLastIndexOf(C)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Returns the last occurrence of the given compound in this store; performs
 a linear search</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#getLastIndexOf(C)" class="member-name-link">getLastIndexOf(C)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#getLastIndexOf(C)" class="member-name-link">getLastIndexOf(C)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Returns the length of the Sequence if the given compound was equal to
 the one given during construction.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getLastIndexOf(C)" class="member-name-link">getLastIndexOf(C)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/Sequence.html#getLastIndexOf(C)" class="member-name-link">getLastIndexOf(C)</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Scans through the Sequence looking for the last occurrence of the given
 compound</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html#getLastIndexOf(C)" class="member-name-link">getLastIndexOf(C)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/views/ComplementSequenceView.html#getLastIndexOf(C)" class="member-name-link">getLastIndexOf(C)</a> - Method in class org.biojava.nbio.core.sequence.views.<a href="../org/biojava/nbio/core/sequence/views/ComplementSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">ComplementSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/views/RnaSequenceView.html#getLastIndexOf(org.biojava.nbio.core.sequence.compound.NucleotideCompound)" class="member-name-link">getLastIndexOf(NucleotideCompound)</a> - Method in class org.biojava.nbio.core.sequence.views.<a href="../org/biojava/nbio/core/sequence/views/RnaSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#getLastUpdatedDate()" class="member-name-link">getLastUpdatedDate()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>
<div class="block">The DT (DaTe) line shows when an entry was last updated in the database.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#getLength()" class="member-name-link">getLength()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html#getLength()" class="member-name-link">getLength()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.html#getLength()" class="member-name-link">getLength()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the number of columns in the alignment profile.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/CDSSequence.html#getLength()" class="member-name-link">getLength()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/CDSSequence.html" title="class in org.biojava.nbio.core.sequence">CDSSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/ExonSequence.html#getLength()" class="member-name-link">getLength()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/ExonSequence.html" title="class in org.biojava.nbio.core.sequence">ExonSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/GeneSequence.html#getLength()" class="member-name-link">getLength()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/IntronSequence.html#getLength()" class="member-name-link">getLength()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/IntronSequence.html" title="class in org.biojava.nbio.core.sequence">IntronSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#getLength()" class="member-name-link">getLength()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#getLength()" class="member-name-link">getLength()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getLength()" class="member-name-link">getLength()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">The sequence length</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#getLength()" class="member-name-link">getLength()</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/Location.html#getLength()" class="member-name-link">getLength()</a> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/Location.html" title="interface in org.biojava.nbio.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Returns the length of the outer bounds of this location</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#getLength()" class="member-name-link">getLength()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava.nbio.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">Get the length of the MSA where it is assumed that
 all sequence position</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/StartCodonSequence.html#getLength()" class="member-name-link">getLength()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/StartCodonSequence.html" title="class in org.biojava.nbio.core.sequence">StartCodonSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/StopCodonSequence.html#getLength()" class="member-name-link">getLength()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/StopCodonSequence.html" title="class in org.biojava.nbio.core.sequence">StopCodonSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#getLength()" class="member-name-link">getLength()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#getLength()" class="member-name-link">getLength()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#getLength()" class="member-name-link">getLength()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Returns the length of the sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#getLength()" class="member-name-link">getLength()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#getLength()" class="member-name-link">getLength()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Returns the length given during construction</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getLength()" class="member-name-link">getLength()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/LightweightProfile.html#getLength()" class="member-name-link">getLength()</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/LightweightProfile.html" title="interface in org.biojava.nbio.core.sequence.template">LightweightProfile</a></dt>
<dd>
<div class="block">Returns the number of columns in the alignment profile.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/Sequence.html#getLength()" class="member-name-link">getLength()</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Returns the length of the Sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html#getLength()" class="member-name-link">getLength()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranscriptSequence.html#getLength()" class="member-name-link">getLength()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/views/WindowedSequence.html#getLength()" class="member-name-link">getLength()</a> - Method in class org.biojava.nbio.core.sequence.views.<a href="../org/biojava/nbio/core/sequence/views/WindowedSequence.html" title="class in org.biojava.nbio.core.sequence.views">WindowedSequence</a></dt>
<dd>
<div class="block">Returns the size of the windowed sequence which is the length by the
 window size.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaGeneWriter.html#getLineLength()" class="member-name-link">getLineLength()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaGeneWriter.html" title="class in org.biojava.nbio.core.sequence.io">FastaGeneWriter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaWriter.html#getLineLength()" class="member-name-link">getLineLength()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaWriter.html" title="class in org.biojava.nbio.core.sequence.io">FastaWriter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankWriter.html#getLineLength()" class="member-name-link">getLineLength()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankWriter.html" title="class in org.biojava.nbio.core.sequence.io">GenbankWriter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/util/ClasspathResource.html#getList()" class="member-name-link">getList()</a> - Method in class org.biojava.nbio.core.sequence.io.util.<a href="../org/biojava/nbio/core/sequence/io/util/ClasspathResource.html" title="class in org.biojava.nbio.core.sequence.io.util">ClasspathResource</a></dt>
<dd>
<div class="block">Returns this resource as a list of Strings</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html#getList(java.io.BufferedReader)" class="member-name-link">getList(BufferedReader)</a> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Returns the contents of a buffered reader as a list of strings</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html#getList(java.io.File)" class="member-name-link">getList(File)</a> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Delegates to <a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html#getList(java.io.InputStream)"><code>IOUtils.getList(InputStream)</code></a> by wrapping the File
 in a valid stream.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html#getList(java.io.InputStream)" class="member-name-link">getList(InputStream)</a> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Delegates to <a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html#getList(java.io.BufferedReader)"><code>IOUtils.getList(BufferedReader)</code></a> by wrapping the InputStream
 in a valid reader.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#getLocationInAlignment()" class="member-name-link">getLocationInAlignment()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html#getLocationInAlignment()" class="member-name-link">getLocationInAlignment()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns the <a href="../org/biojava/nbio/core/sequence/location/template/Location.html" title="interface in org.biojava.nbio.core.sequence.location.template"><code>Location</code></a> of the original <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a> within an alignment.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html#getLocations()" class="member-name-link">getLocations()</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">A feature could be a single sequence position like a mutation or a post translational modification of an amino acid.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#getLocations()" class="member-name-link">getLocations()</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html#getLocations()" class="member-name-link">getLocations()</a> - Method in interface org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">The location(s) of this feature where the location should contain a reference to parent and sequence etc.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/TargetedLocationParserTest.html#getLocations()" class="member-name-link">getLocations()</a> - Static method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/TargetedLocationParserTest.html" title="class in org.biojava.nbio.core.sequence.location">TargetedLocationParserTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html#getLongName()" class="member-name-link">getLongName()</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html#getLongName()" class="member-name-link">getLongName()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/Compound.html#getLongName()" class="member-name-link">getLongName()</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#getLongName()" class="member-name-link">getLongName()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html#getLongName()" class="member-name-link">getLongName()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/AAIndexFileParser.html#getMatrices()" class="member-name-link">getMatrices()</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/AAIndexFileParser.html" title="class in org.biojava.nbio.core.alignment.matrices">AAIndexFileParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html#getMatrix()" class="member-name-link">getMatrix()</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html#getMatrix()" class="member-name-link">getMatrix()</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/SubstitutionMatrix.html#getMatrix()" class="member-name-link">getMatrix()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava.nbio.core.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns entire matrix.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/AAIndexProvider.html#getMatrix(java.lang.String)" class="member-name-link">getMatrix(String)</a> - Method in interface org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/AAIndexProvider.html" title="interface in org.biojava.nbio.core.alignment.matrices">AAIndexProvider</a></dt>
<dd>
<div class="block">Gets a substitution matrix by its name.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/DefaultAAIndexProvider.html#getMatrix(java.lang.String)" class="member-name-link">getMatrix(String)</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/DefaultAAIndexProvider.html" title="class in org.biojava.nbio.core.alignment.matrices">DefaultAAIndexProvider</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html#getMatrixAsString()" class="member-name-link">getMatrixAsString()</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html#getMatrixAsString()" class="member-name-link">getMatrixAsString()</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/SubstitutionMatrix.html#getMatrixAsString()" class="member-name-link">getMatrixAsString()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava.nbio.core.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns this matrix as a formatted String with <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Compound</code></a> labels along the axes.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html#getMatrixFromAAINDEX(java.lang.String)" class="member-name-link">getMatrixFromAAINDEX(String)</a> - Static method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.core.alignment.matrices">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns any matrix from the AAINDEX database file</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html#getMax()" class="member-name-link">getMax()</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/LocationHelper.html#getMax(java.util.List)" class="member-name-link">getMax(List&lt;Location&gt;)</a> - Static method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/LocationHelper.html" title="class in org.biojava.nbio.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Scans through a list of locations to find the Location with the
 highest end</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/Location.Tools.html#getMax(java.util.List)" class="member-name-link">getMax(List&lt;Location&gt;)</a> - Static method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/Location.Tools.html" title="class in org.biojava.nbio.core.sequence.location.template">Location.Tools</a></dt>
<dd>
<div class="block">Scans through a list of locations to find the Location with the
 highest end</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#getMaxSingleCompoundStringLength()" class="member-name-link">getMaxSingleCompoundStringLength()</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#getMaxSingleCompoundStringLength()" class="member-name-link">getMaxSingleCompoundStringLength()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/CompoundSet.html#getMaxSingleCompoundStringLength()" class="member-name-link">getMaxSingleCompoundStringLength()</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundSet</a></dt>
<dd>
<div class="block">Returns the maximum size of a compound String this set holds</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html#getMaxValue()" class="member-name-link">getMaxValue()</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html#getMaxValue()" class="member-name-link">getMaxValue()</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/SubstitutionMatrix.html#getMaxValue()" class="member-name-link">getMaxValue()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava.nbio.core.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns the maximum value in this matrix.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html#getMin()" class="member-name-link">getMin()</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/LocationHelper.html#getMin(java.util.List)" class="member-name-link">getMin(List&lt;Location&gt;)</a> - Static method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/LocationHelper.html" title="class in org.biojava.nbio.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Scans through a list of locations to find the Location with the
 lowest start</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/Location.Tools.html#getMin(java.util.List)" class="member-name-link">getMin(List&lt;Location&gt;)</a> - Static method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/Location.Tools.html" title="class in org.biojava.nbio.core.sequence.location.template">Location.Tools</a></dt>
<dd>
<div class="block">Scans through a list of locations to find the Location with the
 lowest start</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html#getMinValue()" class="member-name-link">getMinValue()</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html#getMinValue()" class="member-name-link">getMinValue()</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/SubstitutionMatrix.html#getMinValue()" class="member-name-link">getMinValue()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava.nbio.core.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns the minimum value in this matrix.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hsp.html#getMismatchCount()" class="member-name-link">getMismatchCount()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hsp.html" title="class in org.biojava.nbio.core.search.io">Hsp</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html#getMolecularWeight()" class="member-name-link">getMolecularWeight()</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html#getMolecularWeight()" class="member-name-link">getMolecularWeight()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/Compound.html#getMolecularWeight()" class="member-name-link">getMolecularWeight()</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#getMolecularWeight()" class="member-name-link">getMolecularWeight()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html#getMolecularWeight()" class="member-name-link">getMolecularWeight()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblId.html#getMoleculeType()" class="member-name-link">getMoleculeType()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblId.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblId</a></dt>
<dd>
<div class="block">Molecule type this represents the type of molecule as stored</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html#getName()" class="member-name-link">getName()</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html#getName()" class="member-name-link">getName()</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/SubstitutionMatrix.html#getName()" class="member-name-link">getName()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava.nbio.core.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns the name (short description) of this matrix.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/Qualifier.html#getName()" class="member-name-link">getName()</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/Qualifier.html" title="class in org.biojava.nbio.core.sequence.features">Qualifier</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html#getName()" class="member-name-link">getName()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser.IUPACTable</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/PrettyXMLWriterProduct2.html#getNamespaceBindings()" class="member-name-link">getNamespaceBindings()</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/PrettyXMLWriterProduct2.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriterProduct2</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/XMLHelper.html#getNewDocument()" class="member-name-link">getNewDocument()</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/XMLHelper.html" title="class in org.biojava.nbio.core.util">XMLHelper</a></dt>
<dd>
<div class="block">Create a new, empty org.w3c.dom.Document</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getNotesList()" class="member-name-link">getNotesList()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html#getNuc4_2()" class="member-name-link">getNuc4_2()</a> - Static method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.core.alignment.matrices">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Nuc 4.2 matrix by Lowe
 Only the first nucleotide sequence to align can contain ambiguous nucleotides</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html#getNuc4_4()" class="member-name-link">getNuc4_4()</a> - Static method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.core.alignment.matrices">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Nuc 4.4 matrix by Lowe
 Both of the nucleotide sequences to align can contain ambiguous nucleotides</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/Strand.html#getNumericRepresentation()" class="member-name-link">getNumericRepresentation()</a> - Method in enum org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/Strand.html" title="enum in org.biojava.nbio.core.sequence">Strand</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#getNumGapPositions()" class="member-name-link">getNumGapPositions()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html#getNumGapPositions()" class="member-name-link">getNumGapPositions()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns number of gap positions (gap openings and extensions) in the sequence.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#getNumGaps()" class="member-name-link">getNumGaps()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html#getNumGaps()" class="member-name-link">getNumGaps()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns number of gaps (gap openings) in the sequence.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html#getNumIdenticals()" class="member-name-link">getNumIdenticals()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/SequencePair.html#getNumIdenticals()" class="member-name-link">getNumIdenticals()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/SequencePair.html" title="interface in org.biojava.nbio.core.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the number of indices for which both the query and target sequences have an identical <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Compound</code></a>.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/SingleLinkageClustererProduct.html#getNumItems()" class="member-name-link">getNumItems()</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SingleLinkageClustererProduct.html" title="class in org.biojava.nbio.core.util">SingleLinkageClustererProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html#getNumSimilars()" class="member-name-link">getNumSimilars()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/SequencePair.html#getNumSimilars()" class="member-name-link">getNumSimilars()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/SequencePair.html" title="interface in org.biojava.nbio.core.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the number of indices for which both the query and target sequences have a similar <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Compound</code></a>.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html#getOne()" class="member-name-link">getOne()</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html#getOne()" class="member-name-link">getOne()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.CaseInsensitiveTriplet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html#getOne()" class="member-name-link">getOne()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#getOrGanelle()" class="member-name-link">getOrGanelle()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>
<div class="block">The OG (OrGanelle) linetype indicates the sub-cellular location of non-nuclear
 sequences.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#getOrganismClassification()" class="member-name-link">getOrganismClassification()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>
<div class="block">The OC (Organism Classification) lines contain the taxonomic classification
 Of the source organism</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getOrganismName()" class="member-name-link">getOrganismName()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">Get the organism name assigned to this sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#getOrganismSpecies()" class="member-name-link">getOrganismSpecies()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>
<div class="block">The OS (Organism Species) line specifies the preferred scientific name of
 the organism which was the source of the stored sequence.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getOriginalHeader()" class="member-name-link">getOriginalHeader()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#getOriginalSequence()" class="member-name-link">getOriginalSequence()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html#getOriginalSequence()" class="member-name-link">getOriginalSequence()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns the original <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a> before alignment.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html#getOriginalSequences()" class="member-name-link">getOriginalSequences()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.html#getOriginalSequences()" class="member-name-link">getOriginalSequences()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns a <a href="https://docs.oracle.com/javase/8/docs/api/java/util/List.html" title="class or interface in java.util" class="external-link"><code>List</code></a> containing the original <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a>s used for alignment.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#getOverlapCount()" class="member-name-link">getOverlapCount()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html#getOverlapCount()" class="member-name-link">getOverlapCount()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns the maximum number of elements contributed to a column of an alignment by this <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a>.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html#getPAM250()" class="member-name-link">getPAM250()</a> - Static method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.core.alignment.matrices">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns PAM 250 matrix by Dayhoff</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/GeneSequence.html#getParentChromosomeSequence()" class="member-name-link">getParentChromosomeSequence()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">The parent ChromosomeSequence which contains the actual DNA sequence data</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html#getParentFeature()" class="member-name-link">getParentFeature()</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">Get the parent Feature</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#getParentFeature()" class="member-name-link">getParentFeature()</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html#getParentFeature()" class="member-name-link">getParentFeature()</a> - Method in interface org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Get the parent feature</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getParentSequence()" class="member-name-link">getParentSequence()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocationProduct.html#getPartialOn3prime()" class="member-name-link">getPartialOn3prime()</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocationProduct.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocationProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocationProduct.html#getPartialOn5prime()" class="member-name-link">getPartialOn5prime()</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocationProduct.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocationProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html#getPDBCharacter(boolean,char,char,boolean,char)" class="member-name-link">getPDBCharacter(boolean, char, char, boolean, char)</a> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Creates formatted String for a single character of PDB output</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html#getPDBConservation(boolean,char,char,boolean)" class="member-name-link">getPDBConservation(boolean, char, char, boolean)</a> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Creates formatted String for displaying conservation in PDB output</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html#getPDBLegend()" class="member-name-link">getPDBLegend()</a> - Static method in class org.biojava.nbio.core.sequence.io.util.<a href="../org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Creates formatted String for displaying conservation legend in PDB output</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Hsp.html#getPercentageIdentity()" class="member-name-link">getPercentageIdentity()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Hsp.html" title="class in org.biojava.nbio.core.search.io">Hsp</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html#getPercentageOfIdentity(boolean)" class="member-name-link">getPercentageOfIdentity(boolean)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePair</a></dt>
<dd>
<div class="block">Returns the percentage of identity between the two sequences in the alignment as a fraction between 0 and 1.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/SequencePair.html#getPercentageOfIdentity(boolean)" class="member-name-link">getPercentageOfIdentity(boolean)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/SequencePair.html" title="interface in org.biojava.nbio.core.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the percentage of identity between the two sequences in the alignment as a fraction between 0 and 1.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/CDSSequence.html#getPhase()" class="member-name-link">getPhase()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/CDSSequence.html" title="class in org.biojava.nbio.core.sequence">CDSSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/FuzzyPoint.html#getPosition()" class="member-name-link">getPosition()</a> - Method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/FuzzyPoint.html" title="class in org.biojava.nbio.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SimplePoint.html#getPosition()" class="member-name-link">getPosition()</a> - Method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/Point.html#getPosition()" class="member-name-link">getPosition()</a> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/Point.html" title="interface in org.biojava.nbio.core.sequence.location.template">Point</a></dt>
<dd>
<div class="block">Returns the position held by this object</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblId.html#getPrimaryAccession()" class="member-name-link">getPrimaryAccession()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblId.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblId</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Result.html#getProgram()" class="member-name-link">getProgram()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Result.html" title="class in org.biojava.nbio.core.search.io">Result</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Result.html#getProgramSpecificParameter(java.lang.String)" class="member-name-link">getProgramSpecificParameter(String)</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Result.html" title="class in org.biojava.nbio.core.search.io">Result</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Result.html#getProgramSpecificParametersList()" class="member-name-link">getProgramSpecificParametersList()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Result.html" title="class in org.biojava.nbio.core.search.io">Result</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#getProjectIdentifier()" class="member-name-link">getProjectIdentifier()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/DBReferenceInfo.html#getProperties()" class="member-name-link">getProperties()</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">DBReferenceInfo</a></dt>
<dd>
<div class="block">Get the properties</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getProteinAliases()" class="member-name-link">getProteinAliases()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">Pull uniprot protein aliases associated with this sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReaderProduct2.html#getProteinAliases()" class="member-name-link">getProteinAliases()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReaderProduct2.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReaderProduct2</a></dt>
<dd>
<div class="block">Pull uniprot protein aliases associated with this sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReaderProduct2.html#getProteinAliasesFromElement(java.util.ArrayList,org.w3c.dom.Element)" class="member-name-link">getProteinAliasesFromElement(ArrayList&lt;String&gt;, Element)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReaderProduct2.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReaderProduct2</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReaderProduct2.html#getProteinAliasesFromNameGroup(java.util.ArrayList,org.w3c.dom.Element)" class="member-name-link">getProteinAliasesFromNameGroup(ArrayList&lt;String&gt;, Element)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReaderProduct2.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReaderProduct2</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranscriptSequence.html#getProteinCDSSequences()" class="member-name-link">getProteinCDSSequences()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a></dt>
<dd>
<div class="block">Return a list of protein sequences based on each CDS sequence
 where the phase shift between two CDS sequences is assigned to the
 CDS sequence that starts the triplet.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranscriptSequenceProduct.html#getProteinCDSSequences(org.biojava.nbio.core.sequence.TranscriptSequence)" class="member-name-link">getProteinCDSSequences(TranscriptSequence)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranscriptSequenceProduct.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequenceProduct</a></dt>
<dd>
<div class="block">Return a list of protein sequences based on each CDS sequence where the phase shift between two CDS sequences is assigned to the CDS sequence that starts the triplet.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/RNASequence.html#getProteinSequence()" class="member-name-link">getProteinSequence()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/RNASequence.html" title="class in org.biojava.nbio.core.sequence">RNASequence</a></dt>
<dd>
<div class="block">Get the ProteinSequence from the RNA sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranscriptSequence.html#getProteinSequence()" class="member-name-link">getProteinSequence()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a></dt>
<dd>
<div class="block">Get the protein sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/RNASequence.html#getProteinSequence(org.biojava.nbio.core.sequence.transcription.TranscriptionEngine)" class="member-name-link">getProteinSequence(TranscriptionEngine)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/RNASequence.html" title="class in org.biojava.nbio.core.sequence">RNASequence</a></dt>
<dd>
<div class="block">Get the ProteinSequence from the RNA sequence with user-defined
 transcription engine</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranscriptSequence.html#getProteinSequence(org.biojava.nbio.core.sequence.transcription.TranscriptionEngine)" class="member-name-link">getProteinSequence(TranscriptionEngine)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a></dt>
<dd>
<div class="block">Get the protein sequence with user defined TranscriptEngine</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranscriptSequenceProduct.html#getProteinSequence(org.biojava.nbio.core.sequence.transcription.TranscriptionEngine,org.biojava.nbio.core.sequence.TranscriptSequence)" class="member-name-link">getProteinSequence(TranscriptionEngine, TranscriptSequence)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranscriptSequenceProduct.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequenceProduct</a></dt>
<dd>
<div class="block">Get the protein sequence with user defined TranscriptEngine</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#getProteinSequenceCreator()" class="member-name-link">getProteinSequenceCreator()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AccesionedLocation.html#getProxySequenceReader()" class="member-name-link">getProxySequenceReader()</a> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AccesionedLocation.html" title="interface in org.biojava.nbio.core.sequence.location.template">AccesionedLocation</a></dt>
<dd>
<div class="block">Return the proxy reader used to get sequence for this location.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getProxySequenceReader()" class="member-name-link">getProxySequenceReader()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/ABITrace.html#getQcalls()" class="member-name-link">getQcalls()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/ABITrace.html" title="class in org.biojava.nbio.core.sequence.io">ABITrace</a></dt>
<dd>
<div class="block">Returns an <code>int[]</code> array that represents the quality - each int in the
 array corresponds to an quality value 90-255) in the graph at a base location).</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html#getQualifiers()" class="member-name-link">getQualifiers()</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#getQualifiers()" class="member-name-link">getQualifiers()</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html#getQualifiers()" class="member-name-link">getQualifiers()</a> - Method in interface org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Get the qualifiers for this feature</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/QualityFeature.html#getQualities()" class="member-name-link">getQualities()</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/QualityFeature.html" title="class in org.biojava.nbio.core.sequence.features">QualityFeature</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/QualityFeature.html#getQualities(int,int)" class="member-name-link">getQualities(int, int)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/QualityFeature.html" title="class in org.biojava.nbio.core.sequence.features">QualityFeature</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/QualityFeature.html#getQualityAt(int)" class="member-name-link">getQualityAt(int)</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/QualityFeature.html" title="class in org.biojava.nbio.core.sequence.features">QualityFeature</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/QuantityFeature.html#getQuantities()" class="member-name-link">getQuantities()</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/QuantityFeature.html" title="class in org.biojava.nbio.core.sequence.features">QuantityFeature</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfilePair.html#getQuery()" class="member-name-link">getQuery()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfilePair.html" title="class in org.biojava.nbio.core.alignment">SimpleProfilePair</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html#getQuery()" class="member-name-link">getQuery()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/ProfilePair.html#getQuery()" class="member-name-link">getQuery()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/ProfilePair.html" title="interface in org.biojava.nbio.core.alignment.template">ProfilePair</a></dt>
<dd>
<div class="block">Returns the first <a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template"><code>Profile</code></a> of the pair.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/SequencePair.html#getQuery()" class="member-name-link">getQuery()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/SequencePair.html" title="interface in org.biojava.nbio.core.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the first <a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template"><code>AlignedSequence</code></a> of the pair.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Result.html#getQueryDef()" class="member-name-link">getQueryDef()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Result.html" title="class in org.biojava.nbio.core.search.io">Result</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Result.html#getQueryID()" class="member-name-link">getQueryID()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Result.html" title="class in org.biojava.nbio.core.search.io">Result</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Result.html#getQueryLength()" class="member-name-link">getQueryLength()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Result.html" title="class in org.biojava.nbio.core.search.io">Result</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserProduct.html#getQueryReferences()" class="member-name-link">getQueryReferences()</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserProduct.html" title="class in org.biojava.nbio.core.search.io.blast">BlastXMLParserProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserProduct.html#getQueryReferencesMap()" class="member-name-link">getQueryReferencesMap()</a> - Method in class org.biojava.nbio.core.search.io.blast.<a href="../org/biojava/nbio/core/search/io/blast/BlastXMLParserProduct.html" title="class in org.biojava.nbio.core.search.io.blast">BlastXMLParserProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Result.html#getQuerySequence()" class="member-name-link">getQuerySequence()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Result.html" title="class in org.biojava.nbio.core.search.io">Result</a></dt>
<dd>
<div class="block">returns the reference to the original and whole sequence used to query the database.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Result.html#getReference()" class="member-name-link">getReference()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Result.html" title="class in org.biojava.nbio.core.search.io">Result</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html#getReferenceAuthor()" class="member-name-link">getReferenceAuthor()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReference</a></dt>
<dd>
<div class="block">The RA (Reference Author) lines list the authors of the paper (or other
 work) cited.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html#getReferenceComment()" class="member-name-link">getReferenceComment()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReference</a></dt>
<dd>
<div class="block">The RC (Reference Comment) linetype is an optional linetype which appears if
 The reference has a comment.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html#getReferenceCrossReference()" class="member-name-link">getReferenceCrossReference()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReference</a></dt>
<dd>
<div class="block">The RX (reference cross-reference) linetype is
 an optional linetype which appears if
 one or more contiguous base spans of the
 presented sequence can be attributed
 to the reference in question.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html#getReferenceGroup()" class="member-name-link">getReferenceGroup()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReference</a></dt>
<dd>
<div class="block">The RG (Reference Group) lines list the working groups/consortia that
 produced the record.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html#getReferenceLocation()" class="member-name-link">getReferenceLocation()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReference</a></dt>
<dd>
<div class="block">The RL (Reference Location) line contains the conventional citation
 information for the reference.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html#getReferenceNumber()" class="member-name-link">getReferenceNumber()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReference</a></dt>
<dd>
<div class="block">The RN (Reference Number) line gives a unique number to each reference
 Citation within an entry.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html#getReferencePosition()" class="member-name-link">getReferencePosition()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReference</a></dt>
<dd>
<div class="block">The RP (Reference Position) linetype is
 an optional linetype which appears if
 one or more contiguous base spans of
 the presented sequence can be attributed
 to the reference in question.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getReferences()" class="member-name-link">getReferences()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html#getReferenceTitle()" class="member-name-link">getReferenceTitle()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblReference.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReference</a></dt>
<dd>
<div class="block">The RT (Reference Title) lines give the title of the paper (or other work) as
 exactly as is possible given the limitations of computer character sets.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#getRelevantSubLocations()" class="member-name-link">getRelevantSubLocations()</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>
<div class="block">Returns the normalised list of sub locations i.e. only those locations
 which do not have a sub location.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/Location.html#getRelevantSubLocations()" class="member-name-link">getRelevantSubLocations()</a> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/Location.html" title="interface in org.biojava.nbio.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">An extension to <a href="../org/biojava/nbio/core/sequence/location/template/Location.html#getSubLocations()"><code>Location.getSubLocations()</code></a> which returns sub-locations
 of sub-locations; this will continue until it runs out of those locations.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#getRelevantSubSequence(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">getRelevantSubSequence(Sequence&lt;C&gt;)</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/Location.html#getRelevantSubSequence(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">getRelevantSubSequence(Sequence&lt;C&gt;)</a> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/Location.html" title="interface in org.biojava.nbio.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Will return a SequenceReader object which offers a view of all resolved
 locations i.e. those locations which are not complex and define the
 true Sequence represented</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNASequence.html#getReverse()" class="member-name-link">getReverse()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Returns a Sequence which runs in the current reverse order</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/Strand.html#getReverse()" class="member-name-link">getReverse()</a> - Method in enum org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/Strand.html" title="enum in org.biojava.nbio.core.sequence">Strand</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNASequence.html#getReverseComplement()" class="member-name-link">getReverseComplement()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Delegates to <a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getInverse()"><code>AbstractSequence.getInverse()</code></a> for the reverse complement</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/RNASequence.html#getReverseComplement()" class="member-name-link">getReverseComplement()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/RNASequence.html" title="class in org.biojava.nbio.core.sequence">RNASequence</a></dt>
<dd>
<div class="block">Get reverse complement view of the sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Frame.html#getReverseFrames()" class="member-name-link">getReverseFrames()</a> - Static method in enum org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Frame.html" title="enum in org.biojava.nbio.core.sequence.transcription">Frame</a></dt>
<dd>
<div class="block">Returns all frames which are in the reverse orientation</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#getRnaAminoAcidTranslator()" class="member-name-link">getRnaAminoAcidTranslator()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#getRnaCompounds()" class="member-name-link">getRnaCompounds()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AmbiguityRNACompoundSet.html#getRNACompoundSet()" class="member-name-link">getRNACompoundSet()</a> - Static method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/AmbiguityRNACompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AmbiguityRNACompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/RNACompoundSet.html#getRNACompoundSet()" class="member-name-link">getRNACompoundSet()</a> - Static method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/RNACompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">RNACompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNASequence.html#getRNASequence()" class="member-name-link">getRNASequence()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Return the RNASequence equivalent of the DNASequence using default Transcription Engine.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNASequence.html#getRNASequence(org.biojava.nbio.core.sequence.transcription.Frame)" class="member-name-link">getRNASequence(Frame)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Allows the user to pass in the Frame shift.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNASequence.html#getRNASequence(org.biojava.nbio.core.sequence.transcription.TranscriptionEngine)" class="member-name-link">getRNASequence(TranscriptionEngine)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Allow a user to pass in a rules engine to do the DNA to RNA translation</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/DNASequence.html#getRNASequence(org.biojava.nbio.core.sequence.transcription.TranscriptionEngine,org.biojava.nbio.core.sequence.transcription.Frame)" class="member-name-link">getRNASequence(TranscriptionEngine, Frame)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#getRnaSequenceCreator()" class="member-name-link">getRnaSequenceCreator()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/views/RnaSequenceView.html#getRnaToDna()" class="member-name-link">getRnaToDna()</a> - Method in class org.biojava.nbio.core.sequence.views.<a href="../org/biojava/nbio/core/sequence/views/RnaSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html#getRow(C)" class="member-name-link">getRow(C)</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/SubstitutionMatrix.html#getRow(C)" class="member-name-link">getRow(C)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava.nbio.core.alignment.template">SubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html#getRow(org.biojava.nbio.core.sequence.compound.AminoAcidCompound)" class="member-name-link">getRow(AminoAcidCompound)</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html#getRows()" class="member-name-link">getRows()</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html#getScale()" class="member-name-link">getScale()</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/SingleLinkageClusterer.LinkedPair.html#getSecond()" class="member-name-link">getSecond()</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SingleLinkageClusterer.LinkedPair.html" title="class in org.biojava.nbio.core.util">SingleLinkageClusterer.LinkedPair</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/ABITrace.html#getSequence()" class="member-name-link">getSequence()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/ABITrace.html" title="class in org.biojava.nbio.core.sequence.io">ABITrace</a></dt>
<dd>
<div class="block">Returns the original programmatically determined (unedited) sequence as a <code>AbstractSequence<NucleotideCompound></code>.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#getSequence()" class="member-name-link">getSequence()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>
<div class="block">The Sequence Data Line</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/SequenceLocation.html#getSequence()" class="member-name-link">getSequence()</a> - Method in class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/SequenceLocation.html" title="class in org.biojava.nbio.core.sequence.location">SequenceLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FastaSequenceParser.html#getSequence(java.io.BufferedReader,int)" class="member-name-link">getSequence(BufferedReader, int)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FastaSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">FastaSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#getSequence(java.io.BufferedReader,int)" class="member-name-link">getSequence(BufferedReader, int)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/template/SequenceParserInterface.html#getSequence(java.io.BufferedReader,int)" class="member-name-link">getSequence(BufferedReader, int)</a> - Method in interface org.biojava.nbio.core.sequence.io.template.<a href="../org/biojava/nbio/core/sequence/io/template/SequenceParserInterface.html" title="interface in org.biojava.nbio.core.sequence.io.template">SequenceParserInterface</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#getSequence(java.lang.String,long)" class="member-name-link">getSequence(String, long)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">CasePreservingProteinSequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/DNASequenceCreator.html#getSequence(java.lang.String,long)" class="member-name-link">getSequence(String, long)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/DNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">DNASequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#getSequence(java.lang.String,long)" class="member-name-link">getSequence(String, long)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">FileProxyDNASequenceCreator</a></dt>
<dd>
<div class="block">Even though we are passing in the sequence we really only care about the length of the sequence and the offset
 index in the fasta file.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#getSequence(java.lang.String,long)" class="member-name-link">getSequence(String, long)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">FileProxyProteinSequenceCreator</a></dt>
<dd>
<div class="block">Even though we are passing in the sequence we really only care about the length of the sequence and the offset
 index in the fasta file.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FileProxyRNASequenceCreator.html#getSequence(java.lang.String,long)" class="member-name-link">getSequence(String, long)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FileProxyRNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">FileProxyRNASequenceCreator</a></dt>
<dd>
<div class="block">Even though we are passing in the sequence we really only care about the length of the sequence and the offset
 index in the fasta file.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html#getSequence(java.lang.String,long)" class="member-name-link">getSequence(String, long)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">ProteinSequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#getSequence(java.lang.String,long)" class="member-name-link">getSequence(String, long)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/RNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">RNASequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#getSequence(java.lang.String,long)" class="member-name-link">getSequence(String, long)</a> - Method in interface org.biojava.nbio.core.sequence.io.template.<a href="../org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html" title="interface in org.biojava.nbio.core.sequence.io.template">SequenceCreatorInterface</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#getSequence(java.util.List)" class="member-name-link">getSequence(List&lt;C&gt;)</a> - Method in interface org.biojava.nbio.core.sequence.io.template.<a href="../org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html" title="interface in org.biojava.nbio.core.sequence.io.template">SequenceCreatorInterface</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#getSequence(java.util.List)" class="member-name-link">getSequence(List&lt;AminoAcidCompound&gt;)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">CasePreservingProteinSequenceCreator</a></dt>
<dd>
<div class="block">Assumes all compounds were uppercase</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#getSequence(java.util.List)" class="member-name-link">getSequence(List&lt;AminoAcidCompound&gt;)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">FileProxyProteinSequenceCreator</a></dt>
<dd>
<div class="block">Not sure of use case and currently not supported</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html#getSequence(java.util.List)" class="member-name-link">getSequence(List&lt;AminoAcidCompound&gt;)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">ProteinSequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/DNASequenceCreator.html#getSequence(java.util.List)" class="member-name-link">getSequence(List&lt;NucleotideCompound&gt;)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/DNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">DNASequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#getSequence(java.util.List)" class="member-name-link">getSequence(List&lt;NucleotideCompound&gt;)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">FileProxyDNASequenceCreator</a></dt>
<dd>
<div class="block">Not sure of use case and currently not supported</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FileProxyRNASequenceCreator.html#getSequence(java.util.List)" class="member-name-link">getSequence(List&lt;NucleotideCompound&gt;)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FileProxyRNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">FileProxyRNASequenceCreator</a></dt>
<dd>
<div class="block">Not sure of use case and currently not supported</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#getSequence(java.util.List)" class="member-name-link">getSequence(List&lt;NucleotideCompound&gt;)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/RNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">RNASequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html#getSequence(org.biojava.nbio.core.sequence.template.ProxySequenceReader,long)" class="member-name-link">getSequence(ProxySequenceReader&lt;C&gt;, long)</a> - Method in interface org.biojava.nbio.core.sequence.io.template.<a href="../org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html" title="interface in org.biojava.nbio.core.sequence.io.template">SequenceCreatorInterface</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html#getSequence(org.biojava.nbio.core.sequence.template.ProxySequenceReader,long)" class="member-name-link">getSequence(ProxySequenceReader&lt;AminoAcidCompound&gt;, long)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">CasePreservingProteinSequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html#getSequence(org.biojava.nbio.core.sequence.template.ProxySequenceReader,long)" class="member-name-link">getSequence(ProxySequenceReader&lt;AminoAcidCompound&gt;, long)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">FileProxyProteinSequenceCreator</a></dt>
<dd>
<div class="block">Should be able to extend the same concept to a remote URL call or database connection.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html#getSequence(org.biojava.nbio.core.sequence.template.ProxySequenceReader,long)" class="member-name-link">getSequence(ProxySequenceReader&lt;AminoAcidCompound&gt;, long)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">ProteinSequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/DNASequenceCreator.html#getSequence(org.biojava.nbio.core.sequence.template.ProxySequenceReader,long)" class="member-name-link">getSequence(ProxySequenceReader&lt;NucleotideCompound&gt;, long)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/DNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">DNASequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html#getSequence(org.biojava.nbio.core.sequence.template.ProxySequenceReader,long)" class="member-name-link">getSequence(ProxySequenceReader&lt;NucleotideCompound&gt;, long)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">FileProxyDNASequenceCreator</a></dt>
<dd>
<div class="block">Should be able to extend the same concept to a remote URL call or database connection.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/FileProxyRNASequenceCreator.html#getSequence(org.biojava.nbio.core.sequence.template.ProxySequenceReader,long)" class="member-name-link">getSequence(ProxySequenceReader&lt;NucleotideCompound&gt;, long)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/FileProxyRNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">FileProxyRNASequenceCreator</a></dt>
<dd>
<div class="block">Should be able to extend the same concept to a remote URL call or database connection.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/RNASequenceCreator.html#getSequence(org.biojava.nbio.core.sequence.template.ProxySequenceReader,long)" class="member-name-link">getSequence(ProxySequenceReader&lt;NucleotideCompound&gt;, long)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/RNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">RNASequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/GeneSequence.html#getSequence5PrimeTo3Prime()" class="member-name-link">getSequence5PrimeTo3Prime()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Try to give method clarity where you want a DNASequence coding in the 5' to 3' direction
 Returns the DNASequence representative of the 5' and 3' reading based on strand</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#getSequenceAsString()" class="member-name-link">getSequenceAsString()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#getSequenceAsString()" class="member-name-link">getSequenceAsString()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#getSequenceAsString()" class="member-name-link">getSequenceAsString()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getSequenceAsString()" class="member-name-link">getSequenceAsString()</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#getSequenceAsString()" class="member-name-link">getSequenceAsString()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#getSequenceAsString()" class="member-name-link">getSequenceAsString()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Returns the sequence as a String</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#getSequenceAsString()" class="member-name-link">getSequenceAsString()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#getSequenceAsString()" class="member-name-link">getSequenceAsString()</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Delegates to <a href="../org/biojava/nbio/core/sequence/template/SequenceMixin.html#toList(org.biojava.nbio.core.sequence.template.Sequence)"><code>SequenceMixin.toList(org.biojava.nbio.core.sequence.template.Sequence)</code></a></div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getSequenceAsString()" class="member-name-link">getSequenceAsString()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Default case is to assume strand is positive because only CDSSequence can be either positive or negative Strand.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/Sequence.html#getSequenceAsString()" class="member-name-link">getSequenceAsString()</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Returns the String representation of the Sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html#getSequenceAsString()" class="member-name-link">getSequenceAsString()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/views/ComplementSequenceView.html#getSequenceAsString()" class="member-name-link">getSequenceAsString()</a> - Method in class org.biojava.nbio.core.sequence.views.<a href="../org/biojava/nbio/core/sequence/views/ComplementSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">ComplementSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/views/ReversedSequenceView.html#getSequenceAsString()" class="member-name-link">getSequenceAsString()</a> - Method in class org.biojava.nbio.core.sequence.views.<a href="../org/biojava/nbio/core/sequence/views/ReversedSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">ReversedSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/views/RnaSequenceView.html#getSequenceAsString()" class="member-name-link">getSequenceAsString()</a> - Method in class org.biojava.nbio.core.sequence.views.<a href="../org/biojava/nbio/core/sequence/views/RnaSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#getSequenceAsString(java.lang.Integer,java.lang.Integer,org.biojava.nbio.core.sequence.Strand)" class="member-name-link">getSequenceAsString(Integer, Integer, Strand)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#getSequenceAsString(java.lang.Integer,java.lang.Integer,org.biojava.nbio.core.sequence.Strand)" class="member-name-link">getSequenceAsString(Integer, Integer, Strand)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getSequenceAsString(java.lang.Integer,java.lang.Integer,org.biojava.nbio.core.sequence.Strand)" class="member-name-link">getSequenceAsString(Integer, Integer, Strand)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#getSequenceAsString(java.lang.Integer,java.lang.Integer,org.biojava.nbio.core.sequence.Strand)" class="member-name-link">getSequenceAsString(Integer, Integer, Strand)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getSequenceAsString(java.lang.Integer,java.lang.Integer,org.biojava.nbio.core.sequence.Strand)" class="member-name-link">getSequenceAsString(Integer, Integer, Strand)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html#getSequenceAsString(java.util.List,org.biojava.nbio.core.sequence.template.CompoundSet,java.lang.Integer,java.lang.Integer,org.biojava.nbio.core.sequence.Strand)" class="member-name-link">getSequenceAsString(List&lt;C&gt;, CompoundSet&lt;C&gt;, Integer, Integer, Strand)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html" title="class in org.biojava.nbio.core.sequence.storage">SequenceAsStringHelper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#getSequenceCollection()" class="member-name-link">getSequenceCollection()</a> - Static method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/SequenceOptimizationHints.html" title="class in org.biojava.nbio.core.sequence">SequenceOptimizationHints</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#getSequenceDescription()" class="member-name-link">getSequenceDescription()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>
<div class="block">The DE (Description) lines contain general descriptive information about the
 sequence stored.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#getSequenceFromAlignment()" class="member-name-link">getSequenceFromAlignment()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html#getSequenceFromAlignment()" class="member-name-link">getSequenceFromAlignment()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns the sequence positions at each alignment index</div>
</dd>
<dt><a href="../org/biojava/nbio/core/util/SequenceTools.html#getSequenceFromString(java.lang.String)" class="member-name-link">getSequenceFromString(String)</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/SequenceTools.html" title="class in org.biojava.nbio.core.util">SequenceTools</a></dt>
<dd>
<div class="block">Attempts to parse String as a DNA sequence first.<br/>
 If this fails it tries to  parse as a ProteinSequence.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html#getSequenceHeader()" class="member-name-link">getSequenceHeader()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></dt>
<dd>
<div class="block">The SQ (SeQuence header) line marks the beginning of the sequence data and
 Gives a summary of its content.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html#getSequenceHeaderParser()" class="member-name-link">getSequenceHeaderParser()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#getSequenceIndexAt(int)" class="member-name-link">getSequenceIndexAt(int)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html#getSequenceIndexAt(int)" class="member-name-link">getSequenceIndexAt(int)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns the index in the original <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a> corresponding to the given index within an alignment.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/ABITrace.html#getSequenceLength()" class="member-name-link">getSequenceLength()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/ABITrace.html" title="class in org.biojava.nbio.core.sequence.io">ABITrace</a></dt>
<dd>
<div class="block">Returns the length of the sequence (number of bases) in this trace.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblId.html#getSequenceLength()" class="member-name-link">getSequenceLength()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblId.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblId</a></dt>
<dd>
<div class="block">Sequence length The last item on the ID line is the length of the
 sequence (the total number of bases in the sequence).</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getSequenceScore()" class="member-name-link">getSequenceScore()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Provide place holder for a metric that indicate a score associated with the sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/SequenceOptimizationHints.html#getSequenceUsage()" class="member-name-link">getSequenceUsage()</a> - Static method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/SequenceOptimizationHints.html" title="class in org.biojava.nbio.core.sequence">SequenceOptimizationHints</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblId.html#getSequenceVersion()" class="member-name-link">getSequenceVersion()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblId.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblId</a></dt>
<dd>
<div class="block">return the sequence version</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html#getShortDescription()" class="member-name-link">getShortDescription()</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#getShortDescription()" class="member-name-link">getShortDescription()</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html#getShortDescription()" class="member-name-link">getShortDescription()</a> - Method in interface org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Get the short description that can be used to describe the feature</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html#getShortName()" class="member-name-link">getShortName()</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/NucleotideCompound.html#getShortName()" class="member-name-link">getShortName()</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound">NucleotideCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html#getShortName()" class="member-name-link">getShortName()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/Compound.html#getShortName()" class="member-name-link">getShortName()</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#getShortName()" class="member-name-link">getShortName()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html#getShortName()" class="member-name-link">getShortName()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html#getSize()" class="member-name-link">getSize()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.html#getSize()" class="member-name-link">getSize()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the number of rows in this profile.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html#getSize()" class="member-name-link">getSize()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava.nbio.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">Get the number of sequences in the MSA</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/LightweightProfile.html#getSize()" class="member-name-link">getSize()</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/LightweightProfile.html" title="interface in org.biojava.nbio.core.sequence.template">LightweightProfile</a></dt>
<dd>
<div class="block">Returns the number of rows in this profile.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html#getSource()" class="member-name-link">getSource()</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">The feature source</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#getSource()" class="member-name-link">getSource()</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html#getSource()" class="member-name-link">getSource()</a> - Method in interface org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">The source of the feature.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getSource()" class="member-name-link">getSource()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Added support for the source of this sequence for GFF3 export
 If a sub sequence doesn't have  source then check for parent source</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#getStart()" class="member-name-link">getStart()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html#getStart()" class="member-name-link">getStart()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns the <a href="../org/biojava/nbio/core/sequence/location/template/Point.html" title="interface in org.biojava.nbio.core.sequence.location.template"><code>Point</code></a> within an alignment of the first element of the original <a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a>.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/ProfileView.html#getStart()" class="member-name-link">getStart()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/ProfileView.html" title="interface in org.biojava.nbio.core.alignment.template">ProfileView</a></dt>
<dd>
<div class="block">Returns the column index of the viewed <a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template"><code>Profile</code></a> corresponding to the first element in this view</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html#getStart()" class="member-name-link">getStart()</a> - Method in class org.biojava.nbio.core.sequence.edits.<a href="../org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.AbstractEdit</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#getStart()" class="member-name-link">getStart()</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/Location.html#getStart()" class="member-name-link">getStart()</a> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/Location.html" title="interface in org.biojava.nbio.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Start of the location</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranscriptSequence.html#getStartCodonSequence()" class="member-name-link">getStartCodonSequence()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranscriptSequenceProduct.html#getStartCodonSequence()" class="member-name-link">getStartCodonSequence()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranscriptSequenceProduct.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequenceProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranscriptSequence.html#getStopCodonSequence()" class="member-name-link">getStopCodonSequence()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranscriptSequenceProduct.html#getStopCodonSequence()" class="member-name-link">getStopCodonSequence()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranscriptSequenceProduct.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequenceProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/CDSSequence.html#getStrand()" class="member-name-link">getStrand()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/CDSSequence.html" title="class in org.biojava.nbio.core.sequence">CDSSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/GeneSequence.html#getStrand()" class="member-name-link">getStrand()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">A gene should have Strand</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#getStrand()" class="member-name-link">getStrand()</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/Location.html#getStrand()" class="member-name-link">getStrand()</a> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/Location.html" title="interface in org.biojava.nbio.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Strand which the location is located on</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranscriptSequence.html#getStrand()" class="member-name-link">getStrand()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html#getStringForCompound(C)" class="member-name-link">getStringForCompound(C)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/CompoundSet.html#getStringForCompound(C)" class="member-name-link">getStringForCompound(C)</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html#getStringForCompound(org.biojava.nbio.core.sequence.compound.AminoAcidCompound)" class="member-name-link">getStringForCompound(AminoAcidCompound)</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/Strand.html#getStringRepresentation()" class="member-name-link">getStringRepresentation()</a> - Method in enum org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/Strand.html" title="enum in org.biojava.nbio.core.sequence">Strand</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#getSubLocations()" class="member-name-link">getSubLocations()</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/Location.html#getSubLocations()" class="member-name-link">getSubLocations()</a> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/Location.html" title="interface in org.biojava.nbio.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Gives access to the sub locations for this location.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfile.html#getSubProfile(org.biojava.nbio.core.sequence.location.template.Location)" class="member-name-link">getSubProfile(Location)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfile.html" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/Profile.html#getSubProfile(org.biojava.nbio.core.sequence.location.template.Location)" class="member-name-link">getSubProfile(Location)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns a <a href="../org/biojava/nbio/core/alignment/template/ProfileView.html" title="interface in org.biojava.nbio.core.alignment.template"><code>ProfileView</code></a> windowed to contain only the given <a href="../org/biojava/nbio/core/sequence/location/template/Location.html" title="interface in org.biojava.nbio.core.sequence.location.template"><code>Location</code></a>.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#getSubSequence(int,int)" class="member-name-link">getSubSequence(int, int)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Returns a sub sequence view</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html#getSubSequence(java.lang.Integer,java.lang.Integer)" class="member-name-link">getSubSequence(Integer, Integer)</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html#getSubSequence(java.lang.Integer,java.lang.Integer)" class="member-name-link">getSubSequence(Integer, Integer)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html#getSubSequence(java.lang.Integer,java.lang.Integer)" class="member-name-link">getSubSequence(Integer, Integer)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getSubSequence(java.lang.Integer,java.lang.Integer)" class="member-name-link">getSubSequence(Integer, Integer)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html#getSubSequence(java.lang.Integer,java.lang.Integer)" class="member-name-link">getSubSequence(Integer, Integer)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html#getSubSequence(java.lang.Integer,java.lang.Integer)" class="member-name-link">getSubSequence(Integer, Integer)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html#getSubSequence(java.lang.Integer,java.lang.Integer)" class="member-name-link">getSubSequence(Integer, Integer)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html#getSubSequence(java.lang.Integer,java.lang.Integer)" class="member-name-link">getSubSequence(Integer, Integer)</a> - Method in class org.biojava.nbio.core.sequence.storage.<a href="../org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Creates a <a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template"><code>SequenceProxyView</code></a> for the given coordinates</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getSubSequence(java.lang.Integer,java.lang.Integer)" class="member-name-link">getSubSequence(Integer, Integer)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/Sequence.html#getSubSequence(java.lang.Integer,java.lang.Integer)" class="member-name-link">getSubSequence(Integer, Integer)</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Returns a portion of the sequence from the different positions.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html#getSubSequence(java.lang.Integer,java.lang.Integer)" class="member-name-link">getSubSequence(Integer, Integer)</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html#getSubSequence(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">getSubSequence(Sequence&lt;C&gt;)</a> - Method in class org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></dt>
<dd>
<div class="block">If circular this will return the sequence represented by the sub
 locations joined.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/template/Location.html#getSubSequence(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">getSubSequence(Sequence&lt;C&gt;)</a> - Method in interface org.biojava.nbio.core.sequence.location.template.<a href="../org/biojava/nbio/core/sequence/location/template/Location.html" title="interface in org.biojava.nbio.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Will return a SequenceReader object which represents the outer bounds
 of this Location</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html#getTable()" class="member-name-link">getTable()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACParser.html#getTable(java.lang.Integer)" class="member-name-link">getTable(Integer)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/IUPACParser.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser</a></dt>
<dd>
<div class="block">Returns a table by its identifier i.e. 1 means universal codon tables</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACParser.html#getTable(java.lang.String)" class="member-name-link">getTable(String)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/IUPACParser.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser</a></dt>
<dd>
<div class="block">Returns a table by its name</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACParserProduct2.html#getTable(java.lang.String,org.biojava.nbio.core.sequence.io.IUPACParser)" class="member-name-link">getTable(String, IUPACParser)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/IUPACParserProduct2.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParserProduct2</a></dt>
<dd>
<div class="block">Returns a table by its name</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACParser.html#getTables()" class="member-name-link">getTables()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/IUPACParser.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser</a></dt>
<dd>
<div class="block">Returns a list of all available IUPAC tables</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/IUPACParserProduct.html#getTables()" class="member-name-link">getTables()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/IUPACParserProduct.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParserProduct</a></dt>
<dd>
<div class="block">Returns a list of all available IUPAC tables</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleProfilePair.html#getTarget()" class="member-name-link">getTarget()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleProfilePair.html" title="class in org.biojava.nbio.core.alignment">SimpleProfilePair</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html#getTarget()" class="member-name-link">getTarget()</a> - Method in class org.biojava.nbio.core.alignment.<a href="../org/biojava/nbio/core/alignment/SimpleSequencePair.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/ProfilePair.html#getTarget()" class="member-name-link">getTarget()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/ProfilePair.html" title="interface in org.biojava.nbio.core.alignment.template">ProfilePair</a></dt>
<dd>
<div class="block">Returns the second <a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template"><code>Profile</code></a> of the pair.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/SequencePair.html#getTarget()" class="member-name-link">getTarget()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/SequencePair.html" title="interface in org.biojava.nbio.core.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the second <a href="../org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template"><code>AlignedSequence</code></a> of the pair.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html#getTargetSequence(org.biojava.nbio.core.sequence.template.Sequence)" class="member-name-link">getTargetSequence(Sequence&lt;C&gt;)</a> - Method in class org.biojava.nbio.core.sequence.edits.<a href="../org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.AbstractEdit</a></dt>
<dd>
<div class="block">Returns the Sequence which is our edit.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblId.html#getTaxonomicDivision()" class="member-name-link">getTaxonomicDivision()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblId.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblId</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getTaxonomy()" class="member-name-link">getTaxonomy()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/ConcurrencyTools.html#getThreadPool()" class="member-name-link">getThreadPool()</a> - Static method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/ConcurrencyTools.html" title="class in org.biojava.nbio.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Returns current shared thread pool.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html#getThree()" class="member-name-link">getThree()</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html#getThree()" class="member-name-link">getThree()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.CaseInsensitiveTriplet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html#getThree()" class="member-name-link">getThree()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/reference/AbstractReference.html#getTitle()" class="member-name-link">getTitle()</a> - Method in class org.biojava.nbio.core.sequence.reference.<a href="../org/biojava/nbio/core/sequence/reference/AbstractReference.html" title="class in org.biojava.nbio.core.sequence.reference">AbstractReference</a></dt>
<dd>
<div class="block">The title that retrieved from the Reference section.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/reference/GenbankReference.html#getTitle()" class="member-name-link">getTitle()</a> - Method in class org.biojava.nbio.core.sequence.reference.<a href="../org/biojava/nbio/core/sequence/reference/GenbankReference.html" title="class in org.biojava.nbio.core.sequence.reference">GenbankReference</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/reference/ReferenceInterface.html#getTitle()" class="member-name-link">getTitle()</a> - Method in interface org.biojava.nbio.core.sequence.reference.<a href="../org/biojava/nbio/core/sequence/reference/ReferenceInterface.html" title="interface in org.biojava.nbio.core.sequence.reference">ReferenceInterface</a></dt>
<dd>
<div class="block">Get the title that retrieved from Reference section.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html#getToCompoundSet()" class="member-name-link">getToCompoundSet()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblId.html#getTopology()" class="member-name-link">getTopology()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblId.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblId</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/ABITrace.html#getTrace(java.lang.String)" class="member-name-link">getTrace(String)</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/ABITrace.html" title="class in org.biojava.nbio.core.sequence.io">ABITrace</a></dt>
<dd>
<div class="block">Returns one of the four traces - all of the y-coordinate values,
 each of which correspond to a single x-coordinate relative to the
 position in the array, so that if element 4 in the array is 972, then
 x is 4 and y is 972 for that point.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/ABITrace.html#getTraceLength()" class="member-name-link">getTraceLength()</a> - Method in class org.biojava.nbio.core.sequence.io.<a href="../org/biojava/nbio/core/sequence/io/ABITrace.html" title="class in org.biojava.nbio.core.sequence.io">ABITrace</a></dt>
<dd>
<div class="block">Returns the length of the trace (number of x-coordinate points in the graph).</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/GeneSequence.html#getTranscript(java.lang.String)" class="member-name-link">getTranscript(String)</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Get the transcript sequence by accession</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/GeneSequence.html#getTranscripts()" class="member-name-link">getTranscripts()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Get the collection of transcription sequences assigned to this gene</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html#getTriplet()" class="member-name-link">getTriplet()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html#getTwo()" class="member-name-link">getTwo()</a> - Method in class org.biojava.nbio.core.sequence.compound.<a href="../org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html#getTwo()" class="member-name-link">getTwo()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.CaseInsensitiveTriplet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html#getTwo()" class="member-name-link">getTwo()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html#getType()" class="member-name-link">getType()</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">The feature type</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#getType()" class="member-name-link">getType()</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html#getType()" class="member-name-link">getType()</a> - Method in interface org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">The type of the feature.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html#getUnderlyingCompound()" class="member-name-link">getUnderlyingCompound()</a> - Method in class org.biojava.nbio.core.sequence.transcription.<a href="../org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getUniprotbaseURL()" class="member-name-link">getUniprotbaseURL()</a> - Static method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">The current UniProt URL to deal with caching issues. www.uniprot.org is load balanced
 but you can access pir.uniprot.org directly.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getUniprotDirectoryCache()" class="member-name-link">getUniprotDirectoryCache()</a> - Static method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">Local directory cache of XML that can be downloaded</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html#getUniprotXML(java.lang.String)" class="member-name-link">getUniprotXML(String)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReaderProduct.html#getUniprotXML(java.lang.String)" class="member-name-link">getUniprotXML(String)</a> - Method in class org.biojava.nbio.core.sequence.loader.<a href="../org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReaderProduct.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReaderProduct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/TranslationTest.html#getUniversal()" class="member-name-link">getUniversal()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/TranslationTest.html" title="class in org.biojava.nbio.core.sequence">TranslationTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html#getUpperedBase()" class="member-name-link">getUpperedBase()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html#getUserCollection()" class="member-name-link">getUserCollection()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Provided for convince if the developer needs to associate data with a sequence</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html#getUserObject()" class="member-name-link">getUserObject()</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html#getUserObject()" class="member-name-link">getUserObject()</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html#getUserObject()" class="member-name-link">getUserObject()</a> - Method in interface org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/features/Qualifier.html#getValue()" class="member-name-link">getValue()</a> - Method in class org.biojava.nbio.core.sequence.features.<a href="../org/biojava/nbio/core/sequence/features/Qualifier.html" title="class in org.biojava.nbio.core.sequence.features">Qualifier</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/CRC64Checksum.html#getValue()" class="member-name-link">getValue()</a> - Method in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/CRC64Checksum.html" title="class in org.biojava.nbio.core.util">CRC64Checksum</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html#getValue(C,C)" class="member-name-link">getValue(C, C)</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/SubstitutionMatrix.html#getValue(C,C)" class="member-name-link">getValue(C, C)</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava.nbio.core.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns value in matrix for conversion from first <a href="../org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Compound</code></a> to the second.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html#getValue(org.biojava.nbio.core.sequence.compound.AminoAcidCompound,org.biojava.nbio.core.sequence.compound.AminoAcidCompound)" class="member-name-link">getValue(AminoAcidCompound, AminoAcidCompound)</a> - Method in class org.biojava.nbio.core.alignment.matrices.<a href="../org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/search/io/Result.html#getVersion()" class="member-name-link">getVersion()</a> - Method in class org.biojava.nbio.core.search.io.<a href="../org/biojava/nbio/core/search/io/Result.html" title="class in org.biojava.nbio.core.search.io">Result</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/AccessionID.html#getVersion()" class="member-name-link">getVersion()</a> - Method in class org.biojava.nbio.core.sequence.<a href="../org/biojava/nbio/core/sequence/AccessionID.html" title="class in org.biojava.nbio.core.sequence">AccessionID</a></dt>
<dd>
<div class="block">In case if the <a href="../org/biojava/nbio/core/sequence/AccessionID.html#getID()"><code>AccessionID.getID()</code></a> is not unique keeps the id version.</div>
</dd>
<dt><a href="../org/biojava/nbio/core/alignment/template/ProfileView.html#getViewedProfile()" class="member-name-link">getViewedProfile()</a> - Method in interface org.biojava.nbio.core.alignment.template.<a href="../org/biojava/nbio/core/alignment/template/ProfileView.html" title="interface in org.biojava.nbio.core.alignment.template">ProfileView</a></dt>
<dd>
<div class="block">Returns the entire <a href="../org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template"><code>Profile</code></a> being viewed</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html#getViewedSequence()" class="member-name-link">getViewedSequence()</a> - Method in class org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/template/SequenceView.html#getViewedSequence()" class="member-name-link">getViewedSequence()</a> - Method in interface org.biojava.nbio.core.sequence.template.<a href="../org/biojava/nbio/core/sequence/template/SequenceView.html" title="interface in org.biojava.nbio.core.sequence.template">SequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/views/WindowedSequence.html#getWindowSize()" class="member-name-link">getWindowSize()</a> - Method in class org.biojava.nbio.core.sequence.views.<a href="../org/biojava/nbio/core/sequence/views/WindowedSequence.html" title="class in org.biojava.nbio.core.sequence.views">WindowedSequence</a></dt>
<dd>
<div class="block">Access the current window size</div>
</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblReaderTest.html#givenAnEmilFileWhenProcessEmilFileThanTheSequenceShouldReturnAsExpected()" class="member-name-link">givenAnEmilFileWhenProcessEmilFileThanTheSequenceShouldReturnAsExpected()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblReaderTest.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReaderTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblReaderTest.html#givenDirectoryWhenProcessEmblFileThenThrowException()" class="member-name-link">givenDirectoryWhenProcessEmblFileThenThrowException()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblReaderTest.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReaderTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/io/embl/EmblReaderTest.html#givenNullFileParameterWhenProcessEmblFileThenThrowException()" class="member-name-link">givenNullFileParameterWhenProcessEmblFileThenThrowException()</a> - Method in class org.biojava.nbio.core.sequence.io.embl.<a href="../org/biojava/nbio/core/sequence/io/embl/EmblReaderTest.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReaderTest</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/UncompressInputStreamData.html#got" class="member-name-link">got</a> - Variable in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/UncompressInputStreamData.html" title="class in org.biojava.nbio.core.util">UncompressInputStreamData</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html#%3Cinit%3E(boolean,org.biojava.nbio.core.sequence.location.template.Location...)" class="member-name-link">GroupLocation(boolean, Location...)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.GroupLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html#%3Cinit%3E(int,int,org.biojava.nbio.core.sequence.Strand,java.util.List)" class="member-name-link">GroupLocation(int, int, Strand, List&lt;Location&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.GroupLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html#%3Cinit%3E(int,int,org.biojava.nbio.core.sequence.Strand,org.biojava.nbio.core.sequence.location.template.Location...)" class="member-name-link">GroupLocation(int, int, Strand, Location...)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.GroupLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html#%3Cinit%3E(java.util.List)" class="member-name-link">GroupLocation(List&lt;Location&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.GroupLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html#%3Cinit%3E(org.biojava.nbio.core.sequence.location.template.Location...)" class="member-name-link">GroupLocation(Location...)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.GroupLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html#%3Cinit%3E(org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.Strand,boolean,java.util.List)" class="member-name-link">GroupLocation(Point, Point, Strand, boolean, List&lt;Location&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.GroupLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html#%3Cinit%3E(org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.Strand,boolean,org.biojava.nbio.core.sequence.location.template.Location...)" class="member-name-link">GroupLocation(Point, Point, Strand, boolean, Location...)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.GroupLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html#%3Cinit%3E(org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.Strand,java.util.List)" class="member-name-link">GroupLocation(Point, Point, Strand, List&lt;Location&gt;)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.GroupLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html#%3Cinit%3E(org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.location.template.Point,org.biojava.nbio.core.sequence.Strand,org.biojava.nbio.core.sequence.location.template.Location...)" class="member-name-link">GroupLocation(Point, Point, Strand, Location...)</a> - Constructor for class org.biojava.nbio.core.sequence.location.<a href="../org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.GroupLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../org/biojava/nbio/core/util/InputStreamProvider.html#GZIP_MAGIC" class="member-name-link">GZIP_MAGIC</a> - Static variable in class org.biojava.nbio.core.util.<a href="../org/biojava/nbio/core/util/InputStreamProvider.html" title="class in org.biojava.nbio.core.util">InputStreamProvider</a></dt>
<dd>
<div class="block">The magic number found at the start of a GZIP stream.</div>
</dd>
</dl>
<a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Z</a>&nbsp;<br><a href="../allclasses-index.html">All&nbsp;Classes&nbsp;and&nbsp;Interfaces</a><span class="vertical-separator">|</span><a href="../allpackages-index.html">All&nbsp;Packages</a><span class="vertical-separator">|</span><a href="../constant-values.html">Constant&nbsp;Field&nbsp;Values</a><span class="vertical-separator">|</span><a href="../serialized-form.html">Serialized&nbsp;Form</a></main>
</div>
</div>
</body>
</html>
