<!DOCTYPE HTML>
<html lang="pt">
<head>
<!-- Generated by javadoc (17) on Sun May 15 18:36:12 WEST 2022 -->
<title>TranscriptSequenceProduct</title>
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta name="dc.created" content="2022-05-15">
<meta name="description" content="declaration: package: org.biojava.nbio.core.sequence, class: TranscriptSequenceProduct">
<meta name="generator" content="javadoc/ClassWriterImpl">
<link rel="stylesheet" type="text/css" href="../../../../../stylesheet.css" title="Style">
<link rel="stylesheet" type="text/css" href="../../../../../script-dir/jquery-ui.min.css" title="Style">
<link rel="stylesheet" type="text/css" href="../../../../../jquery-ui.overrides.css" title="Style">
<script type="text/javascript" src="../../../../../script.js"></script>
<script type="text/javascript" src="../../../../../script-dir/jquery-3.5.1.min.js"></script>
<script type="text/javascript" src="../../../../../script-dir/jquery-ui.min.js"></script>
</head>
<body class="class-declaration-page">
<script type="text/javascript">var evenRowColor = "even-row-color";
var oddRowColor = "odd-row-color";
var tableTab = "table-tab";
var activeTableTab = "active-table-tab";
var pathtoroot = "../../../../../";
loadScripts(document, 'script');</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<div class="flex-box">
<header role="banner" class="flex-header">
<nav role="navigation">
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="top-nav" id="navbar-top">
<div class="skip-nav"><a href="#skip-navbar-top" title="Skip navigation links">Skip navigation links</a></div>
<ul id="navbar-top-firstrow" class="nav-list" title="Navigation">
<li><a href="../../../../../index.html">Overview</a></li>
<li><a href="package-summary.html">Package</a></li>
<li class="nav-bar-cell1-rev">Class</li>
<li><a href="class-use/TranscriptSequenceProduct.html">Use</a></li>
<li><a href="package-tree.html">Tree</a></li>
<li><a href="../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../index-files/index-1.html">Index</a></li>
<li><a href="../../../../../help-doc.html#class">Help</a></li>
</ul>
</div>
<div class="sub-nav">
<div>
<ul class="sub-nav-list">
<li>Summary:&nbsp;</li>
<li>Nested&nbsp;|&nbsp;</li>
<li>Field&nbsp;|&nbsp;</li>
<li><a href="#constructor-summary">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method-summary">Method</a></li>
</ul>
<ul class="sub-nav-list">
<li>Detail:&nbsp;</li>
<li>Field&nbsp;|&nbsp;</li>
<li><a href="#constructor-detail">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method-detail">Method</a></li>
</ul>
</div>
<div class="nav-list-search"><label for="search-input">SEARCH:</label>
<input type="text" id="search-input" value="search" disabled="disabled">
<input type="reset" id="reset-button" value="reset" disabled="disabled">
</div>
</div>
<!-- ========= END OF TOP NAVBAR ========= -->
<span class="skip-nav" id="skip-navbar-top"></span></nav>
</header>
<div class="flex-content">
<main role="main">
<!-- ======== START OF CLASS DATA ======== -->
<div class="header">
<div class="sub-title"><span class="package-label-in-type">Package</span>&nbsp;<a href="package-summary.html">org.biojava.nbio.core.sequence</a></div>
<h1 title="Class TranscriptSequenceProduct" class="title">Class TranscriptSequenceProduct</h1>
</div>
<div class="inheritance" title="Inheritance Tree"><a href="https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html" title="class or interface in java.lang" class="external-link">java.lang.Object</a>
<div class="inheritance">org.biojava.nbio.core.sequence.TranscriptSequenceProduct</div>
</div>
<section class="class-description" id="class-description">
<hr>
<div class="type-signature"><span class="modifiers">public class </span><span class="element-name type-name-label">TranscriptSequenceProduct</span>
<span class="extends-implements">extends <a href="https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html" title="class or interface in java.lang" class="external-link">Object</a></span></div>
</section>
<section class="summary">
<ul class="summary-list">
<!-- ======== CONSTRUCTOR SUMMARY ======== -->
<li>
<section class="constructor-summary" id="constructor-summary">
<h2>Constructor Summary</h2>
<div class="caption"><span>Constructors</span></div>
<div class="summary-table two-column-summary">
<div class="table-header col-first">Constructor</div>
<div class="table-header col-last">Description</div>
<div class="col-constructor-name even-row-color"><code><a href="#%3Cinit%3E()" class="member-name-link">TranscriptSequenceProduct</a>()</code></div>
<div class="col-last even-row-color">&nbsp;</div>
</div>
</section>
</li>
<!-- ========== METHOD SUMMARY =========== -->
<li>
<section class="method-summary" id="method-summary">
<h2>Method Summary</h2>
<div id="method-summary-table">
<div class="table-tabs" role="tablist" aria-orientation="horizontal"><button id="method-summary-table-tab0" role="tab" aria-selected="true" aria-controls="method-summary-table.tabpanel" tabindex="0" onkeydown="switchTab(event)" onclick="show('method-summary-table', 'method-summary-table', 3)" class="active-table-tab">All Methods</button><button id="method-summary-table-tab2" role="tab" aria-selected="false" aria-controls="method-summary-table.tabpanel" tabindex="-1" onkeydown="switchTab(event)" onclick="show('method-summary-table', 'method-summary-table-tab2', 3)" class="table-tab">Instance Methods</button><button id="method-summary-table-tab4" role="tab" aria-selected="false" aria-controls="method-summary-table.tabpanel" tabindex="-1" onkeydown="switchTab(event)" onclick="show('method-summary-table', 'method-summary-table-tab4', 3)" class="table-tab">Concrete Methods</button></div>
<div id="method-summary-table.tabpanel" role="tabpanel">
<div class="summary-table three-column-summary" aria-labelledby="method-summary-table-tab0">
<div class="table-header col-first">Modifier and Type</div>
<div class="table-header col-second">Method</div>
<div class="table-header col-last">Description</div>
<div class="col-first even-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4"><code><a href="CDSSequence.html" title="class in org.biojava.nbio.core.sequence">CDSSequence</a></code></div>
<div class="col-second even-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4"><code><a href="#addCDS(org.biojava.nbio.core.sequence.AccessionID,int,int,int,org.biojava.nbio.core.sequence.TranscriptSequence)" class="member-name-link">addCDS</a><wbr>(<a href="AccessionID.html" title="class in org.biojava.nbio.core.sequence">AccessionID</a>&nbsp;accession,
 int&nbsp;begin,
 int&nbsp;end,
 int&nbsp;phase,
 <a href="TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a>&nbsp;transcriptSequence)</code></div>
<div class="col-last even-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4">
<div class="block">Add a Coding Sequence region with phase to the transcript sequence</div>
</div>
<div class="col-first odd-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4"><code>void</code></div>
<div class="col-second odd-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4"><code><a href="#addStartCodonSequence(org.biojava.nbio.core.sequence.AccessionID,int,int,org.biojava.nbio.core.sequence.TranscriptSequence)" class="member-name-link">addStartCodonSequence</a><wbr>(<a href="AccessionID.html" title="class in org.biojava.nbio.core.sequence">AccessionID</a>&nbsp;accession,
 int&nbsp;begin,
 int&nbsp;end,
 <a href="TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a>&nbsp;transcriptSequence)</code></div>
<div class="col-last odd-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4">
<div class="block">Sets the start codon sequence at given begin /  end location.</div>
</div>
<div class="col-first even-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4"><code>void</code></div>
<div class="col-second even-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4"><code><a href="#addStopCodonSequence(org.biojava.nbio.core.sequence.AccessionID,int,int,org.biojava.nbio.core.sequence.TranscriptSequence)" class="member-name-link">addStopCodonSequence</a><wbr>(<a href="AccessionID.html" title="class in org.biojava.nbio.core.sequence">AccessionID</a>&nbsp;accession,
 int&nbsp;begin,
 int&nbsp;end,
 <a href="TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a>&nbsp;transcriptSequence)</code></div>
<div class="col-last even-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4">
<div class="block">Sets the stop codon sequence at given begin /  end location.</div>
</div>
<div class="col-first odd-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4"><code><a href="https://docs.oracle.com/javase/8/docs/api/java/util/LinkedHashMap.html" title="class or interface in java.util" class="external-link">LinkedHashMap</a>&lt;<a href="https://docs.oracle.com/javase/8/docs/api/java/lang/String.html" title="class or interface in java.lang" class="external-link">String</a>,<wbr><a href="CDSSequence.html" title="class in org.biojava.nbio.core.sequence">CDSSequence</a>&gt;</code></div>
<div class="col-second odd-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4"><code><a href="#getCdsSequenceHashMap()" class="member-name-link">getCdsSequenceHashMap</a>()</code></div>
<div class="col-last odd-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4">&nbsp;</div>
<div class="col-first even-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4"><code><a href="DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></code></div>
<div class="col-second even-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4"><code><a href="#getDNACodingSequence(org.biojava.nbio.core.sequence.TranscriptSequence)" class="member-name-link">getDNACodingSequence</a><wbr>(<a href="TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a>&nbsp;transcriptSequence)</code></div>
<div class="col-last even-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4">
<div class="block">Get the stitched together CDS sequences then maps to the cDNA</div>
</div>
<div class="col-first odd-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4"><code><a href="https://docs.oracle.com/javase/8/docs/api/java/util/ArrayList.html" title="class or interface in java.util" class="external-link">ArrayList</a>&lt;<a href="ProteinSequence.html" title="class in org.biojava.nbio.core.sequence">ProteinSequence</a>&gt;</code></div>
<div class="col-second odd-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4"><code><a href="#getProteinCDSSequences(org.biojava.nbio.core.sequence.TranscriptSequence)" class="member-name-link">getProteinCDSSequences</a><wbr>(<a href="TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a>&nbsp;transcriptSequence)</code></div>
<div class="col-last odd-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4">
<div class="block">Return a list of protein sequences based on each CDS sequence where the phase shift between two CDS sequences is assigned to the CDS sequence that starts the triplet.</div>
</div>
<div class="col-first even-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4"><code><a href="ProteinSequence.html" title="class in org.biojava.nbio.core.sequence">ProteinSequence</a></code></div>
<div class="col-second even-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4"><code><a href="#getProteinSequence(org.biojava.nbio.core.sequence.transcription.TranscriptionEngine,org.biojava.nbio.core.sequence.TranscriptSequence)" class="member-name-link">getProteinSequence</a><wbr>(<a href="transcription/TranscriptionEngine.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine</a>&nbsp;engine,
 <a href="TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a>&nbsp;transcriptSequence)</code></div>
<div class="col-last even-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4">
<div class="block">Get the protein sequence with user defined TranscriptEngine</div>
</div>
<div class="col-first odd-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4"><code><a href="StartCodonSequence.html" title="class in org.biojava.nbio.core.sequence">StartCodonSequence</a></code></div>
<div class="col-second odd-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4"><code><a href="#getStartCodonSequence()" class="member-name-link">getStartCodonSequence</a>()</code></div>
<div class="col-last odd-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4">&nbsp;</div>
<div class="col-first even-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4"><code><a href="StopCodonSequence.html" title="class in org.biojava.nbio.core.sequence">StopCodonSequence</a></code></div>
<div class="col-second even-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4"><code><a href="#getStopCodonSequence()" class="member-name-link">getStopCodonSequence</a>()</code></div>
<div class="col-last even-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4">&nbsp;</div>
<div class="col-first odd-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4"><code><a href="CDSSequence.html" title="class in org.biojava.nbio.core.sequence">CDSSequence</a></code></div>
<div class="col-second odd-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4"><code><a href="#removeCDS(java.lang.String)" class="member-name-link">removeCDS</a><wbr>(<a href="https://docs.oracle.com/javase/8/docs/api/java/lang/String.html" title="class or interface in java.lang" class="external-link">String</a>&nbsp;accession)</code></div>
<div class="col-last odd-row-color method-summary-table method-summary-table-tab2 method-summary-table-tab4">
<div class="block">Remove a CDS or coding sequence from the transcript sequence</div>
</div>
</div>
</div>
</div>
<div class="inherited-list">
<h3 id="methods-inherited-from-class-java.lang.Object">Methods inherited from class&nbsp;java.lang.<a href="https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html" title="class or interface in java.lang" class="external-link">Object</a></h3>
<code><a href="https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#equals(java.lang.Object)" title="class or interface in java.lang" class="external-link">equals</a>, <a href="https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#getClass()" title="class or interface in java.lang" class="external-link">getClass</a>, <a href="https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#hashCode()" title="class or interface in java.lang" class="external-link">hashCode</a>, <a href="https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#notify()" title="class or interface in java.lang" class="external-link">notify</a>, <a href="https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#notifyAll()" title="class or interface in java.lang" class="external-link">notifyAll</a>, <a href="https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#toString()" title="class or interface in java.lang" class="external-link">toString</a>, <a href="https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#wait()" title="class or interface in java.lang" class="external-link">wait</a>, <a href="https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#wait(long)" title="class or interface in java.lang" class="external-link">wait</a>, <a href="https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#wait(long,int)" title="class or interface in java.lang" class="external-link">wait</a></code></div>
</section>
</li>
</ul>
</section>
<section class="details">
<ul class="details-list">
<!-- ========= CONSTRUCTOR DETAIL ======== -->
<li>
<section class="constructor-details" id="constructor-detail">
<h2>Constructor Details</h2>
<ul class="member-list">
<li>
<section class="detail" id="&lt;init&gt;()">
<h3>TranscriptSequenceProduct</h3>
<div class="member-signature"><span class="modifiers">public</span>&nbsp;<span class="element-name">TranscriptSequenceProduct</span>()</div>
</section>
</li>
</ul>
</section>
</li>
<!-- ============ METHOD DETAIL ========== -->
<li>
<section class="method-details" id="method-detail">
<h2>Method Details</h2>
<ul class="member-list">
<li>
<section class="detail" id="getCdsSequenceHashMap()">
<h3>getCdsSequenceHashMap</h3>
<div class="member-signature"><span class="modifiers">public</span>&nbsp;<span class="return-type"><a href="https://docs.oracle.com/javase/8/docs/api/java/util/LinkedHashMap.html" title="class or interface in java.util" class="external-link">LinkedHashMap</a>&lt;<a href="https://docs.oracle.com/javase/8/docs/api/java/lang/String.html" title="class or interface in java.lang" class="external-link">String</a>,<wbr><a href="CDSSequence.html" title="class in org.biojava.nbio.core.sequence">CDSSequence</a>&gt;</span>&nbsp;<span class="element-name">getCdsSequenceHashMap</span>()</div>
</section>
</li>
<li>
<section class="detail" id="getStartCodonSequence()">
<h3>getStartCodonSequence</h3>
<div class="member-signature"><span class="modifiers">public</span>&nbsp;<span class="return-type"><a href="StartCodonSequence.html" title="class in org.biojava.nbio.core.sequence">StartCodonSequence</a></span>&nbsp;<span class="element-name">getStartCodonSequence</span>()</div>
</section>
</li>
<li>
<section class="detail" id="getStopCodonSequence()">
<h3>getStopCodonSequence</h3>
<div class="member-signature"><span class="modifiers">public</span>&nbsp;<span class="return-type"><a href="StopCodonSequence.html" title="class in org.biojava.nbio.core.sequence">StopCodonSequence</a></span>&nbsp;<span class="element-name">getStopCodonSequence</span>()</div>
</section>
</li>
<li>
<section class="detail" id="removeCDS(java.lang.String)">
<h3>removeCDS</h3>
<div class="member-signature"><span class="modifiers">public</span>&nbsp;<span class="return-type"><a href="CDSSequence.html" title="class in org.biojava.nbio.core.sequence">CDSSequence</a></span>&nbsp;<span class="element-name">removeCDS</span><wbr><span class="parameters">(<a href="https://docs.oracle.com/javase/8/docs/api/java/lang/String.html" title="class or interface in java.lang" class="external-link">String</a>&nbsp;accession)</span></div>
<div class="block">Remove a CDS or coding sequence from the transcript sequence</div>
<dl class="notes">
<dt>Parameters:</dt>
<dd><code>accession</code> - </dd>
<dt>Returns:</dt>
</dl>
</section>
</li>
<li>
<section class="detail" id="addCDS(org.biojava.nbio.core.sequence.AccessionID,int,int,int,org.biojava.nbio.core.sequence.TranscriptSequence)">
<h3>addCDS</h3>
<div class="member-signature"><span class="modifiers">public</span>&nbsp;<span class="return-type"><a href="CDSSequence.html" title="class in org.biojava.nbio.core.sequence">CDSSequence</a></span>&nbsp;<span class="element-name">addCDS</span><wbr><span class="parameters">(<a href="AccessionID.html" title="class in org.biojava.nbio.core.sequence">AccessionID</a>&nbsp;accession,
 int&nbsp;begin,
 int&nbsp;end,
 int&nbsp;phase,
 <a href="TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a>&nbsp;transcriptSequence)</span>
                   throws <span class="exceptions"><a href="https://docs.oracle.com/javase/8/docs/api/java/lang/Exception.html" title="class or interface in java.lang" class="external-link">Exception</a></span></div>
<div class="block">Add a Coding Sequence region with phase to the transcript sequence</div>
<dl class="notes">
<dt>Parameters:</dt>
<dd><code>accession</code> - </dd>
<dd><code>begin</code> - </dd>
<dd><code>end</code> - </dd>
<dd><code>phase</code> - 0,1,2</dd>
<dt>Returns:</dt>
<dt>Throws:</dt>
<dd><code><a href="https://docs.oracle.com/javase/8/docs/api/java/lang/Exception.html" title="class or interface in java.lang" class="external-link">Exception</a></code></dd>
</dl>
</section>
</li>
<li>
<section class="detail" id="getProteinCDSSequences(org.biojava.nbio.core.sequence.TranscriptSequence)">
<h3>getProteinCDSSequences</h3>
<div class="member-signature"><span class="modifiers">public</span>&nbsp;<span class="return-type"><a href="https://docs.oracle.com/javase/8/docs/api/java/util/ArrayList.html" title="class or interface in java.util" class="external-link">ArrayList</a>&lt;<a href="ProteinSequence.html" title="class in org.biojava.nbio.core.sequence">ProteinSequence</a>&gt;</span>&nbsp;<span class="element-name">getProteinCDSSequences</span><wbr><span class="parameters">(<a href="TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a>&nbsp;transcriptSequence)</span></div>
<div class="block">Return a list of protein sequences based on each CDS sequence where the phase shift between two CDS sequences is assigned to the CDS sequence that starts the triplet. This can be used to map a CDS/exon region of a protein sequence back to the DNA sequence If you have a protein sequence and a predicted gene you can take the predict CDS protein sequences and align back to the protein sequence. If you have errors in mapping the predicted protein CDS regions to an the known protein sequence then you can identify possible errors in the prediction</div>
<dl class="notes">
<dt>Returns:</dt>
</dl>
</section>
</li>
<li>
<section class="detail" id="getDNACodingSequence(org.biojava.nbio.core.sequence.TranscriptSequence)">
<h3>getDNACodingSequence</h3>
<div class="member-signature"><span class="modifiers">public</span>&nbsp;<span class="return-type"><a href="DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></span>&nbsp;<span class="element-name">getDNACodingSequence</span><wbr><span class="parameters">(<a href="TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a>&nbsp;transcriptSequence)</span></div>
<div class="block">Get the stitched together CDS sequences then maps to the cDNA</div>
<dl class="notes">
<dt>Returns:</dt>
</dl>
</section>
</li>
<li>
<section class="detail" id="getProteinSequence(org.biojava.nbio.core.sequence.transcription.TranscriptionEngine,org.biojava.nbio.core.sequence.TranscriptSequence)">
<h3>getProteinSequence</h3>
<div class="member-signature"><span class="modifiers">public</span>&nbsp;<span class="return-type"><a href="ProteinSequence.html" title="class in org.biojava.nbio.core.sequence">ProteinSequence</a></span>&nbsp;<span class="element-name">getProteinSequence</span><wbr><span class="parameters">(<a href="transcription/TranscriptionEngine.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine</a>&nbsp;engine,
 <a href="TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a>&nbsp;transcriptSequence)</span></div>
<div class="block">Get the protein sequence with user defined TranscriptEngine</div>
<dl class="notes">
<dt>Parameters:</dt>
<dd><code>engine</code> - </dd>
<dt>Returns:</dt>
</dl>
</section>
</li>
<li>
<section class="detail" id="addStartCodonSequence(org.biojava.nbio.core.sequence.AccessionID,int,int,org.biojava.nbio.core.sequence.TranscriptSequence)">
<h3>addStartCodonSequence</h3>
<div class="member-signature"><span class="modifiers">public</span>&nbsp;<span class="return-type">void</span>&nbsp;<span class="element-name">addStartCodonSequence</span><wbr><span class="parameters">(<a href="AccessionID.html" title="class in org.biojava.nbio.core.sequence">AccessionID</a>&nbsp;accession,
 int&nbsp;begin,
 int&nbsp;end,
 <a href="TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a>&nbsp;transcriptSequence)</span></div>
<div class="block">Sets the start codon sequence at given begin /  end location. Note that calling this method multiple times will replace any existing value.</div>
<dl class="notes">
<dt>Parameters:</dt>
<dd><code>accession</code> - </dd>
<dd><code>begin</code> - </dd>
<dd><code>end</code> - </dd>
</dl>
</section>
</li>
<li>
<section class="detail" id="addStopCodonSequence(org.biojava.nbio.core.sequence.AccessionID,int,int,org.biojava.nbio.core.sequence.TranscriptSequence)">
<h3>addStopCodonSequence</h3>
<div class="member-signature"><span class="modifiers">public</span>&nbsp;<span class="return-type">void</span>&nbsp;<span class="element-name">addStopCodonSequence</span><wbr><span class="parameters">(<a href="AccessionID.html" title="class in org.biojava.nbio.core.sequence">AccessionID</a>&nbsp;accession,
 int&nbsp;begin,
 int&nbsp;end,
 <a href="TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a>&nbsp;transcriptSequence)</span></div>
<div class="block">Sets the stop codon sequence at given begin /  end location. Note that calling this method multiple times will replace any existing value.</div>
<dl class="notes">
<dt>Parameters:</dt>
<dd><code>accession</code> - </dd>
<dd><code>begin</code> - </dd>
<dd><code>end</code> - </dd>
</dl>
</section>
</li>
</ul>
</section>
</li>
</ul>
</section>
<!-- ========= END OF CLASS DATA ========= -->
</main>
</div>
</div>
</body>
</html>
