<!DOCTYPE HTML>
<html lang="pt">
<head>
<!-- Generated by javadoc (17) on Sun May 15 18:36:12 WEST 2022 -->
<title>All Classes and Interfaces</title>
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta name="dc.created" content="2022-05-15">
<meta name="description" content="class index">
<meta name="generator" content="javadoc/AllClassesIndexWriter">
<link rel="stylesheet" type="text/css" href="stylesheet.css" title="Style">
<link rel="stylesheet" type="text/css" href="script-dir/jquery-ui.min.css" title="Style">
<link rel="stylesheet" type="text/css" href="jquery-ui.overrides.css" title="Style">
<script type="text/javascript" src="script.js"></script>
<script type="text/javascript" src="script-dir/jquery-3.5.1.min.js"></script>
<script type="text/javascript" src="script-dir/jquery-ui.min.js"></script>
</head>
<body class="all-classes-index-page">
<script type="text/javascript">var evenRowColor = "even-row-color";
var oddRowColor = "odd-row-color";
var tableTab = "table-tab";
var activeTableTab = "active-table-tab";
var pathtoroot = "./";
loadScripts(document, 'script');</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<div class="flex-box">
<header role="banner" class="flex-header">
<nav role="navigation">
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="top-nav" id="navbar-top">
<div class="skip-nav"><a href="#skip-navbar-top" title="Skip navigation links">Skip navigation links</a></div>
<ul id="navbar-top-firstrow" class="nav-list" title="Navigation">
<li><a href="index.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li>Use</li>
<li><a href="overview-tree.html">Tree</a></li>
<li><a href="deprecated-list.html">Deprecated</a></li>
<li><a href="index-files/index-1.html">Index</a></li>
<li><a href="help-doc.html#all-classes">Help</a></li>
</ul>
</div>
<div class="sub-nav">
<div class="nav-list-search"><label for="search-input">SEARCH:</label>
<input type="text" id="search-input" value="search" disabled="disabled">
<input type="reset" id="reset-button" value="reset" disabled="disabled">
</div>
</div>
<!-- ========= END OF TOP NAVBAR ========= -->
<span class="skip-nav" id="skip-navbar-top"></span></nav>
</header>
<div class="flex-content">
<main role="main">
<div class="header">
<h1 title="All Classes and Interfaces" class="title">All Classes and Interfaces</h1>
</div>
<div id="all-classes-table">
<div class="table-tabs" role="tablist" aria-orientation="horizontal"><button id="all-classes-table-tab0" role="tab" aria-selected="true" aria-controls="all-classes-table.tabpanel" tabindex="0" onkeydown="switchTab(event)" onclick="show('all-classes-table', 'all-classes-table', 2)" class="active-table-tab">All Classes and Interfaces</button><button id="all-classes-table-tab1" role="tab" aria-selected="false" aria-controls="all-classes-table.tabpanel" tabindex="-1" onkeydown="switchTab(event)" onclick="show('all-classes-table', 'all-classes-table-tab1', 2)" class="table-tab">Interfaces</button><button id="all-classes-table-tab2" role="tab" aria-selected="false" aria-controls="all-classes-table.tabpanel" tabindex="-1" onkeydown="switchTab(event)" onclick="show('all-classes-table', 'all-classes-table-tab2', 2)" class="table-tab">Classes</button><button id="all-classes-table-tab3" role="tab" aria-selected="false" aria-controls="all-classes-table.tabpanel" tabindex="-1" onkeydown="switchTab(event)" onclick="show('all-classes-table', 'all-classes-table-tab3', 2)" class="table-tab">Enums</button><button id="all-classes-table-tab5" role="tab" aria-selected="false" aria-controls="all-classes-table.tabpanel" tabindex="-1" onkeydown="switchTab(event)" onclick="show('all-classes-table', 'all-classes-table-tab5', 2)" class="table-tab">Exceptions</button></div>
<div id="all-classes-table.tabpanel" role="tabpanel">
<div class="summary-table two-column-summary" aria-labelledby="all-classes-table-tab0">
<div class="table-header col-first">Class</div>
<div class="table-header col-last">Description</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/alignment/matrices/AAindexFactory.html" title="class in org.biojava.nbio.core.alignment.matrices">AAindexFactory</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Factory class to get Providers for substitution matrices that are provided by
 the AAINDEX database.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/alignment/matrices/AAIndexFileParser.html" title="class in org.biojava.nbio.core.alignment.matrices">AAIndexFileParser</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/alignment/matrices/AAIndexProvider.html" title="interface in org.biojava.nbio.core.alignment.matrices">AAIndexProvider</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab1">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/ABITrace.html" title="class in org.biojava.nbio.core.sequence.io">ABITrace</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Title: ABITrace</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/compound/ABITracerCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">ABITracerCompoundSet</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/ABITracerTest.html" title="class in org.biojava.nbio.core.sequence.io">ABITracerTest</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Test file 3730.ab1 is from https://github.com/biopython/biopython/blob/master/Tests/Abi/3730.ab1 The test data for
 comparing the results from ABITrace.java for file 3730.ab1 is from
 https://github.com/biopython/biopython/blob/master/Tests/Abi/test_data</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/template/AbstractCompound.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompound</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">The details of a Compound</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundSet</a>&lt;<a href="org/biojava/nbio/core/sequence/template/AbstractCompoundSet.html" title="type parameter in AbstractCompoundSet">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava.nbio.core.sequence.template">AbstractCompoundTranslator</a>&lt;<a href="org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="type parameter in AbstractCompoundTranslator">F</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>,<wbr><a href="org/biojava/nbio/core/sequence/template/AbstractCompoundTranslator.html" title="type parameter in AbstractCompoundTranslator">T</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="class in org.biojava.nbio.core.sequence.features">AbstractFeature</a>&lt;<a href="org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="type parameter in AbstractFeature">S</a> extends <a href="org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="type parameter in AbstractFeature">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/sequence/features/AbstractFeature.html" title="type parameter in AbstractFeature">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">A feature is currently any descriptive item that can be associated with a sequence position(s)
 A feature has a type and a source which is currently a string to allow flexibility for the user
 Ideally well defined features should have a class to describe attributes of that feature</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocation</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Base abstraction of a location which encodes for the majority of important
 features about a location such as the start, end and strand</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/location/template/AbstractLocationProduct.html" title="class in org.biojava.nbio.core.sequence.location.template">AbstractLocationProduct</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html" title="class in org.biojava.nbio.core.sequence.template">AbstractNucleotideCompoundSet</a>&lt;<a href="org/biojava/nbio/core/sequence/template/AbstractNucleotideCompoundSet.html" title="type parameter in AbstractNucleotideCompoundSet">C</a> extends <a href="org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound">NucleotideCompound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/reference/AbstractReference.html" title="class in org.biojava.nbio.core.sequence.reference">AbstractReference</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="type parameter in AbstractSequence">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">The base class for DNA, RNA and Protein sequences.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab3"><a href="org/biojava/nbio/core/sequence/template/AbstractSequence.AnnotationType.html" title="enum in org.biojava.nbio.core.sequence.template">AbstractSequence.AnnotationType</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab3">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/sequence/location/template/AccesionedLocation.html" title="interface in org.biojava.nbio.core.sequence.location.template">AccesionedLocation</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab1">
<div class="block">A location which is bound to an AccessionID.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/sequence/template/Accessioned.html" title="interface in org.biojava.nbio.core.sequence.template">Accessioned</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab1">
<div class="block">Indicates an entity is accessioned</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/AccessionID.html" title="class in org.biojava.nbio.core.sequence">AccessionID</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Used in Sequences as the unique identifier.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template">AlignedSequence</a>&lt;<a href="org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="type parameter in AlignedSequence">S</a> extends <a href="org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="type parameter in AlignedSequence">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="type parameter in AlignedSequence">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab1">
<div class="block">Defines a data structure for a <a href="org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a> within an alignment.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab3"><a href="org/biojava/nbio/core/alignment/template/AlignedSequence.Step.html" title="enum in org.biojava.nbio.core.alignment.template">AlignedSequence.Step</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab3">
<div class="block">Defines an alignment step in order to pass alignment information from an <code>Aligner</code> to a constructor.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/compound/AmbiguityDNACompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AmbiguityDNACompoundSet</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/compound/AmbiguityDNACompoundTest.html" title="class in org.biojava.nbio.core.sequence.compound">AmbiguityDNACompoundTest</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/compound/AmbiguityDNARNAHybridCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AmbiguityDNARNAHybridCompoundSet</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Ambiguity set for hybrid DNA/RNA sequences.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/compound/AmbiguityRNACompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AmbiguityRNACompoundSet</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompound</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Used to describe an Amino Acid.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">AminoAcidCompoundSet</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Set of proteinogenic amino acids.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/loader/ArrayListProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">ArrayListProxySequenceReader</a>&lt;<a href="org/biojava/nbio/core/sequence/loader/ArrayListProxySequenceReader.html" title="type parameter in ArrayListProxySequenceReader">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">ArrayListSequenceReader</a>&lt;<a href="org/biojava/nbio/core/sequence/storage/ArrayListSequenceReader.html" title="type parameter in ArrayListSequenceReader">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Stores a Sequence as a collection of compounds in an ArrayList</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/BasicSequence.html" title="class in org.biojava.nbio.core.sequence">BasicSequence</a>&lt;<a href="org/biojava/nbio/core/sequence/BasicSequence.html" title="type parameter in BasicSequence">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Bare bones version of the Sequence object to be used sparingly.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader</a>&lt;<a href="org/biojava/nbio/core/sequence/storage/BitSequenceReader.html" title="type parameter in BitSequenceReader">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">An implementation of the popular bit encodings.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">BitSequenceReader.BitArrayWorker</a>&lt;<a href="org/biojava/nbio/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="type parameter in BitSequenceReader.BitArrayWorker">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">The logic of working with a bit has been separated out into this class
 to help developers create the bit data structures without having to
 put the code into an intermediate format and to also use the format
 without the need to copy this code.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/search/io/blast/BlastHitBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHitBuilder</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Designed by Paolo Pavan.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/search/io/blast/BlastHspBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastHspBuilder</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Designed by Paolo Pavan.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/search/io/blast/BlastResult.html" title="class in org.biojava.nbio.core.search.io.blast">BlastResult</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">This class models a Blast/Blast plus result.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/search/io/blast/BlastResultBuilder.html" title="class in org.biojava.nbio.core.search.io.blast">BlastResultBuilder</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Designed by Paolo Pavan.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/search/io/blast/BlastTabularParser.html" title="class in org.biojava.nbio.core.search.io.blast">BlastTabularParser</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Designed by Paolo Pavan.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/search/io/blast/BlastTabularParserTest.html" title="class in org.biojava.nbio.core.search.io.blast">BlastTabularParserTest</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/search/io/blast/BlastXMLParser.html" title="class in org.biojava.nbio.core.search.io.blast">BlastXMLParser</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Re-designed by Paolo Pavan on the footprint of:
 org.biojava.nbio.genome.query.BlastXMLQuery by Scooter Willis <willishf at gmail dot com>

 You may want to find my contacts on Github and LinkedIn for code info
 or discuss major changes.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/search/io/blast/BlastXMLParserProduct.html" title="class in org.biojava.nbio.core.search.io.blast">BlastXMLParserProduct</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/search/io/blast/BlastXMLParserTest.html" title="class in org.biojava.nbio.core.search.io.blast">BlastXMLParserTest</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava.nbio.core.sequence.io">BufferedReaderBytesRead</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Need to keep track of actual bytes read and take advantage of buffered reader
 performance.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava.nbio.core.sequence.transcription">CaseInsensitiveCompound</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Attempts to wrap compounds so it is possible to view them
 in a case insensitive manner</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/CasePreservingProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">CasePreservingProteinSequenceCreator</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">A sequence creator which preserves the case of its input string in
 the user collection of the returned ProteinSequence.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/CDSComparator.html" title="class in org.biojava.nbio.core.sequence">CDSComparator</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/CDSSequence.html" title="class in org.biojava.nbio.core.sequence">CDSSequence</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Represents a exon or coding sequence in a gene.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/ChromosomeSequence.html" title="class in org.biojava.nbio.core.sequence">ChromosomeSequence</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">A ChromosomeSequence is a DNASequence but keeps track of geneSequences</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/util/ClasspathResource.html" title="class in org.biojava.nbio.core.sequence.io.util">ClasspathResource</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">This object represents a classpath resource on the local system.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/compound/CodonCompound.html" title="class in org.biojava.nbio.core.sequence.compound">CodonCompound</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Define a codon</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/sequence/template/ComplementCompound.html" title="interface in org.biojava.nbio.core.sequence.template">ComplementCompound</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab1">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/views/ComplementSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">ComplementSequenceView</a>&lt;<a href="org/biojava/nbio/core/sequence/views/ComplementSequenceView.html" title="type parameter in ComplementSequenceView">C</a> extends <a href="org/biojava/nbio/core/sequence/template/ComplementCompound.html" title="interface in org.biojava.nbio.core.sequence.template">ComplementCompound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">For a given sequence this class will create a view over the top of it
 and for every request the code will return the complement of the underlying
 base e.g. base A will become base T</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab1">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab5"><a href="org/biojava/nbio/core/exceptions/CompoundNotFoundException.html" title="class in org.biojava.nbio.core.exceptions">CompoundNotFoundException</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab5">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/sequence/template/CompoundSet.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundSet</a>&lt;<a href="org/biojava/nbio/core/sequence/template/CompoundSet.html" title="type parameter in CompoundSet">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab1">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/sequence/template/CompoundTranslator.html" title="interface in org.biojava.nbio.core.sequence.template">CompoundTranslator</a>&lt;<a href="org/biojava/nbio/core/sequence/template/CompoundTranslator.html" title="type parameter in CompoundTranslator">F</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>,<wbr><a href="org/biojava/nbio/core/sequence/template/CompoundTranslator.html" title="type parameter in CompoundTranslator">T</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab1">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/util/ConcurrencyTools.html" title="class in org.biojava.nbio.core.util">ConcurrencyTools</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Static utility to easily share a thread pool for concurrent/parallel/lazy execution.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/util/CRC64Checksum.html" title="class in org.biojava.nbio.core.util">CRC64Checksum</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Utility class that calculates a CRC64 checksum on a stream of bytes.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/sequence/features/DatabaseReferenceInterface.html" title="interface in org.biojava.nbio.core.sequence.features">DatabaseReferenceInterface</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab1">
<div class="block">If a SequenceProxyReader implements this interface then that external source
 has a list of cross reference id(s)</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab3"><a href="org/biojava/nbio/core/sequence/DataSource.html" title="enum in org.biojava.nbio.core.sequence">DataSource</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab3">
<div class="block">
 GenBank                           gi|gi-number|gb|accession|locus
 ENA Data Library                  gi|gi-number|emb|accession|locus
 DDBJ, DNA Database of Japan       gi|gi-number|dbj|accession|locus
 NBRF PIR                          pir||entry
 Protein Research Foundation       prf||name
 SWISS-PROT UNIPROT                sp|accession|name
 Brookhaven Protein Data Bank (1)  pdb|entry|chain
 Brookhaven Protein Data Bank (2)  entry:chain|PDBID|CHAIN|SEQUENCE
 Patents                           pat|country|number
 GenInfo Backbone Id               bbs|number
 General database identifier       gnl|database|identifier
 NCBI Reference Sequence           ref|accession|locus
 Local Sequence identifier         lcl|identifier
 </pr></div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">DBReferenceInfo</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">If you have a uniprot ID then it is possible to get a collection
 of other id(s) that the protein is known by.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/alignment/matrices/DefaultAAIndexProvider.html" title="class in org.biojava.nbio.core.alignment.matrices">DefaultAAIndexProvider</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">The default provider for AAINDEX loads substitution matrices from the AAINDEX file in the resources directory</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="demo/DemoSixFrameTranslation.html" title="class in demo">DemoSixFrameTranslation</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Created by andreas on 8/10/15.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/compound/DNACompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">DNACompoundSet</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/DNASequence.html" title="class in org.biojava.nbio.core.sequence">DNASequence</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">This is class should model the attributes associated with a DNA sequence</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab3"><a href="org/biojava/nbio/core/sequence/DNASequence.DNAType.html" title="enum in org.biojava.nbio.core.sequence">DNASequence.DNAType</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab3">
<div class="block">The type of DNA sequence</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/DNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">DNASequenceCreator</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">A helper class that allows different ways to read a string and create a DNA sequence.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/DNATest.html" title="class in org.biojava.nbio.core.sequence">DNATest</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/transcription/DNAToRNATranslator.html" title="class in org.biojava.nbio.core.sequence.transcription">DNAToRNATranslator</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Performs the first stage of transcription by going from DNA to RNA.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/sequence/edits/Edit.html" title="interface in org.biojava.nbio.core.sequence.edits">Edit</a>&lt;<a href="org/biojava/nbio/core/sequence/edits/Edit.html" title="type parameter in Edit">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab1">
<div class="block">Interface for carrying out edit operations on a Sequence.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.AbstractEdit</a>&lt;<a href="org/biojava/nbio/core/sequence/edits/Edit.AbstractEdit.html" title="type parameter in Edit.AbstractEdit">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Abstract class which defines all edit operations as a call to discover
 what 5' and 3' ends of an editing Sequence should be joined together
 with a target Sequence.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/edits/Edit.Delete.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Delete</a>&lt;<a href="org/biojava/nbio/core/sequence/edits/Edit.Delete.html" title="type parameter in Edit.Delete">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Implementation which allows for the deletion of bases from a Sequence</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/edits/Edit.Insert.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Insert</a>&lt;<a href="org/biojava/nbio/core/sequence/edits/Edit.Insert.html" title="type parameter in Edit.Insert">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Edit implementation which allows us to insert a base at any position
 in a Sequence.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/edits/Edit.Substitute.html" title="class in org.biojava.nbio.core.sequence.edits">Edit.Substitute</a>&lt;<a href="org/biojava/nbio/core/sequence/edits/Edit.Substitute.html" title="type parameter in Edit.Substitute">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Allows for the substitution of bases into an existing Sequence.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/EditSequenceTest.html" title="class in org.biojava.nbio.core.sequence">EditSequenceTest</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/embl/EmblId.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblId</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">This class contains the processed data of embl file
 Primary accession number
 Sequence version number
 Topology: 'circular' or 'linear'
 Molecule type
 Data class
 Taxonomic division
 Sequence length</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/embl/EmblReader.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReader</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">This class should process the data of embl file</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/embl/EmblReaderTest.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReaderTest</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">This class should test the parser of EmblReader class</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/embl/EmblRecord.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblRecord</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">this class contains the parsed data of embl file</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/embl/EmblReference.html" title="class in org.biojava.nbio.core.sequence.io.embl">EmblReference</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">This class contains the processed data of embl file that
 contains the referenceNumber, referenceComment, referencePosition
 referenceCrossReference, referenceGroup, referenceAuthor
 referenceTitle, referenceLocation</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/util/Equals.html" title="class in org.biojava.nbio.core.util">Equals</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">A set of helper methods which return true if the two parameters are
 equal to each other.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/ExonComparator.html" title="class in org.biojava.nbio.core.sequence">ExonComparator</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Sort Exon where it is a little confusing if exons should always be ordered left to right
 where a negative stranded gene should go the other direction.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/ExonSequence.html" title="class in org.biojava.nbio.core.sequence">ExonSequence</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">A gene contains a collection of Exon sequences</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/FastaGeneWriter.html" title="class in org.biojava.nbio.core.sequence.io">FastaGeneWriter</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">A Gene sequence has a Positive or Negative Strand where we want to write out to a stream the 5 to 3 prime version.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/sequence/io/template/FastaHeaderFormatInterface.html" title="interface in org.biojava.nbio.core.sequence.io.template">FastaHeaderFormatInterface</a>&lt;<a href="org/biojava/nbio/core/sequence/io/template/FastaHeaderFormatInterface.html" title="type parameter in FastaHeaderFormatInterface">S</a> extends <a href="org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;?&gt;,<wbr><a href="org/biojava/nbio/core/sequence/io/template/FastaHeaderFormatInterface.html" title="type parameter in FastaHeaderFormatInterface">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab1">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/FastaReader.html" title="class in org.biojava.nbio.core.sequence.io">FastaReader</a>&lt;<a href="org/biojava/nbio/core/sequence/io/FastaReader.html" title="type parameter in FastaReader">S</a> extends <a href="org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;?&gt;,<wbr><a href="org/biojava/nbio/core/sequence/io/FastaReader.html" title="type parameter in FastaReader">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Use FastaReaderHelper as an example of how to use this class where FastaReaderHelper should be the
 primary class used to read Fasta files</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/FastaReaderHelper.html" title="class in org.biojava.nbio.core.sequence.io">FastaReaderHelper</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/FastaReaderTest.html" title="class in org.biojava.nbio.core.sequence.io">FastaReaderTest</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/FastaSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">FastaSequenceParser</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Used to parse a stream of a fasta file to get the sequence</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/FastaWriter.html" title="class in org.biojava.nbio.core.sequence.io">FastaWriter</a>&lt;<a href="org/biojava/nbio/core/sequence/io/FastaWriter.html" title="type parameter in FastaWriter">S</a> extends <a href="org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;?&gt;,<wbr><a href="org/biojava/nbio/core/sequence/io/FastaWriter.html" title="type parameter in FastaWriter">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">The FastaWriter writes a collection of sequences to an outputStream.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/FastaWriterHelper.html" title="class in org.biojava.nbio.core.sequence.io">FastaWriterHelper</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">The class that should be used to write out fasta file of a sequence collection</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/FastaWriterTest.html" title="class in org.biojava.nbio.core.sequence.io">FastaWriterTest</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features">FeatureDbReferenceInfo</a>&lt;<a href="org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="type parameter in FeatureDbReferenceInfo">S</a> extends <a href="org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="type parameter in FeatureDbReferenceInfo">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/sequence/features/FeatureDbReferenceInfo.html" title="type parameter in FeatureDbReferenceInfo">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">It is <a href="org/biojava/nbio/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava.nbio.core.sequence.features"><code>DBReferenceInfo</code></a> which implements <a href="org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features"><code>FeatureInterface</code></a>.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureInterface</a>&lt;<a href="org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="type parameter in FeatureInterface">S</a> extends <a href="org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="type parameter in FeatureInterface">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/sequence/features/FeatureInterface.html" title="type parameter in FeatureInterface">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab1">
<div class="block">Interface class to handle describing arbitrary features.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/sequence/features/FeatureRetriever.html" title="interface in org.biojava.nbio.core.sequence.features">FeatureRetriever</a>&lt;<a href="org/biojava/nbio/core/sequence/features/FeatureRetriever.html" title="type parameter in FeatureRetriever">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab1">
<div class="block">If a SequenceProxyReader implements this interface then that external source
 has a list features</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/sequence/features/FeaturesKeyWordInterface.html" title="interface in org.biojava.nbio.core.sequence.features">FeaturesKeyWordInterface</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab1">
<div class="block">Models the keywords that are annotated for a protein sequence at Uniprot.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/util/FileDownloadUtils.html" title="class in org.biojava.nbio.core.util">FileDownloadUtils</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/FileProxyDNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">FileProxyDNASequenceCreator</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">This class is a good example of using the SequenceCreatorInterface where during parsing of the stream
 the sequence and the offset index are passed to create a Protein sequence that will be loaded in lazily.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/FileProxyProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">FileProxyProteinSequenceCreator</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">This class is a good example of using the SequenceCreatorInterface where during parsing of the stream
 the sequence and the offset index are passed to create a Protein sequence that will be loaded in lazily.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/FileProxyRNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">FileProxyRNASequenceCreator</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">This class is a good example of using the SequenceCreatorInterface where during parsing of the stream
 the sequence and the offset index are passed to create a Protein sequence that will be loaded in lazily.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/util/FlatFileCache.html" title="class in org.biojava.nbio.core.util">FlatFileCache</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Provides a cache for storing multiple small files in memory.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">FourBitSequenceReader</a>&lt;<a href="org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.html" title="type parameter in FourBitSequenceReader">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Four bit encoding of the bit formats.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">FourBitSequenceReader.FourBitArrayWorker</a>&lt;<a href="org/biojava/nbio/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="type parameter in FourBitSequenceReader.FourBitArrayWorker">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">A four bit per compound implementation of the bit array worker code.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab3"><a href="org/biojava/nbio/core/sequence/transcription/Frame.html" title="enum in org.biojava.nbio.core.sequence.transcription">Frame</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab3">
<div class="block">Indicates a way of translating a sequence.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/location/FuzzyPoint.html" title="class in org.biojava.nbio.core.sequence.location">FuzzyPoint</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Implementation for resolving fuzzy locations.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/GenbankCookbookTest.html" title="class in org.biojava.nbio.core.sequence.io">GenbankCookbookTest</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html" title="interface in org.biojava.nbio.core.sequence.io.template">GenbankHeaderFormatInterface</a>&lt;<a href="org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html" title="type parameter in GenbankHeaderFormatInterface">S</a> extends <a href="org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;?&gt;,<wbr><a href="org/biojava/nbio/core/sequence/io/template/GenbankHeaderFormatInterface.html" title="type parameter in GenbankHeaderFormatInterface">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab1">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReader</a>&lt;<a href="org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReader.html" title="type parameter in GenbankProxySequenceReader">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReaderProduct.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReaderProduct</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReaderTest.html" title="class in org.biojava.nbio.core.sequence.loader">GenbankProxySequenceReaderTest</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Testing example for issue #834</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/GenbankReader.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReader</a>&lt;<a href="org/biojava/nbio/core/sequence/io/GenbankReader.html" title="type parameter in GenbankReader">S</a> extends <a href="org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="org/biojava/nbio/core/sequence/io/GenbankReader.html" title="type parameter in GenbankReader">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/sequence/io/GenbankReader.html" title="type parameter in GenbankReader">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Use <a href="org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html" title="class in org.biojava.nbio.core.sequence.io"><code>GenbankReaderHelper</code></a> as an example of how to use this class where <a href="org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html" title="class in org.biojava.nbio.core.sequence.io"><code>GenbankReaderHelper</code></a> should be the
 primary class used to read Genbank files</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/GenbankReaderHelper.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReaderHelper</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/GenbankReaderTest.html" title="class in org.biojava.nbio.core.sequence.io">GenbankReaderTest</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/reference/GenbankReference.html" title="class in org.biojava.nbio.core.sequence.reference">GenbankReference</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">For Genbank format file only.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="class in org.biojava.nbio.core.sequence.io">GenbankSequenceParser</a>&lt;<a href="org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="type parameter in GenbankSequenceParser">S</a> extends <a href="org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="type parameter in GenbankSequenceParser">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/sequence/io/GenbankSequenceParser.html" title="type parameter in GenbankSequenceParser">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/GenbankWriter.html" title="class in org.biojava.nbio.core.sequence.io">GenbankWriter</a>&lt;<a href="org/biojava/nbio/core/sequence/io/GenbankWriter.html" title="type parameter in GenbankWriter">S</a> extends <a href="org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;?&gt;,<wbr><a href="org/biojava/nbio/core/sequence/io/GenbankWriter.html" title="type parameter in GenbankWriter">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/GenbankWriterHelper.html" title="class in org.biojava.nbio.core.sequence.io">GenbankWriterHelper</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">The class that should be used to write out genbank file of a sequence
 collection</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/GenbankWriterTest.html" title="class in org.biojava.nbio.core.sequence.io">GenbankWriterTest</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.html" title="class in org.biojava.nbio.core.sequence.io">GenericFastaHeaderFormat</a>&lt;<a href="org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.html" title="type parameter in GenericFastaHeaderFormat">S</a> extends <a href="org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;?&gt;,<wbr><a href="org/biojava/nbio/core/sequence/io/GenericFastaHeaderFormat.html" title="type parameter in GenericFastaHeaderFormat">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">We store the original header if the sequence is parsed from a fasta file and will use that exact
 sequence if we write out the sequences to a fasta file.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">GenericFastaHeaderParser</a>&lt;<a href="org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html" title="type parameter in GenericFastaHeaderParser">S</a> extends <a href="org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html" title="type parameter in GenericFastaHeaderParser">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/sequence/io/GenericFastaHeaderParser.html" title="type parameter in GenericFastaHeaderParser">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">The default fasta header parser where some headers are well defined based on the source
 database which allows us to set the source of the protein sequence and the identifier
 that can be used in future implementations to load features from external sources

 If the user has a custom header with local data then they can create their own implementation
 of a FastaHeaderParserInterface</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/GenericFastaHeaderParserTest.html" title="class in org.biojava.nbio.core.sequence.io">GenericFastaHeaderParserTest</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html" title="class in org.biojava.nbio.core.sequence.io">GenericGenbankHeaderFormat</a>&lt;<a href="org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html" title="type parameter in GenericGenbankHeaderFormat">S</a> extends <a href="org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html" title="type parameter in GenericGenbankHeaderFormat">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.html" title="type parameter in GenericGenbankHeaderFormat">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">GenericGenbankHeaderParser</a>&lt;<a href="org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" title="type parameter in GenericGenbankHeaderParser">S</a> extends <a href="org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" title="type parameter in GenericGenbankHeaderParser">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.html" title="type parameter in GenericGenbankHeaderParser">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html" title="class in org.biojava.nbio.core.sequence.io">GenericInsdcHeaderFormat</a>&lt;<a href="org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html" title="type parameter in GenericInsdcHeaderFormat">S</a> extends <a href="org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html" title="type parameter in GenericInsdcHeaderFormat">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/sequence/io/GenericInsdcHeaderFormat.html" title="type parameter in GenericInsdcHeaderFormat">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/GeneSequence.html" title="class in org.biojava.nbio.core.sequence">GeneSequence</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/util/Hashcoder.html" title="class in org.biojava.nbio.core.util">Hashcoder</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Contains helper methods for generating a HashCode without having to resort to
 the commons lang hashcode builders.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/search/io/Hit.html" title="class in org.biojava.nbio.core.search.io">Hit</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">This class models a search Hit.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/search/io/Hsp.html" title="class in org.biojava.nbio.core.search.io">Hsp</a>&lt;<a href="org/biojava/nbio/core/search/io/Hsp.html" title="type parameter in Hsp">S</a> extends <a href="org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="org/biojava/nbio/core/search/io/Hsp.html" title="type parameter in Hsp">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/search/io/Hsp.html" title="type parameter in Hsp">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">This class models a search Hsp.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/search/io/HspTest.html" title="class in org.biojava.nbio.core.search.io">HspTest</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/util/InputStreamProvider.html" title="class in org.biojava.nbio.core.util">InputStreamProvider</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">A class that provides an InputStream from a File.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/location/InsdcLocations.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">A collection of locations which are used whenever we work with INSDC; some
 of which could be deprecated (from INSDC's point of view) yet appear
 in records.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/location/InsdcLocations.BondLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.BondLocation</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Used to represent bond locations equivalent to bond(7,8) or bond(7).</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/location/InsdcLocations.GroupLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.GroupLocation</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Deprecated in INSDC yet still appears; equivalent to the order()
 directive except no 5' to 3' ordering is defined.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/location/InsdcLocations.OneOfLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.OneOfLocation</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Deprecated in INSDC; refers to a set of locations of which one
 location could be valid e.g. one-of(location, location, location).</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/location/InsdcLocations.OrderLocation.html" title="class in org.biojava.nbio.core.sequence.location">InsdcLocations.OrderLocation</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Used to describe a 5' to 3' ordering but no firm assurance it is correct</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/location/InsdcParser.html" title="class in org.biojava.nbio.core.sequence.location">InsdcParser</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Parser for working with INSDC style locations.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/location/InsdcParserTest.html" title="class in org.biojava.nbio.core.sequence.location">InsdcParserTest</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/IntronSequence.html" title="class in org.biojava.nbio.core.sequence">IntronSequence</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/util/IOUtils.html" title="class in org.biojava.nbio.core.sequence.io.util">IOUtils</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/sequence/io/util/IOUtils.ReaderProcessor.html" title="interface in org.biojava.nbio.core.sequence.io.util">IOUtils.ReaderProcessor</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab1">
<div class="block">Closure interface used when working with
 <code>IOUtils#processReader(String)</code>.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/IUPACParser.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Available translations

 
 1 - UNIVERSAL
 2 - VERTEBRATE_MITOCHONDRIAL
 3 - YEAST_MITOCHONDRIAL
 4 - MOLD_MITOCHONDRIAL
 5 - INVERTEBRATE_MITOCHONDRIAL
 6 - CILIATE_NUCLEAR
 9 - ECHINODERM_MITOCHONDRIAL
 10 - EUPLOTID_NUCLEAR
 11 - BACTERIAL
 12 - ALTERNATIVE_YEAST_NUCLEAR
 13 - ASCIDIAN_MITOCHONDRIAL
 14 - FLATWORM_MITOCHONDRIAL
 15 - BLEPHARISMA_MACRONUCLEAR
 16 - 2CHLOROPHYCEAN_MITOCHONDRIAL
 21 - TREMATODE_MITOCHONDRIAL
 23 - SCENEDESMUS_MITOCHONDRIAL
 

 Taken from <a href="https://www.ncbi.nlm.nih.gov/Taxonomy/Utils/wprintgc.cgi?mode=c">NCBI</a> with slight modification and put into the classpath resource.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParser.IUPACTable</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Holds the concept of a codon table from the IUPAC format</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/IUPACParserProduct.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParserProduct</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/IUPACParserProduct2.html" title="class in org.biojava.nbio.core.sequence.io">IUPACParserProduct2</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/IUPACTableProduct.html" title="class in org.biojava.nbio.core.sequence.io">IUPACTableProduct</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">JoiningSequenceReader</a>&lt;<a href="org/biojava/nbio/core/sequence/storage/JoiningSequenceReader.html" title="type parameter in JoiningSequenceReader">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">This reader actually proxies onto multiple types of sequence in order
 to allow a number of sequence objects to act as if they are one sequence.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/JoiningSequenceReaderTest.html" title="class in org.biojava.nbio.core.sequence">JoiningSequenceReaderTest</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/sequence/template/LightweightProfile.html" title="interface in org.biojava.nbio.core.sequence.template">LightweightProfile</a>&lt;<a href="org/biojava/nbio/core/sequence/template/LightweightProfile.html" title="type parameter in LightweightProfile">S</a> extends <a href="org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="org/biojava/nbio/core/sequence/template/LightweightProfile.html" title="type parameter in LightweightProfile">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/sequence/template/LightweightProfile.html" title="type parameter in LightweightProfile">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab1">
<div class="block">Defines a minimal data structure for reading and writing a sequence alignment.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab3"><a href="org/biojava/nbio/core/sequence/template/LightweightProfile.StringFormat.html" title="enum in org.biojava.nbio.core.sequence.template">LightweightProfile.StringFormat</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab3">
<div class="block">List of output formats.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/sequence/location/template/Location.html" title="interface in org.biojava.nbio.core.sequence.location.template">Location</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab1">
<div class="block">Sets of integers used to represent the location of features on sequence.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/location/template/Location.Tools.html" title="class in org.biojava.nbio.core.sequence.location.template">Location.Tools</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Helper methods for use with the Location classes.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/location/LocationHelper.html" title="class in org.biojava.nbio.core.sequence.location">LocationHelper</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Helper methods for use with the Location classes.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/location/LocationParserTest.html" title="class in org.biojava.nbio.core.sequence.location">LocationParserTest</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">TODO: Temporary test is switched off.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/location/LocationTest.html" title="class in org.biojava.nbio.core.sequence.location">LocationTest</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/exceptions/Messages.html" title="class in org.biojava.nbio.core.exceptions">Messages</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava.nbio.core.sequence">MultipleSequenceAlignment</a>&lt;<a href="org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="type parameter in MultipleSequenceAlignment">S</a> extends <a href="org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="type parameter in MultipleSequenceAlignment">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/sequence/MultipleSequenceAlignment.html" title="type parameter in MultipleSequenceAlignment">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Implements a minimal data structure for reading and writing a sequence alignment.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/alignment/template/MutableAlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template">MutableAlignedSequence</a>&lt;<a href="org/biojava/nbio/core/alignment/template/MutableAlignedSequence.html" title="type parameter in MutableAlignedSequence">S</a> extends <a href="org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="org/biojava/nbio/core/alignment/template/MutableAlignedSequence.html" title="type parameter in MutableAlignedSequence">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/alignment/template/MutableAlignedSequence.html" title="type parameter in MutableAlignedSequence">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab1">
<div class="block">Defines a mutable (editable) data structure for an <a href="org/biojava/nbio/core/alignment/template/AlignedSequence.html" title="interface in org.biojava.nbio.core.alignment.template"><code>AlignedSequence</code></a>.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/alignment/template/MutableProfile.html" title="interface in org.biojava.nbio.core.alignment.template">MutableProfile</a>&lt;<a href="org/biojava/nbio/core/alignment/template/MutableProfile.html" title="type parameter in MutableProfile">S</a> extends <a href="org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="org/biojava/nbio/core/alignment/template/MutableProfile.html" title="type parameter in MutableProfile">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/alignment/template/MutableProfile.html" title="type parameter in MutableProfile">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab1">
<div class="block">Defines a mutable (editable) data structure for a <a href="org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template"><code>Profile</code></a>.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/alignment/template/MutableProfilePair.html" title="interface in org.biojava.nbio.core.alignment.template">MutableProfilePair</a>&lt;<a href="org/biojava/nbio/core/alignment/template/MutableProfilePair.html" title="type parameter in MutableProfilePair">S</a> extends <a href="org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="org/biojava/nbio/core/alignment/template/MutableProfilePair.html" title="type parameter in MutableProfilePair">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/alignment/template/MutableProfilePair.html" title="type parameter in MutableProfilePair">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab1">
<div class="block">Defines a mutable (editable) data structure for a <a href="org/biojava/nbio/core/alignment/template/ProfilePair.html" title="interface in org.biojava.nbio.core.alignment.template"><code>ProfilePair</code></a>.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/alignment/template/MutableSequencePair.html" title="interface in org.biojava.nbio.core.alignment.template">MutableSequencePair</a>&lt;<a href="org/biojava/nbio/core/alignment/template/MutableSequencePair.html" title="type parameter in MutableSequencePair">S</a> extends <a href="org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="org/biojava/nbio/core/alignment/template/MutableSequencePair.html" title="type parameter in MutableSequencePair">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/alignment/template/MutableSequencePair.html" title="type parameter in MutableSequencePair">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab1">
<div class="block">Defines a mutable (editable) data structure for the results of pairwise sequence alignment.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound">NucleotideCompound</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="demo/ParseFastaFileDemo.html" title="class in demo">ParseFastaFileDemo</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Created by andreas on 6/17/15.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab5"><a href="org/biojava/nbio/core/exceptions/ParserException.html" title="class in org.biojava.nbio.core.exceptions">ParserException</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab5">
<div class="block">General abstraction of different parsing errors</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html" title="class in org.biojava.nbio.core.sequence.io">PlainFastaHeaderParser</a>&lt;<a href="org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html" title="type parameter in PlainFastaHeaderParser">S</a> extends <a href="org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html" title="type parameter in PlainFastaHeaderParser">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/sequence/io/PlainFastaHeaderParser.html" title="type parameter in PlainFastaHeaderParser">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">The plain fasta header takes everything in the header as a single entity.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/sequence/location/template/Point.html" title="interface in org.biojava.nbio.core.sequence.location.template">Point</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab1">
<div class="block">Holds a single point part of a location</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/sequence/location/template/Point.Resolver.html" title="interface in org.biojava.nbio.core.sequence.location.template">Point.Resolver</a>&lt;<a href="org/biojava/nbio/core/sequence/location/template/Point.Resolver.html" title="type parameter in Point.Resolver">T</a> extends <a href="org/biojava/nbio/core/sequence/location/template/Point.html" title="interface in org.biojava.nbio.core.sequence.location.template">Point</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab1">
<div class="block">Used to resolve a position about a point</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/util/PrettyXMLWriter.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriter</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Implementation of XMLWriter which emits nicely formatted documents
 to a PrintWriter.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/util/PrettyXMLWriterProduct.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriterProduct</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/util/PrettyXMLWriterProduct2.html" title="class in org.biojava.nbio.core.util">PrettyXMLWriterProduct2</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template">Profile</a>&lt;<a href="org/biojava/nbio/core/alignment/template/Profile.html" title="type parameter in Profile">S</a> extends <a href="org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="org/biojava/nbio/core/alignment/template/Profile.html" title="type parameter in Profile">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/alignment/template/Profile.html" title="type parameter in Profile">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab1">
<div class="block">Defines a data structure for the results of sequence alignment.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab3"><a href="org/biojava/nbio/core/alignment/template/Profile.StringFormat.html" title="enum in org.biojava.nbio.core.alignment.template">Profile.StringFormat</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab3">
<div class="block">List of output formats.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/alignment/template/ProfilePair.html" title="interface in org.biojava.nbio.core.alignment.template">ProfilePair</a>&lt;<a href="org/biojava/nbio/core/alignment/template/ProfilePair.html" title="type parameter in ProfilePair">S</a> extends <a href="org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="org/biojava/nbio/core/alignment/template/ProfilePair.html" title="type parameter in ProfilePair">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/alignment/template/ProfilePair.html" title="type parameter in ProfilePair">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab1">
<div class="block">Defines a data structure for the results of the alignment of a pair of <a href="org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template"><code>Profile</code></a>s.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/alignment/template/ProfileView.html" title="interface in org.biojava.nbio.core.alignment.template">ProfileView</a>&lt;<a href="org/biojava/nbio/core/alignment/template/ProfileView.html" title="type parameter in ProfileView">S</a> extends <a href="org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="org/biojava/nbio/core/alignment/template/ProfileView.html" title="type parameter in ProfileView">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/alignment/template/ProfileView.html" title="type parameter in ProfileView">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab1">
<div class="block">Defines a data structure for a view of sequence alignment.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/ProteinSequence.html" title="class in org.biojava.nbio.core.sequence">ProteinSequence</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">The representation of a ProteinSequence</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/ProteinSequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">ProteinSequenceCreator</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Used to create a ProteinSequence from a String to allow for details
 about the location of the sequence etc.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/ProteinSequenceTest.html" title="class in org.biojava.nbio.core.sequence">ProteinSequenceTest</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/sequence/template/ProxySequenceReader.html" title="interface in org.biojava.nbio.core.sequence.template">ProxySequenceReader</a>&lt;<a href="org/biojava/nbio/core/sequence/template/ProxySequenceReader.html" title="type parameter in ProxySequenceReader">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab1">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/features/Qualifier.html" title="class in org.biojava.nbio.core.sequence.features">Qualifier</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/features/QualityFeature.html" title="class in org.biojava.nbio.core.sequence.features">QualityFeature</a>&lt;<a href="org/biojava/nbio/core/sequence/features/QualityFeature.html" title="type parameter in QualityFeature">S</a> extends <a href="org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="org/biojava/nbio/core/sequence/features/QualityFeature.html" title="type parameter in QualityFeature">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/sequence/features/QualityFeature.html" title="type parameter in QualityFeature">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">DNA Sequences produced by modern sequencers usually have quality informaion
 attached to them.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/features/QuantityFeature.html" title="class in org.biojava.nbio.core.sequence.features">QuantityFeature</a>&lt;<a href="org/biojava/nbio/core/sequence/features/QuantityFeature.html" title="type parameter in QuantityFeature">S</a> extends <a href="org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="org/biojava/nbio/core/sequence/features/QuantityFeature.html" title="type parameter in QuantityFeature">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/sequence/features/QuantityFeature.html" title="type parameter in QuantityFeature">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">It is common to have a numerical value or values associated with a feature.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/sequence/reference/ReferenceInterface.html" title="interface in org.biojava.nbio.core.sequence.reference">ReferenceInterface</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab1">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/search/io/Result.html" title="class in org.biojava.nbio.core.search.io">Result</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">This class models a search result.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/search/io/ResultFactory.html" title="interface in org.biojava.nbio.core.search.io">ResultFactory</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab1">
<div class="block">Designed by Paolo Pavan.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/views/ReversedSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">ReversedSequenceView</a>&lt;<a href="org/biojava/nbio/core/sequence/views/ReversedSequenceView.html" title="type parameter in ReversedSequenceView">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">For a given sequence this class will return the base at the reversed
 position i.e. in a sequence of size 10, if you request base 2 you will get
 back the base at position 9.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/compound/RNACompoundSet.html" title="class in org.biojava.nbio.core.sequence.compound">RNACompoundSet</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/RNASequence.html" title="class in org.biojava.nbio.core.sequence">RNASequence</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">RNASequence where RNACompoundSet are the allowed values</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/io/RNASequenceCreator.html" title="class in org.biojava.nbio.core.sequence.io">RNASequenceCreator</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Used to create a RNA sequence</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/views/RnaSequenceView.html" title="class in org.biojava.nbio.core.sequence.views">RnaSequenceView</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Attempts to do on the fly translation of RNA by not requesting the compounds
 until asked.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/transcription/RNAToAminoAcidTranslator.html" title="class in org.biojava.nbio.core.sequence.transcription">RNAToAminoAcidTranslator</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Takes a <a href="org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a> of <a href="org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound"><code>NucleotideCompound</code></a> which should represent
 an RNA sequence (<a href="org/biojava/nbio/core/sequence/RNASequence.html" title="class in org.biojava.nbio.core.sequence"><code>RNASequence</code></a> is good for this) and returns a list of
 <a href="org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a> which hold <a href="org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava.nbio.core.sequence.compound"><code>AminoAcidCompound</code></a>.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/alignment/matrices/ScaledSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">ScaledSubstitutionMatrix</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">The biojava-alignment module represents substitution matrices with short
 values.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/search/io/SearchIO.html" title="class in org.biojava.nbio.core.search.io">SearchIO</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Designed by Paolo Pavan.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/search/io/SearchIOTest.html" title="class in org.biojava.nbio.core.search.io">SearchIOTest</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Designed by Paolo Pavan.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="org/biojava/nbio/core/sequence/template/Sequence.html" title="type parameter in Sequence">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab1">
<div class="block">Main interface for defining a collection of Compounds and accessing them
 using biological indexes</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html" title="class in org.biojava.nbio.core.sequence.storage">SequenceAsStringHelper</a>&lt;<a href="org/biojava/nbio/core/sequence/storage/SequenceAsStringHelper.html" title="type parameter in SequenceAsStringHelper">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">This is a common method that can be used across multiple storage/proxy implementations to
 handle Negative strand and other interesting elements of sequence data.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/SequenceComparator.html" title="class in org.biojava.nbio.core.sequence">SequenceComparator</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Used to sort sequences in ascending order of  bioBegin property.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html" title="interface in org.biojava.nbio.core.sequence.io.template">SequenceCreatorInterface</a>&lt;<a href="org/biojava/nbio/core/sequence/io/template/SequenceCreatorInterface.html" title="type parameter in SequenceCreatorInterface">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab1">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava.nbio.core.sequence.loader">SequenceFileProxyLoader</a>&lt;<a href="org/biojava/nbio/core/sequence/loader/SequenceFileProxyLoader.html" title="type parameter in SequenceFileProxyLoader">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">This class represents the storage container of a sequence stored in a fasta file where
 the initial parsing of the file we store the offset and length of the sequence.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/sequence/io/template/SequenceHeaderParserInterface.html" title="interface in org.biojava.nbio.core.sequence.io.template">SequenceHeaderParserInterface</a>&lt;<a href="org/biojava/nbio/core/sequence/io/template/SequenceHeaderParserInterface.html" title="type parameter in SequenceHeaderParserInterface">S</a> extends <a href="org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;?&gt;,<wbr><a href="org/biojava/nbio/core/sequence/io/template/SequenceHeaderParserInterface.html" title="type parameter in SequenceHeaderParserInterface">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab1">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/location/SequenceLocation.html" title="class in org.biojava.nbio.core.sequence.location">SequenceLocation</a>&lt;<a href="org/biojava/nbio/core/sequence/location/SequenceLocation.html" title="type parameter in SequenceLocation">S</a> extends <a href="org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="org/biojava/nbio/core/sequence/location/SequenceLocation.html" title="type parameter in SequenceLocation">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/sequence/location/SequenceLocation.html" title="type parameter in SequenceLocation">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">A location in a sequence that keeps a reference to its parent sequence</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/template/SequenceMixin.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Provides a set of static methods to be used as static imports when needed
 across multiple Sequence implementations but inheritance gets in the way.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/template/SequenceMixin.SequenceIterator.html" title="class in org.biojava.nbio.core.sequence.template">SequenceMixin.SequenceIterator</a>&lt;<a href="org/biojava/nbio/core/sequence/template/SequenceMixin.SequenceIterator.html" title="type parameter in SequenceMixin.SequenceIterator">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">A basic sequence iterator which iterates over the given Sequence by
 biological index.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/SequenceOptimizationHints.html" title="class in org.biojava.nbio.core.sequence">SequenceOptimizationHints</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">A static class that provides optimization hints for memory or performance handling of sequence data.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab3"><a href="org/biojava/nbio/core/sequence/SequenceOptimizationHints.SequenceCollection.html" title="enum in org.biojava.nbio.core.sequence">SequenceOptimizationHints.SequenceCollection</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab3">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab3"><a href="org/biojava/nbio/core/sequence/SequenceOptimizationHints.SequenceUsage.html" title="enum in org.biojava.nbio.core.sequence">SequenceOptimizationHints.SequenceUsage</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab3">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/alignment/template/SequencePair.html" title="interface in org.biojava.nbio.core.alignment.template">SequencePair</a>&lt;<a href="org/biojava/nbio/core/alignment/template/SequencePair.html" title="type parameter in SequencePair">S</a> extends <a href="org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="org/biojava/nbio/core/alignment/template/SequencePair.html" title="type parameter in SequencePair">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/alignment/template/SequencePair.html" title="type parameter in SequencePair">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab1">
<div class="block">Defines a data structure for the results of pairwise sequence alignment.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/sequence/io/template/SequenceParserInterface.html" title="interface in org.biojava.nbio.core.sequence.io.template">SequenceParserInterface</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab1">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="class in org.biojava.nbio.core.sequence.template">SequenceProxyView</a>&lt;<a href="org/biojava/nbio/core/sequence/template/SequenceProxyView.html" title="type parameter in SequenceProxyView">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/sequence/template/SequenceReader.html" title="interface in org.biojava.nbio.core.sequence.template">SequenceReader</a>&lt;<a href="org/biojava/nbio/core/sequence/template/SequenceReader.html" title="type parameter in SequenceReader">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab1">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/SequenceTestUtils.html" title="class in org.biojava.nbio.core.sequence">SequenceTestUtils</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Test utility methods for classes in this package</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/util/SequenceTools.html" title="class in org.biojava.nbio.core.util">SequenceTools</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/sequence/template/SequenceView.html" title="interface in org.biojava.nbio.core.sequence.template">SequenceView</a>&lt;<a href="org/biojava/nbio/core/sequence/template/SequenceView.html" title="type parameter in SequenceView">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab1">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/SequenceViewTest.html" title="class in org.biojava.nbio.core.sequence">SequenceViewTest</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequence</a>&lt;<a href="org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="type parameter in SimpleAlignedSequence">S</a> extends <a href="org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="type parameter in SimpleAlignedSequence">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/alignment/SimpleAlignedSequence.html" title="type parameter in SimpleAlignedSequence">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Implements a data structure for a <a href="org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a> within an alignment.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/alignment/SimpleAlignedSequenceTest.html" title="class in org.biojava.nbio.core.alignment">SimpleAlignedSequenceTest</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/loader/SimpleGenbankProxySequenceReaderTest.html" title="class in org.biojava.nbio.core.sequence.loader">SimpleGenbankProxySequenceReaderTest</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Testing example for issue #834</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/location/SimpleLocation.html" title="class in org.biojava.nbio.core.sequence.location">SimpleLocation</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Very basic implementation of the Location interface which defines a series
 of simple constructors.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/location/SimplePoint.html" title="class in org.biojava.nbio.core.sequence.location">SimplePoint</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Basic implementation of the Point interface.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/alignment/SimpleProfile.html" title="class in org.biojava.nbio.core.alignment">SimpleProfile</a>&lt;<a href="org/biojava/nbio/core/alignment/SimpleProfile.html" title="type parameter in SimpleProfile">S</a> extends <a href="org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="org/biojava/nbio/core/alignment/SimpleProfile.html" title="type parameter in SimpleProfile">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/alignment/SimpleProfile.html" title="type parameter in SimpleProfile">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Implements a data structure for the results of sequence alignment.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/alignment/SimpleProfilePair.html" title="class in org.biojava.nbio.core.alignment">SimpleProfilePair</a>&lt;<a href="org/biojava/nbio/core/alignment/SimpleProfilePair.html" title="type parameter in SimpleProfilePair">S</a> extends <a href="org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="org/biojava/nbio/core/alignment/SimpleProfilePair.html" title="type parameter in SimpleProfilePair">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/alignment/SimpleProfilePair.html" title="type parameter in SimpleProfilePair">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Implements a data structure for the results of the alignment of a pair of <a href="org/biojava/nbio/core/alignment/template/Profile.html" title="interface in org.biojava.nbio.core.alignment.template"><code>Profile</code></a>s.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/alignment/SimpleProfileTest.html" title="class in org.biojava.nbio.core.alignment">SimpleProfileTest</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/alignment/SimpleSequencePair.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePair</a>&lt;<a href="org/biojava/nbio/core/alignment/SimpleSequencePair.html" title="type parameter in SimpleSequencePair">S</a> extends <a href="org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template">Sequence</a>&lt;<a href="org/biojava/nbio/core/alignment/SimpleSequencePair.html" title="type parameter in SimpleSequencePair">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/alignment/SimpleSequencePair.html" title="type parameter in SimpleSequencePair">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Implements a data structure for the results of pairwise sequence alignment.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/alignment/SimpleSequencePairTest.html" title="class in org.biojava.nbio.core.alignment">SimpleSequencePairTest</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html" title="class in org.biojava.nbio.core.alignment.matrices">SimpleSubstitutionMatrix</a>&lt;<a href="org/biojava/nbio/core/alignment/matrices/SimpleSubstitutionMatrix.html" title="type parameter in SimpleSubstitutionMatrix">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Implements a data structure which holds the score (penalty or bonus) given during alignment for the exchange of one
 <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Compound</code></a> in a sequence for another.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">SingleCompoundSequenceReader</a>&lt;<a href="org/biojava/nbio/core/sequence/storage/SingleCompoundSequenceReader.html" title="type parameter in SingleCompoundSequenceReader">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">An implementation of the SequenceReader interface which for every
 call will return only 1 compound (given to it during construction; a String
 is also valid but will require a CompoundSet).</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/util/SingleLinkageClusterer.html" title="class in org.biojava.nbio.core.util">SingleLinkageClusterer</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">An implementation of a single linkage clusterer

 See http://en.wikipedia.org/wiki/Single-linkage_clustering</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/util/SingleLinkageClustererProduct.html" title="class in org.biojava.nbio.core.util">SingleLinkageClustererProduct</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/util/SingleLinkageClustererProductProduct.html" title="class in org.biojava.nbio.core.util">SingleLinkageClustererProductProduct</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/util/SoftHashMap.html" title="class in org.biojava.nbio.core.util">SoftHashMap</a>&lt;<a href="org/biojava/nbio/core/util/SoftHashMap.html" title="type parameter in SoftHashMap">K</a>,<wbr><a href="org/biojava/nbio/core/util/SoftHashMap.html" title="type parameter in SoftHashMap">V</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">A in memory cache using soft references.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/StartCodonSequence.html" title="class in org.biojava.nbio.core.sequence">StartCodonSequence</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Used to map the start codon feature on a gene</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/StopCodonSequence.html" title="class in org.biojava.nbio.core.sequence">StopCodonSequence</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Used to map the stop codon sequence on a gene</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab3"><a href="org/biojava/nbio/core/sequence/Strand.html" title="enum in org.biojava.nbio.core.sequence">Strand</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab3">
<div class="block">Provides a way of representing the strand of a sequence, location
 hit or feature.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/util/StringManipulationHelper.html" title="class in org.biojava.nbio.core.util">StringManipulationHelper</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">A utility class for common <a href="https://docs.oracle.com/javase/8/docs/api/java/lang/String.html" title="class or interface in java.lang" class="external-link"><code>String</code></a> manipulation tasks.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">StringProxySequenceReader</a>&lt;<a href="org/biojava/nbio/core/sequence/loader/StringProxySequenceReader.html" title="type parameter in StringProxySequenceReader">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">An example of a ProxySequenceReader that is created from a String.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava.nbio.core.alignment.template">SubstitutionMatrix</a>&lt;<a href="org/biojava/nbio/core/alignment/template/SubstitutionMatrix.html" title="type parameter in SubstitutionMatrix">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab1">
<div class="block">Defines a data structure which holds the score (penalty or bonus) given during alignment for the exchange of one
 <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Compound</code></a> in a sequence for another.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/alignment/matrices/SubstitutionMatrixHelper.html" title="class in org.biojava.nbio.core.alignment.matrices">SubstitutionMatrixHelper</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Static utility to access substitution matrices that come bundled with BioJava.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/sequence/transcription/Table.html" title="interface in org.biojava.nbio.core.sequence.transcription">Table</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab1">
<div class="block">Provides a way of separating us from the specific <a href="org/biojava/nbio/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava.nbio.core.sequence.io"><code>IUPACParser.IUPACTable</code></a> even
 though this is the only implementing class for the interface.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.CaseInsensitiveTriplet</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Class used to hold three nucleotides together and allow for equality
 to be assessed in a case insensitive manner.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/transcription/Table.Codon.html" title="class in org.biojava.nbio.core.sequence.transcription">Table.Codon</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Instance of a Codon which is 3 <a href="org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound"><code>NucleotideCompound</code></a>s, its
 corresponding <a href="org/biojava/nbio/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava.nbio.core.sequence.compound"><code>AminoAcidCompound</code></a> and if it is a start or stop codon.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/location/TargetedLocationParserTest.html" title="class in org.biojava.nbio.core.sequence.location">TargetedLocationParserTest</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/location/TargetedLocationParserTest.Data.html" title="class in org.biojava.nbio.core.sequence.location">TargetedLocationParserTest.Data</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/TaxonomyID.html" title="class in org.biojava.nbio.core.sequence">TaxonomyID</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">A sequence can be associated with a species or Taxonomy ID</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/TestAmbiguityCompoundSet.html" title="class in org.biojava.nbio.core">TestAmbiguityCompoundSet</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">A Test case for https://github.com/biojava/biojava/issues/344

 Created by andreas on 12/4/15.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/fasta/TestFASTAReader.html" title="class in org.biojava.nbio.core.fasta">TestFASTAReader</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/TestSequenceEquals.html" title="class in org.biojava.nbio.core.sequence">TestSequenceEquals</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Created by yana on 3/27/17.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/util/TestUncompressInputStream.html" title="class in org.biojava.nbio.core.util">TestUncompressInputStream</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/features/TextFeature.html" title="class in org.biojava.nbio.core.sequence.features">TextFeature</a>&lt;<a href="org/biojava/nbio/core/sequence/features/TextFeature.html" title="type parameter in TextFeature">S</a> extends <a href="org/biojava/nbio/core/sequence/template/AbstractSequence.html" title="class in org.biojava.nbio.core.sequence.template">AbstractSequence</a>&lt;<a href="org/biojava/nbio/core/sequence/features/TextFeature.html" title="type parameter in TextFeature">C</a>&gt;,<wbr><a href="org/biojava/nbio/core/sequence/features/TextFeature.html" title="type parameter in TextFeature">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">A implmentation of AbstractFeature</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Used as a way of encapsulating the data structures required to parse DNA to a
 Protein sequence.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava.nbio.core.sequence.transcription">TranscriptionEngine.Builder</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">This class is the way to create a <code>TranslationEngine</code>.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/TranscriptSequence.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequence</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">This is the sequence if you want to go from a gene sequence to a protein sequence.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/TranscriptSequenceProduct.html" title="class in org.biojava.nbio.core.sequence">TranscriptSequenceProduct</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab5"><a href="org/biojava/nbio/core/exceptions/TranslationException.html" title="class in org.biojava.nbio.core.exceptions">TranslationException</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab5">
<div class="block">Thrown from AbstractCompundTranslator</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/TranslationTest.html" title="class in org.biojava.nbio.core.sequence">TranslationTest</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html" title="class in org.biojava.nbio.core.sequence.storage">TwoBitSequenceReader</a>&lt;<a href="org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.html" title="type parameter in TwoBitSequenceReader">C</a> extends <a href="org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound">NucleotideCompound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Implementation of the 2bit encoding.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="class in org.biojava.nbio.core.sequence.storage">TwoBitSequenceReader.TwoBitArrayWorker</a>&lt;<a href="org/biojava/nbio/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="type parameter in TwoBitSequenceReader.TwoBitArrayWorker">C</a> extends <a href="org/biojava/nbio/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava.nbio.core.sequence.compound">NucleotideCompound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Extension of the BitArrayWorker which provides the 2bit implementation
 code.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="demo/UncompressFile.html" title="class in demo">UncompressFile</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Uncompresses a single tarred or zipped file, writing output to stdandard out</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/util/UncompressInputStream.html" title="class in org.biojava.nbio.core.util">UncompressInputStream</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">This class decompresses an input stream containing data compressed with
 the unix "compress" utility (LZC, a LZW variant).</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/util/UncompressInputStreamData.html" title="class in org.biojava.nbio.core.util">UncompressInputStreamData</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReader</a>&lt;<a href="org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.html" title="type parameter in UniprotProxySequenceReader">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">
<div class="block">Pass in a Uniprot ID and this ProxySequenceReader when passed to a ProteinSequence will get the sequence data and other data elements
 associated with the ProteinSequence by Uniprot.</div>
</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReaderProduct.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReaderProduct</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReaderProduct2.html" title="class in org.biojava.nbio.core.sequence.loader">UniprotProxySequenceReaderProduct2</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/views/WindowedSequence.html" title="class in org.biojava.nbio.core.sequence.views">WindowedSequence</a>&lt;<a href="org/biojava/nbio/core/sequence/views/WindowedSequence.html" title="type parameter in WindowedSequence">C</a> extends <a href="org/biojava/nbio/core/sequence/template/Compound.html" title="interface in org.biojava.nbio.core.sequence.template">Compound</a>&gt;</div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">A sliding window view of a sequence which does not implement any
 interfaces like <a href="org/biojava/nbio/core/sequence/template/Sequence.html" title="interface in org.biojava.nbio.core.sequence.template"><code>Sequence</code></a> because they do not fit how this works.</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/sequence/views/WindowViewTests.html" title="class in org.biojava.nbio.core.sequence.views">WindowViewTests</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab2">&nbsp;</div>
<div class="col-first odd-row-color all-classes-table all-classes-table-tab2"><a href="org/biojava/nbio/core/util/XMLHelper.html" title="class in org.biojava.nbio.core.util">XMLHelper</a></div>
<div class="col-last odd-row-color all-classes-table all-classes-table-tab2">
<div class="block">Helper methods to simplify boilerplate XML parsing code for  org.w3c.dom XML objects</div>
</div>
<div class="col-first even-row-color all-classes-table all-classes-table-tab1"><a href="org/biojava/nbio/core/util/XMLWriter.html" title="interface in org.biojava.nbio.core.util">XMLWriter</a></div>
<div class="col-last even-row-color all-classes-table all-classes-table-tab1">
<div class="block">Simple interface for building XML documents.</div>
</div>
</div>
</div>
</div>
</main>
</div>
</div>
</body>
</html>
